{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [],
   "source": [
    "# !pip install numpy pandas scikit-allel matplotlib\n",
    "import gzip\n",
    "import pandas as pd\n",
    "from pandas_plink import read_plink\n",
    "from sklearn.preprocessing import StandardScaler\n",
    "import os\n",
    "# import allel\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "from sklearn.decomposition import PCA\n",
    "import seaborn as sns\n",
    "from sklearn.linear_model import LinearRegression\n",
    "import numpy as np\n",
    "from sklearn.impute import SimpleImputer\n",
    "from scipy.stats import pearsonr\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4005050199.py:3: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4005050199.py:3: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 47.68it/s]\n"
     ]
    }
   ],
   "source": [
    "# Load genotype data using pandas-plink\n",
    "plink_file_prefix = './chr22_data'  # Modify for other chromosomes if needed\n",
    "(bim, fam, bed) = read_plink(plink_file_prefix)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FAM file: \n",
      "  fid         iid father mother gender trait  i\n",
      "0   0  GTEX-111CU      0      0      0    -9  0\n",
      "1   0  GTEX-111FC      0      0      0    -9  1\n",
      "2   0  GTEX-111VG      0      0      0    -9  2\n",
      "3   0  GTEX-111YS      0      0      0    -9  3\n",
      "4   0  GTEX-1122O      0      0      0    -9  4\n"
     ]
    }
   ],
   "source": [
    "print(f\"FAM file: \\n{fam.head()}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FAM file: \n",
      "  chrom         snp   cm       pos a0 a1  i\n",
      "0    22    rs131538  0.0  16390411  A  G  0\n",
      "1    22   rs7287144  0.0  16406147  G  A  1\n",
      "2    22   rs5747620  0.0  16551808  C  T  2\n",
      "3    22  rs17430900  0.0  16573137  A  G  3\n",
      "4    22   rs9605903  0.0  16573830  C  T  4\n"
     ]
    }
   ],
   "source": [
    "print(f\"FAM file: \\n{bim.head()}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract the sample IDs from the FAM file\n",
    "genotype_sample_ids = fam['iid'].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load the gene expression sample IDs (similar to before)\n",
    "gene_expression_file = 'Whole_Blood.v8.normalized_expression.bed'\n",
    "\n",
    "gene_expression_data = pd.read_csv(gene_expression_file, sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "# extract chr22\n",
    "gene_expression_data=gene_expression_data[gene_expression_data['#chr'] == 'chr22']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#chr</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>GTEX-111YS</th>\n",
       "      <th>GTEX-1122O</th>\n",
       "      <th>GTEX-1128S</th>\n",
       "      <th>GTEX-113IC</th>\n",
       "      <th>GTEX-113JC</th>\n",
       "      <th>GTEX-117XS</th>\n",
       "      <th>...</th>\n",
       "      <th>GTEX-ZV7C</th>\n",
       "      <th>GTEX-ZVE2</th>\n",
       "      <th>GTEX-ZVP2</th>\n",
       "      <th>GTEX-ZVT2</th>\n",
       "      <th>GTEX-ZVT3</th>\n",
       "      <th>GTEX-ZVT4</th>\n",
       "      <th>GTEX-ZVZP</th>\n",
       "      <th>GTEX-ZVZQ</th>\n",
       "      <th>GTEX-ZXES</th>\n",
       "      <th>GTEX-ZXG5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>19122</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16601886</td>\n",
       "      <td>16601887</td>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>1.570734</td>\n",
       "      <td>-0.770282</td>\n",
       "      <td>-1.353622</td>\n",
       "      <td>0.512862</td>\n",
       "      <td>-0.853391</td>\n",
       "      <td>-0.560321</td>\n",
       "      <td>...</td>\n",
       "      <td>1.120983</td>\n",
       "      <td>-1.821345</td>\n",
       "      <td>2.214046</td>\n",
       "      <td>1.217151</td>\n",
       "      <td>-0.902828</td>\n",
       "      <td>0.376684</td>\n",
       "      <td>1.265581</td>\n",
       "      <td>-0.954578</td>\n",
       "      <td>0.001868</td>\n",
       "      <td>0.919804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19123</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16612581</td>\n",
       "      <td>16612582</td>\n",
       "      <td>ENSG00000283633.1</td>\n",
       "      <td>1.747926</td>\n",
       "      <td>-0.996665</td>\n",
       "      <td>-0.582310</td>\n",
       "      <td>-1.545582</td>\n",
       "      <td>0.372677</td>\n",
       "      <td>0.231802</td>\n",
       "      <td>...</td>\n",
       "      <td>1.073249</td>\n",
       "      <td>-1.040597</td>\n",
       "      <td>1.178870</td>\n",
       "      <td>1.114008</td>\n",
       "      <td>-0.534285</td>\n",
       "      <td>2.259617</td>\n",
       "      <td>1.079918</td>\n",
       "      <td>-0.133007</td>\n",
       "      <td>0.144322</td>\n",
       "      <td>1.209352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19124</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16821698</td>\n",
       "      <td>16821699</td>\n",
       "      <td>ENSG00000172967.7</td>\n",
       "      <td>0.869633</td>\n",
       "      <td>-0.417104</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>0.735562</td>\n",
       "      <td>-0.254890</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>...</td>\n",
       "      <td>0.948702</td>\n",
       "      <td>-0.392775</td>\n",
       "      <td>1.382185</td>\n",
       "      <td>0.740467</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.293681</td>\n",
       "      <td>1.714493</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.095404</td>\n",
       "      <td>0.061677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19125</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17036569</td>\n",
       "      <td>17036570</td>\n",
       "      <td>ENSG00000237438.7</td>\n",
       "      <td>1.232973</td>\n",
       "      <td>-1.034199</td>\n",
       "      <td>-0.931270</td>\n",
       "      <td>-0.182182</td>\n",
       "      <td>-1.432461</td>\n",
       "      <td>0.595640</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.720952</td>\n",
       "      <td>-1.073249</td>\n",
       "      <td>0.095404</td>\n",
       "      <td>-1.765402</td>\n",
       "      <td>0.573481</td>\n",
       "      <td>-1.498020</td>\n",
       "      <td>0.328973</td>\n",
       "      <td>-0.336870</td>\n",
       "      <td>0.227967</td>\n",
       "      <td>-0.340827</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19126</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17067820</td>\n",
       "      <td>17067821</td>\n",
       "      <td>ENSG00000273203.1</td>\n",
       "      <td>-0.696937</td>\n",
       "      <td>-0.750331</td>\n",
       "      <td>-0.069165</td>\n",
       "      <td>0.687444</td>\n",
       "      <td>-0.491673</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>...</td>\n",
       "      <td>0.388743</td>\n",
       "      <td>0.099157</td>\n",
       "      <td>0.960486</td>\n",
       "      <td>0.348756</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>-0.243330</td>\n",
       "      <td>1.558035</td>\n",
       "      <td>-0.325032</td>\n",
       "      <td>-0.151876</td>\n",
       "      <td>0.569083</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 674 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        #chr     start       end             gene_id  GTEX-111YS  GTEX-1122O  \\\n",
       "19122  chr22  16601886  16601887  ENSG00000100181.22    1.570734   -0.770282   \n",
       "19123  chr22  16612581  16612582   ENSG00000283633.1    1.747926   -0.996665   \n",
       "19124  chr22  16821698  16821699   ENSG00000172967.7    0.869633   -0.417104   \n",
       "19125  chr22  17036569  17036570   ENSG00000237438.7    1.232973   -1.034199   \n",
       "19126  chr22  17067820  17067821   ENSG00000273203.1   -0.696937   -0.750331   \n",
       "\n",
       "       GTEX-1128S  GTEX-113IC  GTEX-113JC  GTEX-117XS  ...  GTEX-ZV7C  \\\n",
       "19122   -1.353622    0.512862   -0.853391   -0.560321  ...   1.120983   \n",
       "19123   -0.582310   -1.545582    0.372677    0.231802  ...   1.073249   \n",
       "19124   -1.197791    0.735562   -0.254890   -1.197791  ...   0.948702   \n",
       "19125   -0.931270   -0.182182   -1.432461    0.595640  ...  -0.720952   \n",
       "19126   -0.069165    0.687444   -0.491673   -1.396898  ...   0.388743   \n",
       "\n",
       "       GTEX-ZVE2  GTEX-ZVP2  GTEX-ZVT2  GTEX-ZVT3  GTEX-ZVT4  GTEX-ZVZP  \\\n",
       "19122  -1.821345   2.214046   1.217151  -0.902828   0.376684   1.265581   \n",
       "19123  -1.040597   1.178870   1.114008  -0.534285   2.259617   1.079918   \n",
       "19124  -0.392775   1.382185   0.740467  -1.197791  -0.293681   1.714493   \n",
       "19125  -1.073249   0.095404  -1.765402   0.573481  -1.498020   0.328973   \n",
       "19126   0.099157   0.960486   0.348756  -1.396898  -0.243330   1.558035   \n",
       "\n",
       "       GTEX-ZVZQ  GTEX-ZXES  GTEX-ZXG5  \n",
       "19122  -0.954578   0.001868   0.919804  \n",
       "19123  -0.133007   0.144322   1.209352  \n",
       "19124  -1.197791  -0.095404   0.061677  \n",
       "19125  -0.336870   0.227967  -0.340827  \n",
       "19126  -0.325032  -0.151876   0.569083  \n",
       "\n",
       "[5 rows x 674 columns]"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_expression_data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/874599569.py:3: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  genes_on_chrom['window_start'] = genes_on_chrom['start'].apply(lambda x: max(0, x - 500000))\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/874599569.py:4: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  genes_on_chrom['window_end'] = genes_on_chrom['end'] + 500000\n"
     ]
    }
   ],
   "source": [
    "# Define ±500 kb windows for each gene\n",
    "genes_on_chrom = gene_expression_data[['gene_id', \"#chr\", 'start', 'end']]\n",
    "genes_on_chrom['window_start'] = genes_on_chrom['start'].apply(lambda x: max(0, x - 500000))\n",
    "genes_on_chrom['window_end'] = genes_on_chrom['end'] + 500000"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#chr</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>GTEX-111YS</th>\n",
       "      <th>GTEX-1122O</th>\n",
       "      <th>GTEX-1128S</th>\n",
       "      <th>GTEX-113IC</th>\n",
       "      <th>GTEX-113JC</th>\n",
       "      <th>GTEX-117XS</th>\n",
       "      <th>...</th>\n",
       "      <th>GTEX-ZV7C</th>\n",
       "      <th>GTEX-ZVE2</th>\n",
       "      <th>GTEX-ZVP2</th>\n",
       "      <th>GTEX-ZVT2</th>\n",
       "      <th>GTEX-ZVT3</th>\n",
       "      <th>GTEX-ZVT4</th>\n",
       "      <th>GTEX-ZVZP</th>\n",
       "      <th>GTEX-ZVZQ</th>\n",
       "      <th>GTEX-ZXES</th>\n",
       "      <th>GTEX-ZXG5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>19122</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16601886</td>\n",
       "      <td>16601887</td>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>1.570734</td>\n",
       "      <td>-0.770282</td>\n",
       "      <td>-1.353622</td>\n",
       "      <td>0.512862</td>\n",
       "      <td>-0.853391</td>\n",
       "      <td>-0.560321</td>\n",
       "      <td>...</td>\n",
       "      <td>1.120983</td>\n",
       "      <td>-1.821345</td>\n",
       "      <td>2.214046</td>\n",
       "      <td>1.217151</td>\n",
       "      <td>-0.902828</td>\n",
       "      <td>0.376684</td>\n",
       "      <td>1.265581</td>\n",
       "      <td>-0.954578</td>\n",
       "      <td>0.001868</td>\n",
       "      <td>0.919804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19123</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16612581</td>\n",
       "      <td>16612582</td>\n",
       "      <td>ENSG00000283633.1</td>\n",
       "      <td>1.747926</td>\n",
       "      <td>-0.996665</td>\n",
       "      <td>-0.582310</td>\n",
       "      <td>-1.545582</td>\n",
       "      <td>0.372677</td>\n",
       "      <td>0.231802</td>\n",
       "      <td>...</td>\n",
       "      <td>1.073249</td>\n",
       "      <td>-1.040597</td>\n",
       "      <td>1.178870</td>\n",
       "      <td>1.114008</td>\n",
       "      <td>-0.534285</td>\n",
       "      <td>2.259617</td>\n",
       "      <td>1.079918</td>\n",
       "      <td>-0.133007</td>\n",
       "      <td>0.144322</td>\n",
       "      <td>1.209352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19124</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16821698</td>\n",
       "      <td>16821699</td>\n",
       "      <td>ENSG00000172967.7</td>\n",
       "      <td>0.869633</td>\n",
       "      <td>-0.417104</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>0.735562</td>\n",
       "      <td>-0.254890</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>...</td>\n",
       "      <td>0.948702</td>\n",
       "      <td>-0.392775</td>\n",
       "      <td>1.382185</td>\n",
       "      <td>0.740467</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.293681</td>\n",
       "      <td>1.714493</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.095404</td>\n",
       "      <td>0.061677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19125</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17036569</td>\n",
       "      <td>17036570</td>\n",
       "      <td>ENSG00000237438.7</td>\n",
       "      <td>1.232973</td>\n",
       "      <td>-1.034199</td>\n",
       "      <td>-0.931270</td>\n",
       "      <td>-0.182182</td>\n",
       "      <td>-1.432461</td>\n",
       "      <td>0.595640</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.720952</td>\n",
       "      <td>-1.073249</td>\n",
       "      <td>0.095404</td>\n",
       "      <td>-1.765402</td>\n",
       "      <td>0.573481</td>\n",
       "      <td>-1.498020</td>\n",
       "      <td>0.328973</td>\n",
       "      <td>-0.336870</td>\n",
       "      <td>0.227967</td>\n",
       "      <td>-0.340827</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19126</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17067820</td>\n",
       "      <td>17067821</td>\n",
       "      <td>ENSG00000273203.1</td>\n",
       "      <td>-0.696937</td>\n",
       "      <td>-0.750331</td>\n",
       "      <td>-0.069165</td>\n",
       "      <td>0.687444</td>\n",
       "      <td>-0.491673</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>...</td>\n",
       "      <td>0.388743</td>\n",
       "      <td>0.099157</td>\n",
       "      <td>0.960486</td>\n",
       "      <td>0.348756</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>-0.243330</td>\n",
       "      <td>1.558035</td>\n",
       "      <td>-0.325032</td>\n",
       "      <td>-0.151876</td>\n",
       "      <td>0.569083</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19127</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17084953</td>\n",
       "      <td>17084954</td>\n",
       "      <td>ENSG00000177663.13</td>\n",
       "      <td>0.564697</td>\n",
       "      <td>1.980120</td>\n",
       "      <td>-0.266485</td>\n",
       "      <td>-0.569083</td>\n",
       "      <td>-0.151876</td>\n",
       "      <td>-0.984462</td>\n",
       "      <td>...</td>\n",
       "      <td>2.099561</td>\n",
       "      <td>-0.106668</td>\n",
       "      <td>0.247180</td>\n",
       "      <td>1.596922</td>\n",
       "      <td>-1.980120</td>\n",
       "      <td>-0.227967</td>\n",
       "      <td>1.765402</td>\n",
       "      <td>-2.099561</td>\n",
       "      <td>0.289783</td>\n",
       "      <td>0.039235</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19128</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17093938</td>\n",
       "      <td>17093939</td>\n",
       "      <td>ENSG00000235328.1</td>\n",
       "      <td>-0.604585</td>\n",
       "      <td>1.257303</td>\n",
       "      <td>0.826808</td>\n",
       "      <td>0.392775</td>\n",
       "      <td>0.564697</td>\n",
       "      <td>-0.755291</td>\n",
       "      <td>...</td>\n",
       "      <td>1.624253</td>\n",
       "      <td>1.021528</td>\n",
       "      <td>-0.148098</td>\n",
       "      <td>0.902828</td>\n",
       "      <td>-1.802066</td>\n",
       "      <td>0.886109</td>\n",
       "      <td>0.790544</td>\n",
       "      <td>-1.265581</td>\n",
       "      <td>0.148098</td>\n",
       "      <td>-0.170799</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19129</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17121366</td>\n",
       "      <td>17121367</td>\n",
       "      <td>ENSG00000183307.3</td>\n",
       "      <td>0.780373</td>\n",
       "      <td>1.120983</td>\n",
       "      <td>-0.445806</td>\n",
       "      <td>0.396813</td>\n",
       "      <td>-0.293681</td>\n",
       "      <td>-0.765266</td>\n",
       "      <td>...</td>\n",
       "      <td>1.521370</td>\n",
       "      <td>-0.483259</td>\n",
       "      <td>0.663975</td>\n",
       "      <td>0.352728</td>\n",
       "      <td>-1.201627</td>\n",
       "      <td>-0.609076</td>\n",
       "      <td>2.099561</td>\n",
       "      <td>-1.610437</td>\n",
       "      <td>1.624253</td>\n",
       "      <td>-0.197397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19130</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17121585</td>\n",
       "      <td>17121586</td>\n",
       "      <td>ENSG00000235478.4</td>\n",
       "      <td>0.591185</td>\n",
       "      <td>0.054194</td>\n",
       "      <td>-1.047038</td>\n",
       "      <td>0.474879</td>\n",
       "      <td>-0.600107</td>\n",
       "      <td>-0.061677</td>\n",
       "      <td>...</td>\n",
       "      <td>1.372539</td>\n",
       "      <td>0.121709</td>\n",
       "      <td>0.864194</td>\n",
       "      <td>0.254890</td>\n",
       "      <td>-0.437569</td>\n",
       "      <td>0.091652</td>\n",
       "      <td>1.906141</td>\n",
       "      <td>-0.144322</td>\n",
       "      <td>1.475458</td>\n",
       "      <td>0.740467</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19131</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17165267</td>\n",
       "      <td>17165268</td>\n",
       "      <td>ENSG00000069998.12</td>\n",
       "      <td>1.164031</td>\n",
       "      <td>-0.282000</td>\n",
       "      <td>-2.259617</td>\n",
       "      <td>1.120983</td>\n",
       "      <td>-1.282401</td>\n",
       "      <td>-0.254890</td>\n",
       "      <td>...</td>\n",
       "      <td>0.133007</td>\n",
       "      <td>0.521402</td>\n",
       "      <td>2.514430</td>\n",
       "      <td>0.117946</td>\n",
       "      <td>-0.696937</td>\n",
       "      <td>0.148098</td>\n",
       "      <td>-0.529981</td>\n",
       "      <td>-1.422116</td>\n",
       "      <td>0.301491</td>\n",
       "      <td>0.344789</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>10 rows × 674 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        #chr     start       end             gene_id  GTEX-111YS  GTEX-1122O  \\\n",
       "19122  chr22  16601886  16601887  ENSG00000100181.22    1.570734   -0.770282   \n",
       "19123  chr22  16612581  16612582   ENSG00000283633.1    1.747926   -0.996665   \n",
       "19124  chr22  16821698  16821699   ENSG00000172967.7    0.869633   -0.417104   \n",
       "19125  chr22  17036569  17036570   ENSG00000237438.7    1.232973   -1.034199   \n",
       "19126  chr22  17067820  17067821   ENSG00000273203.1   -0.696937   -0.750331   \n",
       "19127  chr22  17084953  17084954  ENSG00000177663.13    0.564697    1.980120   \n",
       "19128  chr22  17093938  17093939   ENSG00000235328.1   -0.604585    1.257303   \n",
       "19129  chr22  17121366  17121367   ENSG00000183307.3    0.780373    1.120983   \n",
       "19130  chr22  17121585  17121586   ENSG00000235478.4    0.591185    0.054194   \n",
       "19131  chr22  17165267  17165268  ENSG00000069998.12    1.164031   -0.282000   \n",
       "\n",
       "       GTEX-1128S  GTEX-113IC  GTEX-113JC  GTEX-117XS  ...  GTEX-ZV7C  \\\n",
       "19122   -1.353622    0.512862   -0.853391   -0.560321  ...   1.120983   \n",
       "19123   -0.582310   -1.545582    0.372677    0.231802  ...   1.073249   \n",
       "19124   -1.197791    0.735562   -0.254890   -1.197791  ...   0.948702   \n",
       "19125   -0.931270   -0.182182   -1.432461    0.595640  ...  -0.720952   \n",
       "19126   -0.069165    0.687444   -0.491673   -1.396898  ...   0.388743   \n",
       "19127   -0.266485   -0.569083   -0.151876   -0.984462  ...   2.099561   \n",
       "19128    0.826808    0.392775    0.564697   -0.755291  ...   1.624253   \n",
       "19129   -0.445806    0.396813   -0.293681   -0.765266  ...   1.521370   \n",
       "19130   -1.047038    0.474879   -0.600107   -0.061677  ...   1.372539   \n",
       "19131   -2.259617    1.120983   -1.282401   -0.254890  ...   0.133007   \n",
       "\n",
       "       GTEX-ZVE2  GTEX-ZVP2  GTEX-ZVT2  GTEX-ZVT3  GTEX-ZVT4  GTEX-ZVZP  \\\n",
       "19122  -1.821345   2.214046   1.217151  -0.902828   0.376684   1.265581   \n",
       "19123  -1.040597   1.178870   1.114008  -0.534285   2.259617   1.079918   \n",
       "19124  -0.392775   1.382185   0.740467  -1.197791  -0.293681   1.714493   \n",
       "19125  -1.073249   0.095404  -1.765402   0.573481  -1.498020   0.328973   \n",
       "19126   0.099157   0.960486   0.348756  -1.396898  -0.243330   1.558035   \n",
       "19127  -0.106668   0.247180   1.596922  -1.980120  -0.227967   1.765402   \n",
       "19128   1.021528  -0.148098   0.902828  -1.802066   0.886109   0.790544   \n",
       "19129  -0.483259   0.663975   0.352728  -1.201627  -0.609076   2.099561   \n",
       "19130   0.121709   0.864194   0.254890  -0.437569   0.091652   1.906141   \n",
       "19131   0.521402   2.514430   0.117946  -0.696937   0.148098  -0.529981   \n",
       "\n",
       "       GTEX-ZVZQ  GTEX-ZXES  GTEX-ZXG5  \n",
       "19122  -0.954578   0.001868   0.919804  \n",
       "19123  -0.133007   0.144322   1.209352  \n",
       "19124  -1.197791  -0.095404   0.061677  \n",
       "19125  -0.336870   0.227967  -0.340827  \n",
       "19126  -0.325032  -0.151876   0.569083  \n",
       "19127  -2.099561   0.289783   0.039235  \n",
       "19128  -1.265581   0.148098  -0.170799  \n",
       "19129  -1.610437   1.624253  -0.197397  \n",
       "19130  -0.144322   1.475458   0.740467  \n",
       "19131  -1.422116   0.301491   0.344789  \n",
       "\n",
       "[10 rows x 674 columns]"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_expression_data.head(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['GTEX-111YS', 'GTEX-1122O', 'GTEX-1128S', 'GTEX-113IC', 'GTEX-113JC',\n",
       "       'GTEX-117XS', 'GTEX-117YW', 'GTEX-1192W', 'GTEX-1192X', 'GTEX-11DXX',\n",
       "       ...\n",
       "       'GTEX-ZV7C', 'GTEX-ZVE2', 'GTEX-ZVP2', 'GTEX-ZVT2', 'GTEX-ZVT3',\n",
       "       'GTEX-ZVT4', 'GTEX-ZVZP', 'GTEX-ZVZQ', 'GTEX-ZXES', 'GTEX-ZXG5'],\n",
       "      dtype='object', length=670)"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Gene expression sample columns (excluding the metadata columns)\n",
    "expression_sample_ids = gene_expression_data.columns[4:]  # Adjust if necessary\n",
    "expression_sample_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get the sample IDs from the .fam file\n",
    "fam_lin_sample_ids = fam['iid'].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "common_sample_ids = [sid for sid in expression_sample_ids if sid in genotype_sample_ids]\n",
    "\n",
    "# Get the indices of the common samples in the genotype matrix\n",
    "common_sample_indices = [genotype_sample_ids.index(sid) for sid in common_sample_ids]\n",
    "\n",
    "# Filter genotype matrix for common samples (489 individuals and 185 SNPs)\n",
    "genotype_matrix = bed.compute()[:, common_sample_indices]  # Filter genotype matrix based on common samples\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "# Filter gene expression values for the same common samples\n",
    "y = gene_expression_data[common_sample_ids].values#.flatten()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[ 1.57073416e+00, -7.70281989e-01, -1.35362223e+00, ...,\n",
       "        -9.54577587e-01,  1.86783140e-03,  9.19804155e-01],\n",
       "       [ 1.74792566e+00, -9.96665348e-01, -5.82310320e-01, ...,\n",
       "        -1.33007082e-01,  1.44322386e-01,  1.20935218e+00],\n",
       "       [ 8.69633462e-01, -4.17104346e-01, -1.19779070e+00, ...,\n",
       "        -1.19779070e+00, -9.54038745e-02,  6.16774826e-02],\n",
       "       ...,\n",
       "       [-1.11400811e+00, -2.24134360e-01, -2.05020784e-01, ...,\n",
       "         4.49935320e-01,  9.33928731e-03, -1.03419883e+00],\n",
       "       [ 3.44788535e-01, -1.14185294e-01, -2.13468333e+00, ...,\n",
       "         1.33517774e+00, -2.42841806e-02,  1.48098193e-01],\n",
       "       [-4.29361443e-01, -6.13579409e-01,  8.64193921e-01, ...,\n",
       "         1.37253909e+00, -2.58751315e-01,  1.12801294e+00]])"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "y"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Filtered genotype matrix shape: (15209, 574)\n"
     ]
    }
   ],
   "source": [
    "\n",
    "print(f\"Filtered genotype matrix shape: {genotype_matrix.shape}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "574"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(y)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>gene_id</th>\n",
       "      <th>#chr</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>window_start</th>\n",
       "      <th>window_end</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>19122</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>chr22</td>\n",
       "      <td>16601886</td>\n",
       "      <td>16601887</td>\n",
       "      <td>16101886</td>\n",
       "      <td>17101887</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19123</th>\n",
       "      <td>ENSG00000283633.1</td>\n",
       "      <td>chr22</td>\n",
       "      <td>16612581</td>\n",
       "      <td>16612582</td>\n",
       "      <td>16112581</td>\n",
       "      <td>17112582</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19124</th>\n",
       "      <td>ENSG00000172967.7</td>\n",
       "      <td>chr22</td>\n",
       "      <td>16821698</td>\n",
       "      <td>16821699</td>\n",
       "      <td>16321698</td>\n",
       "      <td>17321699</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19125</th>\n",
       "      <td>ENSG00000237438.7</td>\n",
       "      <td>chr22</td>\n",
       "      <td>17036569</td>\n",
       "      <td>17036570</td>\n",
       "      <td>16536569</td>\n",
       "      <td>17536570</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19126</th>\n",
       "      <td>ENSG00000273203.1</td>\n",
       "      <td>chr22</td>\n",
       "      <td>17067820</td>\n",
       "      <td>17067821</td>\n",
       "      <td>16567820</td>\n",
       "      <td>17567821</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                  gene_id   #chr     start       end  window_start  window_end\n",
       "19122  ENSG00000100181.22  chr22  16601886  16601887      16101886    17101887\n",
       "19123   ENSG00000283633.1  chr22  16612581  16612582      16112581    17112582\n",
       "19124   ENSG00000172967.7  chr22  16821698  16821699      16321698    17321699\n",
       "19125   ENSG00000237438.7  chr22  17036569  17036570      16536569    17536570\n",
       "19126   ENSG00000273203.1  chr22  17067820  17067821      16567820    17567821"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "genes_on_chrom.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Scale UP cis-eQTL"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100181.22.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100181.22_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100181.22\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100181.22.nosex .\n",
      "--extract range: 15015 variants excluded.\n",
      "--extract range: 194 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "194 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100181.22.bed +\n",
      "./output/chr22_data.ENSG00000100181.22.bim +\n",
      "./output/chr22_data.ENSG00000100181.22.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100181.22.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000283633.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000283633.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000283633.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000283633.1.nosex .\n",
      "--extract range: 15006 variants excluded.\n",
      "--extract range: 203 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "203 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000283633.1.bed +\n",
      "./output/chr22_data.ENSG00000283633.1.bim +\n",
      "./output/chr22_data.ENSG00000283633.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000283633.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000172967.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000172967.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000172967.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000172967.7.nosex .\n",
      "--extract range: 14890 variants excluded.\n",
      "--extract range: 319 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "319 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000172967.7.bed +\n",
      "./output/chr22_data.ENSG00000172967.7.bim +\n",
      "./output/chr22_data.ENSG00000172967.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000172967.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237438.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237438.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237438.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237438.7.nosex .\n",
      "--extract range: 14784 variants excluded.\n",
      "--extract range: 425 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "425 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237438.7.bed +\n",
      "./output/chr22_data.ENSG00000237438.7.bim +\n",
      "./output/chr22_data.ENSG00000237438.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237438.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273203.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273203.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273203.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273203.1.nosex .\n",
      "--extract range: 14764 variants excluded.\n",
      "--extract range: 445 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "445 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273203.1.bed +\n",
      "./output/chr22_data.ENSG00000273203.1.bim +\n",
      "./output/chr22_data.ENSG00000273203.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273203.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000177663.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000177663.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000177663.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000177663.13.nosex .\n",
      "--extract range: 14765 variants excluded.\n",
      "--extract range: 444 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "444 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000177663.13.bed +\n",
      "./output/chr22_data.ENSG00000177663.13.bim +\n",
      "./output/chr22_data.ENSG00000177663.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000177663.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235328.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235328.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235328.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235328.1.nosex .\n",
      "--extract range: 14766 variants excluded.\n",
      "--extract range: 443 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "443 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235328.1.bed +\n",
      "./output/chr22_data.ENSG00000235328.1.bim +\n",
      "./output/chr22_data.ENSG00000235328.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235328.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183307.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183307.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183307.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183307.3.nosex .\n",
      "--extract range: 14755 variants excluded.\n",
      "--extract range: 454 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "454 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183307.3.bed +\n",
      "./output/chr22_data.ENSG00000183307.3.bim +\n",
      "./output/chr22_data.ENSG00000183307.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183307.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235478.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235478.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235478.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235478.4.nosex .\n",
      "--extract range: 14755 variants excluded.\n",
      "--extract range: 454 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "454 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235478.4.bed +\n",
      "./output/chr22_data.ENSG00000235478.4.bim +\n",
      "./output/chr22_data.ENSG00000235478.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235478.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000069998.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000069998.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000069998.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000069998.12.nosex .\n",
      "--extract range: 14747 variants excluded.\n",
      "--extract range: 462 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "462 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000069998.12.bed +\n",
      "./output/chr22_data.ENSG00000069998.12.bim +\n",
      "./output/chr22_data.ENSG00000069998.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000069998.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000093072.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000093072.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000093072.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000093072.15.nosex .\n",
      "--extract range: 14738 variants excluded.\n",
      "--extract range: 471 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "471 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000093072.15.bed +\n",
      "./output/chr22_data.ENSG00000093072.15.bim +\n",
      "./output/chr22_data.ENSG00000093072.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000093072.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182902.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182902.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182902.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182902.13.nosex .\n",
      "--extract range: 14696 variants excluded.\n",
      "--extract range: 513 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "513 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182902.13.bed +\n",
      "./output/chr22_data.ENSG00000182902.13.bim +\n",
      "./output/chr22_data.ENSG00000182902.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182902.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000236754.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000236754.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000236754.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000236754.5.nosex .\n",
      "--extract range: 14687 variants excluded.\n",
      "--extract range: 522 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "522 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000236754.5.bed +\n",
      "./output/chr22_data.ENSG00000236754.5.bim +\n",
      "./output/chr22_data.ENSG00000236754.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000236754.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000131100.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000131100.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000131100.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000131100.12.nosex .\n",
      "--extract range: 14695 variants excluded.\n",
      "--extract range: 514 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "514 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000131100.12.bed +\n",
      "./output/chr22_data.ENSG00000131100.12.bim +\n",
      "./output/chr22_data.ENSG00000131100.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000131100.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099968.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099968.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099968.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099968.17.nosex .\n",
      "--extract range: 14695 variants excluded.\n",
      "--extract range: 514 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "514 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099968.17.bed +\n",
      "./output/chr22_data.ENSG00000099968.17.bim +\n",
      "./output/chr22_data.ENSG00000099968.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099968.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000015475.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000015475.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000015475.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000015475.18.nosex .\n",
      "--extract range: 14756 variants excluded.\n",
      "--extract range: 453 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "453 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000015475.18.bed +\n",
      "./output/chr22_data.ENSG00000015475.18.bim +\n",
      "./output/chr22_data.ENSG00000015475.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000015475.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000269220.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000269220.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000269220.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000269220.1.nosex .\n",
      "--extract range: 14756 variants excluded.\n",
      "--extract range: 453 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "453 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000269220.1.bed +\n",
      "./output/chr22_data.ENSG00000269220.1.bim +\n",
      "./output/chr22_data.ENSG00000269220.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000269220.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000243156.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000243156.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000243156.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000243156.7.nosex .\n",
      "--extract range: 14879 variants excluded.\n",
      "--extract range: 330 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "330 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000243156.7.bed +\n",
      "./output/chr22_data.ENSG00000243156.7.bim +\n",
      "./output/chr22_data.ENSG00000243156.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000243156.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225335.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225335.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225335.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225335.3.nosex .\n",
      "--extract range: 14909 variants excluded.\n",
      "--extract range: 300 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "300 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225335.3.bed +\n",
      "./output/chr22_data.ENSG00000225335.3.bim +\n",
      "./output/chr22_data.ENSG00000225335.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225335.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000215193.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000215193.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000215193.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000215193.12.nosex .\n",
      "--extract range: 14909 variants excluded.\n",
      "--extract range: 300 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "300 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000215193.12.bed +\n",
      "./output/chr22_data.ENSG00000215193.12.bim +\n",
      "./output/chr22_data.ENSG00000215193.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000215193.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183785.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183785.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183785.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183785.14.nosex .\n",
      "--extract range: 14921 variants excluded.\n",
      "--extract range: 288 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "288 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183785.14.bed +\n",
      "./output/chr22_data.ENSG00000183785.14.bim +\n",
      "./output/chr22_data.ENSG00000183785.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183785.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184979.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184979.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184979.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184979.9.nosex .\n",
      "--extract range: 14938 variants excluded.\n",
      "--extract range: 271 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "271 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184979.9.bed +\n",
      "./output/chr22_data.ENSG00000184979.9.bim +\n",
      "./output/chr22_data.ENSG00000184979.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184979.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278558.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278558.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278558.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278558.4.nosex .\n",
      "--extract range: 15108 variants excluded.\n",
      "--extract range: 101 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999917.\n",
      "101 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278558.4.bed +\n",
      "./output/chr22_data.ENSG00000278558.4.bim +\n",
      "./output/chr22_data.ENSG00000278558.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278558.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000274602.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000274602.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000274602.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000274602.4.nosex .\n",
      "--extract range: 15120 variants excluded.\n",
      "--extract range: 89 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999921.\n",
      "89 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000274602.4.bed +\n",
      "./output/chr22_data.ENSG00000274602.4.bim +\n",
      "./output/chr22_data.ENSG00000274602.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000274602.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000275793.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000275793.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000275793.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000275793.1.nosex .\n",
      "--extract range: 15123 variants excluded.\n",
      "--extract range: 86 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999919.\n",
      "86 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000275793.1.bed +\n",
      "./output/chr22_data.ENSG00000275793.1.bim +\n",
      "./output/chr22_data.ENSG00000275793.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000275793.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183628.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183628.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183628.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183628.12.nosex .\n",
      "--extract range: 14997 variants excluded.\n",
      "--extract range: 212 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999941.\n",
      "212 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183628.12.bed +\n",
      "./output/chr22_data.ENSG00000183628.12.bim +\n",
      "./output/chr22_data.ENSG00000183628.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183628.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100033.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100033.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100033.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100033.16.nosex .\n",
      "--extract range: 14991 variants excluded.\n",
      "--extract range: 218 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999942.\n",
      "218 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100033.16.bed +\n",
      "./output/chr22_data.ENSG00000100033.16.bim +\n",
      "./output/chr22_data.ENSG00000100033.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100033.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273032.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273032.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273032.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273032.1.nosex .\n",
      "--extract range: 14946 variants excluded.\n",
      "--extract range: 263 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999952.\n",
      "263 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273032.1.bed +\n",
      "./output/chr22_data.ENSG00000273032.1.bim +\n",
      "./output/chr22_data.ENSG00000273032.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273032.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273164.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273164.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273164.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273164.1.nosex .\n",
      "--extract range: 14940 variants excluded.\n",
      "--extract range: 269 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999953.\n",
      "269 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273164.1.bed +\n",
      "./output/chr22_data.ENSG00000273164.1.bim +\n",
      "./output/chr22_data.ENSG00000273164.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273164.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241527.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241527.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241527.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241527.1.nosex .\n",
      "--extract range: 14935 variants excluded.\n",
      "--extract range: 274 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999954.\n",
      "274 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241527.1.bed +\n",
      "./output/chr22_data.ENSG00000241527.1.bim +\n",
      "./output/chr22_data.ENSG00000241527.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241527.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273311.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273311.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273311.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273311.1.nosex .\n",
      "--extract range: 14932 variants excluded.\n",
      "--extract range: 277 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "277 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273311.1.bed +\n",
      "./output/chr22_data.ENSG00000273311.1.bim +\n",
      "./output/chr22_data.ENSG00000273311.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273311.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000281530.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000281530.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000281530.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000281530.1.nosex .\n",
      "--extract range: 14928 variants excluded.\n",
      "--extract range: 281 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "281 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000281530.1.bed +\n",
      "./output/chr22_data.ENSG00000281530.1.bim +\n",
      "./output/chr22_data.ENSG00000281530.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000281530.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000223461.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000223461.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000223461.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000223461.1.nosex .\n",
      "--extract range: 14894 variants excluded.\n",
      "--extract range: 315 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99996.\n",
      "315 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000223461.1.bed +\n",
      "./output/chr22_data.ENSG00000223461.1.bim +\n",
      "./output/chr22_data.ENSG00000223461.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000223461.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000070413.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000070413.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000070413.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000070413.19.nosex .\n",
      "--extract range: 14892 variants excluded.\n",
      "--extract range: 317 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99996.\n",
      "317 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000070413.19.bed +\n",
      "./output/chr22_data.ENSG00000070413.19.bim +\n",
      "./output/chr22_data.ENSG00000070413.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000070413.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272682.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272682.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272682.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272682.1.nosex .\n",
      "--extract range: 14890 variants excluded.\n",
      "--extract range: 319 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999961.\n",
      "319 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272682.1.bed +\n",
      "./output/chr22_data.ENSG00000272682.1.bim +\n",
      "./output/chr22_data.ENSG00000272682.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272682.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100056.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100056.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100056.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100056.11.nosex .\n",
      "--extract range: 14878 variants excluded.\n",
      "--extract range: 331 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "331 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100056.11.bed +\n",
      "./output/chr22_data.ENSG00000100056.11.bim +\n",
      "./output/chr22_data.ENSG00000100056.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100056.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000260924.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000260924.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000260924.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000260924.2.nosex .\n",
      "--extract range: 14867 variants excluded.\n",
      "--extract range: 342 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "342 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000260924.2.bed +\n",
      "./output/chr22_data.ENSG00000260924.2.bim +\n",
      "./output/chr22_data.ENSG00000260924.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000260924.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100075.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100075.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100075.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100075.9.nosex .\n",
      "--extract range: 14865 variants excluded.\n",
      "--extract range: 344 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "344 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100075.9.bed +\n",
      "./output/chr22_data.ENSG00000100075.9.bim +\n",
      "./output/chr22_data.ENSG00000100075.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100075.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000070371.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000070371.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000070371.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000070371.15.nosex .\n",
      "--extract range: 14814 variants excluded.\n",
      "--extract range: 395 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "395 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000070371.15.bed +\n",
      "./output/chr22_data.ENSG00000070371.15.bim +\n",
      "./output/chr22_data.ENSG00000070371.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000070371.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100084.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100084.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100084.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100084.14.nosex .\n",
      "--extract range: 14738 variants excluded.\n",
      "--extract range: 471 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999973.\n",
      "471 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100084.14.bed +\n",
      "./output/chr22_data.ENSG00000100084.14.bim +\n",
      "./output/chr22_data.ENSG00000100084.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100084.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185608.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185608.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185608.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185608.8.nosex .\n",
      "--extract range: 14738 variants excluded.\n",
      "--extract range: 471 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999973.\n",
      "471 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185608.8.bed +\n",
      "./output/chr22_data.ENSG00000185608.8.bim +\n",
      "./output/chr22_data.ENSG00000185608.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185608.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000242259.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000242259.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000242259.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000242259.8.nosex .\n",
      "--extract range: 14722 variants excluded.\n",
      "--extract range: 487 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "487 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000242259.8.bed +\n",
      "./output/chr22_data.ENSG00000242259.8.bim +\n",
      "./output/chr22_data.ENSG00000242259.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000242259.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185065.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185065.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185065.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185065.6.nosex .\n",
      "--extract range: 14722 variants excluded.\n",
      "--extract range: 487 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "487 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185065.6.bed +\n",
      "./output/chr22_data.ENSG00000185065.6.bim +\n",
      "./output/chr22_data.ENSG00000185065.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185065.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273300.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273300.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273300.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273300.1.nosex .\n",
      "--extract range: 14720 variants excluded.\n",
      "--extract range: 489 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "489 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273300.1.bed +\n",
      "./output/chr22_data.ENSG00000273300.1.bim +\n",
      "./output/chr22_data.ENSG00000273300.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273300.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000070010.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000070010.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000070010.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000070010.18.nosex .\n",
      "--extract range: 14700 variants excluded.\n",
      "--extract range: 509 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "509 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000070010.18.bed +\n",
      "./output/chr22_data.ENSG00000070010.18.bim +\n",
      "./output/chr22_data.ENSG00000070010.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000070010.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000093009.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000093009.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000093009.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000093009.9.nosex .\n",
      "--extract range: 14701 variants excluded.\n",
      "--extract range: 508 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "508 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000093009.9.bed +\n",
      "./output/chr22_data.ENSG00000093009.9.bim +\n",
      "./output/chr22_data.ENSG00000093009.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000093009.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184113.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184113.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184113.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184113.9.nosex .\n",
      "--extract range: 14680 variants excluded.\n",
      "--extract range: 529 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "529 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184113.9.bed +\n",
      "./output/chr22_data.ENSG00000184113.9.bim +\n",
      "./output/chr22_data.ENSG00000184113.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184113.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184702.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184702.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184702.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184702.17.nosex .\n",
      "--extract range: 14688 variants excluded.\n",
      "--extract range: 521 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "521 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184702.17.bed +\n",
      "./output/chr22_data.ENSG00000184702.17.bim +\n",
      "./output/chr22_data.ENSG00000184702.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184702.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184058.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184058.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184058.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184058.12.nosex .\n",
      "--extract range: 14697 variants excluded.\n",
      "--extract range: 512 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "512 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184058.12.bed +\n",
      "./output/chr22_data.ENSG00000184058.12.bim +\n",
      "./output/chr22_data.ENSG00000184058.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184058.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235776.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235776.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235776.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235776.2.nosex .\n",
      "--extract range: 14702 variants excluded.\n",
      "--extract range: 507 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "507 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235776.2.bed +\n",
      "./output/chr22_data.ENSG00000235776.2.bim +\n",
      "./output/chr22_data.ENSG00000235776.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235776.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000215012.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000215012.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000215012.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000215012.8.nosex .\n",
      "--extract range: 14715 variants excluded.\n",
      "--extract range: 494 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "494 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000215012.8.bed +\n",
      "./output/chr22_data.ENSG00000215012.8.bim +\n",
      "./output/chr22_data.ENSG00000215012.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000215012.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185838.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185838.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185838.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185838.13.nosex .\n",
      "--extract range: 14716 variants excluded.\n",
      "--extract range: 493 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "493 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185838.13.bed +\n",
      "./output/chr22_data.ENSG00000185838.13.bim +\n",
      "./output/chr22_data.ENSG00000185838.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185838.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000232926.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000232926.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000232926.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000232926.1.nosex .\n",
      "--extract range: 14717 variants excluded.\n",
      "--extract range: 492 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "492 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000232926.1.bed +\n",
      "./output/chr22_data.ENSG00000232926.1.bim +\n",
      "./output/chr22_data.ENSG00000232926.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000232926.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000093010.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000093010.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000093010.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000093010.13.nosex .\n",
      "--extract range: 14695 variants excluded.\n",
      "--extract range: 514 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "514 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000093010.13.bed +\n",
      "./output/chr22_data.ENSG00000093010.13.bim +\n",
      "./output/chr22_data.ENSG00000093010.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000093010.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184470.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184470.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184470.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184470.20.nosex .\n",
      "--extract range: 14694 variants excluded.\n",
      "--extract range: 515 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "515 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184470.20.bed +\n",
      "./output/chr22_data.ENSG00000184470.20.bim +\n",
      "./output/chr22_data.ENSG00000184470.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184470.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099889.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099889.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099889.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099889.13.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099889.13.bed +\n",
      "./output/chr22_data.ENSG00000099889.13.bim +\n",
      "./output/chr22_data.ENSG00000099889.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099889.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183597.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183597.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183597.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183597.15.nosex .\n",
      "--extract range: 14711 variants excluded.\n",
      "--extract range: 498 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "498 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183597.15.bed +\n",
      "./output/chr22_data.ENSG00000183597.15.bim +\n",
      "./output/chr22_data.ENSG00000183597.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183597.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000236540.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000236540.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000236540.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000236540.7.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000236540.7.bed +\n",
      "./output/chr22_data.ENSG00000236540.7.bim +\n",
      "./output/chr22_data.ENSG00000236540.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000236540.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128191.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128191.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128191.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128191.15.nosex .\n",
      "--extract range: 14709 variants excluded.\n",
      "--extract range: 500 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "500 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128191.15.bed +\n",
      "./output/chr22_data.ENSG00000128191.15.bim +\n",
      "./output/chr22_data.ENSG00000128191.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128191.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099901.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099901.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099901.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099901.16.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099901.16.bed +\n",
      "./output/chr22_data.ENSG00000099901.16.bim +\n",
      "./output/chr22_data.ENSG00000099901.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099901.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099899.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099899.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099899.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099899.14.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099899.14.bed +\n",
      "./output/chr22_data.ENSG00000099899.14.bim +\n",
      "./output/chr22_data.ENSG00000099899.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099899.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099904.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099904.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099904.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099904.15.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099904.15.bed +\n",
      "./output/chr22_data.ENSG00000099904.15.bim +\n",
      "./output/chr22_data.ENSG00000099904.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099904.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000234409.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000234409.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000234409.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000234409.6.nosex .\n",
      "--extract range: 14700 variants excluded.\n",
      "--extract range: 509 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "509 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000234409.6.bed +\n",
      "./output/chr22_data.ENSG00000234409.6.bim +\n",
      "./output/chr22_data.ENSG00000234409.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000234409.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000236499.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000236499.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000236499.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000236499.2.nosex .\n",
      "--extract range: 14710 variants excluded.\n",
      "--extract range: 499 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "499 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000236499.2.bed +\n",
      "./output/chr22_data.ENSG00000236499.2.bim +\n",
      "./output/chr22_data.ENSG00000236499.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000236499.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000040608.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000040608.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000040608.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000040608.13.nosex .\n",
      "--extract range: 14707 variants excluded.\n",
      "--extract range: 502 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "502 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000040608.13.bed +\n",
      "./output/chr22_data.ENSG00000040608.13.bim +\n",
      "./output/chr22_data.ENSG00000040608.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000040608.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273343.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273343.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273343.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273343.1.nosex .\n",
      "--extract range: 14699 variants excluded.\n",
      "--extract range: 510 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "510 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273343.1.bed +\n",
      "./output/chr22_data.ENSG00000273343.1.bim +\n",
      "./output/chr22_data.ENSG00000273343.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273343.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128185.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128185.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128185.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128185.9.nosex .\n",
      "--extract range: 14698 variants excluded.\n",
      "--extract range: 511 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "511 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128185.9.bed +\n",
      "./output/chr22_data.ENSG00000128185.9.bim +\n",
      "./output/chr22_data.ENSG00000128185.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128185.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185252.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185252.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185252.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185252.18.nosex .\n",
      "--extract range: 14721 variants excluded.\n",
      "--extract range: 488 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "488 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185252.18.bed +\n",
      "./output/chr22_data.ENSG00000185252.18.bim +\n",
      "./output/chr22_data.ENSG00000185252.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185252.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000244486.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000244486.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000244486.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000244486.8.nosex .\n",
      "--extract range: 14733 variants excluded.\n",
      "--extract range: 476 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "476 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000244486.8.bed +\n",
      "./output/chr22_data.ENSG00000244486.8.bim +\n",
      "./output/chr22_data.ENSG00000244486.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000244486.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000215493.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000215493.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000215493.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000215493.3.nosex .\n",
      "--extract range: 14735 variants excluded.\n",
      "--extract range: 474 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "474 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000215493.3.bed +\n",
      "./output/chr22_data.ENSG00000215493.3.bim +\n",
      "./output/chr22_data.ENSG00000215493.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000215493.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000236670.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000236670.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000236670.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000236670.1.nosex .\n",
      "--extract range: 14733 variants excluded.\n",
      "--extract range: 476 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "476 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000236670.1.bed +\n",
      "./output/chr22_data.ENSG00000236670.1.bim +\n",
      "./output/chr22_data.ENSG00000236670.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000236670.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099910.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099910.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099910.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099910.16.nosex .\n",
      "--extract range: 14734 variants excluded.\n",
      "--extract range: 475 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "475 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099910.16.bed +\n",
      "./output/chr22_data.ENSG00000099910.16.bim +\n",
      "./output/chr22_data.ENSG00000099910.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099910.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099917.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099917.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099917.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099917.17.nosex .\n",
      "--extract range: 14734 variants excluded.\n",
      "--extract range: 475 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "475 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099917.17.bed +\n",
      "./output/chr22_data.ENSG00000099917.17.bim +\n",
      "./output/chr22_data.ENSG00000099917.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099917.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000223553.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000223553.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000223553.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000223553.6.nosex .\n",
      "--extract range: 14742 variants excluded.\n",
      "--extract range: 467 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "467 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000223553.6.bed +\n",
      "./output/chr22_data.ENSG00000223553.6.bim +\n",
      "./output/chr22_data.ENSG00000223553.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000223553.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000229107.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000229107.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000229107.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000229107.2.nosex .\n",
      "--extract range: 14748 variants excluded.\n",
      "--extract range: 461 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "461 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000229107.2.bed +\n",
      "./output/chr22_data.ENSG00000229107.2.bim +\n",
      "./output/chr22_data.ENSG00000229107.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000229107.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226287.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226287.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226287.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226287.8.nosex .\n",
      "--extract range: 14772 variants excluded.\n",
      "--extract range: 437 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "437 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226287.8.bed +\n",
      "./output/chr22_data.ENSG00000226287.8.bim +\n",
      "./output/chr22_data.ENSG00000226287.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226287.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099937.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099937.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099937.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099937.10.nosex .\n",
      "--extract range: 14801 variants excluded.\n",
      "--extract range: 408 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "408 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099937.10.bed +\n",
      "./output/chr22_data.ENSG00000099937.10.bim +\n",
      "./output/chr22_data.ENSG00000099937.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099937.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099940.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099940.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099940.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099940.11.nosex .\n",
      "--extract range: 14815 variants excluded.\n",
      "--extract range: 394 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "394 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099940.11.bed +\n",
      "./output/chr22_data.ENSG00000099940.11.bim +\n",
      "./output/chr22_data.ENSG00000099940.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099940.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241973.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241973.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241973.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241973.10.nosex .\n",
      "--extract range: 14815 variants excluded.\n",
      "--extract range: 394 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "394 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241973.10.bed +\n",
      "./output/chr22_data.ENSG00000241973.10.bim +\n",
      "./output/chr22_data.ENSG00000241973.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241973.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099942.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099942.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099942.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099942.12.nosex .\n",
      "--extract range: 14841 variants excluded.\n",
      "--extract range: 368 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "368 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099942.12.bed +\n",
      "./output/chr22_data.ENSG00000099942.12.bim +\n",
      "./output/chr22_data.ENSG00000099942.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099942.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237476.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237476.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237476.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237476.1.nosex .\n",
      "--extract range: 14862 variants excluded.\n",
      "--extract range: 347 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "347 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237476.1.bed +\n",
      "./output/chr22_data.ENSG00000237476.1.bim +\n",
      "./output/chr22_data.ENSG00000237476.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237476.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183773.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183773.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183773.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183773.15.nosex .\n",
      "--extract range: 14863 variants excluded.\n",
      "--extract range: 346 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "346 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183773.15.bed +\n",
      "./output/chr22_data.ENSG00000183773.15.bim +\n",
      "./output/chr22_data.ENSG00000183773.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183773.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099949.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099949.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099949.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099949.18.nosex .\n",
      "--extract range: 14869 variants excluded.\n",
      "--extract range: 340 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "340 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099949.18.bed +\n",
      "./output/chr22_data.ENSG00000099949.18.bim +\n",
      "./output/chr22_data.ENSG00000099949.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099949.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272829.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272829.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272829.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272829.1.nosex .\n",
      "--extract range: 14871 variants excluded.\n",
      "--extract range: 338 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "338 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272829.1.bed +\n",
      "./output/chr22_data.ENSG00000272829.1.bim +\n",
      "./output/chr22_data.ENSG00000272829.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272829.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184436.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184436.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184436.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184436.11.nosex .\n",
      "--extract range: 14877 variants excluded.\n",
      "--extract range: 332 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "332 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184436.11.bed +\n",
      "./output/chr22_data.ENSG00000184436.11.bim +\n",
      "./output/chr22_data.ENSG00000184436.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184436.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000230513.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000230513.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000230513.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000230513.1.nosex .\n",
      "--extract range: 14877 variants excluded.\n",
      "--extract range: 332 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "332 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000230513.1.bed +\n",
      "./output/chr22_data.ENSG00000230513.1.bim +\n",
      "./output/chr22_data.ENSG00000230513.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000230513.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099957.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099957.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099957.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099957.16.nosex .\n",
      "--extract range: 14880 variants excluded.\n",
      "--extract range: 329 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "329 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099957.16.bed +\n",
      "./output/chr22_data.ENSG00000099957.16.bim +\n",
      "./output/chr22_data.ENSG00000099957.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099957.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000169635.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000169635.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000169635.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000169635.9.nosex .\n",
      "--extract range: 14939 variants excluded.\n",
      "--extract range: 270 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "270 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000169635.9.bed +\n",
      "./output/chr22_data.ENSG00000169635.9.bim +\n",
      "./output/chr22_data.ENSG00000169635.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000169635.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000206140.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000206140.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000206140.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000206140.10.nosex .\n",
      "--extract range: 14942 variants excluded.\n",
      "--extract range: 267 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "267 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000206140.10.bed +\n",
      "./output/chr22_data.ENSG00000206140.10.bim +\n",
      "./output/chr22_data.ENSG00000206140.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000206140.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183506.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183506.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183506.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183506.17.nosex .\n",
      "--extract range: 14965 variants excluded.\n",
      "--extract range: 244 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "244 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183506.17.bed +\n",
      "./output/chr22_data.ENSG00000183506.17.bim +\n",
      "./output/chr22_data.ENSG00000183506.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183506.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185651.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185651.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185651.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185651.14.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185651.14.bed +\n",
      "./output/chr22_data.ENSG00000185651.14.bim +\n",
      "./output/chr22_data.ENSG00000185651.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185651.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000161179.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000161179.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000161179.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000161179.13.nosex .\n",
      "--extract range: 14959 variants excluded.\n",
      "--extract range: 250 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "250 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000161179.13.bed +\n",
      "./output/chr22_data.ENSG00000161179.13.bim +\n",
      "./output/chr22_data.ENSG00000161179.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000161179.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000161180.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000161180.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000161180.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000161180.10.nosex .\n",
      "--extract range: 14959 variants excluded.\n",
      "--extract range: 250 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "250 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000161180.10.bed +\n",
      "./output/chr22_data.ENSG00000161180.10.bim +\n",
      "./output/chr22_data.ENSG00000161180.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000161180.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128228.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128228.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128228.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128228.4.nosex .\n",
      "--extract range: 14958 variants excluded.\n",
      "--extract range: 251 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "251 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128228.4.bed +\n",
      "./output/chr22_data.ENSG00000128228.4.bim +\n",
      "./output/chr22_data.ENSG00000128228.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128228.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272954.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272954.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272954.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272954.1.nosex .\n",
      "--extract range: 14948 variants excluded.\n",
      "--extract range: 261 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "261 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272954.1.bed +\n",
      "./output/chr22_data.ENSG00000272954.1.bim +\n",
      "./output/chr22_data.ENSG00000272954.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272954.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100023.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100023.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100023.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100023.18.nosex .\n",
      "--extract range: 14937 variants excluded.\n",
      "--extract range: 272 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "272 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100023.18.bed +\n",
      "./output/chr22_data.ENSG00000100023.18.bim +\n",
      "./output/chr22_data.ENSG00000100023.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100023.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100027.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100027.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100027.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100027.14.nosex .\n",
      "--extract range: 14893 variants excluded.\n",
      "--extract range: 316 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "316 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100027.14.bed +\n",
      "./output/chr22_data.ENSG00000100027.14.bim +\n",
      "./output/chr22_data.ENSG00000100027.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100027.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100030.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100030.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100030.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100030.14.nosex .\n",
      "--extract range: 14879 variants excluded.\n",
      "--extract range: 330 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "330 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100030.14.bed +\n",
      "./output/chr22_data.ENSG00000100030.14.bim +\n",
      "./output/chr22_data.ENSG00000100030.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100030.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000224086.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000224086.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000224086.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000224086.5.nosex .\n",
      "--extract range: 14869 variants excluded.\n",
      "--extract range: 340 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "340 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000224086.5.bed +\n",
      "./output/chr22_data.ENSG00000224086.5.bim +\n",
      "./output/chr22_data.ENSG00000224086.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000224086.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100034.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100034.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100034.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100034.13.nosex .\n",
      "--extract range: 14869 variants excluded.\n",
      "--extract range: 340 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "340 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100034.13.bed +\n",
      "./output/chr22_data.ENSG00000100034.13.bim +\n",
      "./output/chr22_data.ENSG00000100034.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100034.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100038.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100038.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100038.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100038.19.nosex .\n",
      "--extract range: 14876 variants excluded.\n",
      "--extract range: 333 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "333 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100038.19.bed +\n",
      "./output/chr22_data.ENSG00000100038.19.bim +\n",
      "./output/chr22_data.ENSG00000100038.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100038.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000197549.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000197549.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000197549.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000197549.9.nosex .\n",
      "--extract range: 14870 variants excluded.\n",
      "--extract range: 339 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "339 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000197549.9.bed +\n",
      "./output/chr22_data.ENSG00000197549.9.bim +\n",
      "./output/chr22_data.ENSG00000197549.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000197549.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000253239.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000253239.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000253239.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000253239.1.nosex .\n",
      "--extract range: 14870 variants excluded.\n",
      "--extract range: 339 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "339 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000253239.1.bed +\n",
      "./output/chr22_data.ENSG00000253239.1.bim +\n",
      "./output/chr22_data.ENSG00000253239.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000253239.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211637.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211637.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211637.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211637.2.nosex .\n",
      "--extract range: 14870 variants excluded.\n",
      "--extract range: 339 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "339 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211637.2.bed +\n",
      "./output/chr22_data.ENSG00000211637.2.bim +\n",
      "./output/chr22_data.ENSG00000211637.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211637.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211638.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211638.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211638.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211638.2.nosex .\n",
      "--extract range: 14878 variants excluded.\n",
      "--extract range: 331 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "331 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211638.2.bed +\n",
      "./output/chr22_data.ENSG00000211638.2.bim +\n",
      "./output/chr22_data.ENSG00000211638.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211638.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000263366.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000263366.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000263366.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000263366.2.nosex .\n",
      "--extract range: 14880 variants excluded.\n",
      "--extract range: 329 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "329 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000263366.2.bed +\n",
      "./output/chr22_data.ENSG00000263366.2.bim +\n",
      "./output/chr22_data.ENSG00000263366.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000263366.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211639.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211639.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211639.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211639.2.nosex .\n",
      "--extract range: 14898 variants excluded.\n",
      "--extract range: 311 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "311 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211639.2.bed +\n",
      "./output/chr22_data.ENSG00000211639.2.bim +\n",
      "./output/chr22_data.ENSG00000211639.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211639.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211640.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211640.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211640.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211640.4.nosex .\n",
      "--extract range: 14910 variants excluded.\n",
      "--extract range: 299 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "299 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211640.4.bed +\n",
      "./output/chr22_data.ENSG00000211640.4.bim +\n",
      "./output/chr22_data.ENSG00000211640.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211640.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211642.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211642.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211642.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211642.3.nosex .\n",
      "--extract range: 14920 variants excluded.\n",
      "--extract range: 289 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "289 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211642.3.bed +\n",
      "./output/chr22_data.ENSG00000211642.3.bim +\n",
      "./output/chr22_data.ENSG00000211642.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211642.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000228050.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000228050.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000228050.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000228050.1.nosex .\n",
      "--extract range: 14921 variants excluded.\n",
      "--extract range: 288 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "288 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000228050.1.bed +\n",
      "./output/chr22_data.ENSG00000228050.1.bim +\n",
      "./output/chr22_data.ENSG00000228050.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000228050.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279278.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279278.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279278.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279278.1.nosex .\n",
      "--extract range: 14939 variants excluded.\n",
      "--extract range: 270 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "270 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279278.1.bed +\n",
      "./output/chr22_data.ENSG00000279278.1.bim +\n",
      "./output/chr22_data.ENSG00000279278.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279278.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000234630.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000234630.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000234630.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000234630.1.nosex .\n",
      "--extract range: 14949 variants excluded.\n",
      "--extract range: 260 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "260 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000234630.1.bed +\n",
      "./output/chr22_data.ENSG00000234630.1.bim +\n",
      "./output/chr22_data.ENSG00000234630.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000234630.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272779.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272779.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272779.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272779.1.nosex .\n",
      "--extract range: 14952 variants excluded.\n",
      "--extract range: 257 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "257 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272779.1.bed +\n",
      "./output/chr22_data.ENSG00000272779.1.bim +\n",
      "./output/chr22_data.ENSG00000272779.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272779.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211643.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211643.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211643.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211643.2.nosex .\n",
      "--extract range: 14953 variants excluded.\n",
      "--extract range: 256 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "256 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211643.2.bed +\n",
      "./output/chr22_data.ENSG00000211643.2.bim +\n",
      "./output/chr22_data.ENSG00000211643.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211643.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211644.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211644.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211644.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211644.3.nosex .\n",
      "--extract range: 14954 variants excluded.\n",
      "--extract range: 255 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "255 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211644.3.bed +\n",
      "./output/chr22_data.ENSG00000211644.3.bim +\n",
      "./output/chr22_data.ENSG00000211644.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211644.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211645.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211645.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211645.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211645.2.nosex .\n",
      "--extract range: 14955 variants excluded.\n",
      "--extract range: 254 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "254 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211645.2.bed +\n",
      "./output/chr22_data.ENSG00000211645.2.bim +\n",
      "./output/chr22_data.ENSG00000211645.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211645.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000223350.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000223350.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000223350.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000223350.2.nosex .\n",
      "--extract range: 14962 variants excluded.\n",
      "--extract range: 247 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "247 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000223350.2.bed +\n",
      "./output/chr22_data.ENSG00000223350.2.bim +\n",
      "./output/chr22_data.ENSG00000223350.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000223350.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211647.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211647.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211647.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211647.1.nosex .\n",
      "--extract range: 14965 variants excluded.\n",
      "--extract range: 244 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "244 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211647.1.bed +\n",
      "./output/chr22_data.ENSG00000211647.1.bim +\n",
      "./output/chr22_data.ENSG00000211647.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211647.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211648.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211648.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211648.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211648.2.nosex .\n",
      "--extract range: 14966 variants excluded.\n",
      "--extract range: 243 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "243 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211648.2.bed +\n",
      "./output/chr22_data.ENSG00000211648.2.bim +\n",
      "./output/chr22_data.ENSG00000211648.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211648.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211649.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211649.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211649.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211649.3.nosex .\n",
      "--extract range: 14968 variants excluded.\n",
      "--extract range: 241 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "241 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211649.3.bed +\n",
      "./output/chr22_data.ENSG00000211649.3.bim +\n",
      "./output/chr22_data.ENSG00000211649.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211649.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211650.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211650.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211650.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211650.2.nosex .\n",
      "--extract range: 14968 variants excluded.\n",
      "--extract range: 241 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "241 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211650.2.bed +\n",
      "./output/chr22_data.ENSG00000211650.2.bim +\n",
      "./output/chr22_data.ENSG00000211650.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211650.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211651.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211651.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211651.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211651.3.nosex .\n",
      "--extract range: 14968 variants excluded.\n",
      "--extract range: 241 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "241 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211651.3.bed +\n",
      "./output/chr22_data.ENSG00000211651.3.bim +\n",
      "./output/chr22_data.ENSG00000211651.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211651.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211652.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211652.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211652.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211652.2.nosex .\n",
      "--extract range: 14969 variants excluded.\n",
      "--extract range: 240 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "240 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211652.2.bed +\n",
      "./output/chr22_data.ENSG00000211652.2.bim +\n",
      "./output/chr22_data.ENSG00000211652.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211652.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000253818.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000253818.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000253818.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000253818.1.nosex .\n",
      "--extract range: 14970 variants excluded.\n",
      "--extract range: 239 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "239 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000253818.1.bed +\n",
      "./output/chr22_data.ENSG00000253818.1.bim +\n",
      "./output/chr22_data.ENSG00000253818.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000253818.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211653.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211653.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211653.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211653.2.nosex .\n",
      "--extract range: 14971 variants excluded.\n",
      "--extract range: 238 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "238 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211653.2.bed +\n",
      "./output/chr22_data.ENSG00000211653.2.bim +\n",
      "./output/chr22_data.ENSG00000211653.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211653.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211654.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211654.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211654.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211654.2.nosex .\n",
      "--extract range: 14963 variants excluded.\n",
      "--extract range: 246 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "246 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211654.2.bed +\n",
      "./output/chr22_data.ENSG00000211654.2.bim +\n",
      "./output/chr22_data.ENSG00000211654.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211654.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211655.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211655.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211655.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211655.3.nosex .\n",
      "--extract range: 14965 variants excluded.\n",
      "--extract range: 244 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "244 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211655.3.bed +\n",
      "./output/chr22_data.ENSG00000211655.3.bim +\n",
      "./output/chr22_data.ENSG00000211655.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211655.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000275004.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000275004.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000275004.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000275004.3.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000275004.3.bed +\n",
      "./output/chr22_data.ENSG00000275004.3.bim +\n",
      "./output/chr22_data.ENSG00000275004.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000275004.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185686.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185686.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185686.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185686.17.nosex .\n",
      "--extract range: 14973 variants excluded.\n",
      "--extract range: 236 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "236 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185686.17.bed +\n",
      "./output/chr22_data.ENSG00000185686.17.bim +\n",
      "./output/chr22_data.ENSG00000185686.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185686.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211658.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211658.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211658.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211658.2.nosex .\n",
      "--extract range: 14989 variants excluded.\n",
      "--extract range: 220 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999994.\n",
      "220 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211658.2.bed +\n",
      "./output/chr22_data.ENSG00000211658.2.bim +\n",
      "./output/chr22_data.ENSG00000211658.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211658.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211659.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211659.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211659.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211659.2.nosex .\n",
      "--extract range: 15007 variants excluded.\n",
      "--extract range: 202 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "202 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211659.2.bed +\n",
      "./output/chr22_data.ENSG00000211659.2.bim +\n",
      "./output/chr22_data.ENSG00000211659.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211659.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211660.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211660.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211660.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211660.3.nosex .\n",
      "--extract range: 15009 variants excluded.\n",
      "--extract range: 200 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "200 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211660.3.bed +\n",
      "./output/chr22_data.ENSG00000211660.3.bim +\n",
      "./output/chr22_data.ENSG00000211660.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211660.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211662.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211662.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211662.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211662.2.nosex .\n",
      "--extract range: 15012 variants excluded.\n",
      "--extract range: 197 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "197 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211662.2.bed +\n",
      "./output/chr22_data.ENSG00000211662.2.bim +\n",
      "./output/chr22_data.ENSG00000211662.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211662.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211663.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211663.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211663.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211663.2.nosex .\n",
      "--extract range: 15013 variants excluded.\n",
      "--extract range: 196 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "196 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211663.2.bed +\n",
      "./output/chr22_data.ENSG00000211663.2.bim +\n",
      "./output/chr22_data.ENSG00000211663.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211663.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211664.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211664.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211664.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211664.2.nosex .\n",
      "--extract range: 15012 variants excluded.\n",
      "--extract range: 197 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "197 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211664.2.bed +\n",
      "./output/chr22_data.ENSG00000211664.2.bim +\n",
      "./output/chr22_data.ENSG00000211664.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211664.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211665.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211665.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211665.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211665.2.nosex .\n",
      "--extract range: 15014 variants excluded.\n",
      "--extract range: 195 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "195 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211665.2.bed +\n",
      "./output/chr22_data.ENSG00000211665.2.bim +\n",
      "./output/chr22_data.ENSG00000211665.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211665.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211666.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211666.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211666.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211666.2.nosex .\n",
      "--extract range: 15011 variants excluded.\n",
      "--extract range: 198 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "198 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211666.2.bed +\n",
      "./output/chr22_data.ENSG00000211666.2.bim +\n",
      "./output/chr22_data.ENSG00000211666.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211666.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211667.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211667.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211667.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211667.2.nosex .\n",
      "--extract range: 15008 variants excluded.\n",
      "--extract range: 201 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "201 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211667.2.bed +\n",
      "./output/chr22_data.ENSG00000211667.2.bim +\n",
      "./output/chr22_data.ENSG00000211667.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211667.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000275772.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000275772.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000275772.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000275772.1.nosex .\n",
      "--extract range: 14994 variants excluded.\n",
      "--extract range: 215 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "215 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000275772.1.bed +\n",
      "./output/chr22_data.ENSG00000275772.1.bim +\n",
      "./output/chr22_data.ENSG00000275772.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000275772.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211668.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211668.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211668.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211668.2.nosex .\n",
      "--extract range: 14985 variants excluded.\n",
      "--extract range: 224 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "224 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211668.2.bed +\n",
      "./output/chr22_data.ENSG00000211668.2.bim +\n",
      "./output/chr22_data.ENSG00000211668.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211668.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211669.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211669.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211669.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211669.2.nosex .\n",
      "--extract range: 14970 variants excluded.\n",
      "--extract range: 239 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "239 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211669.2.bed +\n",
      "./output/chr22_data.ENSG00000211669.2.bim +\n",
      "./output/chr22_data.ENSG00000211669.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211669.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211670.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211670.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211670.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211670.2.nosex .\n",
      "--extract range: 14970 variants excluded.\n",
      "--extract range: 239 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "239 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211670.2.bed +\n",
      "./output/chr22_data.ENSG00000211670.2.bim +\n",
      "./output/chr22_data.ENSG00000211670.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211670.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278196.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278196.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278196.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278196.3.nosex .\n",
      "--extract range: 14970 variants excluded.\n",
      "--extract range: 239 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "239 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278196.3.bed +\n",
      "./output/chr22_data.ENSG00000278196.3.bim +\n",
      "./output/chr22_data.ENSG00000278196.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278196.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000253234.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000253234.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000253234.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000253234.1.nosex .\n",
      "--extract range: 14971 variants excluded.\n",
      "--extract range: 238 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "238 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000253234.1.bed +\n",
      "./output/chr22_data.ENSG00000253234.1.bim +\n",
      "./output/chr22_data.ENSG00000253234.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000253234.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211672.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211672.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211672.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211672.2.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211672.2.bed +\n",
      "./output/chr22_data.ENSG00000211672.2.bim +\n",
      "./output/chr22_data.ENSG00000211672.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211672.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211673.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211673.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211673.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211673.2.nosex .\n",
      "--extract range: 14961 variants excluded.\n",
      "--extract range: 248 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "248 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211673.2.bed +\n",
      "./output/chr22_data.ENSG00000211673.2.bim +\n",
      "./output/chr22_data.ENSG00000211673.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211673.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211674.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211674.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211674.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211674.2.nosex .\n",
      "--extract range: 14956 variants excluded.\n",
      "--extract range: 253 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "253 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211674.2.bed +\n",
      "./output/chr22_data.ENSG00000211674.2.bim +\n",
      "./output/chr22_data.ENSG00000211674.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211674.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211675.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211675.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211675.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211675.2.nosex .\n",
      "--extract range: 14956 variants excluded.\n",
      "--extract range: 253 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "253 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211675.2.bed +\n",
      "./output/chr22_data.ENSG00000211675.2.bim +\n",
      "./output/chr22_data.ENSG00000211675.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211675.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211676.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211676.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211676.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211676.2.nosex .\n",
      "--extract range: 14957 variants excluded.\n",
      "--extract range: 252 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "252 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211676.2.bed +\n",
      "./output/chr22_data.ENSG00000211676.2.bim +\n",
      "./output/chr22_data.ENSG00000211676.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211676.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211677.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211677.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211677.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211677.2.nosex .\n",
      "--extract range: 14956 variants excluded.\n",
      "--extract range: 253 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "253 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211677.2.bed +\n",
      "./output/chr22_data.ENSG00000211677.2.bim +\n",
      "./output/chr22_data.ENSG00000211677.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211677.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211678.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211678.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211678.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211678.2.nosex .\n",
      "--extract range: 14956 variants excluded.\n",
      "--extract range: 253 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "253 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211678.2.bed +\n",
      "./output/chr22_data.ENSG00000211678.2.bim +\n",
      "./output/chr22_data.ENSG00000211678.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211678.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211679.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211679.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211679.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211679.2.nosex .\n",
      "--extract range: 14955 variants excluded.\n",
      "--extract range: 254 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "254 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211679.2.bed +\n",
      "./output/chr22_data.ENSG00000211679.2.bim +\n",
      "./output/chr22_data.ENSG00000211679.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211679.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000254030.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000254030.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000254030.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000254030.1.nosex .\n",
      "--extract range: 14951 variants excluded.\n",
      "--extract range: 258 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "258 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000254030.1.bed +\n",
      "./output/chr22_data.ENSG00000254030.1.bim +\n",
      "./output/chr22_data.ENSG00000254030.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000254030.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000222037.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000222037.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000222037.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000222037.5.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000222037.5.bed +\n",
      "./output/chr22_data.ENSG00000222037.5.bim +\n",
      "./output/chr22_data.ENSG00000222037.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000222037.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000211685.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000211685.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000211685.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000211685.3.nosex .\n",
      "--extract range: 14943 variants excluded.\n",
      "--extract range: 266 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "266 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000211685.3.bed +\n",
      "./output/chr22_data.ENSG00000211685.3.bim +\n",
      "./output/chr22_data.ENSG00000211685.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000211685.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128266.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128266.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128266.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128266.8.nosex .\n",
      "--extract range: 14899 variants excluded.\n",
      "--extract range: 310 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "310 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128266.8.bed +\n",
      "./output/chr22_data.ENSG00000128266.8.bim +\n",
      "./output/chr22_data.ENSG00000128266.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128266.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100218.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100218.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100218.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100218.11.nosex .\n",
      "--extract range: 14876 variants excluded.\n",
      "--extract range: 333 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "333 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100218.11.bed +\n",
      "./output/chr22_data.ENSG00000100218.11.bim +\n",
      "./output/chr22_data.ENSG00000100218.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100218.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100228.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100228.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100228.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100228.12.nosex .\n",
      "--extract range: 14876 variants excluded.\n",
      "--extract range: 333 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "333 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100228.12.bed +\n",
      "./output/chr22_data.ENSG00000100228.12.bim +\n",
      "./output/chr22_data.ENSG00000100228.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100228.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000186716.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000186716.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000186716.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000186716.20.nosex .\n",
      "--extract range: 14871 variants excluded.\n",
      "--extract range: 338 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "338 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000186716.20.bed +\n",
      "./output/chr22_data.ENSG00000186716.20.bim +\n",
      "./output/chr22_data.ENSG00000186716.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000186716.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000240160.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000240160.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000240160.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000240160.3.nosex .\n",
      "--extract range: 14838 variants excluded.\n",
      "--extract range: 371 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "371 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000240160.3.bed +\n",
      "./output/chr22_data.ENSG00000240160.3.bim +\n",
      "./output/chr22_data.ENSG00000240160.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000240160.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000230701.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000230701.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000230701.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000230701.2.nosex .\n",
      "--extract range: 14838 variants excluded.\n",
      "--extract range: 371 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "371 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000230701.2.bed +\n",
      "./output/chr22_data.ENSG00000230701.2.bim +\n",
      "./output/chr22_data.ENSG00000230701.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000230701.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128322.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128322.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128322.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128322.6.nosex .\n",
      "--extract range: 14767 variants excluded.\n",
      "--extract range: 442 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "442 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128322.6.bed +\n",
      "./output/chr22_data.ENSG00000128322.6.bim +\n",
      "./output/chr22_data.ENSG00000128322.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128322.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272733.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272733.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272733.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272733.1.nosex .\n",
      "--extract range: 14770 variants excluded.\n",
      "--extract range: 439 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "439 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272733.1.bed +\n",
      "./output/chr22_data.ENSG00000272733.1.bim +\n",
      "./output/chr22_data.ENSG00000272733.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272733.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000189269.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000189269.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000189269.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000189269.12.nosex .\n",
      "--extract range: 14772 variants excluded.\n",
      "--extract range: 437 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "437 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000189269.12.bed +\n",
      "./output/chr22_data.ENSG00000189269.12.bim +\n",
      "./output/chr22_data.ENSG00000189269.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000189269.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000234353.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000234353.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000234353.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000234353.2.nosex .\n",
      "--extract range: 14774 variants excluded.\n",
      "--extract range: 435 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "435 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000234353.2.bed +\n",
      "./output/chr22_data.ENSG00000234353.2.bim +\n",
      "./output/chr22_data.ENSG00000234353.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000234353.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000244723.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000244723.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000244723.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000244723.3.nosex .\n",
      "--extract range: 14773 variants excluded.\n",
      "--extract range: 436 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "436 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000244723.3.bed +\n",
      "./output/chr22_data.ENSG00000244723.3.bim +\n",
      "./output/chr22_data.ENSG00000244723.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000244723.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000159496.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000159496.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000159496.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000159496.14.nosex .\n",
      "--extract range: 14786 variants excluded.\n",
      "--extract range: 423 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "423 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000159496.14.bed +\n",
      "./output/chr22_data.ENSG00000159496.14.bim +\n",
      "./output/chr22_data.ENSG00000159496.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000159496.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272578.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272578.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272578.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272578.5.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272578.5.bed +\n",
      "./output/chr22_data.ENSG00000272578.5.bim +\n",
      "./output/chr22_data.ENSG00000272578.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272578.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000187792.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000187792.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000187792.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000187792.4.nosex .\n",
      "--extract range: 14781 variants excluded.\n",
      "--extract range: 428 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "428 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000187792.4.bed +\n",
      "./output/chr22_data.ENSG00000187792.4.bim +\n",
      "./output/chr22_data.ENSG00000187792.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000187792.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128218.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128218.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128218.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128218.7.nosex .\n",
      "--extract range: 14781 variants excluded.\n",
      "--extract range: 428 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "428 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128218.7.bed +\n",
      "./output/chr22_data.ENSG00000128218.7.bim +\n",
      "./output/chr22_data.ENSG00000128218.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128218.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000169314.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000169314.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000169314.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000169314.14.nosex .\n",
      "--extract range: 14784 variants excluded.\n",
      "--extract range: 425 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "425 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000169314.14.bed +\n",
      "./output/chr22_data.ENSG00000169314.14.bim +\n",
      "./output/chr22_data.ENSG00000169314.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000169314.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000250479.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000250479.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000250479.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000250479.8.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000250479.8.bed +\n",
      "./output/chr22_data.ENSG00000250479.8.bim +\n",
      "./output/chr22_data.ENSG00000250479.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000250479.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099953.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099953.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099953.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099953.9.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099953.9.bed +\n",
      "./output/chr22_data.ENSG00000099953.9.bim +\n",
      "./output/chr22_data.ENSG00000099953.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099953.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099956.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099956.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099956.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099956.18.nosex .\n",
      "--extract range: 14795 variants excluded.\n",
      "--extract range: 414 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "414 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099956.18.bed +\n",
      "./output/chr22_data.ENSG00000099956.18.bim +\n",
      "./output/chr22_data.ENSG00000099956.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099956.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099958.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099958.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099958.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099958.14.nosex .\n",
      "--extract range: 14813 variants excluded.\n",
      "--extract range: 396 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "396 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099958.14.bed +\n",
      "./output/chr22_data.ENSG00000099958.14.bim +\n",
      "./output/chr22_data.ENSG00000099958.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099958.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133460.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133460.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133460.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133460.19.nosex .\n",
      "--extract range: 14810 variants excluded.\n",
      "--extract range: 399 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "399 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133460.19.bed +\n",
      "./output/chr22_data.ENSG00000133460.19.bim +\n",
      "./output/chr22_data.ENSG00000133460.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133460.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000240972.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000240972.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000240972.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000240972.1.nosex .\n",
      "--extract range: 14825 variants excluded.\n",
      "--extract range: 384 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "384 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000240972.1.bed +\n",
      "./output/chr22_data.ENSG00000240972.1.bim +\n",
      "./output/chr22_data.ENSG00000240972.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000240972.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000218537.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000218537.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000218537.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000218537.1.nosex .\n",
      "--extract range: 14824 variants excluded.\n",
      "--extract range: 385 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "385 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000218537.1.bed +\n",
      "./output/chr22_data.ENSG00000218537.1.bim +\n",
      "./output/chr22_data.ENSG00000218537.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000218537.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273295.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273295.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273295.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273295.1.nosex .\n",
      "--extract range: 14825 variants excluded.\n",
      "--extract range: 384 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "384 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273295.1.bed +\n",
      "./output/chr22_data.ENSG00000273295.1.bim +\n",
      "./output/chr22_data.ENSG00000273295.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273295.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225282.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225282.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225282.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225282.1.nosex .\n",
      "--extract range: 14834 variants excluded.\n",
      "--extract range: 375 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "375 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225282.1.bed +\n",
      "./output/chr22_data.ENSG00000225282.1.bim +\n",
      "./output/chr22_data.ENSG00000225282.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225282.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099974.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099974.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099974.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099974.7.nosex .\n",
      "--extract range: 14823 variants excluded.\n",
      "--extract range: 386 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "386 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099974.7.bed +\n",
      "./output/chr22_data.ENSG00000099974.7.bim +\n",
      "./output/chr22_data.ENSG00000099974.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099974.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272787.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272787.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272787.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272787.1.nosex .\n",
      "--extract range: 14824 variants excluded.\n",
      "--extract range: 385 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "385 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272787.1.bed +\n",
      "./output/chr22_data.ENSG00000272787.1.bim +\n",
      "./output/chr22_data.ENSG00000272787.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272787.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099977.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099977.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099977.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099977.13.nosex .\n",
      "--extract range: 14829 variants excluded.\n",
      "--extract range: 380 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "380 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099977.13.bed +\n",
      "./output/chr22_data.ENSG00000099977.13.bim +\n",
      "./output/chr22_data.ENSG00000099977.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099977.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099991.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099991.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099991.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099991.17.nosex .\n",
      "--extract range: 14832 variants excluded.\n",
      "--extract range: 377 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "377 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099991.17.bed +\n",
      "./output/chr22_data.ENSG00000099991.17.bim +\n",
      "./output/chr22_data.ENSG00000099991.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099991.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099994.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099994.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099994.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099994.10.nosex .\n",
      "--extract range: 14860 variants excluded.\n",
      "--extract range: 349 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "349 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099994.10.bed +\n",
      "./output/chr22_data.ENSG00000099994.10.bim +\n",
      "./output/chr22_data.ENSG00000099994.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099994.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099998.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099998.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099998.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099998.17.nosex .\n",
      "--extract range: 14862 variants excluded.\n",
      "--extract range: 347 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "347 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099998.17.bed +\n",
      "./output/chr22_data.ENSG00000099998.17.bim +\n",
      "./output/chr22_data.ENSG00000099998.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099998.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100014.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100014.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100014.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100014.19.nosex .\n",
      "--extract range: 14858 variants excluded.\n",
      "--extract range: 351 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "351 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100014.19.bed +\n",
      "./output/chr22_data.ENSG00000100014.19.bim +\n",
      "./output/chr22_data.ENSG00000100014.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100014.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128271.21.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128271.21_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128271.21\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128271.21.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128271.21.bed +\n",
      "./output/chr22_data.ENSG00000128271.21.bim +\n",
      "./output/chr22_data.ENSG00000128271.21.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128271.21.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000178803.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000178803.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000178803.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000178803.10.nosex .\n",
      "--extract range: 14854 variants excluded.\n",
      "--extract range: 355 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "355 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000178803.10.bed +\n",
      "./output/chr22_data.ENSG00000178803.10.bim +\n",
      "./output/chr22_data.ENSG00000178803.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000178803.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100024.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100024.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100024.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100024.14.nosex .\n",
      "--extract range: 14848 variants excluded.\n",
      "--extract range: 361 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "361 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100024.14.bed +\n",
      "./output/chr22_data.ENSG00000100024.14.bim +\n",
      "./output/chr22_data.ENSG00000100024.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100024.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000228923.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000228923.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000228923.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000228923.1.nosex .\n",
      "--extract range: 14843 variants excluded.\n",
      "--extract range: 366 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "366 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000228923.1.bed +\n",
      "./output/chr22_data.ENSG00000228923.1.bim +\n",
      "./output/chr22_data.ENSG00000228923.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000228923.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000138867.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000138867.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000138867.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000138867.16.nosex .\n",
      "--extract range: 14859 variants excluded.\n",
      "--extract range: 350 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "350 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000138867.16.bed +\n",
      "./output/chr22_data.ENSG00000138867.16.bim +\n",
      "./output/chr22_data.ENSG00000138867.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000138867.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100028.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100028.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100028.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100028.11.nosex .\n",
      "--extract range: 14859 variants excluded.\n",
      "--extract range: 350 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "350 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100028.11.bed +\n",
      "./output/chr22_data.ENSG00000100028.11.bim +\n",
      "./output/chr22_data.ENSG00000100028.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100028.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000178026.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000178026.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000178026.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000178026.12.nosex .\n",
      "--extract range: 14857 variants excluded.\n",
      "--extract range: 352 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "352 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000178026.12.bed +\n",
      "./output/chr22_data.ENSG00000178026.12.bim +\n",
      "./output/chr22_data.ENSG00000178026.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000178026.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100031.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100031.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100031.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100031.18.nosex .\n",
      "--extract range: 14857 variants excluded.\n",
      "--extract range: 352 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "352 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100031.18.bed +\n",
      "./output/chr22_data.ENSG00000100031.18.bim +\n",
      "./output/chr22_data.ENSG00000100031.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100031.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000284128.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000284128.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000284128.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000284128.1.nosex .\n",
      "--extract range: 14854 variants excluded.\n",
      "--extract range: 355 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "355 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000284128.1.bed +\n",
      "./output/chr22_data.ENSG00000284128.1.bim +\n",
      "./output/chr22_data.ENSG00000284128.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000284128.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000167037.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000167037.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000167037.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000167037.18.nosex .\n",
      "--extract range: 14812 variants excluded.\n",
      "--extract range: 397 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "397 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000167037.18.bed +\n",
      "./output/chr22_data.ENSG00000167037.18.bim +\n",
      "./output/chr22_data.ENSG00000167037.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000167037.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000197077.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000197077.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000197077.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000197077.13.nosex .\n",
      "--extract range: 14834 variants excluded.\n",
      "--extract range: 375 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "375 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000197077.13.bed +\n",
      "./output/chr22_data.ENSG00000197077.13.bim +\n",
      "./output/chr22_data.ENSG00000197077.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000197077.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000206066.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000206066.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000206066.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000206066.3.nosex .\n",
      "--extract range: 14836 variants excluded.\n",
      "--extract range: 373 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "373 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000206066.3.bed +\n",
      "./output/chr22_data.ENSG00000206066.3.bim +\n",
      "./output/chr22_data.ENSG00000206066.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000206066.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000231466.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000231466.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000231466.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000231466.2.nosex .\n",
      "--extract range: 14830 variants excluded.\n",
      "--extract range: 379 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "379 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000231466.2.bed +\n",
      "./output/chr22_data.ENSG00000231466.2.bim +\n",
      "./output/chr22_data.ENSG00000231466.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000231466.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100068.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100068.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100068.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100068.11.nosex .\n",
      "--extract range: 14803 variants excluded.\n",
      "--extract range: 406 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "406 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100068.11.bed +\n",
      "./output/chr22_data.ENSG00000100068.11.bim +\n",
      "./output/chr22_data.ENSG00000100068.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100068.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100058.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100058.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100058.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100058.12.nosex .\n",
      "--extract range: 14798 variants excluded.\n",
      "--extract range: 411 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "411 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100058.12.bed +\n",
      "./output/chr22_data.ENSG00000100058.12.bim +\n",
      "./output/chr22_data.ENSG00000100058.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100058.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272977.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272977.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272977.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272977.1.nosex .\n",
      "--extract range: 14797 variants excluded.\n",
      "--extract range: 412 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "412 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272977.1.bed +\n",
      "./output/chr22_data.ENSG00000272977.1.bim +\n",
      "./output/chr22_data.ENSG00000272977.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272977.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100077.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100077.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100077.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100077.14.nosex .\n",
      "--extract range: 14752 variants excluded.\n",
      "--extract range: 457 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "457 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100077.14.bed +\n",
      "./output/chr22_data.ENSG00000100077.14.bim +\n",
      "./output/chr22_data.ENSG00000100077.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100077.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000224003.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000224003.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000224003.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000224003.1.nosex .\n",
      "--extract range: 14758 variants excluded.\n",
      "--extract range: 451 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "451 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000224003.1.bed +\n",
      "./output/chr22_data.ENSG00000224003.1.bim +\n",
      "./output/chr22_data.ENSG00000224003.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000224003.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133454.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133454.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133454.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133454.15.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133454.15.bed +\n",
      "./output/chr22_data.ENSG00000133454.15.bim +\n",
      "./output/chr22_data.ENSG00000133454.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133454.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100095.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100095.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100095.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100095.18.nosex .\n",
      "--extract range: 14511 variants excluded.\n",
      "--extract range: 698 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "698 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100095.18.bed +\n",
      "./output/chr22_data.ENSG00000100095.18.bim +\n",
      "./output/chr22_data.ENSG00000100095.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100095.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128203.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128203.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128203.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128203.6.nosex .\n",
      "--extract range: 14530 variants excluded.\n",
      "--extract range: 679 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "679 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128203.6.bed +\n",
      "./output/chr22_data.ENSG00000128203.6.bim +\n",
      "./output/chr22_data.ENSG00000128203.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128203.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100099.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100099.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100099.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100099.20.nosex .\n",
      "--extract range: 14500 variants excluded.\n",
      "--extract range: 709 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "709 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100099.20.bed +\n",
      "./output/chr22_data.ENSG00000100099.20.bim +\n",
      "./output/chr22_data.ENSG00000100099.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100099.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100104.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100104.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100104.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100104.13.nosex .\n",
      "--extract range: 14500 variants excluded.\n",
      "--extract range: 709 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "709 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100104.13.bed +\n",
      "./output/chr22_data.ENSG00000100104.13.bim +\n",
      "./output/chr22_data.ENSG00000100104.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100104.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100109.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100109.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100109.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100109.16.nosex .\n",
      "--extract range: 14500 variants excluded.\n",
      "--extract range: 709 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "709 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100109.16.bed +\n",
      "./output/chr22_data.ENSG00000100109.16.bim +\n",
      "./output/chr22_data.ENSG00000100109.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100109.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000261188.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000261188.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000261188.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000261188.1.nosex .\n",
      "--extract range: 14500 variants excluded.\n",
      "--extract range: 709 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "709 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000261188.1.bed +\n",
      "./output/chr22_data.ENSG00000261188.1.bim +\n",
      "./output/chr22_data.ENSG00000261188.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000261188.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000213707.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000213707.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000213707.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000213707.2.nosex .\n",
      "--extract range: 14461 variants excluded.\n",
      "--extract range: 748 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "748 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000213707.2.bed +\n",
      "./output/chr22_data.ENSG00000213707.2.bim +\n",
      "./output/chr22_data.ENSG00000213707.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000213707.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128294.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128294.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128294.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128294.15.nosex .\n",
      "--extract range: 14425 variants excluded.\n",
      "--extract range: 784 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "784 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128294.15.bed +\n",
      "./output/chr22_data.ENSG00000128294.15.bim +\n",
      "./output/chr22_data.ENSG00000128294.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128294.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100122.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100122.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100122.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100122.5.nosex .\n",
      "--extract range: 14418 variants excluded.\n",
      "--extract range: 791 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "791 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100122.5.bed +\n",
      "./output/chr22_data.ENSG00000100122.5.bim +\n",
      "./output/chr22_data.ENSG00000100122.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100122.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000196431.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000196431.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000196431.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000196431.3.nosex .\n",
      "--extract range: 14418 variants excluded.\n",
      "--extract range: 791 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "791 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000196431.3.bed +\n",
      "./output/chr22_data.ENSG00000196431.3.bim +\n",
      "./output/chr22_data.ENSG00000196431.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000196431.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225783.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225783.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225783.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225783.6.nosex .\n",
      "--extract range: 14407 variants excluded.\n",
      "--extract range: 802 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "802 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225783.6.bed +\n",
      "./output/chr22_data.ENSG00000225783.6.bim +\n",
      "./output/chr22_data.ENSG00000225783.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225783.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000244625.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000244625.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000244625.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000244625.5.nosex .\n",
      "--extract range: 14392 variants excluded.\n",
      "--extract range: 817 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "817 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000244625.5.bed +\n",
      "./output/chr22_data.ENSG00000244625.5.bim +\n",
      "./output/chr22_data.ENSG00000244625.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000244625.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000223704.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000223704.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000223704.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000223704.1.nosex .\n",
      "--extract range: 14421 variants excluded.\n",
      "--extract range: 788 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "788 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000223704.1.bed +\n",
      "./output/chr22_data.ENSG00000223704.1.bim +\n",
      "./output/chr22_data.ENSG00000223704.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000223704.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000231369.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000231369.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000231369.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000231369.1.nosex .\n",
      "--extract range: 14425 variants excluded.\n",
      "--extract range: 784 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "784 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000231369.1.bed +\n",
      "./output/chr22_data.ENSG00000231369.1.bim +\n",
      "./output/chr22_data.ENSG00000231369.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000231369.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235271.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235271.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235271.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235271.5.nosex .\n",
      "--extract range: 14420 variants excluded.\n",
      "--extract range: 789 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "789 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235271.5.bed +\n",
      "./output/chr22_data.ENSG00000235271.5.bim +\n",
      "./output/chr22_data.ENSG00000235271.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235271.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000169184.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000169184.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000169184.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000169184.5.nosex .\n",
      "--extract range: 14668 variants excluded.\n",
      "--extract range: 541 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999969.\n",
      "541 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000169184.5.bed +\n",
      "./output/chr22_data.ENSG00000169184.5.bim +\n",
      "./output/chr22_data.ENSG00000169184.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000169184.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235954.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235954.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235954.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235954.6.nosex .\n",
      "--extract range: 14738 variants excluded.\n",
      "--extract range: 471 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "471 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235954.6.bed +\n",
      "./output/chr22_data.ENSG00000235954.6.bim +\n",
      "./output/chr22_data.ENSG00000235954.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235954.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000180957.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000180957.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000180957.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000180957.17.nosex .\n",
      "--extract range: 14738 variants excluded.\n",
      "--extract range: 471 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "471 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000180957.17.bed +\n",
      "./output/chr22_data.ENSG00000180957.17.bim +\n",
      "./output/chr22_data.ENSG00000180957.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000180957.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100154.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100154.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100154.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100154.14.nosex .\n",
      "--extract range: 14921 variants excluded.\n",
      "--extract range: 288 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "288 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100154.14.bed +\n",
      "./output/chr22_data.ENSG00000100154.14.bim +\n",
      "./output/chr22_data.ENSG00000100154.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100154.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183765.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183765.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183765.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183765.20.nosex .\n",
      "--extract range: 14923 variants excluded.\n",
      "--extract range: 286 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "286 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183765.20.bed +\n",
      "./output/chr22_data.ENSG00000183765.20.bim +\n",
      "./output/chr22_data.ENSG00000183765.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183765.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100209.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100209.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100209.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100209.9.nosex .\n",
      "--extract range: 14922 variants excluded.\n",
      "--extract range: 287 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "287 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100209.9.bed +\n",
      "./output/chr22_data.ENSG00000100209.9.bim +\n",
      "./output/chr22_data.ENSG00000100209.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100209.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000159873.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000159873.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000159873.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000159873.9.nosex .\n",
      "--extract range: 14910 variants excluded.\n",
      "--extract range: 299 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "299 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000159873.9.bed +\n",
      "./output/chr22_data.ENSG00000159873.9.bim +\n",
      "./output/chr22_data.ENSG00000159873.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000159873.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100219.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100219.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100219.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100219.16.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100219.16.bed +\n",
      "./output/chr22_data.ENSG00000100219.16.bim +\n",
      "./output/chr22_data.ENSG00000100219.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100219.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226471.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226471.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226471.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226471.6.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226471.6.bed +\n",
      "./output/chr22_data.ENSG00000226471.6.bim +\n",
      "./output/chr22_data.ENSG00000226471.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226471.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272858.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272858.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272858.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272858.1.nosex .\n",
      "--extract range: 14897 variants excluded.\n",
      "--extract range: 312 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "312 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272858.1.bed +\n",
      "./output/chr22_data.ENSG00000272858.1.bim +\n",
      "./output/chr22_data.ENSG00000272858.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272858.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183579.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183579.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183579.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183579.15.nosex .\n",
      "--extract range: 14885 variants excluded.\n",
      "--extract range: 324 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "324 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183579.15.bed +\n",
      "./output/chr22_data.ENSG00000183579.15.bim +\n",
      "./output/chr22_data.ENSG00000183579.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183579.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183762.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183762.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183762.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183762.12.nosex .\n",
      "--extract range: 14841 variants excluded.\n",
      "--extract range: 368 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "368 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183762.12.bed +\n",
      "./output/chr22_data.ENSG00000183762.12.bim +\n",
      "./output/chr22_data.ENSG00000183762.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183762.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226772.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226772.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226772.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226772.1.nosex .\n",
      "--extract range: 14834 variants excluded.\n",
      "--extract range: 375 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "375 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226772.1.bed +\n",
      "./output/chr22_data.ENSG00000226772.1.bim +\n",
      "./output/chr22_data.ENSG00000226772.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226772.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000186998.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000186998.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000186998.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000186998.15.nosex .\n",
      "--extract range: 14813 variants excluded.\n",
      "--extract range: 396 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "396 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000186998.15.bed +\n",
      "./output/chr22_data.ENSG00000186998.15.bim +\n",
      "./output/chr22_data.ENSG00000186998.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000186998.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237015.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237015.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237015.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237015.1.nosex .\n",
      "--extract range: 14804 variants excluded.\n",
      "--extract range: 405 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "405 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237015.1.bed +\n",
      "./output/chr22_data.ENSG00000237015.1.bim +\n",
      "./output/chr22_data.ENSG00000237015.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237015.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100263.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100263.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100263.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100263.13.nosex .\n",
      "--extract range: 14806 variants excluded.\n",
      "--extract range: 403 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "403 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100263.13.bed +\n",
      "./output/chr22_data.ENSG00000100263.13.bim +\n",
      "./output/chr22_data.ENSG00000100263.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100263.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182944.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182944.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182944.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182944.17.nosex .\n",
      "--extract range: 14806 variants excluded.\n",
      "--extract range: 403 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "403 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182944.17.bed +\n",
      "./output/chr22_data.ENSG00000182944.17.bim +\n",
      "./output/chr22_data.ENSG00000182944.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182944.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185340.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185340.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185340.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185340.15.nosex .\n",
      "--extract range: 14809 variants excluded.\n",
      "--extract range: 400 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "400 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185340.15.bed +\n",
      "./output/chr22_data.ENSG00000185340.15.bim +\n",
      "./output/chr22_data.ENSG00000185340.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185340.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273216.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273216.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273216.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273216.1.nosex .\n",
      "--extract range: 14809 variants excluded.\n",
      "--extract range: 400 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "400 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273216.1.bed +\n",
      "./output/chr22_data.ENSG00000273216.1.bim +\n",
      "./output/chr22_data.ENSG00000273216.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273216.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100276.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100276.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100276.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100276.9.nosex .\n",
      "--extract range: 14810 variants excluded.\n",
      "--extract range: 399 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "399 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100276.9.bed +\n",
      "./output/chr22_data.ENSG00000100276.9.bim +\n",
      "./output/chr22_data.ENSG00000100276.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100276.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100280.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100280.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100280.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100280.16.nosex .\n",
      "--extract range: 14807 variants excluded.\n",
      "--extract range: 402 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "402 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100280.16.bed +\n",
      "./output/chr22_data.ENSG00000100280.16.bim +\n",
      "./output/chr22_data.ENSG00000100280.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100280.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100285.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100285.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100285.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100285.9.nosex .\n",
      "--extract range: 14819 variants excluded.\n",
      "--extract range: 390 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "390 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100285.9.bed +\n",
      "./output/chr22_data.ENSG00000100285.9.bim +\n",
      "./output/chr22_data.ENSG00000100285.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100285.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100296.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100296.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100296.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100296.13.nosex .\n",
      "--extract range: 14824 variants excluded.\n",
      "--extract range: 385 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "385 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100296.13.bed +\n",
      "./output/chr22_data.ENSG00000100296.13.bim +\n",
      "./output/chr22_data.ENSG00000100296.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100296.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184117.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184117.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184117.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184117.11.nosex .\n",
      "--extract range: 14841 variants excluded.\n",
      "--extract range: 368 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999973.\n",
      "368 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184117.11.bed +\n",
      "./output/chr22_data.ENSG00000184117.11.bim +\n",
      "./output/chr22_data.ENSG00000184117.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184117.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000186575.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000186575.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000186575.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000186575.17.nosex .\n",
      "--extract range: 14853 variants excluded.\n",
      "--extract range: 356 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "356 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000186575.17.bed +\n",
      "./output/chr22_data.ENSG00000186575.17.bim +\n",
      "./output/chr22_data.ENSG00000186575.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000186575.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100314.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100314.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100314.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100314.3.nosex .\n",
      "--extract range: 14854 variants excluded.\n",
      "--extract range: 355 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "355 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100314.3.bed +\n",
      "./output/chr22_data.ENSG00000100314.3.bim +\n",
      "./output/chr22_data.ENSG00000100314.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100314.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100319.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100319.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100319.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100319.12.nosex .\n",
      "--extract range: 14859 variants excluded.\n",
      "--extract range: 350 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "350 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100319.12.bed +\n",
      "./output/chr22_data.ENSG00000100319.12.bim +\n",
      "./output/chr22_data.ENSG00000100319.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100319.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184076.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184076.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184076.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184076.13.nosex .\n",
      "--extract range: 14859 variants excluded.\n",
      "--extract range: 350 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "350 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184076.13.bed +\n",
      "./output/chr22_data.ENSG00000184076.13.bim +\n",
      "./output/chr22_data.ENSG00000184076.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184076.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100325.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100325.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100325.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100325.14.nosex .\n",
      "--extract range: 14851 variants excluded.\n",
      "--extract range: 358 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999969.\n",
      "358 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100325.14.bed +\n",
      "./output/chr22_data.ENSG00000100325.14.bim +\n",
      "./output/chr22_data.ENSG00000100325.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100325.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100330.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100330.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100330.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100330.15.nosex .\n",
      "--extract range: 14842 variants excluded.\n",
      "--extract range: 367 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99997.\n",
      "367 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100330.15.bed +\n",
      "./output/chr22_data.ENSG00000100330.15.bim +\n",
      "./output/chr22_data.ENSG00000100330.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100330.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279159.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279159.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279159.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279159.1.nosex .\n",
      "--extract range: 14823 variants excluded.\n",
      "--extract range: 386 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "386 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279159.1.bed +\n",
      "./output/chr22_data.ENSG00000279159.1.bim +\n",
      "./output/chr22_data.ENSG00000279159.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279159.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000227201.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000227201.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000227201.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000227201.2.nosex .\n",
      "--extract range: 14832 variants excluded.\n",
      "--extract range: 377 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "377 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000227201.2.bed +\n",
      "./output/chr22_data.ENSG00000227201.2.bim +\n",
      "./output/chr22_data.ENSG00000227201.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000227201.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128342.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128342.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128342.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128342.4.nosex .\n",
      "--extract range: 14814 variants excluded.\n",
      "--extract range: 395 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "395 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128342.4.bed +\n",
      "./output/chr22_data.ENSG00000128342.4.bim +\n",
      "./output/chr22_data.ENSG00000128342.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128342.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000268812.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000268812.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000268812.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000268812.3.nosex .\n",
      "--extract range: 14813 variants excluded.\n",
      "--extract range: 396 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "396 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000268812.3.bed +\n",
      "./output/chr22_data.ENSG00000268812.3.bim +\n",
      "./output/chr22_data.ENSG00000268812.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000268812.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099985.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099985.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099985.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099985.3.nosex .\n",
      "--extract range: 14819 variants excluded.\n",
      "--extract range: 390 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "390 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099985.3.bed +\n",
      "./output/chr22_data.ENSG00000099985.3.bim +\n",
      "./output/chr22_data.ENSG00000099985.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099985.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279699.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279699.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279699.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279699.1.nosex .\n",
      "--extract range: 14815 variants excluded.\n",
      "--extract range: 394 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "394 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279699.1.bed +\n",
      "./output/chr22_data.ENSG00000279699.1.bim +\n",
      "./output/chr22_data.ENSG00000279699.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279699.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000239282.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000239282.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000239282.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000239282.7.nosex .\n",
      "--extract range: 14820 variants excluded.\n",
      "--extract range: 389 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "389 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000239282.7.bed +\n",
      "./output/chr22_data.ENSG00000239282.7.bim +\n",
      "./output/chr22_data.ENSG00000239282.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000239282.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099992.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099992.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099992.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099992.15.nosex .\n",
      "--extract range: 14817 variants excluded.\n",
      "--extract range: 392 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "392 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099992.15.bed +\n",
      "./output/chr22_data.ENSG00000099992.15.bim +\n",
      "./output/chr22_data.ENSG00000099992.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099992.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099995.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099995.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099995.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099995.18.nosex .\n",
      "--extract range: 14814 variants excluded.\n",
      "--extract range: 395 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "395 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099995.18.bed +\n",
      "./output/chr22_data.ENSG00000099995.18.bim +\n",
      "./output/chr22_data.ENSG00000099995.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099995.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000187860.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000187860.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000187860.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000187860.10.nosex .\n",
      "--extract range: 14813 variants excluded.\n",
      "--extract range: 396 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "396 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000187860.10.bed +\n",
      "./output/chr22_data.ENSG00000187860.10.bim +\n",
      "./output/chr22_data.ENSG00000187860.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000187860.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100003.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100003.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100003.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100003.17.nosex .\n",
      "--extract range: 14803 variants excluded.\n",
      "--extract range: 406 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "406 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100003.17.bed +\n",
      "./output/chr22_data.ENSG00000100003.17.bim +\n",
      "./output/chr22_data.ENSG00000100003.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100003.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000099999.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000099999.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000099999.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000099999.14.nosex .\n",
      "--extract range: 14802 variants excluded.\n",
      "--extract range: 407 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "407 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000099999.14.bed +\n",
      "./output/chr22_data.ENSG00000099999.14.bim +\n",
      "./output/chr22_data.ENSG00000099999.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000099999.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000242114.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000242114.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000242114.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000242114.5.nosex .\n",
      "--extract range: 14802 variants excluded.\n",
      "--extract range: 407 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "407 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000242114.5.bed +\n",
      "./output/chr22_data.ENSG00000242114.5.bim +\n",
      "./output/chr22_data.ENSG00000242114.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000242114.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273428.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273428.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273428.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273428.3.nosex .\n",
      "--extract range: 14803 variants excluded.\n",
      "--extract range: 406 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "406 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273428.3.bed +\n",
      "./output/chr22_data.ENSG00000273428.3.bim +\n",
      "./output/chr22_data.ENSG00000273428.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273428.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100012.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100012.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100012.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100012.11.nosex .\n",
      "--extract range: 14801 variants excluded.\n",
      "--extract range: 408 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "408 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100012.11.bed +\n",
      "./output/chr22_data.ENSG00000100012.11.bim +\n",
      "./output/chr22_data.ENSG00000100012.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100012.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133488.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133488.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133488.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133488.14.nosex .\n",
      "--extract range: 14811 variants excluded.\n",
      "--extract range: 398 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "398 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133488.14.bed +\n",
      "./output/chr22_data.ENSG00000133488.14.bim +\n",
      "./output/chr22_data.ENSG00000133488.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133488.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225774.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225774.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225774.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225774.2.nosex .\n",
      "--extract range: 14821 variants excluded.\n",
      "--extract range: 388 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "388 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225774.2.bed +\n",
      "./output/chr22_data.ENSG00000225774.2.bim +\n",
      "./output/chr22_data.ENSG00000225774.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225774.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128242.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128242.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128242.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128242.12.nosex .\n",
      "--extract range: 14813 variants excluded.\n",
      "--extract range: 396 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "396 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128242.12.bed +\n",
      "./output/chr22_data.ENSG00000128242.12.bim +\n",
      "./output/chr22_data.ENSG00000128242.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128242.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100029.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100029.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100029.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100029.17.nosex .\n",
      "--extract range: 14809 variants excluded.\n",
      "--extract range: 400 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "400 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100029.17.bed +\n",
      "./output/chr22_data.ENSG00000100029.17.bim +\n",
      "./output/chr22_data.ENSG00000100029.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100029.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185339.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185339.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185339.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185339.8.nosex .\n",
      "--extract range: 14809 variants excluded.\n",
      "--extract range: 400 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "400 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185339.8.bed +\n",
      "./output/chr22_data.ENSG00000185339.8.bim +\n",
      "./output/chr22_data.ENSG00000185339.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185339.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100036.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100036.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100036.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100036.12.nosex .\n",
      "--extract range: 14802 variants excluded.\n",
      "--extract range: 407 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "407 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100036.12.bed +\n",
      "./output/chr22_data.ENSG00000100036.12.bim +\n",
      "./output/chr22_data.ENSG00000100036.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100036.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000250318.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000250318.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000250318.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000250318.1.nosex .\n",
      "--extract range: 14795 variants excluded.\n",
      "--extract range: 414 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "414 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000250318.1.bed +\n",
      "./output/chr22_data.ENSG00000250318.1.bim +\n",
      "./output/chr22_data.ENSG00000250318.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000250318.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000167065.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000167065.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000167065.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000167065.13.nosex .\n",
      "--extract range: 14793 variants excluded.\n",
      "--extract range: 416 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "416 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000167065.13.bed +\n",
      "./output/chr22_data.ENSG00000167065.13.bim +\n",
      "./output/chr22_data.ENSG00000167065.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000167065.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184792.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184792.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184792.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184792.15.nosex .\n",
      "--extract range: 14794 variants excluded.\n",
      "--extract range: 415 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "415 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184792.15.bed +\n",
      "./output/chr22_data.ENSG00000184792.15.bim +\n",
      "./output/chr22_data.ENSG00000184792.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184792.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237977.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237977.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237977.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237977.1.nosex .\n",
      "--extract range: 14852 variants excluded.\n",
      "--extract range: 357 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "357 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237977.1.bed +\n",
      "./output/chr22_data.ENSG00000237977.1.bim +\n",
      "./output/chr22_data.ENSG00000237977.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237977.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235989.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235989.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235989.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235989.3.nosex .\n",
      "--extract range: 14859 variants excluded.\n",
      "--extract range: 350 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "350 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235989.3.bed +\n",
      "./output/chr22_data.ENSG00000235989.3.bim +\n",
      "./output/chr22_data.ENSG00000235989.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235989.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133422.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133422.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133422.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133422.12.nosex .\n",
      "--extract range: 14862 variants excluded.\n",
      "--extract range: 347 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "347 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133422.12.bed +\n",
      "./output/chr22_data.ENSG00000133422.12.bim +\n",
      "./output/chr22_data.ENSG00000133422.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133422.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000253352.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000253352.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000253352.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000253352.8.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000253352.8.bed +\n",
      "./output/chr22_data.ENSG00000253352.8.bim +\n",
      "./output/chr22_data.ENSG00000253352.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000253352.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183963.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183963.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183963.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183963.18.nosex .\n",
      "--extract range: 14873 variants excluded.\n",
      "--extract range: 336 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "336 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183963.18.bed +\n",
      "./output/chr22_data.ENSG00000183963.18.bim +\n",
      "./output/chr22_data.ENSG00000183963.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183963.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273387.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273387.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273387.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273387.1.nosex .\n",
      "--extract range: 14872 variants excluded.\n",
      "--extract range: 337 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "337 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273387.1.bed +\n",
      "./output/chr22_data.ENSG00000273387.1.bim +\n",
      "./output/chr22_data.ENSG00000273387.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273387.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198832.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198832.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198832.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198832.10.nosex .\n",
      "--extract range: 14870 variants excluded.\n",
      "--extract range: 339 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "339 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198832.10.bed +\n",
      "./output/chr22_data.ENSG00000198832.10.bim +\n",
      "./output/chr22_data.ENSG00000198832.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198832.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185133.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185133.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185133.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185133.13.nosex .\n",
      "--extract range: 14873 variants excluded.\n",
      "--extract range: 336 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "336 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185133.13.bed +\n",
      "./output/chr22_data.ENSG00000185133.13.bim +\n",
      "./output/chr22_data.ENSG00000185133.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185133.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000138942.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000138942.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000138942.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000138942.15.nosex .\n",
      "--extract range: 14878 variants excluded.\n",
      "--extract range: 331 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "331 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000138942.15.bed +\n",
      "./output/chr22_data.ENSG00000138942.15.bim +\n",
      "./output/chr22_data.ENSG00000138942.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000138942.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182541.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182541.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182541.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182541.17.nosex .\n",
      "--extract range: 14887 variants excluded.\n",
      "--extract range: 322 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "322 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182541.17.bed +\n",
      "./output/chr22_data.ENSG00000182541.17.bim +\n",
      "./output/chr22_data.ENSG00000182541.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182541.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100100.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100100.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100100.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100100.12.nosex .\n",
      "--extract range: 14895 variants excluded.\n",
      "--extract range: 314 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "314 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100100.12.bed +\n",
      "./output/chr22_data.ENSG00000100100.12.bim +\n",
      "./output/chr22_data.ENSG00000100100.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100100.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000228839.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000228839.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000228839.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000228839.5.nosex .\n",
      "--extract range: 14895 variants excluded.\n",
      "--extract range: 314 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "314 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000228839.5.bed +\n",
      "./output/chr22_data.ENSG00000228839.5.bim +\n",
      "./output/chr22_data.ENSG00000228839.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000228839.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100105.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100105.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100105.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100105.17.nosex .\n",
      "--extract range: 14900 variants excluded.\n",
      "--extract range: 309 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "309 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100105.17.bed +\n",
      "./output/chr22_data.ENSG00000100105.17.bim +\n",
      "./output/chr22_data.ENSG00000100105.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100105.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000213888.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000213888.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000213888.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000213888.3.nosex .\n",
      "--extract range: 14900 variants excluded.\n",
      "--extract range: 309 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "309 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000213888.3.bed +\n",
      "./output/chr22_data.ENSG00000213888.3.bim +\n",
      "./output/chr22_data.ENSG00000213888.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000213888.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185721.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185721.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185721.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185721.12.nosex .\n",
      "--extract range: 14903 variants excluded.\n",
      "--extract range: 306 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "306 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185721.12.bed +\n",
      "./output/chr22_data.ENSG00000185721.12.bim +\n",
      "./output/chr22_data.ENSG00000185721.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185721.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000240591.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000240591.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000240591.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000240591.1.nosex .\n",
      "--extract range: 14889 variants excluded.\n",
      "--extract range: 320 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "320 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000240591.1.bed +\n",
      "./output/chr22_data.ENSG00000240591.1.bim +\n",
      "./output/chr22_data.ENSG00000240591.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000240591.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198089.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198089.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198089.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198089.15.nosex .\n",
      "--extract range: 14886 variants excluded.\n",
      "--extract range: 323 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "323 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198089.15.bed +\n",
      "./output/chr22_data.ENSG00000198089.15.bim +\n",
      "./output/chr22_data.ENSG00000198089.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198089.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184708.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184708.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184708.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184708.17.nosex .\n",
      "--extract range: 14885 variants excluded.\n",
      "--extract range: 324 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "324 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184708.17.bed +\n",
      "./output/chr22_data.ENSG00000184708.17.bim +\n",
      "./output/chr22_data.ENSG00000184708.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184708.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000214093.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000214093.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000214093.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000214093.3.nosex .\n",
      "--extract range: 14894 variants excluded.\n",
      "--extract range: 315 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "315 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000214093.3.bed +\n",
      "./output/chr22_data.ENSG00000214093.3.bim +\n",
      "./output/chr22_data.ENSG00000214093.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000214093.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241878.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241878.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241878.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241878.11.nosex .\n",
      "--extract range: 14910 variants excluded.\n",
      "--extract range: 299 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "299 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241878.11.bed +\n",
      "./output/chr22_data.ENSG00000241878.11.bim +\n",
      "./output/chr22_data.ENSG00000241878.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241878.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183530.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183530.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183530.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183530.13.nosex .\n",
      "--extract range: 14879 variants excluded.\n",
      "--extract range: 330 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "330 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183530.13.bed +\n",
      "./output/chr22_data.ENSG00000183530.13.bim +\n",
      "./output/chr22_data.ENSG00000183530.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183530.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100150.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100150.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100150.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100150.16.nosex .\n",
      "--extract range: 14881 variants excluded.\n",
      "--extract range: 328 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "328 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100150.16.bed +\n",
      "./output/chr22_data.ENSG00000100150.16.bim +\n",
      "./output/chr22_data.ENSG00000100150.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100150.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128245.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128245.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128245.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128245.14.nosex .\n",
      "--extract range: 14792 variants excluded.\n",
      "--extract range: 417 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99997.\n",
      "417 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128245.14.bed +\n",
      "./output/chr22_data.ENSG00000128245.14.bim +\n",
      "./output/chr22_data.ENSG00000128245.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128245.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128254.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128254.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128254.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128254.12.nosex .\n",
      "--extract range: 14792 variants excluded.\n",
      "--extract range: 417 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99997.\n",
      "417 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128254.12.bed +\n",
      "./output/chr22_data.ENSG00000128254.12.bim +\n",
      "./output/chr22_data.ENSG00000128254.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128254.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000232346.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000232346.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000232346.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000232346.1.nosex .\n",
      "--extract range: 14757 variants excluded.\n",
      "--extract range: 452 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "452 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000232346.1.bed +\n",
      "./output/chr22_data.ENSG00000232346.1.bim +\n",
      "./output/chr22_data.ENSG00000232346.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000232346.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128253.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128253.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128253.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128253.13.nosex .\n",
      "--extract range: 14735 variants excluded.\n",
      "--extract range: 474 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "474 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128253.13.bed +\n",
      "./output/chr22_data.ENSG00000128253.13.bim +\n",
      "./output/chr22_data.ENSG00000128253.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128253.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000214076.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000214076.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000214076.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000214076.3.nosex .\n",
      "--extract range: 14694 variants excluded.\n",
      "--extract range: 515 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999967.\n",
      "515 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000214076.3.bed +\n",
      "./output/chr22_data.ENSG00000214076.3.bim +\n",
      "./output/chr22_data.ENSG00000214076.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000214076.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205853.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205853.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205853.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205853.10.nosex .\n",
      "--extract range: 14643 variants excluded.\n",
      "--extract range: 566 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "566 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205853.10.bed +\n",
      "./output/chr22_data.ENSG00000205853.10.bim +\n",
      "./output/chr22_data.ENSG00000205853.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205853.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100220.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100220.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100220.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100220.11.nosex .\n",
      "--extract range: 14632 variants excluded.\n",
      "--extract range: 577 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "577 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100220.11.bed +\n",
      "./output/chr22_data.ENSG00000100220.11.bim +\n",
      "./output/chr22_data.ENSG00000100220.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100220.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100225.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100225.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100225.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100225.17.nosex .\n",
      "--extract range: 14644 variants excluded.\n",
      "--extract range: 565 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "565 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100225.17.bed +\n",
      "./output/chr22_data.ENSG00000100225.17.bim +\n",
      "./output/chr22_data.ENSG00000100225.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100225.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100234.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100234.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100234.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100234.11.nosex .\n",
      "--extract range: 14611 variants excluded.\n",
      "--extract range: 598 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "598 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100234.11.bed +\n",
      "./output/chr22_data.ENSG00000100234.11.bim +\n",
      "./output/chr22_data.ENSG00000100234.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100234.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133424.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133424.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133424.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133424.20.nosex .\n",
      "--extract range: 14626 variants excluded.\n",
      "--extract range: 583 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999993.\n",
      "583 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133424.20.bed +\n",
      "./output/chr22_data.ENSG00000133424.20.bim +\n",
      "./output/chr22_data.ENSG00000133424.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133424.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100281.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100281.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100281.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100281.13.nosex .\n",
      "--extract range: 14672 variants excluded.\n",
      "--extract range: 537 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "537 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100281.13.bed +\n",
      "./output/chr22_data.ENSG00000100281.13.bim +\n",
      "./output/chr22_data.ENSG00000100281.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100281.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100284.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100284.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100284.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100284.20.nosex .\n",
      "--extract range: 14689 variants excluded.\n",
      "--extract range: 520 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "520 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100284.20.bed +\n",
      "./output/chr22_data.ENSG00000100284.20.bim +\n",
      "./output/chr22_data.ENSG00000100284.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100284.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100292.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100292.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100292.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100292.16.nosex .\n",
      "--extract range: 14711 variants excluded.\n",
      "--extract range: 498 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "498 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100292.16.bed +\n",
      "./output/chr22_data.ENSG00000100292.16.bim +\n",
      "./output/chr22_data.ENSG00000100292.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100292.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100297.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100297.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100297.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100297.15.nosex .\n",
      "--extract range: 14721 variants excluded.\n",
      "--extract range: 488 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "488 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100297.15.bed +\n",
      "./output/chr22_data.ENSG00000100297.15.bim +\n",
      "./output/chr22_data.ENSG00000100297.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100297.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100302.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100302.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100302.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100302.6.nosex .\n",
      "--extract range: 14793 variants excluded.\n",
      "--extract range: 416 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "416 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100302.6.bed +\n",
      "./output/chr22_data.ENSG00000100302.6.bim +\n",
      "./output/chr22_data.ENSG00000100302.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100302.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198125.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198125.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198125.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198125.12.nosex .\n",
      "--extract range: 14849 variants excluded.\n",
      "--extract range: 360 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "360 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198125.12.bed +\n",
      "./output/chr22_data.ENSG00000198125.12.bim +\n",
      "./output/chr22_data.ENSG00000198125.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198125.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000221963.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000221963.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000221963.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000221963.5.nosex .\n",
      "--extract range: 14850 variants excluded.\n",
      "--extract range: 359 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "359 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000221963.5.bed +\n",
      "./output/chr22_data.ENSG00000221963.5.bim +\n",
      "./output/chr22_data.ENSG00000221963.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000221963.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100320.22.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100320.22_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100320.22\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100320.22.nosex .\n",
      "--extract range: 14778 variants excluded.\n",
      "--extract range: 431 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "431 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100320.22.bed +\n",
      "./output/chr22_data.ENSG00000100320.22.bim +\n",
      "./output/chr22_data.ENSG00000100320.22.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100320.22.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279805.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279805.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279805.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279805.1.nosex .\n",
      "--extract range: 14772 variants excluded.\n",
      "--extract range: 437 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "437 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279805.1.bed +\n",
      "./output/chr22_data.ENSG00000279805.1.bim +\n",
      "./output/chr22_data.ENSG00000279805.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279805.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128284.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128284.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128284.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128284.19.nosex .\n",
      "--extract range: 14786 variants excluded.\n",
      "--extract range: 423 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "423 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128284.19.bed +\n",
      "./output/chr22_data.ENSG00000128284.19.bim +\n",
      "./output/chr22_data.ENSG00000128284.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128284.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100336.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100336.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100336.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100336.17.nosex .\n",
      "--extract range: 14775 variants excluded.\n",
      "--extract range: 434 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "434 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100336.17.bed +\n",
      "./output/chr22_data.ENSG00000100336.17.bim +\n",
      "./output/chr22_data.ENSG00000100336.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100336.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128335.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128335.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128335.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128335.13.nosex .\n",
      "--extract range: 14777 variants excluded.\n",
      "--extract range: 432 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "432 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128335.13.bed +\n",
      "./output/chr22_data.ENSG00000128335.13.bim +\n",
      "./output/chr22_data.ENSG00000128335.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128335.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100342.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100342.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100342.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100342.20.nosex .\n",
      "--extract range: 14774 variants excluded.\n",
      "--extract range: 435 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999997.\n",
      "435 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100342.20.bed +\n",
      "./output/chr22_data.ENSG00000100342.20.bim +\n",
      "./output/chr22_data.ENSG00000100342.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100342.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100345.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100345.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100345.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100345.20.nosex .\n",
      "--extract range: 14729 variants excluded.\n",
      "--extract range: 480 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "480 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100345.20.bed +\n",
      "./output/chr22_data.ENSG00000100345.20.bim +\n",
      "./output/chr22_data.ENSG00000100345.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100345.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237668.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237668.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237668.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237668.1.nosex .\n",
      "--extract range: 14718 variants excluded.\n",
      "--extract range: 491 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "491 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237668.1.bed +\n",
      "./output/chr22_data.ENSG00000237668.1.bim +\n",
      "./output/chr22_data.ENSG00000237668.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237668.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000228719.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000228719.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000228719.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000228719.1.nosex .\n",
      "--extract range: 14700 variants excluded.\n",
      "--extract range: 509 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "509 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000228719.1.bed +\n",
      "./output/chr22_data.ENSG00000228719.1.bim +\n",
      "./output/chr22_data.ENSG00000228719.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000228719.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100348.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100348.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100348.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100348.9.nosex .\n",
      "--extract range: 14678 variants excluded.\n",
      "--extract range: 531 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "531 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100348.9.bed +\n",
      "./output/chr22_data.ENSG00000100348.9.bim +\n",
      "./output/chr22_data.ENSG00000100348.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100348.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100350.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100350.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100350.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100350.14.nosex .\n",
      "--extract range: 14658 variants excluded.\n",
      "--extract range: 551 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "551 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100350.14.bed +\n",
      "./output/chr22_data.ENSG00000100350.14.bim +\n",
      "./output/chr22_data.ENSG00000100350.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100350.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100353.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100353.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100353.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100353.17.nosex .\n",
      "--extract range: 14648 variants excluded.\n",
      "--extract range: 561 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "561 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100353.17.bed +\n",
      "./output/chr22_data.ENSG00000100353.17.bim +\n",
      "./output/chr22_data.ENSG00000100353.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100353.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100360.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100360.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100360.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100360.14.nosex .\n",
      "--extract range: 14629 variants excluded.\n",
      "--extract range: 580 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "580 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100360.14.bed +\n",
      "./output/chr22_data.ENSG00000100360.14.bim +\n",
      "./output/chr22_data.ENSG00000100360.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100360.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100362.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100362.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100362.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100362.12.nosex .\n",
      "--extract range: 14632 variants excluded.\n",
      "--extract range: 577 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "577 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100362.12.bed +\n",
      "./output/chr22_data.ENSG00000100362.12.bim +\n",
      "./output/chr22_data.ENSG00000100362.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100362.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100365.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100365.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100365.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100365.14.nosex .\n",
      "--extract range: 14627 variants excluded.\n",
      "--extract range: 582 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "582 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100365.14.bed +\n",
      "./output/chr22_data.ENSG00000100365.14.bim +\n",
      "./output/chr22_data.ENSG00000100365.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100365.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183822.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183822.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183822.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183822.3.nosex .\n",
      "--extract range: 14625 variants excluded.\n",
      "--extract range: 584 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "584 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183822.3.bed +\n",
      "./output/chr22_data.ENSG00000183822.3.bim +\n",
      "./output/chr22_data.ENSG00000183822.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183822.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100368.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100368.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100368.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100368.13.nosex .\n",
      "--extract range: 14612 variants excluded.\n",
      "--extract range: 597 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "597 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100368.13.bed +\n",
      "./output/chr22_data.ENSG00000100368.13.bim +\n",
      "./output/chr22_data.ENSG00000100368.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100368.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000232254.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000232254.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000232254.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000232254.1.nosex .\n",
      "--extract range: 14600 variants excluded.\n",
      "--extract range: 609 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "609 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000232254.1.bed +\n",
      "./output/chr22_data.ENSG00000232254.1.bim +\n",
      "./output/chr22_data.ENSG00000232254.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000232254.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128309.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128309.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128309.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128309.16.nosex .\n",
      "--extract range: 14616 variants excluded.\n",
      "--extract range: 593 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "593 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128309.16.bed +\n",
      "./output/chr22_data.ENSG00000128309.16.bim +\n",
      "./output/chr22_data.ENSG00000128309.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128309.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128311.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128311.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128311.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128311.13.nosex .\n",
      "--extract range: 14616 variants excluded.\n",
      "--extract range: 593 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "593 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128311.13.bed +\n",
      "./output/chr22_data.ENSG00000128311.13.bim +\n",
      "./output/chr22_data.ENSG00000128311.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128311.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100379.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100379.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100379.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100379.17.nosex .\n",
      "--extract range: 14632 variants excluded.\n",
      "--extract range: 577 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "577 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100379.17.bed +\n",
      "./output/chr22_data.ENSG00000100379.17.bim +\n",
      "./output/chr22_data.ENSG00000100379.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100379.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000187045.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000187045.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000187045.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000187045.16.nosex .\n",
      "--extract range: 14644 variants excluded.\n",
      "--extract range: 565 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "565 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000187045.16.bed +\n",
      "./output/chr22_data.ENSG00000187045.16.bim +\n",
      "./output/chr22_data.ENSG00000187045.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000187045.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100385.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100385.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100385.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100385.13.nosex .\n",
      "--extract range: 14651 variants excluded.\n",
      "--extract range: 558 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "558 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100385.13.bed +\n",
      "./output/chr22_data.ENSG00000100385.13.bim +\n",
      "./output/chr22_data.ENSG00000100385.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100385.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133466.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133466.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133466.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133466.13.nosex .\n",
      "--extract range: 14652 variants excluded.\n",
      "--extract range: 557 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "557 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133466.13.bed +\n",
      "./output/chr22_data.ENSG00000133466.13.bim +\n",
      "./output/chr22_data.ENSG00000133466.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133466.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278195.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278195.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278195.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278195.1.nosex .\n",
      "--extract range: 14654 variants excluded.\n",
      "--extract range: 555 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "555 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278195.1.bed +\n",
      "./output/chr22_data.ENSG00000278195.1.bim +\n",
      "./output/chr22_data.ENSG00000278195.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278195.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128340.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128340.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128340.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128340.14.nosex .\n",
      "--extract range: 14666 variants excluded.\n",
      "--extract range: 543 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "543 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128340.14.bed +\n",
      "./output/chr22_data.ENSG00000128340.14.bim +\n",
      "./output/chr22_data.ENSG00000128340.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128340.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100055.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100055.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100055.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100055.20.nosex .\n",
      "--extract range: 14666 variants excluded.\n",
      "--extract range: 543 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "543 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100055.20.bed +\n",
      "./output/chr22_data.ENSG00000100055.20.bim +\n",
      "./output/chr22_data.ENSG00000100055.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100055.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000166897.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000166897.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000166897.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000166897.14.nosex .\n",
      "--extract range: 14709 variants excluded.\n",
      "--extract range: 500 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "500 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000166897.14.bed +\n",
      "./output/chr22_data.ENSG00000166897.14.bim +\n",
      "./output/chr22_data.ENSG00000166897.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000166897.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100060.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100060.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100060.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100060.17.nosex .\n",
      "--extract range: 14743 variants excluded.\n",
      "--extract range: 466 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "466 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100060.17.bed +\n",
      "./output/chr22_data.ENSG00000100060.17.bim +\n",
      "./output/chr22_data.ENSG00000100060.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100060.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100065.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100065.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100065.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100065.14.nosex .\n",
      "--extract range: 14763 variants excluded.\n",
      "--extract range: 446 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "446 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100065.14.bed +\n",
      "./output/chr22_data.ENSG00000100065.14.bim +\n",
      "./output/chr22_data.ENSG00000100065.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100065.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128283.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128283.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128283.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128283.6.nosex .\n",
      "--extract range: 14763 variants excluded.\n",
      "--extract range: 446 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "446 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128283.6.bed +\n",
      "./output/chr22_data.ENSG00000128283.6.bim +\n",
      "./output/chr22_data.ENSG00000128283.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128283.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100079.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100079.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100079.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100079.6.nosex .\n",
      "--extract range: 14761 variants excluded.\n",
      "--extract range: 448 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "448 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100079.6.bed +\n",
      "./output/chr22_data.ENSG00000100079.6.bim +\n",
      "./output/chr22_data.ENSG00000100079.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100079.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100083.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100083.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100083.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100083.18.nosex .\n",
      "--extract range: 14768 variants excluded.\n",
      "--extract range: 441 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "441 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100083.18.bed +\n",
      "./output/chr22_data.ENSG00000100083.18.bim +\n",
      "./output/chr22_data.ENSG00000100083.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100083.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100092.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100092.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100092.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100092.20.nosex .\n",
      "--extract range: 14776 variants excluded.\n",
      "--extract range: 433 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "433 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100092.20.bed +\n",
      "./output/chr22_data.ENSG00000100092.20.bim +\n",
      "./output/chr22_data.ENSG00000100092.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100092.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000233360.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000233360.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000233360.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000233360.4.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000233360.4.bed +\n",
      "./output/chr22_data.ENSG00000233360.4.bim +\n",
      "./output/chr22_data.ENSG00000233360.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000233360.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241360.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241360.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241360.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241360.1.nosex .\n",
      "--extract range: 14782 variants excluded.\n",
      "--extract range: 427 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "427 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241360.1.bed +\n",
      "./output/chr22_data.ENSG00000241360.1.bim +\n",
      "./output/chr22_data.ENSG00000241360.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241360.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100097.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100097.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100097.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100097.11.nosex .\n",
      "--extract range: 14780 variants excluded.\n",
      "--extract range: 429 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "429 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100097.11.bed +\n",
      "./output/chr22_data.ENSG00000100097.11.bim +\n",
      "./output/chr22_data.ENSG00000100097.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100097.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273899.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273899.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273899.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273899.4.nosex .\n",
      "--extract range: 14783 variants excluded.\n",
      "--extract range: 426 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "426 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273899.4.bed +\n",
      "./output/chr22_data.ENSG00000273899.4.bim +\n",
      "./output/chr22_data.ENSG00000273899.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273899.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100106.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100106.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100106.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100106.19.nosex .\n",
      "--extract range: 14798 variants excluded.\n",
      "--extract range: 411 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "411 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100106.19.bed +\n",
      "./output/chr22_data.ENSG00000100106.19.bim +\n",
      "./output/chr22_data.ENSG00000100106.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100106.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000189060.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000189060.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000189060.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000189060.5.nosex .\n",
      "--extract range: 14836 variants excluded.\n",
      "--extract range: 373 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "373 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000189060.5.bed +\n",
      "./output/chr22_data.ENSG00000189060.5.bim +\n",
      "./output/chr22_data.ENSG00000189060.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000189060.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100116.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100116.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100116.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100116.16.nosex .\n",
      "--extract range: 14833 variants excluded.\n",
      "--extract range: 376 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "376 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100116.16.bed +\n",
      "./output/chr22_data.ENSG00000100116.16.bim +\n",
      "./output/chr22_data.ENSG00000100116.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100116.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100124.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100124.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100124.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100124.14.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100124.14.bed +\n",
      "./output/chr22_data.ENSG00000100124.14.bim +\n",
      "./output/chr22_data.ENSG00000100124.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100124.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100129.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100129.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100129.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100129.17.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100129.17.bed +\n",
      "./output/chr22_data.ENSG00000100129.17.bim +\n",
      "./output/chr22_data.ENSG00000100129.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100129.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279738.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279738.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279738.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279738.1.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279738.1.bed +\n",
      "./output/chr22_data.ENSG00000279738.1.bim +\n",
      "./output/chr22_data.ENSG00000279738.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279738.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100139.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100139.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100139.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100139.13.nosex .\n",
      "--extract range: 14875 variants excluded.\n",
      "--extract range: 334 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "334 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100139.13.bed +\n",
      "./output/chr22_data.ENSG00000100139.13.bim +\n",
      "./output/chr22_data.ENSG00000100139.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100139.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278948.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278948.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278948.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278948.1.nosex .\n",
      "--extract range: 14867 variants excluded.\n",
      "--extract range: 342 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "342 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278948.1.bed +\n",
      "./output/chr22_data.ENSG00000278948.1.bim +\n",
      "./output/chr22_data.ENSG00000278948.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278948.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128346.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128346.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128346.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128346.10.nosex .\n",
      "--extract range: 14868 variants excluded.\n",
      "--extract range: 341 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "341 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128346.10.bed +\n",
      "./output/chr22_data.ENSG00000128346.10.bim +\n",
      "./output/chr22_data.ENSG00000128346.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128346.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100142.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100142.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100142.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100142.14.nosex .\n",
      "--extract range: 14869 variants excluded.\n",
      "--extract range: 340 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "340 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100142.14.bed +\n",
      "./output/chr22_data.ENSG00000100142.14.bim +\n",
      "./output/chr22_data.ENSG00000100142.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100142.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100151.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100151.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100151.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100151.15.nosex .\n",
      "--extract range: 14873 variants excluded.\n",
      "--extract range: 336 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "336 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100151.15.bed +\n",
      "./output/chr22_data.ENSG00000100151.15.bim +\n",
      "./output/chr22_data.ENSG00000100151.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100151.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100156.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100156.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100156.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100156.10.nosex .\n",
      "--extract range: 14882 variants excluded.\n",
      "--extract range: 327 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "327 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100156.10.bed +\n",
      "./output/chr22_data.ENSG00000100156.10.bim +\n",
      "./output/chr22_data.ENSG00000100156.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100156.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272720.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272720.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272720.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272720.1.nosex .\n",
      "--extract range: 14885 variants excluded.\n",
      "--extract range: 324 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "324 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272720.1.bed +\n",
      "./output/chr22_data.ENSG00000272720.1.bim +\n",
      "./output/chr22_data.ENSG00000272720.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272720.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128298.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128298.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128298.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128298.16.nosex .\n",
      "--extract range: 14890 variants excluded.\n",
      "--extract range: 319 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "319 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128298.16.bed +\n",
      "./output/chr22_data.ENSG00000128298.16.bim +\n",
      "./output/chr22_data.ENSG00000128298.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128298.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185022.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185022.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185022.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185022.11.nosex .\n",
      "--extract range: 14894 variants excluded.\n",
      "--extract range: 315 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "315 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185022.11.bed +\n",
      "./output/chr22_data.ENSG00000185022.11.bim +\n",
      "./output/chr22_data.ENSG00000185022.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185022.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184381.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184381.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184381.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184381.18.nosex .\n",
      "--extract range: 14894 variants excluded.\n",
      "--extract range: 315 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "315 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184381.18.bed +\n",
      "./output/chr22_data.ENSG00000184381.18.bim +\n",
      "./output/chr22_data.ENSG00000184381.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184381.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198792.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198792.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198792.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198792.12.nosex .\n",
      "--extract range: 14886 variants excluded.\n",
      "--extract range: 323 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "323 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198792.12.bed +\n",
      "./output/chr22_data.ENSG00000198792.12.bim +\n",
      "./output/chr22_data.ENSG00000198792.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198792.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000213923.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000213923.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000213923.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000213923.11.nosex .\n",
      "--extract range: 14886 variants excluded.\n",
      "--extract range: 323 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "323 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000213923.11.bed +\n",
      "./output/chr22_data.ENSG00000213923.11.bim +\n",
      "./output/chr22_data.ENSG00000213923.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000213923.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000244627.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000244627.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000244627.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000244627.5.nosex .\n",
      "--extract range: 14876 variants excluded.\n",
      "--extract range: 333 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "333 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000244627.5.bed +\n",
      "./output/chr22_data.ENSG00000244627.5.bim +\n",
      "./output/chr22_data.ENSG00000244627.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000244627.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100196.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100196.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100196.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100196.10.nosex .\n",
      "--extract range: 14862 variants excluded.\n",
      "--extract range: 347 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "347 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100196.10.bed +\n",
      "./output/chr22_data.ENSG00000100196.10.bim +\n",
      "./output/chr22_data.ENSG00000100196.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100196.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100201.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100201.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100201.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100201.20.nosex .\n",
      "--extract range: 14872 variants excluded.\n",
      "--extract range: 337 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "337 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100201.20.bed +\n",
      "./output/chr22_data.ENSG00000100201.20.bim +\n",
      "./output/chr22_data.ENSG00000100201.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100201.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100206.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100206.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100206.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100206.9.nosex .\n",
      "--extract range: 14873 variants excluded.\n",
      "--extract range: 336 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "336 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100206.9.bed +\n",
      "./output/chr22_data.ENSG00000100206.9.bim +\n",
      "./output/chr22_data.ENSG00000100206.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100206.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100211.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100211.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100211.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100211.10.nosex .\n",
      "--extract range: 14877 variants excluded.\n",
      "--extract range: 332 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "332 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100211.10.bed +\n",
      "./output/chr22_data.ENSG00000100211.10.bim +\n",
      "./output/chr22_data.ENSG00000100211.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100211.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000228274.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000228274.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000228274.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000228274.3.nosex .\n",
      "--extract range: 14875 variants excluded.\n",
      "--extract range: 334 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "334 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000228274.3.bed +\n",
      "./output/chr22_data.ENSG00000228274.3.bim +\n",
      "./output/chr22_data.ENSG00000228274.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000228274.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100216.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100216.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100216.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100216.5.nosex .\n",
      "--extract range: 14875 variants excluded.\n",
      "--extract range: 334 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "334 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100216.5.bed +\n",
      "./output/chr22_data.ENSG00000100216.5.bim +\n",
      "./output/chr22_data.ENSG00000100216.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100216.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100221.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100221.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100221.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100221.10.nosex .\n",
      "--extract range: 14872 variants excluded.\n",
      "--extract range: 337 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "337 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100221.10.bed +\n",
      "./output/chr22_data.ENSG00000100221.10.bim +\n",
      "./output/chr22_data.ENSG00000100221.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100221.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100226.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100226.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100226.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100226.15.nosex .\n",
      "--extract range: 14872 variants excluded.\n",
      "--extract range: 337 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "337 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100226.15.bed +\n",
      "./output/chr22_data.ENSG00000100226.15.bim +\n",
      "./output/chr22_data.ENSG00000100226.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100226.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000229598.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000229598.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000229598.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000229598.1.nosex .\n",
      "--extract range: 14870 variants excluded.\n",
      "--extract range: 339 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "339 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000229598.1.bed +\n",
      "./output/chr22_data.ENSG00000229598.1.bim +\n",
      "./output/chr22_data.ENSG00000229598.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000229598.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273096.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273096.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273096.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273096.1.nosex .\n",
      "--extract range: 14868 variants excluded.\n",
      "--extract range: 341 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "341 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273096.1.bed +\n",
      "./output/chr22_data.ENSG00000273096.1.bim +\n",
      "./output/chr22_data.ENSG00000273096.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273096.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000230149.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000230149.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000230149.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000230149.2.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000230149.2.bed +\n",
      "./output/chr22_data.ENSG00000230149.2.bim +\n",
      "./output/chr22_data.ENSG00000230149.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000230149.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225450.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225450.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225450.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225450.1.nosex .\n",
      "--extract range: 14865 variants excluded.\n",
      "--extract range: 344 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "344 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225450.1.bed +\n",
      "./output/chr22_data.ENSG00000225450.1.bim +\n",
      "./output/chr22_data.ENSG00000225450.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225450.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272669.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272669.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272669.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272669.1.nosex .\n",
      "--extract range: 14867 variants excluded.\n",
      "--extract range: 342 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "342 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272669.1.bed +\n",
      "./output/chr22_data.ENSG00000272669.1.bim +\n",
      "./output/chr22_data.ENSG00000272669.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272669.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273076.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273076.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273076.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273076.1.nosex .\n",
      "--extract range: 14867 variants excluded.\n",
      "--extract range: 342 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "342 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273076.1.bed +\n",
      "./output/chr22_data.ENSG00000273076.1.bim +\n",
      "./output/chr22_data.ENSG00000273076.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273076.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100242.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100242.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100242.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100242.15.nosex .\n",
      "--extract range: 14876 variants excluded.\n",
      "--extract range: 333 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "333 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100242.15.bed +\n",
      "./output/chr22_data.ENSG00000100242.15.bim +\n",
      "./output/chr22_data.ENSG00000100242.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100242.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100246.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100246.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100246.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100246.12.nosex .\n",
      "--extract range: 14878 variants excluded.\n",
      "--extract range: 331 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "331 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100246.12.bed +\n",
      "./output/chr22_data.ENSG00000100246.12.bim +\n",
      "./output/chr22_data.ENSG00000100246.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100246.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000221890.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000221890.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000221890.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000221890.3.nosex .\n",
      "--extract range: 14871 variants excluded.\n",
      "--extract range: 338 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "338 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000221890.3.bed +\n",
      "./output/chr22_data.ENSG00000221890.3.bim +\n",
      "./output/chr22_data.ENSG00000221890.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000221890.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183741.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183741.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183741.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183741.11.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183741.11.bed +\n",
      "./output/chr22_data.ENSG00000183741.11.bim +\n",
      "./output/chr22_data.ENSG00000183741.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183741.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128383.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128383.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128383.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128383.12.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128383.12.bed +\n",
      "./output/chr22_data.ENSG00000128383.12.bim +\n",
      "./output/chr22_data.ENSG00000128383.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128383.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000179750.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000179750.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000179750.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000179750.15.nosex .\n",
      "--extract range: 14864 variants excluded.\n",
      "--extract range: 345 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "345 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000179750.15.bed +\n",
      "./output/chr22_data.ENSG00000179750.15.bim +\n",
      "./output/chr22_data.ENSG00000179750.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000179750.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000244509.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000244509.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000244509.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000244509.3.nosex .\n",
      "--extract range: 14867 variants excluded.\n",
      "--extract range: 342 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "342 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000244509.3.bed +\n",
      "./output/chr22_data.ENSG00000244509.3.bim +\n",
      "./output/chr22_data.ENSG00000244509.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000244509.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000243811.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000243811.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000243811.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000243811.8.nosex .\n",
      "--extract range: 14865 variants excluded.\n",
      "--extract range: 344 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "344 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000243811.8.bed +\n",
      "./output/chr22_data.ENSG00000243811.8.bim +\n",
      "./output/chr22_data.ENSG00000243811.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000243811.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128394.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128394.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128394.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128394.16.nosex .\n",
      "--extract range: 14864 variants excluded.\n",
      "--extract range: 345 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "345 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128394.16.bed +\n",
      "./output/chr22_data.ENSG00000128394.16.bim +\n",
      "./output/chr22_data.ENSG00000128394.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128394.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000239713.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000239713.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000239713.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000239713.8.nosex .\n",
      "--extract range: 14855 variants excluded.\n",
      "--extract range: 354 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "354 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000239713.8.bed +\n",
      "./output/chr22_data.ENSG00000239713.8.bim +\n",
      "./output/chr22_data.ENSG00000239713.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000239713.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225720.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225720.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225720.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225720.6.nosex .\n",
      "--extract range: 14852 variants excluded.\n",
      "--extract range: 357 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "357 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225720.6.bed +\n",
      "./output/chr22_data.ENSG00000225720.6.bim +\n",
      "./output/chr22_data.ENSG00000225720.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225720.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100298.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100298.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100298.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100298.15.nosex .\n",
      "--extract range: 14850 variants excluded.\n",
      "--extract range: 359 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "359 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100298.15.bed +\n",
      "./output/chr22_data.ENSG00000100298.15.bim +\n",
      "./output/chr22_data.ENSG00000100298.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100298.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279833.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279833.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279833.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279833.1.nosex .\n",
      "--extract range: 14839 variants excluded.\n",
      "--extract range: 370 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999962.\n",
      "370 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279833.1.bed +\n",
      "./output/chr22_data.ENSG00000279833.1.bim +\n",
      "./output/chr22_data.ENSG00000279833.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279833.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100307.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100307.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100307.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100307.12.nosex .\n",
      "--extract range: 14843 variants excluded.\n",
      "--extract range: 366 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999966.\n",
      "366 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100307.12.bed +\n",
      "./output/chr22_data.ENSG00000100307.12.bim +\n",
      "./output/chr22_data.ENSG00000100307.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100307.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100311.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100311.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100311.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100311.16.nosex .\n",
      "--extract range: 14853 variants excluded.\n",
      "--extract range: 356 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "356 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100311.16.bed +\n",
      "./output/chr22_data.ENSG00000100311.16.bim +\n",
      "./output/chr22_data.ENSG00000100311.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100311.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000209480.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000209480.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000209480.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000209480.1.nosex .\n",
      "--extract range: 14857 variants excluded.\n",
      "--extract range: 352 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "352 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000209480.1.bed +\n",
      "./output/chr22_data.ENSG00000209480.1.bim +\n",
      "./output/chr22_data.ENSG00000209480.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000209480.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100316.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100316.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100316.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100316.15.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100316.15.bed +\n",
      "./output/chr22_data.ENSG00000100316.15.bim +\n",
      "./output/chr22_data.ENSG00000100316.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100316.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100321.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100321.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100321.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100321.14.nosex .\n",
      "--extract range: 14871 variants excluded.\n",
      "--extract range: 338 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "338 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100321.14.bed +\n",
      "./output/chr22_data.ENSG00000100321.14.bim +\n",
      "./output/chr22_data.ENSG00000100321.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100321.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100324.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100324.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100324.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100324.13.nosex .\n",
      "--extract range: 14871 variants excluded.\n",
      "--extract range: 338 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "338 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100324.13.bed +\n",
      "./output/chr22_data.ENSG00000100324.13.bim +\n",
      "./output/chr22_data.ENSG00000100324.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100324.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128268.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128268.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128268.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128268.11.nosex .\n",
      "--extract range: 14874 variants excluded.\n",
      "--extract range: 335 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "335 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128268.11.bed +\n",
      "./output/chr22_data.ENSG00000128268.11.bim +\n",
      "./output/chr22_data.ENSG00000128268.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128268.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100335.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100335.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100335.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100335.13.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100335.13.bed +\n",
      "./output/chr22_data.ENSG00000100335.13.bim +\n",
      "./output/chr22_data.ENSG00000100335.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100335.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128272.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128272.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128272.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128272.14.nosex .\n",
      "--extract range: 14865 variants excluded.\n",
      "--extract range: 344 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "344 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128272.14.bed +\n",
      "./output/chr22_data.ENSG00000128272.14.bim +\n",
      "./output/chr22_data.ENSG00000128272.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128272.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000187051.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000187051.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000187051.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000187051.8.nosex .\n",
      "--extract range: 14864 variants excluded.\n",
      "--extract range: 345 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "345 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000187051.8.bed +\n",
      "./output/chr22_data.ENSG00000187051.8.bim +\n",
      "./output/chr22_data.ENSG00000187051.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000187051.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100346.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100346.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100346.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100346.17.nosex .\n",
      "--extract range: 14873 variants excluded.\n",
      "--extract range: 336 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "336 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100346.17.bed +\n",
      "./output/chr22_data.ENSG00000100346.17.bim +\n",
      "./output/chr22_data.ENSG00000100346.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100346.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226085.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226085.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226085.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226085.3.nosex .\n",
      "--extract range: 14940 variants excluded.\n",
      "--extract range: 269 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999995.\n",
      "269 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226085.3.bed +\n",
      "./output/chr22_data.ENSG00000226085.3.bim +\n",
      "./output/chr22_data.ENSG00000226085.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226085.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100351.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100351.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100351.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100351.16.nosex .\n",
      "--extract range: 14942 variants excluded.\n",
      "--extract range: 267 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999995.\n",
      "267 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100351.16.bed +\n",
      "./output/chr22_data.ENSG00000100351.16.bim +\n",
      "./output/chr22_data.ENSG00000100351.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100351.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000225528.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000225528.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000225528.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000225528.3.nosex .\n",
      "--extract range: 14946 variants excluded.\n",
      "--extract range: 263 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999995.\n",
      "263 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000225528.3.bed +\n",
      "./output/chr22_data.ENSG00000225528.3.bim +\n",
      "./output/chr22_data.ENSG00000225528.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000225528.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000133477.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000133477.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000133477.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000133477.16.nosex .\n",
      "--extract range: 14942 variants excluded.\n",
      "--extract range: 267 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999995.\n",
      "267 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000133477.16.bed +\n",
      "./output/chr22_data.ENSG00000133477.16.bim +\n",
      "./output/chr22_data.ENSG00000133477.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000133477.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100354.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100354.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100354.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100354.20.nosex .\n",
      "--extract range: 14944 variants excluded.\n",
      "--extract range: 265 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999995.\n",
      "265 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100354.20.bed +\n",
      "./output/chr22_data.ENSG00000100354.20.bim +\n",
      "./output/chr22_data.ENSG00000100354.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100354.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000239900.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000239900.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000239900.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000239900.12.nosex .\n",
      "--extract range: 14940 variants excluded.\n",
      "--extract range: 269 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "269 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000239900.12.bed +\n",
      "./output/chr22_data.ENSG00000239900.12.bim +\n",
      "./output/chr22_data.ENSG00000239900.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000239900.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100359.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100359.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100359.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100359.20.nosex .\n",
      "--extract range: 14935 variants excluded.\n",
      "--extract range: 274 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "274 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100359.20.bed +\n",
      "./output/chr22_data.ENSG00000100359.20.bim +\n",
      "./output/chr22_data.ENSG00000100359.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100359.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272834.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272834.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272834.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272834.1.nosex .\n",
      "--extract range: 14943 variants excluded.\n",
      "--extract range: 266 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "266 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272834.1.bed +\n",
      "./output/chr22_data.ENSG00000272834.1.bim +\n",
      "./output/chr22_data.ENSG00000272834.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272834.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000227413.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000227413.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000227413.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000227413.1.nosex .\n",
      "--extract range: 14949 variants excluded.\n",
      "--extract range: 260 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "260 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000227413.1.bed +\n",
      "./output/chr22_data.ENSG00000227413.1.bim +\n",
      "./output/chr22_data.ENSG00000227413.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000227413.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000232564.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000232564.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000232564.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000232564.3.nosex .\n",
      "--extract range: 14968 variants excluded.\n",
      "--extract range: 241 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "241 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000232564.3.bed +\n",
      "./output/chr22_data.ENSG00000232564.3.bim +\n",
      "./output/chr22_data.ENSG00000232564.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000232564.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000230071.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000230071.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000230071.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000230071.2.nosex .\n",
      "--extract range: 14959 variants excluded.\n",
      "--extract range: 250 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "250 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000230071.2.bed +\n",
      "./output/chr22_data.ENSG00000230071.2.bim +\n",
      "./output/chr22_data.ENSG00000230071.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000230071.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000196588.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000196588.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000196588.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000196588.14.nosex .\n",
      "--extract range: 14951 variants excluded.\n",
      "--extract range: 258 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "258 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000196588.14.bed +\n",
      "./output/chr22_data.ENSG00000196588.14.bim +\n",
      "./output/chr22_data.ENSG00000196588.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000196588.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128285.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128285.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128285.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128285.4.nosex .\n",
      "--extract range: 14943 variants excluded.\n",
      "--extract range: 266 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "266 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128285.4.bed +\n",
      "./output/chr22_data.ENSG00000128285.4.bim +\n",
      "./output/chr22_data.ENSG00000128285.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128285.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100372.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100372.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100372.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100372.14.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100372.14.bed +\n",
      "./output/chr22_data.ENSG00000100372.14.bim +\n",
      "./output/chr22_data.ENSG00000100372.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100372.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100380.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100380.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100380.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100380.13.nosex .\n",
      "--extract range: 14950 variants excluded.\n",
      "--extract range: 259 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "259 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100380.13.bed +\n",
      "./output/chr22_data.ENSG00000100380.13.bim +\n",
      "./output/chr22_data.ENSG00000100380.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100380.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000196236.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000196236.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000196236.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000196236.12.nosex .\n",
      "--extract range: 14950 variants excluded.\n",
      "--extract range: 259 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "259 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000196236.12.bed +\n",
      "./output/chr22_data.ENSG00000196236.12.bim +\n",
      "./output/chr22_data.ENSG00000196236.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000196236.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100387.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100387.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100387.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100387.8.nosex .\n",
      "--extract range: 14955 variants excluded.\n",
      "--extract range: 254 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "254 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100387.8.bed +\n",
      "./output/chr22_data.ENSG00000100387.8.bim +\n",
      "./output/chr22_data.ENSG00000100387.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100387.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237214.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237214.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237214.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237214.3.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237214.3.bed +\n",
      "./output/chr22_data.ENSG00000237214.3.bim +\n",
      "./output/chr22_data.ENSG00000237214.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237214.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000213857.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000213857.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000213857.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000213857.3.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000213857.3.bed +\n",
      "./output/chr22_data.ENSG00000213857.3.bim +\n",
      "./output/chr22_data.ENSG00000213857.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000213857.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100393.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100393.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100393.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100393.12.nosex .\n",
      "--extract range: 14971 variants excluded.\n",
      "--extract range: 238 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "238 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100393.12.bed +\n",
      "./output/chr22_data.ENSG00000100393.12.bim +\n",
      "./output/chr22_data.ENSG00000100393.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100393.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000231993.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000231993.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000231993.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000231993.1.nosex .\n",
      "--extract range: 14980 variants excluded.\n",
      "--extract range: 229 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "229 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000231993.1.bed +\n",
      "./output/chr22_data.ENSG00000231993.1.bim +\n",
      "./output/chr22_data.ENSG00000231993.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000231993.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100395.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100395.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100395.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100395.14.nosex .\n",
      "--extract range: 14979 variants excluded.\n",
      "--extract range: 230 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "230 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100395.14.bed +\n",
      "./output/chr22_data.ENSG00000100395.14.bim +\n",
      "./output/chr22_data.ENSG00000100395.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100395.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235513.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235513.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235513.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235513.1.nosex .\n",
      "--extract range: 14984 variants excluded.\n",
      "--extract range: 225 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "225 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235513.1.bed +\n",
      "./output/chr22_data.ENSG00000235513.1.bim +\n",
      "./output/chr22_data.ENSG00000235513.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235513.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100399.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100399.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100399.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100399.15.nosex .\n",
      "--extract range: 14987 variants excluded.\n",
      "--extract range: 222 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "222 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100399.15.bed +\n",
      "./output/chr22_data.ENSG00000100399.15.bim +\n",
      "./output/chr22_data.ENSG00000100399.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100399.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100401.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100401.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100401.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100401.19.nosex .\n",
      "--extract range: 14986 variants excluded.\n",
      "--extract range: 223 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "223 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100401.19.bed +\n",
      "./output/chr22_data.ENSG00000100401.19.bim +\n",
      "./output/chr22_data.ENSG00000100401.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100401.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100403.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100403.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100403.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100403.11.nosex .\n",
      "--extract range: 14982 variants excluded.\n",
      "--extract range: 227 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999969.\n",
      "227 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100403.11.bed +\n",
      "./output/chr22_data.ENSG00000100403.11.bim +\n",
      "./output/chr22_data.ENSG00000100403.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100403.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000167074.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000167074.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000167074.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000167074.14.nosex .\n",
      "--extract range: 14987 variants excluded.\n",
      "--extract range: 222 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "222 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000167074.14.bed +\n",
      "./output/chr22_data.ENSG00000167074.14.bim +\n",
      "./output/chr22_data.ENSG00000167074.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000167074.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273424.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273424.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273424.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273424.1.nosex .\n",
      "--extract range: 14974 variants excluded.\n",
      "--extract range: 235 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99997.\n",
      "235 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273424.1.bed +\n",
      "./output/chr22_data.ENSG00000273424.1.bim +\n",
      "./output/chr22_data.ENSG00000273424.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273424.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183864.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183864.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183864.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183864.4.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999971.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183864.4.bed +\n",
      "./output/chr22_data.ENSG00000183864.4.bim +\n",
      "./output/chr22_data.ENSG00000183864.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183864.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100410.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100410.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100410.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100410.7.nosex .\n",
      "--extract range: 14961 variants excluded.\n",
      "--extract range: 248 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "248 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100410.7.bed +\n",
      "./output/chr22_data.ENSG00000100410.7.bim +\n",
      "./output/chr22_data.ENSG00000100410.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100410.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100412.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100412.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100412.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100412.15.nosex .\n",
      "--extract range: 14960 variants excluded.\n",
      "--extract range: 249 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999972.\n",
      "249 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100412.15.bed +\n",
      "./output/chr22_data.ENSG00000100412.15.bim +\n",
      "./output/chr22_data.ENSG00000100412.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100412.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100413.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100413.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100413.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100413.16.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100413.16.bed +\n",
      "./output/chr22_data.ENSG00000100413.16.bim +\n",
      "./output/chr22_data.ENSG00000100413.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100413.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000172346.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000172346.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000172346.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000172346.14.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000172346.14.bed +\n",
      "./output/chr22_data.ENSG00000172346.14.bim +\n",
      "./output/chr22_data.ENSG00000172346.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000172346.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100417.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100417.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100417.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100417.11.nosex .\n",
      "--extract range: 14941 variants excluded.\n",
      "--extract range: 268 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "268 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100417.11.bed +\n",
      "./output/chr22_data.ENSG00000100417.11.bim +\n",
      "./output/chr22_data.ENSG00000100417.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100417.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100418.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100418.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100418.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100418.7.nosex .\n",
      "--extract range: 14939 variants excluded.\n",
      "--extract range: 270 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "270 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100418.7.bed +\n",
      "./output/chr22_data.ENSG00000100418.7.bim +\n",
      "./output/chr22_data.ENSG00000100418.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100418.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000196419.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000196419.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000196419.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000196419.12.nosex .\n",
      "--extract range: 14939 variants excluded.\n",
      "--extract range: 270 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "270 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000196419.12.bed +\n",
      "./output/chr22_data.ENSG00000196419.12.bim +\n",
      "./output/chr22_data.ENSG00000196419.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000196419.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184208.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184208.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184208.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184208.10.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184208.10.bed +\n",
      "./output/chr22_data.ENSG00000184208.10.bim +\n",
      "./output/chr22_data.ENSG00000184208.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184208.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100138.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100138.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100138.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100138.13.nosex .\n",
      "--extract range: 14945 variants excluded.\n",
      "--extract range: 264 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "264 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100138.13.bed +\n",
      "./output/chr22_data.ENSG00000100138.13.bim +\n",
      "./output/chr22_data.ENSG00000100138.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100138.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000167077.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000167077.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000167077.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000167077.12.nosex .\n",
      "--extract range: 14944 variants excluded.\n",
      "--extract range: 265 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "265 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000167077.12.bed +\n",
      "./output/chr22_data.ENSG00000167077.12.bim +\n",
      "./output/chr22_data.ENSG00000167077.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000167077.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100147.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100147.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100147.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100147.13.nosex .\n",
      "--extract range: 14933 variants excluded.\n",
      "--extract range: 276 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "276 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100147.13.bed +\n",
      "./output/chr22_data.ENSG00000100147.13.bim +\n",
      "./output/chr22_data.ENSG00000100147.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100147.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184068.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184068.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184068.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184068.2.nosex .\n",
      "--extract range: 14926 variants excluded.\n",
      "--extract range: 283 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999965.\n",
      "283 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184068.2.bed +\n",
      "./output/chr22_data.ENSG00000184068.2.bim +\n",
      "./output/chr22_data.ENSG00000184068.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184068.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198911.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198911.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198911.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198911.11.nosex .\n",
      "--extract range: 14920 variants excluded.\n",
      "--extract range: 289 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999966.\n",
      "289 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198911.11.bed +\n",
      "./output/chr22_data.ENSG00000198911.11.bim +\n",
      "./output/chr22_data.ENSG00000198911.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198911.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000234965.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000234965.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000234965.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000234965.2.nosex .\n",
      "--extract range: 14907 variants excluded.\n",
      "--extract range: 302 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "302 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000234965.2.bed +\n",
      "./output/chr22_data.ENSG00000234965.2.bim +\n",
      "./output/chr22_data.ENSG00000234965.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000234965.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000159958.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000159958.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000159958.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000159958.6.nosex .\n",
      "--extract range: 14903 variants excluded.\n",
      "--extract range: 306 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "306 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000159958.6.bed +\n",
      "./output/chr22_data.ENSG00000159958.6.bim +\n",
      "./output/chr22_data.ENSG00000159958.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000159958.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100162.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100162.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100162.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100162.14.nosex .\n",
      "--extract range: 14903 variants excluded.\n",
      "--extract range: 306 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "306 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100162.14.bed +\n",
      "./output/chr22_data.ENSG00000100162.14.bim +\n",
      "./output/chr22_data.ENSG00000100162.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100162.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100167.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100167.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100167.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100167.19.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100167.19.bed +\n",
      "./output/chr22_data.ENSG00000100167.19.bim +\n",
      "./output/chr22_data.ENSG00000100167.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100167.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183066.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183066.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183066.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183066.14.nosex .\n",
      "--extract range: 14895 variants excluded.\n",
      "--extract range: 314 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "314 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183066.14.bed +\n",
      "./output/chr22_data.ENSG00000183066.14.bim +\n",
      "./output/chr22_data.ENSG00000183066.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183066.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198951.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198951.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198951.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198951.11.nosex .\n",
      "--extract range: 14908 variants excluded.\n",
      "--extract range: 301 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "301 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198951.11.bed +\n",
      "./output/chr22_data.ENSG00000198951.11.bim +\n",
      "./output/chr22_data.ENSG00000198951.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198951.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000177096.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000177096.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000177096.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000177096.8.nosex .\n",
      "--extract range: 14908 variants excluded.\n",
      "--extract range: 301 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "301 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000177096.8.bed +\n",
      "./output/chr22_data.ENSG00000177096.8.bim +\n",
      "./output/chr22_data.ENSG00000177096.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000177096.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183172.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183172.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183172.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183172.8.nosex .\n",
      "--extract range: 14908 variants excluded.\n",
      "--extract range: 301 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "301 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183172.8.bed +\n",
      "./output/chr22_data.ENSG00000183172.8.bim +\n",
      "./output/chr22_data.ENSG00000183172.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183172.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000270083.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000270083.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000270083.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000270083.1.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000270083.1.bed +\n",
      "./output/chr22_data.ENSG00000270083.1.bim +\n",
      "./output/chr22_data.ENSG00000270083.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000270083.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184983.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184983.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184983.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184983.9.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184983.9.bed +\n",
      "./output/chr22_data.ENSG00000184983.9.bim +\n",
      "./output/chr22_data.ENSG00000184983.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184983.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000237037.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000237037.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000237037.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000237037.9.nosex .\n",
      "--extract range: 14902 variants excluded.\n",
      "--extract range: 307 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "307 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000237037.9.bed +\n",
      "./output/chr22_data.ENSG00000237037.9.bim +\n",
      "./output/chr22_data.ENSG00000237037.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000237037.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100197.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100197.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100197.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100197.20.nosex .\n",
      "--extract range: 14894 variants excluded.\n",
      "--extract range: 315 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "315 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100197.20.bed +\n",
      "./output/chr22_data.ENSG00000100197.20.bim +\n",
      "./output/chr22_data.ENSG00000100197.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100197.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000227370.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000227370.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000227370.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000227370.1.nosex .\n",
      "--extract range: 14893 variants excluded.\n",
      "--extract range: 316 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999987.\n",
      "316 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000227370.1.bed +\n",
      "./output/chr22_data.ENSG00000227370.1.bim +\n",
      "./output/chr22_data.ENSG00000227370.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000227370.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000281538.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000281538.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000281538.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000281538.1.nosex .\n",
      "--extract range: 14890 variants excluded.\n",
      "--extract range: 319 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "319 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000281538.1.bed +\n",
      "./output/chr22_data.ENSG00000281538.1.bim +\n",
      "./output/chr22_data.ENSG00000281538.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000281538.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205702.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205702.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205702.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205702.10.nosex .\n",
      "--extract range: 14891 variants excluded.\n",
      "--extract range: 318 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "318 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205702.10.bed +\n",
      "./output/chr22_data.ENSG00000205702.10.bim +\n",
      "./output/chr22_data.ENSG00000205702.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205702.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226450.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226450.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226450.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226450.2.nosex .\n",
      "--extract range: 14886 variants excluded.\n",
      "--extract range: 323 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "323 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226450.2.bed +\n",
      "./output/chr22_data.ENSG00000226450.2.bim +\n",
      "./output/chr22_data.ENSG00000226450.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226450.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182057.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182057.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182057.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182057.4.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182057.4.bed +\n",
      "./output/chr22_data.ENSG00000182057.4.bim +\n",
      "./output/chr22_data.ENSG00000182057.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182057.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000233903.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000233903.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000233903.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000233903.2.nosex .\n",
      "--extract range: 14862 variants excluded.\n",
      "--extract range: 347 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "347 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000233903.2.bed +\n",
      "./output/chr22_data.ENSG00000233903.2.bim +\n",
      "./output/chr22_data.ENSG00000233903.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000233903.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273366.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273366.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273366.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273366.1.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999984.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273366.1.bed +\n",
      "./output/chr22_data.ENSG00000273366.1.bim +\n",
      "./output/chr22_data.ENSG00000273366.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273366.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100207.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100207.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100207.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100207.18.nosex .\n",
      "--extract range: 14858 variants excluded.\n",
      "--extract range: 351 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "351 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100207.18.bed +\n",
      "./output/chr22_data.ENSG00000100207.18.bim +\n",
      "./output/chr22_data.ENSG00000100207.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100207.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000229891.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000229891.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000229891.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000229891.1.nosex .\n",
      "--extract range: 14853 variants excluded.\n",
      "--extract range: 356 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "356 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000229891.1.bed +\n",
      "./output/chr22_data.ENSG00000229891.1.bim +\n",
      "./output/chr22_data.ENSG00000229891.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000229891.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235568.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235568.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235568.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235568.6.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999989.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235568.6.bed +\n",
      "./output/chr22_data.ENSG00000235568.6.bim +\n",
      "./output/chr22_data.ENSG00000235568.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235568.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000189306.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000189306.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000189306.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000189306.10.nosex .\n",
      "--extract range: 14848 variants excluded.\n",
      "--extract range: 361 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "361 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000189306.10.bed +\n",
      "./output/chr22_data.ENSG00000189306.10.bim +\n",
      "./output/chr22_data.ENSG00000189306.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000189306.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000183569.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000183569.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000183569.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000183569.17.nosex .\n",
      "--extract range: 14847 variants excluded.\n",
      "--extract range: 362 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "362 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000183569.17.bed +\n",
      "./output/chr22_data.ENSG00000183569.17.bim +\n",
      "./output/chr22_data.ENSG00000183569.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000183569.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000202058.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000202058.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000202058.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000202058.1.nosex .\n",
      "--extract range: 14840 variants excluded.\n",
      "--extract range: 369 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "369 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000202058.1.bed +\n",
      "./output/chr22_data.ENSG00000202058.1.bim +\n",
      "./output/chr22_data.ENSG00000202058.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000202058.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182841.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182841.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182841.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182841.12.nosex .\n",
      "--extract range: 14839 variants excluded.\n",
      "--extract range: 370 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "370 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182841.12.bed +\n",
      "./output/chr22_data.ENSG00000182841.12.bim +\n",
      "./output/chr22_data.ENSG00000182841.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182841.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100227.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100227.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100227.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100227.17.nosex .\n",
      "--extract range: 14831 variants excluded.\n",
      "--extract range: 378 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "378 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100227.17.bed +\n",
      "./output/chr22_data.ENSG00000100227.17.bim +\n",
      "./output/chr22_data.ENSG00000100227.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100227.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100243.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100243.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100243.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100243.20.nosex .\n",
      "--extract range: 14820 variants excluded.\n",
      "--extract range: 389 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999986.\n",
      "389 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100243.20.bed +\n",
      "./output/chr22_data.ENSG00000100243.20.bim +\n",
      "./output/chr22_data.ENSG00000100243.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100243.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128274.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128274.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128274.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128274.15.nosex .\n",
      "--extract range: 14785 variants excluded.\n",
      "--extract range: 424 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999964.\n",
      "424 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128274.15.bed +\n",
      "./output/chr22_data.ENSG00000128274.15.bim +\n",
      "./output/chr22_data.ENSG00000128274.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128274.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000234009.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000234009.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000234009.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000234009.1.nosex .\n",
      "--extract range: 14752 variants excluded.\n",
      "--extract range: 457 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999957.\n",
      "457 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000234009.1.bed +\n",
      "./output/chr22_data.ENSG00000234009.1.bim +\n",
      "./output/chr22_data.ENSG00000234009.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000234009.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000274717.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000274717.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000274717.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000274717.1.nosex .\n",
      "--extract range: 14753 variants excluded.\n",
      "--extract range: 456 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999963.\n",
      "456 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000274717.1.bed +\n",
      "./output/chr22_data.ENSG00000274717.1.bim +\n",
      "./output/chr22_data.ENSG00000274717.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000274717.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000242247.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000242247.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000242247.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000242247.10.nosex .\n",
      "--extract range: 14744 variants excluded.\n",
      "--extract range: 465 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999961.\n",
      "465 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000242247.10.bed +\n",
      "./output/chr22_data.ENSG00000242247.10.bim +\n",
      "./output/chr22_data.ENSG00000242247.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000242247.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100266.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100266.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100266.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100266.18.nosex .\n",
      "--extract range: 14729 variants excluded.\n",
      "--extract range: 480 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999953.\n",
      "480 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100266.18.bed +\n",
      "./output/chr22_data.ENSG00000100266.18.bim +\n",
      "./output/chr22_data.ENSG00000100266.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100266.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100271.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100271.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100271.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100271.16.nosex .\n",
      "--extract range: 14713 variants excluded.\n",
      "--extract range: 496 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "496 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100271.16.bed +\n",
      "./output/chr22_data.ENSG00000100271.16.bim +\n",
      "./output/chr22_data.ENSG00000100271.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100271.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100290.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100290.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100290.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100290.2.nosex .\n",
      "--extract range: 14707 variants excluded.\n",
      "--extract range: 502 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "502 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100290.2.bed +\n",
      "./output/chr22_data.ENSG00000100290.2.bim +\n",
      "./output/chr22_data.ENSG00000100290.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100290.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100294.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100294.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100294.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100294.12.nosex .\n",
      "--extract range: 14709 variants excluded.\n",
      "--extract range: 500 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999958.\n",
      "500 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100294.12.bed +\n",
      "./output/chr22_data.ENSG00000100294.12.bim +\n",
      "./output/chr22_data.ENSG00000100294.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100294.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100300.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100300.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100300.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100300.17.nosex .\n",
      "--extract range: 14708 variants excluded.\n",
      "--extract range: 501 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999958.\n",
      "501 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100300.17.bed +\n",
      "./output/chr22_data.ENSG00000100300.17.bim +\n",
      "./output/chr22_data.ENSG00000100300.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100300.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100304.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100304.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100304.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100304.12.nosex .\n",
      "--extract range: 14705 variants excluded.\n",
      "--extract range: 504 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999958.\n",
      "504 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100304.12.bed +\n",
      "./output/chr22_data.ENSG00000100304.12.bim +\n",
      "./output/chr22_data.ENSG00000100304.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100304.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100347.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100347.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100347.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100347.14.nosex .\n",
      "--extract range: 14549 variants excluded.\n",
      "--extract range: 660 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "660 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100347.14.bed +\n",
      "./output/chr22_data.ENSG00000100347.14.bim +\n",
      "./output/chr22_data.ENSG00000100347.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100347.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000188677.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000188677.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000188677.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000188677.14.nosex .\n",
      "--extract range: 14547 variants excluded.\n",
      "--extract range: 662 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999968.\n",
      "662 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000188677.14.bed +\n",
      "./output/chr22_data.ENSG00000188677.14.bim +\n",
      "./output/chr22_data.ENSG00000188677.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000188677.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000280434.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000280434.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000280434.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000280434.1.nosex .\n",
      "--extract range: 14526 variants excluded.\n",
      "--extract range: 683 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999971.\n",
      "683 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000280434.1.bed +\n",
      "./output/chr22_data.ENSG00000280434.1.bim +\n",
      "./output/chr22_data.ENSG00000280434.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000280434.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000138964.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000138964.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000138964.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000138964.16.nosex .\n",
      "--extract range: 14525 variants excluded.\n",
      "--extract range: 684 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "684 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000138964.16.bed +\n",
      "./output/chr22_data.ENSG00000138964.16.bim +\n",
      "./output/chr22_data.ENSG00000138964.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000138964.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279933.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279933.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279933.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279933.1.nosex .\n",
      "--extract range: 14527 variants excluded.\n",
      "--extract range: 682 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "682 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279933.1.bed +\n",
      "./output/chr22_data.ENSG00000279933.1.bim +\n",
      "./output/chr22_data.ENSG00000279933.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279933.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000280011.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000280011.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000280011.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000280011.1.nosex .\n",
      "--extract range: 14514 variants excluded.\n",
      "--extract range: 695 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999956.\n",
      "695 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000280011.1.bed +\n",
      "./output/chr22_data.ENSG00000280011.1.bim +\n",
      "./output/chr22_data.ENSG00000280011.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000280011.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000188636.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000188636.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000188636.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000188636.3.nosex .\n",
      "--extract range: 14524 variants excluded.\n",
      "--extract range: 685 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999957.\n",
      "685 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000188636.3.bed +\n",
      "./output/chr22_data.ENSG00000188636.3.bim +\n",
      "./output/chr22_data.ENSG00000188636.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000188636.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000186654.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000186654.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000186654.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000186654.20.nosex .\n",
      "--extract range: 14562 variants excluded.\n",
      "--extract range: 647 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999955.\n",
      "647 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000186654.20.bed +\n",
      "./output/chr22_data.ENSG00000186654.20.bim +\n",
      "./output/chr22_data.ENSG00000186654.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000186654.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241484.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241484.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241484.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241484.9.nosex .\n",
      "--extract range: 14580 variants excluded.\n",
      "--extract range: 629 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999956.\n",
      "629 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241484.9.bed +\n",
      "./output/chr22_data.ENSG00000241484.9.bim +\n",
      "./output/chr22_data.ENSG00000241484.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241484.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273243.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273243.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273243.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273243.1.nosex .\n",
      "--extract range: 14651 variants excluded.\n",
      "--extract range: 558 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "558 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273243.1.bed +\n",
      "./output/chr22_data.ENSG00000273243.1.bim +\n",
      "./output/chr22_data.ENSG00000273243.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273243.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000226328.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000226328.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000226328.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000226328.6.nosex .\n",
      "--extract range: 14656 variants excluded.\n",
      "--extract range: 553 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "553 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000226328.6.bed +\n",
      "./output/chr22_data.ENSG00000226328.6.bim +\n",
      "./output/chr22_data.ENSG00000226328.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000226328.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000093000.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000093000.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000093000.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000093000.18.nosex .\n",
      "--extract range: 14656 variants excluded.\n",
      "--extract range: 553 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "553 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000093000.18.bed +\n",
      "./output/chr22_data.ENSG00000093000.18.bim +\n",
      "./output/chr22_data.ENSG00000093000.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000093000.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273353.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273353.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273353.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273353.1.nosex .\n",
      "--extract range: 14651 variants excluded.\n",
      "--extract range: 558 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "558 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273353.1.bed +\n",
      "./output/chr22_data.ENSG00000273353.1.bim +\n",
      "./output/chr22_data.ENSG00000273353.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273353.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100364.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100364.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100364.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100364.18.nosex .\n",
      "--extract range: 14676 variants excluded.\n",
      "--extract range: 533 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "533 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100364.18.bed +\n",
      "./output/chr22_data.ENSG00000100364.18.bim +\n",
      "./output/chr22_data.ENSG00000100364.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100364.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100373.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100373.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100373.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100373.9.nosex .\n",
      "--extract range: 14677 variants excluded.\n",
      "--extract range: 532 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "532 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100373.9.bed +\n",
      "./output/chr22_data.ENSG00000100373.9.bim +\n",
      "./output/chr22_data.ENSG00000100373.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100373.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100376.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100376.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100376.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100376.11.nosex .\n",
      "--extract range: 14691 variants excluded.\n",
      "--extract range: 518 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999976.\n",
      "518 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100376.11.bed +\n",
      "./output/chr22_data.ENSG00000100376.11.bim +\n",
      "./output/chr22_data.ENSG00000100376.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100376.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128408.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128408.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128408.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128408.8.nosex .\n",
      "--extract range: 14732 variants excluded.\n",
      "--extract range: 477 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "477 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128408.8.bed +\n",
      "./output/chr22_data.ENSG00000128408.8.bim +\n",
      "./output/chr22_data.ENSG00000128408.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128408.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000077942.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000077942.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000077942.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000077942.18.nosex .\n",
      "--extract range: 14763 variants excluded.\n",
      "--extract range: 446 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "446 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000077942.18.bed +\n",
      "./output/chr22_data.ENSG00000077942.18.bim +\n",
      "./output/chr22_data.ENSG00000077942.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000077942.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000130638.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000130638.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000130638.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000130638.16.nosex .\n",
      "--extract range: 14797 variants excluded.\n",
      "--extract range: 412 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "412 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000130638.16.bed +\n",
      "./output/chr22_data.ENSG00000130638.16.bim +\n",
      "./output/chr22_data.ENSG00000130638.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000130638.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273145.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273145.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273145.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273145.1.nosex .\n",
      "--extract range: 14860 variants excluded.\n",
      "--extract range: 349 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "349 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273145.1.bed +\n",
      "./output/chr22_data.ENSG00000273145.1.bim +\n",
      "./output/chr22_data.ENSG00000273145.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273145.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000231711.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000231711.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000231711.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000231711.2.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000231711.2.bed +\n",
      "./output/chr22_data.ENSG00000231711.2.bim +\n",
      "./output/chr22_data.ENSG00000231711.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000231711.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000241990.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000241990.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000241990.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000241990.5.nosex .\n",
      "--extract range: 14860 variants excluded.\n",
      "--extract range: 349 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "349 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000241990.5.bed +\n",
      "./output/chr22_data.ENSG00000241990.5.bim +\n",
      "./output/chr22_data.ENSG00000241990.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000241990.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182257.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182257.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182257.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182257.6.nosex .\n",
      "--extract range: 14860 variants excluded.\n",
      "--extract range: 349 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "349 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182257.6.bed +\n",
      "./output/chr22_data.ENSG00000182257.6.bim +\n",
      "./output/chr22_data.ENSG00000182257.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182257.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000231010.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000231010.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000231010.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000231010.1.nosex .\n",
      "--extract range: 14858 variants excluded.\n",
      "--extract range: 351 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "351 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000231010.1.bed +\n",
      "./output/chr22_data.ENSG00000231010.1.bim +\n",
      "./output/chr22_data.ENSG00000231010.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000231010.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000197182.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000197182.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000197182.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000197182.14.nosex .\n",
      "--extract range: 14866 variants excluded.\n",
      "--extract range: 343 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "343 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000197182.14.bed +\n",
      "./output/chr22_data.ENSG00000197182.14.bim +\n",
      "./output/chr22_data.ENSG00000197182.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000197182.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000186951.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000186951.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000186951.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000186951.16.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000186951.16.bed +\n",
      "./output/chr22_data.ENSG00000186951.16.bim +\n",
      "./output/chr22_data.ENSG00000186951.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000186951.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205643.10.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205643.10_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205643.10\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205643.10.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205643.10.bed +\n",
      "./output/chr22_data.ENSG00000205643.10.bim +\n",
      "./output/chr22_data.ENSG00000205643.10.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205643.10.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000075234.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000075234.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000075234.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000075234.16.nosex .\n",
      "--extract range: 14849 variants excluded.\n",
      "--extract range: 360 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "360 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000075234.16.bed +\n",
      "./output/chr22_data.ENSG00000075234.16.bim +\n",
      "./output/chr22_data.ENSG00000075234.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000075234.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000277232.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000277232.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000277232.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000277232.2.nosex .\n",
      "--extract range: 14848 variants excluded.\n",
      "--extract range: 361 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "361 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000277232.2.bed +\n",
      "./output/chr22_data.ENSG00000277232.2.bim +\n",
      "./output/chr22_data.ENSG00000277232.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000277232.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000075218.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000075218.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000075218.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000075218.18.nosex .\n",
      "--extract range: 14848 variants excluded.\n",
      "--extract range: 361 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "361 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000075218.18.bed +\n",
      "./output/chr22_data.ENSG00000075218.18.bim +\n",
      "./output/chr22_data.ENSG00000075218.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000075218.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100416.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100416.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100416.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100416.12.nosex .\n",
      "--extract range: 14852 variants excluded.\n",
      "--extract range: 357 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999996.\n",
      "357 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100416.12.bed +\n",
      "./output/chr22_data.ENSG00000100416.12.bim +\n",
      "./output/chr22_data.ENSG00000100416.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100416.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000075275.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000075275.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000075275.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000075275.16.nosex .\n",
      "--extract range: 14785 variants excluded.\n",
      "--extract range: 424 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "424 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000075275.16.bed +\n",
      "./output/chr22_data.ENSG00000075275.16.bim +\n",
      "./output/chr22_data.ENSG00000075275.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000075275.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000075240.16.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000075240.16_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000075240.16\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000075240.16.nosex .\n",
      "--extract range: 14777 variants excluded.\n",
      "--extract range: 432 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "432 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000075240.16.bed +\n",
      "./output/chr22_data.ENSG00000075240.16.bim +\n",
      "./output/chr22_data.ENSG00000075240.16.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000075240.16.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100422.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100422.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100422.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100422.13.nosex .\n",
      "--extract range: 14756 variants excluded.\n",
      "--extract range: 453 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "453 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100422.13.bed +\n",
      "./output/chr22_data.ENSG00000100422.13.bim +\n",
      "./output/chr22_data.ENSG00000100422.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100422.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000260708.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000260708.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000260708.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000260708.1.nosex .\n",
      "--extract range: 14744 variants excluded.\n",
      "--extract range: 465 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "465 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000260708.1.bed +\n",
      "./output/chr22_data.ENSG00000260708.1.bim +\n",
      "./output/chr22_data.ENSG00000260708.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000260708.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000054611.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000054611.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000054611.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000054611.13.nosex .\n",
      "--extract range: 14744 variants excluded.\n",
      "--extract range: 465 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999991.\n",
      "465 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000054611.13.bed +\n",
      "./output/chr22_data.ENSG00000054611.13.bim +\n",
      "./output/chr22_data.ENSG00000054611.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000054611.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000280080.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000280080.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000280080.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000280080.2.nosex .\n",
      "--extract range: 14694 variants excluded.\n",
      "--extract range: 515 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999992.\n",
      "515 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000280080.2.bed +\n",
      "./output/chr22_data.ENSG00000280080.2.bim +\n",
      "./output/chr22_data.ENSG00000280080.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000280080.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278881.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278881.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278881.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278881.2.nosex .\n",
      "--extract range: 14721 variants excluded.\n",
      "--extract range: 488 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "488 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278881.2.bed +\n",
      "./output/chr22_data.ENSG00000278881.2.bim +\n",
      "./output/chr22_data.ENSG00000278881.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278881.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000218357.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000218357.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000218357.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000218357.3.nosex .\n",
      "--extract range: 14653 variants excluded.\n",
      "--extract range: 556 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "556 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000218357.3.bed +\n",
      "./output/chr22_data.ENSG00000218357.3.bim +\n",
      "./output/chr22_data.ENSG00000218357.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000218357.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000213279.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000213279.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000213279.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000213279.2.nosex .\n",
      "--extract range: 14643 variants excluded.\n",
      "--extract range: 566 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "566 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000213279.2.bed +\n",
      "./output/chr22_data.ENSG00000213279.2.bim +\n",
      "./output/chr22_data.ENSG00000213279.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000213279.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000235111.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000235111.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000235111.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000235111.1.nosex .\n",
      "--extract range: 14639 variants excluded.\n",
      "--extract range: 570 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99999.\n",
      "570 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000235111.1.bed +\n",
      "./output/chr22_data.ENSG00000235111.1.bim +\n",
      "./output/chr22_data.ENSG00000235111.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000235111.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000188511.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000188511.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000188511.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000188511.12.nosex .\n",
      "--extract range: 14645 variants excluded.\n",
      "--extract range: 564 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "564 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000188511.12.bed +\n",
      "./output/chr22_data.ENSG00000188511.12.bim +\n",
      "./output/chr22_data.ENSG00000188511.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000188511.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000279345.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000279345.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000279345.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000279345.1.nosex .\n",
      "--extract range: 14747 variants excluded.\n",
      "--extract range: 462 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "462 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000279345.1.bed +\n",
      "./output/chr22_data.ENSG00000279345.1.bim +\n",
      "./output/chr22_data.ENSG00000279345.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000279345.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100425.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100425.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100425.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100425.18.nosex .\n",
      "--extract range: 14757 variants excluded.\n",
      "--extract range: 452 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999978.\n",
      "452 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100425.18.bed +\n",
      "./output/chr22_data.ENSG00000100425.18.bim +\n",
      "./output/chr22_data.ENSG00000100425.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100425.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100426.6.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100426.6_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100426.6\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100426.6.nosex .\n",
      "--extract range: 14781 variants excluded.\n",
      "--extract range: 428 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "428 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100426.6.bed +\n",
      "./output/chr22_data.ENSG00000100426.6.bim +\n",
      "./output/chr22_data.ENSG00000100426.6.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100426.6.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273192.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273192.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273192.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273192.1.nosex .\n",
      "--extract range: 14812 variants excluded.\n",
      "--extract range: 397 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "397 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273192.1.bed +\n",
      "./output/chr22_data.ENSG00000273192.1.bim +\n",
      "./output/chr22_data.ENSG00000273192.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273192.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184164.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184164.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184164.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184164.14.nosex .\n",
      "--extract range: 14827 variants excluded.\n",
      "--extract range: 382 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "382 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184164.14.bed +\n",
      "./output/chr22_data.ENSG00000184164.14.bim +\n",
      "./output/chr22_data.ENSG00000184164.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184164.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000182858.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000182858.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000182858.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000182858.13.nosex .\n",
      "--extract range: 14828 variants excluded.\n",
      "--extract range: 381 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "381 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000182858.13.bed +\n",
      "./output/chr22_data.ENSG00000182858.13.bim +\n",
      "./output/chr22_data.ENSG00000182858.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000182858.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000278869.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000278869.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000278869.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000278869.1.nosex .\n",
      "--extract range: 14825 variants excluded.\n",
      "--extract range: 384 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999975.\n",
      "384 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000278869.1.bed +\n",
      "./output/chr22_data.ENSG00000278869.1.bim +\n",
      "./output/chr22_data.ENSG00000278869.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000278869.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000198355.4.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000198355.4_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000198355.4\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000198355.4.nosex .\n",
      "--extract range: 14827 variants excluded.\n",
      "--extract range: 382 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999974.\n",
      "382 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000198355.4.bed +\n",
      "./output/chr22_data.ENSG00000198355.4.bim +\n",
      "./output/chr22_data.ENSG00000198355.4.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000198355.4.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100427.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100427.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100427.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100427.15.nosex .\n",
      "--extract range: 14838 variants excluded.\n",
      "--extract range: 371 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "371 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100427.15.bed +\n",
      "./output/chr22_data.ENSG00000100427.15.bim +\n",
      "./output/chr22_data.ENSG00000100427.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100427.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000073150.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000073150.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000073150.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000073150.13.nosex .\n",
      "--extract range: 14825 variants excluded.\n",
      "--extract range: 384 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "384 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000073150.13.bed +\n",
      "./output/chr22_data.ENSG00000073150.13.bim +\n",
      "./output/chr22_data.ENSG00000073150.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000073150.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000170638.9.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000170638.9_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000170638.9\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000170638.9.nosex .\n",
      "--extract range: 14832 variants excluded.\n",
      "--extract range: 377 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "377 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000170638.9.bed +\n",
      "./output/chr22_data.ENSG00000170638.9.bim +\n",
      "./output/chr22_data.ENSG00000170638.9.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000170638.9.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273188.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273188.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273188.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273188.1.nosex .\n",
      "--extract range: 14834 variants excluded.\n",
      "--extract range: 375 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "375 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273188.1.bed +\n",
      "./output/chr22_data.ENSG00000273188.1.bim +\n",
      "./output/chr22_data.ENSG00000273188.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273188.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273253.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273253.2_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273253.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273253.2.nosex .\n",
      "--extract range: 14840 variants excluded.\n",
      "--extract range: 369 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "369 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273253.2.bed +\n",
      "./output/chr22_data.ENSG00000273253.2.bim +\n",
      "./output/chr22_data.ENSG00000273253.2.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273253.2.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000073169.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000073169.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000073169.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000073169.13.nosex .\n",
      "--extract range: 14840 variants excluded.\n",
      "--extract range: 369 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "369 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000073169.13.bed +\n",
      "./output/chr22_data.ENSG00000073169.13.bim +\n",
      "./output/chr22_data.ENSG00000073169.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000073169.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272836.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272836.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272836.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272836.1.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272836.1.bed +\n",
      "./output/chr22_data.ENSG00000272836.1.bim +\n",
      "./output/chr22_data.ENSG00000272836.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272836.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000273137.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000273137.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000273137.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000273137.1.nosex .\n",
      "--extract range: 14843 variants excluded.\n",
      "--extract range: 366 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "366 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000273137.1.bed +\n",
      "./output/chr22_data.ENSG00000273137.1.bim +\n",
      "./output/chr22_data.ENSG00000273137.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000273137.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128159.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128159.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128159.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128159.11.nosex .\n",
      "--extract range: 14843 variants excluded.\n",
      "--extract range: 366 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "366 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128159.11.bed +\n",
      "./output/chr22_data.ENSG00000128159.11.bim +\n",
      "./output/chr22_data.ENSG00000128159.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128159.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100429.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100429.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100429.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100429.17.nosex .\n",
      "--extract range: 14844 variants excluded.\n",
      "--extract range: 365 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "365 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100429.17.bed +\n",
      "./output/chr22_data.ENSG00000100429.17.bim +\n",
      "./output/chr22_data.ENSG00000100429.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100429.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000188130.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000188130.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000188130.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000188130.13.nosex .\n",
      "--extract range: 14846 variants excluded.\n",
      "--extract range: 363 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "363 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000188130.13.bed +\n",
      "./output/chr22_data.ENSG00000188130.13.bim +\n",
      "./output/chr22_data.ENSG00000188130.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000188130.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000185386.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000185386.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000185386.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000185386.14.nosex .\n",
      "--extract range: 14848 variants excluded.\n",
      "--extract range: 361 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "361 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000185386.14.bed +\n",
      "./output/chr22_data.ENSG00000185386.14.bim +\n",
      "./output/chr22_data.ENSG00000185386.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000185386.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000196576.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000196576.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000196576.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000196576.14.nosex .\n",
      "--extract range: 14856 variants excluded.\n",
      "--extract range: 353 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "353 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000196576.14.bed +\n",
      "./output/chr22_data.ENSG00000196576.14.bim +\n",
      "./output/chr22_data.ENSG00000196576.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000196576.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205593.11.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205593.11_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205593.11\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205593.11.nosex .\n",
      "--extract range: 14860 variants excluded.\n",
      "--extract range: 349 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "349 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205593.11.bed +\n",
      "./output/chr22_data.ENSG00000205593.11.bim +\n",
      "./output/chr22_data.ENSG00000205593.11.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205593.11.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100239.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100239.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100239.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100239.15.nosex .\n",
      "--extract range: 14861 variants excluded.\n",
      "--extract range: 348 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "348 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100239.15.bed +\n",
      "./output/chr22_data.ENSG00000100239.15.bim +\n",
      "./output/chr22_data.ENSG00000100239.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100239.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100241.20.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100241.20_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100241.20\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100241.20.nosex .\n",
      "--extract range: 14893 variants excluded.\n",
      "--extract range: 316 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "316 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100241.20.bed +\n",
      "./output/chr22_data.ENSG00000100241.20.bim +\n",
      "./output/chr22_data.ENSG00000100241.20.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100241.20.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000128165.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000128165.8_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000128165.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000128165.8.nosex .\n",
      "--extract range: 14893 variants excluded.\n",
      "--extract range: 316 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "316 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000128165.8.bed +\n",
      "./output/chr22_data.ENSG00000128165.8.bim +\n",
      "./output/chr22_data.ENSG00000128165.8.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000128165.8.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100253.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100253.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100253.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100253.12.nosex .\n",
      "--extract range: 14893 variants excluded.\n",
      "--extract range: 316 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "316 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100253.12.bed +\n",
      "./output/chr22_data.ENSG00000100253.12.bim +\n",
      "./output/chr22_data.ENSG00000100253.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100253.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100258.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100258.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100258.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100258.17.nosex .\n",
      "--extract range: 14898 variants excluded.\n",
      "--extract range: 311 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "311 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100258.17.bed +\n",
      "./output/chr22_data.ENSG00000100258.17.bim +\n",
      "./output/chr22_data.ENSG00000100258.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100258.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000025770.18.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000025770.18_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000025770.18\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000025770.18.nosex .\n",
      "--extract range: 14899 variants excluded.\n",
      "--extract range: 310 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "310 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000025770.18.bed +\n",
      "./output/chr22_data.ENSG00000025770.18.bim +\n",
      "./output/chr22_data.ENSG00000025770.18.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000025770.18.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272821.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272821.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272821.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272821.1.nosex .\n",
      "--extract range: 14906 variants excluded.\n",
      "--extract range: 303 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999982.\n",
      "303 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272821.1.bed +\n",
      "./output/chr22_data.ENSG00000272821.1.bim +\n",
      "./output/chr22_data.ENSG00000272821.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272821.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000130489.14.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000130489.14_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000130489.14\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000130489.14.nosex .\n",
      "--extract range: 14907 variants excluded.\n",
      "--extract range: 302 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "302 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000130489.14.bed +\n",
      "./output/chr22_data.ENSG00000130489.14.bim +\n",
      "./output/chr22_data.ENSG00000130489.14.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000130489.14.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000025708.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000025708.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000025708.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000025708.13.nosex .\n",
      "--extract range: 14909 variants excluded.\n",
      "--extract range: 300 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "300 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000025708.13.bed +\n",
      "./output/chr22_data.ENSG00000025708.13.bim +\n",
      "./output/chr22_data.ENSG00000025708.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000025708.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000177989.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000177989.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000177989.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000177989.13.nosex .\n",
      "--extract range: 14910 variants excluded.\n",
      "--extract range: 299 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "299 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000177989.13.bed +\n",
      "./output/chr22_data.ENSG00000177989.13.bim +\n",
      "./output/chr22_data.ENSG00000177989.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000177989.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000272666.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000272666.1_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000272666.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000272666.1.nosex .\n",
      "--extract range: 14915 variants excluded.\n",
      "--extract range: 294 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "294 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000272666.1.bed +\n",
      "./output/chr22_data.ENSG00000272666.1.bim +\n",
      "./output/chr22_data.ENSG00000272666.1.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000272666.1.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000130487.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000130487.5_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000130487.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000130487.5.nosex .\n",
      "--extract range: 14916 variants excluded.\n",
      "--extract range: 293 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "293 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000130487.5.bed +\n",
      "./output/chr22_data.ENSG00000130487.5.bim +\n",
      "./output/chr22_data.ENSG00000130487.5.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000130487.5.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000217442.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000217442.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000217442.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000217442.3.nosex .\n",
      "--extract range: 14921 variants excluded.\n",
      "--extract range: 288 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999981.\n",
      "288 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000217442.3.bed +\n",
      "./output/chr22_data.ENSG00000217442.3.bim +\n",
      "./output/chr22_data.ENSG00000217442.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000217442.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205560.12.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205560.12_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205560.12\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205560.12.nosex .\n",
      "--extract range: 14928 variants excluded.\n",
      "--extract range: 281 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "281 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205560.12.bed +\n",
      "./output/chr22_data.ENSG00000205560.12.bim +\n",
      "./output/chr22_data.ENSG00000205560.12.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205560.12.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000205559.3.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000205559.3_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000205559.3\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000205559.3.nosex .\n",
      "--extract range: 14930 variants excluded.\n",
      "--extract range: 279 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.99998.\n",
      "279 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000205559.3.bed +\n",
      "./output/chr22_data.ENSG00000205559.3.bim +\n",
      "./output/chr22_data.ENSG00000205559.3.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000205559.3.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000008735.13.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000008735.13_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000008735.13\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000008735.13.nosex .\n",
      "--extract range: 14940 variants excluded.\n",
      "--extract range: 269 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "269 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000008735.13.bed +\n",
      "./output/chr22_data.ENSG00000008735.13.bim +\n",
      "./output/chr22_data.ENSG00000008735.13.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000008735.13.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100288.19.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100288.19_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100288.19\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100288.19.nosex .\n",
      "--extract range: 14940 variants excluded.\n",
      "--extract range: 269 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999979.\n",
      "269 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100288.19.bed +\n",
      "./output/chr22_data.ENSG00000100288.19.bim +\n",
      "./output/chr22_data.ENSG00000100288.19.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100288.19.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000100299.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000100299.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000100299.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000100299.17.nosex .\n",
      "--extract range: 14964 variants excluded.\n",
      "--extract range: 245 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999977.\n",
      "245 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000100299.17.bed +\n",
      "./output/chr22_data.ENSG00000100299.17.bim +\n",
      "./output/chr22_data.ENSG00000100299.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000100299.17.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000251322.7.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000251322.7_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000251322.7\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000251322.7.nosex .\n",
      "--extract range: 14985 variants excluded.\n",
      "--extract range: 224 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999988.\n",
      "224 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000251322.7.bed +\n",
      "./output/chr22_data.ENSG00000251322.7.bim +\n",
      "./output/chr22_data.ENSG00000251322.7.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000251322.7.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000184319.15.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000184319.15_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000184319.15\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000184319.15.nosex .\n",
      "--extract range: 15020 variants excluded.\n",
      "--extract range: 189 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999985.\n",
      "189 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000184319.15.bed +\n",
      "./output/chr22_data.ENSG00000184319.15.bim +\n",
      "./output/chr22_data.ENSG00000184319.15.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000184319.15.\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/chr22_data.ENSG00000079974.17.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract range ./output/ENSG00000079974.17_coord.txt\n",
      "  --make-bed\n",
      "  --out ./output/chr22_data.ENSG00000079974.17\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/chr22_data.ENSG00000079974.17.nosex .\n",
      "--extract range: 15025 variants excluded.\n",
      "--extract range: 184 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "184 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--make-bed to ./output/chr22_data.ENSG00000079974.17.bed +\n",
      "./output/chr22_data.ENSG00000079974.17.bim +\n",
      "./output/chr22_data.ENSG00000079974.17.fam ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
      "SNPs extracted successfully for gene ENSG00000079974.17.\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "\n",
    "plink_file_prefix = './chr22_data'\n",
    "output_dir = './output'\n",
    "\n",
    "# Initialize a list to store genes with no SNPs in the range\n",
    "skip_genes = []\n",
    "\n",
    "# Loop over each gene and create a PLINK input file\n",
    "for index, row in genes_on_chrom.iterrows():\n",
    "    gene_name = row['gene_id']\n",
    "    chr_num = row['#chr']\n",
    "    window_start = row['window_start']\n",
    "    window_end = row['window_end']\n",
    "    \n",
    "    # Create coordinate file for PLINK\n",
    "    coord_file = os.path.join(output_dir, f\"{gene_name}_coord.txt\")\n",
    "    with open(coord_file, 'w') as f:\n",
    "        f.write(f\"{chr_num} {window_start} {window_end} {gene_name}\\n\")\n",
    "    \n",
    "    # Define PLINK command and output prefix\n",
    "    plink_output_prefix = os.path.join(output_dir, f\"chr22_data.{gene_name}\")\n",
    "    plink_command = f\"/Users/nancyh/Desktop/hartwell/plink --bfile {plink_file_prefix} --extract range {coord_file} --out {plink_output_prefix} --make-bed\"\n",
    "    \n",
    "    # Run PLINK command and capture the output\n",
    "    result = os.system(plink_command)\n",
    "    \n",
    "    # Check if PLINK generated the output files\n",
    "    bed_file = f\"{plink_output_prefix}.bed\"\n",
    "    if not os.path.exists(bed_file):\n",
    "        # Add gene to skip list if no SNPs were extracted\n",
    "        print(f\"No SNPs found for gene {gene_name} in range {window_start}-{window_end}. Skipping this gene.\")\n",
    "        skip_genes.append(gene_name)\n",
    "    else:\n",
    "        print(f\"SNPs extracted successfully for gene {gene_name}.\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 102.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.07it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 437.89it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.66it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 129.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.47it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 129.35it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.82it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.44it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 128.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 249.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.61it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.30it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.25it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 130.06it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.27it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.41it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.61it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.88it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.76it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.18it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 110.91it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.22it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.88it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.32it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.64it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.81it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 489.57it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.35it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.98it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.95it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.67it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.14it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.12it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.68it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.52it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.49it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.26it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.83it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.91it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.86it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 52.43it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 130.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 112.57it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 431.57it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.53it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.44it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 108.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.43it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.65it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.69it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 112.86it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.92it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.04it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 364.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.53it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 66.33it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.62it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 64.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 107.44it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 87.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.38it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 499.52it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 169.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.92it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.10it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 60.25it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.57it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.33it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 112.68it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.24it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.62it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.97it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.57it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.03it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 98.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 468.39it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 442.67it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.03it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 432.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.81it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.33it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 464.47it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 432.52it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.97it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 80.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.53it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 390.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.15it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.59it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.24it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.39it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 455.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.02it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.87it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.02it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.91it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.68it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 452.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.53it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.06it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.09it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 76.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 461.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 436.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.28it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.52it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.56it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 456.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.02it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 86.99it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 105.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 451.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 457.89it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.53it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 463.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.65it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.64it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.85it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 372.58it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 428.81it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 488.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.39it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 431.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.87it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.99it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 457.31it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.80it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 448.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.42it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 113.34it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.43it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 110.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.07it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.25it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.27it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.42it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.82it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 460.19it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.47it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 435.49it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 486.62it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.62it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.73it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.88it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 112.68it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.17it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 454.47it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.33it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.34it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 472.95it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 61.46it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.16it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.88it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.60it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.33it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.30it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.49it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.11it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.67it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.67it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.45it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.67it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 481.85it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.15it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 113.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 80.89it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.02it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.60it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.67it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.61it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 66.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.56it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 244.04it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.07it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.20it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 59.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 109.67it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 143.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.57it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 75.49it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 72.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.92it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 498.49it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 128.73it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.85it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.31it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 129.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 431.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.73it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 99.83it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.99it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 58.34it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.25it/s]\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:58: RuntimeWarning: divide by zero encountered in scalar divide\n",
      "  standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.59it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.65it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.83it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 79.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 342.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.38it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 95.33it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 99.16it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.24it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 328.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.06it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 81.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.32it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 82.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 110.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.58it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 60.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.83it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.16it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.30it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.07it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 383.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 58.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.92it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.07it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 109.71it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 62.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 468.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.04it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 175.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.30it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 58.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.42it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.87it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 61.09it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.69it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.38it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 440.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.30it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 130.04it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 76.16it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.63it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.65it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.82it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 465.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 70.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.09it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.99it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.69it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.82it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.02it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.55it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 71.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.09it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.09it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 515.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.59it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 57.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.46it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.56it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 487.16it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 83.20it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 110.83it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.42it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 427.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.64it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 70.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.19it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.12it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.01it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.91it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 91.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.32it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 56.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.28it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.64it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.04it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.27it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 72.63it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.25it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 128.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 58.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.38it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 106.61it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 35.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.10it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.78it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 451.32it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 58.00it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 128.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.71it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.03it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 69.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.60it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.71it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.20it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.31it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 110.20it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.15it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.45it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 107.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 468.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 442.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 461.88it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 113.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 507.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.54it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 126.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.91it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 56.71it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.43it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 191.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 433.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.05it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.36it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.91it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.29it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.06it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.44it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files:  33%|███▎      | 1/3 [00:00<00:00,  8.83it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 21.79it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.22it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.21it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 57.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.59it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.65it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 55.11it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.77it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 66.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.41it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 113.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.86it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.90it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 418.89it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 72.92it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.67it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 84.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.76it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 269.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.14it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.51it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.87it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.28it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 430.48it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.97it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 112.15it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.82it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.66it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.15it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.13it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 113.99it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.42it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.26it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 123.40it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 114.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.80it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.28it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 65.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.66it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.84it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 55.89it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 122.37it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 117.34it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 125.46it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 84.69it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 111.39it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.72it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.93it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 55.43it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 127.18it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 433.27it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 74.95it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.38it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.64it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 67.17it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.86it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 119.98it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 73.23it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.24it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.75it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 74.70it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 451.86it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 115.74it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.30it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 68.44it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.83it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 120.59it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 124.94it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 76.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 440.39it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 118.08it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 121.50it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 55.96it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 116.80it/s]\n",
      "Mapping files:   0%|          | 0/3 [00:00<?, ?it/s]/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4241259851.py:16: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  (bim, fam, bed) = read_plink(plink_file_prefix)\n",
      "Mapping files: 100%|██████████| 3/3 [00:00<00:00, 456.00it/s]\n"
     ]
    }
   ],
   "source": [
    "from sklearn.linear_model import LinearRegression\n",
    "import numpy as np\n",
    "from scipy import stats\n",
    "\n",
    "# Placeholder to store results for all genes\n",
    "all_results = []\n",
    "\n",
    "for index, row in genes_on_chrom.iterrows():\n",
    "    gene_name = row['gene_id']\n",
    "    # Skip genes with no SNPs\n",
    "    if gene_name in skip_genes:\n",
    "        continue\n",
    "\n",
    "    # Load PLINK genotype data for this gene\n",
    "    plink_file_prefix = os.path.join(output_dir, f\"chr22_data.{gene_name}\")\n",
    "    (bim, fam, bed) = read_plink(plink_file_prefix)\n",
    "    \n",
    "    # Filter gene expression for the gene of interest\n",
    "    expression_values = gene_expression_data[gene_expression_data['gene_id'] == gene_name]\n",
    "    if expression_values.empty:\n",
    "        continue\n",
    "    \n",
    "    # Get common samples between gene expression and genotype data\n",
    "    expression_sample_ids = gene_expression_data.columns[4:]\n",
    "    genotype_sample_ids = fam['iid'].tolist()\n",
    "    common_sample_ids = [sid for sid in expression_sample_ids if sid in genotype_sample_ids]\n",
    "    common_sample_indices = [genotype_sample_ids.index(sid) for sid in common_sample_ids]\n",
    "    \n",
    "    # Filter genotype matrix and gene expression values for common samples\n",
    "    genotype_matrix = bed.compute()[:, common_sample_indices]\n",
    "    y = expression_values[common_sample_ids].values.flatten()\n",
    "    \n",
    "    # Perform linear regression for each SNP and store results\n",
    "    for i in range(genotype_matrix.shape[0]):  # Loop through SNPs (rows)\n",
    "        X = genotype_matrix[i, :].reshape(-1, 1)\n",
    "\n",
    "        # Identify finite values in X and y\n",
    "        finite_mask = np.isfinite(X).flatten() & np.isfinite(y).flatten()\n",
    "\n",
    "        # Filter X and y to include only finite values\n",
    "        X_filtered = X[finite_mask]\n",
    "        y_filtered = y[finite_mask]\n",
    "\n",
    "        # Skip if insufficient data after filtering\n",
    "        if len(y_filtered) < 2:\n",
    "            continue\n",
    "\n",
    "        model = LinearRegression()\n",
    "        model.fit(X_filtered, y_filtered)\n",
    "        \n",
    "        beta = model.coef_[0]\n",
    "        r_squared = model.score(X_filtered, y_filtered)\n",
    "\n",
    "        # Calculate residuals and standard error of the coefficient\n",
    "        y_pred = model.predict(X_filtered)\n",
    "        residuals = y_filtered - y_pred\n",
    "        residual_sum_of_squares = np.sum(residuals**2)\n",
    "        standard_error = np.sqrt(residual_sum_of_squares / (X_filtered.shape[0] - 2)) / np.sqrt(np.sum((X_filtered - np.mean(X_filtered))**2))\n",
    "        \n",
    "        # Calculate t-statistic and p-value\n",
    "        t_statistic = beta / standard_error\n",
    "        p_value = 2 * (1 - stats.t.cdf(abs(t_statistic), df=X_filtered.shape[0] - 2))\n",
    "        \n",
    "        # Append results for this SNP\n",
    "        all_results.append({\n",
    "            'Gene': gene_name,\n",
    "            'SNP': bim['snp'][i],\n",
    "            'CHR': bim['chrom'][i],\n",
    "            'BP': bim['pos'][i],\n",
    "            'Beta': beta,\n",
    "            'R-Squared': r_squared,\n",
    "            'P-Value': p_value\n",
    "        })\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>SNP</th>\n",
       "      <th>CHR</th>\n",
       "      <th>BP</th>\n",
       "      <th>Beta</th>\n",
       "      <th>R-Squared</th>\n",
       "      <th>P-Value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>rs131538</td>\n",
       "      <td>22</td>\n",
       "      <td>16390411</td>\n",
       "      <td>-0.132131</td>\n",
       "      <td>0.001941</td>\n",
       "      <td>0.291989</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>rs7287144</td>\n",
       "      <td>22</td>\n",
       "      <td>16406147</td>\n",
       "      <td>0.189270</td>\n",
       "      <td>0.016168</td>\n",
       "      <td>0.002272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>rs5747620</td>\n",
       "      <td>22</td>\n",
       "      <td>16551808</td>\n",
       "      <td>0.100802</td>\n",
       "      <td>0.005627</td>\n",
       "      <td>0.072532</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>rs17430900</td>\n",
       "      <td>22</td>\n",
       "      <td>16573137</td>\n",
       "      <td>-0.235998</td>\n",
       "      <td>0.000964</td>\n",
       "      <td>0.457945</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>rs9605903</td>\n",
       "      <td>22</td>\n",
       "      <td>16573830</td>\n",
       "      <td>-0.225223</td>\n",
       "      <td>0.022217</td>\n",
       "      <td>0.000339</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219072</th>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>rs2285395</td>\n",
       "      <td>22</td>\n",
       "      <td>50739662</td>\n",
       "      <td>-0.387023</td>\n",
       "      <td>0.016158</td>\n",
       "      <td>0.002280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219073</th>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>rs13056621</td>\n",
       "      <td>22</td>\n",
       "      <td>50743331</td>\n",
       "      <td>-0.067351</td>\n",
       "      <td>0.001198</td>\n",
       "      <td>0.407877</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219074</th>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>rs3888396</td>\n",
       "      <td>22</td>\n",
       "      <td>50772964</td>\n",
       "      <td>0.029802</td>\n",
       "      <td>0.000244</td>\n",
       "      <td>0.708766</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219075</th>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>rs2238837</td>\n",
       "      <td>22</td>\n",
       "      <td>50774447</td>\n",
       "      <td>-0.052825</td>\n",
       "      <td>0.001320</td>\n",
       "      <td>0.384965</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219076</th>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>rs28729663</td>\n",
       "      <td>22</td>\n",
       "      <td>50780578</td>\n",
       "      <td>-0.215157</td>\n",
       "      <td>0.011643</td>\n",
       "      <td>0.009680</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>219077 rows × 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                      Gene         SNP CHR        BP      Beta  R-Squared  \\\n",
       "0       ENSG00000100181.22    rs131538  22  16390411 -0.132131   0.001941   \n",
       "1       ENSG00000100181.22   rs7287144  22  16406147  0.189270   0.016168   \n",
       "2       ENSG00000100181.22   rs5747620  22  16551808  0.100802   0.005627   \n",
       "3       ENSG00000100181.22  rs17430900  22  16573137 -0.235998   0.000964   \n",
       "4       ENSG00000100181.22   rs9605903  22  16573830 -0.225223   0.022217   \n",
       "...                    ...         ...  ..       ...       ...        ...   \n",
       "219072  ENSG00000079974.17   rs2285395  22  50739662 -0.387023   0.016158   \n",
       "219073  ENSG00000079974.17  rs13056621  22  50743331 -0.067351   0.001198   \n",
       "219074  ENSG00000079974.17   rs3888396  22  50772964  0.029802   0.000244   \n",
       "219075  ENSG00000079974.17   rs2238837  22  50774447 -0.052825   0.001320   \n",
       "219076  ENSG00000079974.17  rs28729663  22  50780578 -0.215157   0.011643   \n",
       "\n",
       "         P-Value  \n",
       "0       0.291989  \n",
       "1       0.002272  \n",
       "2       0.072532  \n",
       "3       0.457945  \n",
       "4       0.000339  \n",
       "...          ...  \n",
       "219072  0.002280  \n",
       "219073  0.407877  \n",
       "219074  0.708766  \n",
       "219075  0.384965  \n",
       "219076  0.009680  \n",
       "\n",
       "[219077 rows x 7 columns]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Convert to DataFrame\n",
    "all_results_df = pd.DataFrame(all_results)\n",
    "all_results_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                 Gene         SNP CHR        BP      Beta  R-Squared   P-Value\n",
      "0  ENSG00000100181.22    rs131538  22  16390411 -0.132131   0.001941  0.291989\n",
      "1  ENSG00000100181.22   rs7287144  22  16406147  0.189270   0.016168  0.002272\n",
      "2  ENSG00000100181.22   rs5747620  22  16551808  0.100802   0.005627  0.072532\n",
      "3  ENSG00000100181.22  rs17430900  22  16573137 -0.235997   0.000964  0.457945\n",
      "4  ENSG00000100181.22   rs9605903  22  16573830 -0.225223   0.022217  0.000339\n"
     ]
    }
   ],
   "source": [
    "# Save to CSV\n",
    "all_results_df.to_csv('output/all_cis_eQTL_results.csv', index=False)\n",
    "print(all_results_df.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PRS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1271313970.py:3: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  bim_df = pd.read_csv(bim_file, delim_whitespace=True, header=None, names=['CHR', 'SNP', 'GD', 'BP', 'A1', 'A0'])\n"
     ]
    }
   ],
   "source": [
    "# Load the BIM file\n",
    "bim_file = './chr22_data.bim' \n",
    "bim_df = pd.read_csv(bim_file, delim_whitespace=True, header=None, names=['CHR', 'SNP', 'GD', 'BP', 'A1', 'A0'])\n",
    "\n",
    "# Keep only relevant columns: SNP and A0 (reference allele)\n",
    "bim_df = bim_df[['SNP', 'A0']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SNP</th>\n",
       "      <th>A0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>rs131538</td>\n",
       "      <td>G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>rs7287144</td>\n",
       "      <td>A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>rs5747620</td>\n",
       "      <td>T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>rs17430900</td>\n",
       "      <td>G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rs9605903</td>\n",
       "      <td>T</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "          SNP A0\n",
       "0    rs131538  G\n",
       "1   rs7287144  A\n",
       "2   rs5747620  T\n",
       "3  rs17430900  G\n",
       "4   rs9605903  T"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "bim_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SNP</th>\n",
       "      <th>P</th>\n",
       "      <th>BETA</th>\n",
       "      <th>A0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>rs139432</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>0.277155</td>\n",
       "      <td>G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>rs139438</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>0.277155</td>\n",
       "      <td>C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>rs3818003</td>\n",
       "      <td>0.000042</td>\n",
       "      <td>-0.254546</td>\n",
       "      <td>C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>rs139450</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>0.277155</td>\n",
       "      <td>T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>rs139451</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>0.277155</td>\n",
       "      <td>G</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "         SNP         P      BETA A0\n",
       "0   rs139432  0.000003  0.277155  G\n",
       "1   rs139438  0.000003  0.277155  C\n",
       "2  rs3818003  0.000042 -0.254546  C\n",
       "3   rs139450  0.000003  0.277155  T\n",
       "4   rs139451  0.000003  0.277155  G"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Load the summary statistics\n",
    "summary_stats_file = 'output/all_cis_eQTL_results.csv'  # Adjust to your file path\n",
    "summary_stats = pd.read_csv(summary_stats_file)\n",
    "\n",
    "gene_of_interest = 'ENSG00000167077.12'   \n",
    "\n",
    "summary_stats = summary_stats[summary_stats['Gene']==gene_of_interest]\n",
    "\n",
    "summary_stats.rename(columns={'P-Value':'P', 'Beta':\"BETA\"}, inplace=True)\n",
    "\n",
    "# Filter out rows with P-value equal to 1\n",
    "summary_stats = summary_stats[summary_stats['P'] < 1]\n",
    "\n",
    "# Keep only necessary columns (SNP, P-value, BETA)\n",
    "summary_stats = summary_stats[['SNP', 'P', 'BETA']]\n",
    "# Merge the A0 allele from the BIM file into the summary statistics\n",
    "summary_stats = pd.merge(summary_stats, bim_df, on='SNP', how='inner')\n",
    "\n",
    "summary_stats.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [],
   "source": [
    "#a0 reference (REF) in 1kG"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [],
   "source": [
    "# # Save the transformed file for PLINK clumping\n",
    "summary_stats.to_csv('output/eQTL_Results_Transformed.txt', sep='\\t', index=False)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Perform Clumping"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to output/clumped_results.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-kb 250\n",
      "  --clump-p1 1\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --out output/clumped_results\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to output/clumped_results.nosex .\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is 0.999983.\n",
      "15209 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 22 clumps formed from 265 top variants.\n",
      "Results written to output/clumped_results.clumped .\n",
      "PLINK clumping completed successfully.\n"
     ]
    }
   ],
   "source": [
    "# Define file paths\n",
    "plink_path = \"/Users/nancyh/Desktop/hartwell/./plink\"  # Path to PLINK executable\n",
    "genotype_prefix = './chr22_data'  # Adjust to your genotype file prefix (no extension)\n",
    "summary_stats_file = \"output/eQTL_Results_Transformed.txt\"  # Summary statistics file\n",
    "output_prefix = \"output/clumped_results\"  # Output prefix for clumped results\n",
    "\n",
    "# Construct the PLINK command for clumping\n",
    "plink_command = (\n",
    "    f\"{plink_path} \"\n",
    "    f\"--bfile {genotype_prefix} \"\n",
    "    f\"--clump {summary_stats_file} \"\n",
    "    f\"--clump-p1 1 \"\n",
    "    f\"--clump-r2 0.1 \"\n",
    "    f\"--clump-kb 250 \"\n",
    "    f\"--clump-snp-field SNP \"\n",
    "    f\"--clump-field P \"\n",
    "    f\"--out {output_prefix}\"\n",
    ")\n",
    "\n",
    "# Run the PLINK command and capture the result\n",
    "result = os.system(plink_command)\n",
    "\n",
    "# Check if PLINK ran successfully\n",
    "if result == 0:\n",
    "    print(\"PLINK clumping completed successfully.\")\n",
    "else:\n",
    "    print(\"PLINK clumping encountered an error.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Extract Valid SNPs After Clumping"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipping malformed line: \n",
      "Skipping malformed line: \n",
      "Valid SNPs saved to output/clumped_results_valid_snps.txt\n"
     ]
    }
   ],
   "source": [
    "# Define the path to the clumped file and output SNP list file\n",
    "clumped_file = f\"{output_prefix}.clumped\"\n",
    "valid_snps_file = f\"{output_prefix}_valid_snps.txt\"\n",
    "\n",
    "# Extract the SNP IDs (3rd column) from the clumped file and save them\n",
    "with open(clumped_file, 'r') as infile, open(valid_snps_file, 'w') as outfile:\n",
    "    next(infile)  # Skip header\n",
    "    for line in infile:\n",
    "        parts = line.split()\n",
    "        if len(parts) >= 3:  # Check if there are at least three columns\n",
    "            snp_id = parts[2]  # SNP ID is in the third column\n",
    "            outfile.write(f\"{snp_id}\\n\")\n",
    "        else:\n",
    "            print(f\"Skipping malformed line: {line.strip()}\")\n",
    "\n",
    "print(f\"Valid SNPs saved to {valid_snps_file}\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Define the P-value thresholds file for PRS calculation\n",
    "with open(\"./range_list\", \"w\") as f:\n",
    "    thresholds = [0.001, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5]\n",
    "    for t in thresholds:\n",
    "        f.write(f\"{t} 0 {t}\\n\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### awk '{print $1,$2}' eQTL_Results_Transformed.txt > SNP.pvalue"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/PRS_results.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract output/clumped_results_valid_snps.txt\n",
      "  --out ./output/PRS_results\n",
      "  --q-score-range ./range_list ./output/SNP.pvalue\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/PRS_results.nosex .\n",
      "--extract: 22 variants remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 715 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate is exactly 1.\n",
      "22 variants and 715 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 22 valid predictors loaded.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: 243 lines skipped in --score file (243 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/PRS_results.nopred for details.\n",
      "Error: No valid entries in --q-score-range data file.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "768"
      ]
     },
     "execution_count": 48,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Construct the PLINK command for PRS calculation\n",
    "prs_command = (\n",
    "    f\"{plink_path} \"\n",
    "    f\"--bfile {genotype_prefix} \"\n",
    "    f\"--score {summary_stats_file} 1 4 3 header \"\n",
    "    f\"--q-score-range ./range_list ./output/SNP.pvalue \"\n",
    "    f\"--extract {valid_snps_file} \"\n",
    "    f\"--out ./output/PRS_results\"\n",
    ")\n",
    "\n",
    "# Run PLINK command for PRS calculation\n",
    "os.system(prs_command)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   FID         IID  PHENO  CNT  CNT2     SCORE\n",
      "0    0  GTEX-111CU     -9   86    69  0.063374\n",
      "1    0  GTEX-111FC     -9   86    70  0.074360\n",
      "2    0  GTEX-111VG     -9   86    68  0.079669\n",
      "3    0  GTEX-111YS     -9   86    69  0.088883\n",
      "4    0  GTEX-1122O     -9   86    69  0.074422\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/133411089.py:3: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_df = pd.read_csv(prs_file, delim_whitespace=True)\n"
     ]
    }
   ],
   "source": [
    "# Load PRS results for a specific threshold (e.g., 0.05)\n",
    "prs_file = 'output/PRS_results.0.5.profile'\n",
    "prs_df = pd.read_csv(prs_file, delim_whitespace=True)\n",
    "\n",
    "# View the PRS scores for each individual\n",
    "print(prs_df.head())\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Cross-Val"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>#chr</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>gene_id</th>\n",
       "      <th>GTEX-111YS</th>\n",
       "      <th>GTEX-1122O</th>\n",
       "      <th>GTEX-1128S</th>\n",
       "      <th>GTEX-113IC</th>\n",
       "      <th>GTEX-113JC</th>\n",
       "      <th>GTEX-117XS</th>\n",
       "      <th>...</th>\n",
       "      <th>GTEX-ZV7C</th>\n",
       "      <th>GTEX-ZVE2</th>\n",
       "      <th>GTEX-ZVP2</th>\n",
       "      <th>GTEX-ZVT2</th>\n",
       "      <th>GTEX-ZVT3</th>\n",
       "      <th>GTEX-ZVT4</th>\n",
       "      <th>GTEX-ZVZP</th>\n",
       "      <th>GTEX-ZVZQ</th>\n",
       "      <th>GTEX-ZXES</th>\n",
       "      <th>GTEX-ZXG5</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>19122</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16601886</td>\n",
       "      <td>16601887</td>\n",
       "      <td>ENSG00000100181.22</td>\n",
       "      <td>1.570734</td>\n",
       "      <td>-0.770282</td>\n",
       "      <td>-1.353622</td>\n",
       "      <td>0.512862</td>\n",
       "      <td>-0.853391</td>\n",
       "      <td>-0.560321</td>\n",
       "      <td>...</td>\n",
       "      <td>1.120983</td>\n",
       "      <td>-1.821345</td>\n",
       "      <td>2.214046</td>\n",
       "      <td>1.217151</td>\n",
       "      <td>-0.902828</td>\n",
       "      <td>0.376684</td>\n",
       "      <td>1.265581</td>\n",
       "      <td>-0.954578</td>\n",
       "      <td>0.001868</td>\n",
       "      <td>0.919804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19123</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16612581</td>\n",
       "      <td>16612582</td>\n",
       "      <td>ENSG00000283633.1</td>\n",
       "      <td>1.747926</td>\n",
       "      <td>-0.996665</td>\n",
       "      <td>-0.582310</td>\n",
       "      <td>-1.545582</td>\n",
       "      <td>0.372677</td>\n",
       "      <td>0.231802</td>\n",
       "      <td>...</td>\n",
       "      <td>1.073249</td>\n",
       "      <td>-1.040597</td>\n",
       "      <td>1.178870</td>\n",
       "      <td>1.114008</td>\n",
       "      <td>-0.534285</td>\n",
       "      <td>2.259617</td>\n",
       "      <td>1.079918</td>\n",
       "      <td>-0.133007</td>\n",
       "      <td>0.144322</td>\n",
       "      <td>1.209352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19124</th>\n",
       "      <td>chr22</td>\n",
       "      <td>16821698</td>\n",
       "      <td>16821699</td>\n",
       "      <td>ENSG00000172967.7</td>\n",
       "      <td>0.869633</td>\n",
       "      <td>-0.417104</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>0.735562</td>\n",
       "      <td>-0.254890</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>...</td>\n",
       "      <td>0.948702</td>\n",
       "      <td>-0.392775</td>\n",
       "      <td>1.382185</td>\n",
       "      <td>0.740467</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.293681</td>\n",
       "      <td>1.714493</td>\n",
       "      <td>-1.197791</td>\n",
       "      <td>-0.095404</td>\n",
       "      <td>0.061677</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19125</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17036569</td>\n",
       "      <td>17036570</td>\n",
       "      <td>ENSG00000237438.7</td>\n",
       "      <td>1.232973</td>\n",
       "      <td>-1.034199</td>\n",
       "      <td>-0.931270</td>\n",
       "      <td>-0.182182</td>\n",
       "      <td>-1.432461</td>\n",
       "      <td>0.595640</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.720952</td>\n",
       "      <td>-1.073249</td>\n",
       "      <td>0.095404</td>\n",
       "      <td>-1.765402</td>\n",
       "      <td>0.573481</td>\n",
       "      <td>-1.498020</td>\n",
       "      <td>0.328973</td>\n",
       "      <td>-0.336870</td>\n",
       "      <td>0.227967</td>\n",
       "      <td>-0.340827</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19126</th>\n",
       "      <td>chr22</td>\n",
       "      <td>17067820</td>\n",
       "      <td>17067821</td>\n",
       "      <td>ENSG00000273203.1</td>\n",
       "      <td>-0.696937</td>\n",
       "      <td>-0.750331</td>\n",
       "      <td>-0.069165</td>\n",
       "      <td>0.687444</td>\n",
       "      <td>-0.491673</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>...</td>\n",
       "      <td>0.388743</td>\n",
       "      <td>0.099157</td>\n",
       "      <td>0.960486</td>\n",
       "      <td>0.348756</td>\n",
       "      <td>-1.396898</td>\n",
       "      <td>-0.243330</td>\n",
       "      <td>1.558035</td>\n",
       "      <td>-0.325032</td>\n",
       "      <td>-0.151876</td>\n",
       "      <td>0.569083</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19691</th>\n",
       "      <td>chr22</td>\n",
       "      <td>50601454</td>\n",
       "      <td>50601455</td>\n",
       "      <td>ENSG00000100288.19</td>\n",
       "      <td>0.212657</td>\n",
       "      <td>-1.382185</td>\n",
       "      <td>0.445806</td>\n",
       "      <td>0.659325</td>\n",
       "      <td>0.297584</td>\n",
       "      <td>-0.491673</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.908458</td>\n",
       "      <td>1.442962</td>\n",
       "      <td>-0.604585</td>\n",
       "      <td>-1.027843</td>\n",
       "      <td>0.425269</td>\n",
       "      <td>-0.725805</td>\n",
       "      <td>1.282401</td>\n",
       "      <td>-0.035496</td>\n",
       "      <td>-0.842686</td>\n",
       "      <td>0.328973</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19692</th>\n",
       "      <td>chr22</td>\n",
       "      <td>50628172</td>\n",
       "      <td>50628173</td>\n",
       "      <td>ENSG00000100299.17</td>\n",
       "      <td>0.309319</td>\n",
       "      <td>-0.035496</td>\n",
       "      <td>0.364680</td>\n",
       "      <td>-1.021528</td>\n",
       "      <td>-0.364680</td>\n",
       "      <td>0.805956</td>\n",
       "      <td>...</td>\n",
       "      <td>0.760269</td>\n",
       "      <td>-0.487461</td>\n",
       "      <td>-1.027843</td>\n",
       "      <td>0.227967</td>\n",
       "      <td>-0.336870</td>\n",
       "      <td>-0.384717</td>\n",
       "      <td>1.273946</td>\n",
       "      <td>0.551601</td>\n",
       "      <td>0.730675</td>\n",
       "      <td>0.197397</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19693</th>\n",
       "      <td>chr22</td>\n",
       "      <td>50674414</td>\n",
       "      <td>50674415</td>\n",
       "      <td>ENSG00000251322.7</td>\n",
       "      <td>-1.114008</td>\n",
       "      <td>-0.224134</td>\n",
       "      <td>-0.205021</td>\n",
       "      <td>-1.186387</td>\n",
       "      <td>0.364680</td>\n",
       "      <td>0.117946</td>\n",
       "      <td>...</td>\n",
       "      <td>0.024284</td>\n",
       "      <td>0.720952</td>\n",
       "      <td>-0.937049</td>\n",
       "      <td>0.437569</td>\n",
       "      <td>-0.760269</td>\n",
       "      <td>1.086635</td>\n",
       "      <td>-0.795660</td>\n",
       "      <td>0.449935</td>\n",
       "      <td>0.009339</td>\n",
       "      <td>-1.034199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19694</th>\n",
       "      <td>chr22</td>\n",
       "      <td>50756947</td>\n",
       "      <td>50756948</td>\n",
       "      <td>ENSG00000184319.15</td>\n",
       "      <td>0.344789</td>\n",
       "      <td>-0.114185</td>\n",
       "      <td>-2.134683</td>\n",
       "      <td>0.500121</td>\n",
       "      <td>-2.749905</td>\n",
       "      <td>0.216479</td>\n",
       "      <td>...</td>\n",
       "      <td>-0.400858</td>\n",
       "      <td>-0.178385</td>\n",
       "      <td>0.336870</td>\n",
       "      <td>-1.241002</td>\n",
       "      <td>-1.021528</td>\n",
       "      <td>-0.479065</td>\n",
       "      <td>0.529981</td>\n",
       "      <td>1.335178</td>\n",
       "      <td>-0.024284</td>\n",
       "      <td>0.148098</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19695</th>\n",
       "      <td>chr22</td>\n",
       "      <td>50783662</td>\n",
       "      <td>50783663</td>\n",
       "      <td>ENSG00000079974.17</td>\n",
       "      <td>-0.429361</td>\n",
       "      <td>-0.613579</td>\n",
       "      <td>0.864194</td>\n",
       "      <td>1.714493</td>\n",
       "      <td>0.031758</td>\n",
       "      <td>0.613579</td>\n",
       "      <td>...</td>\n",
       "      <td>-1.009019</td>\n",
       "      <td>0.301491</td>\n",
       "      <td>-1.326123</td>\n",
       "      <td>-0.129239</td>\n",
       "      <td>0.785448</td>\n",
       "      <td>-0.159439</td>\n",
       "      <td>-2.134683</td>\n",
       "      <td>1.372539</td>\n",
       "      <td>-0.258751</td>\n",
       "      <td>1.128013</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>574 rows × 674 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        #chr     start       end             gene_id  GTEX-111YS  GTEX-1122O  \\\n",
       "19122  chr22  16601886  16601887  ENSG00000100181.22    1.570734   -0.770282   \n",
       "19123  chr22  16612581  16612582   ENSG00000283633.1    1.747926   -0.996665   \n",
       "19124  chr22  16821698  16821699   ENSG00000172967.7    0.869633   -0.417104   \n",
       "19125  chr22  17036569  17036570   ENSG00000237438.7    1.232973   -1.034199   \n",
       "19126  chr22  17067820  17067821   ENSG00000273203.1   -0.696937   -0.750331   \n",
       "...      ...       ...       ...                 ...         ...         ...   \n",
       "19691  chr22  50601454  50601455  ENSG00000100288.19    0.212657   -1.382185   \n",
       "19692  chr22  50628172  50628173  ENSG00000100299.17    0.309319   -0.035496   \n",
       "19693  chr22  50674414  50674415   ENSG00000251322.7   -1.114008   -0.224134   \n",
       "19694  chr22  50756947  50756948  ENSG00000184319.15    0.344789   -0.114185   \n",
       "19695  chr22  50783662  50783663  ENSG00000079974.17   -0.429361   -0.613579   \n",
       "\n",
       "       GTEX-1128S  GTEX-113IC  GTEX-113JC  GTEX-117XS  ...  GTEX-ZV7C  \\\n",
       "19122   -1.353622    0.512862   -0.853391   -0.560321  ...   1.120983   \n",
       "19123   -0.582310   -1.545582    0.372677    0.231802  ...   1.073249   \n",
       "19124   -1.197791    0.735562   -0.254890   -1.197791  ...   0.948702   \n",
       "19125   -0.931270   -0.182182   -1.432461    0.595640  ...  -0.720952   \n",
       "19126   -0.069165    0.687444   -0.491673   -1.396898  ...   0.388743   \n",
       "...           ...         ...         ...         ...  ...        ...   \n",
       "19691    0.445806    0.659325    0.297584   -0.491673  ...  -0.908458   \n",
       "19692    0.364680   -1.021528   -0.364680    0.805956  ...   0.760269   \n",
       "19693   -0.205021   -1.186387    0.364680    0.117946  ...   0.024284   \n",
       "19694   -2.134683    0.500121   -2.749905    0.216479  ...  -0.400858   \n",
       "19695    0.864194    1.714493    0.031758    0.613579  ...  -1.009019   \n",
       "\n",
       "       GTEX-ZVE2  GTEX-ZVP2  GTEX-ZVT2  GTEX-ZVT3  GTEX-ZVT4  GTEX-ZVZP  \\\n",
       "19122  -1.821345   2.214046   1.217151  -0.902828   0.376684   1.265581   \n",
       "19123  -1.040597   1.178870   1.114008  -0.534285   2.259617   1.079918   \n",
       "19124  -0.392775   1.382185   0.740467  -1.197791  -0.293681   1.714493   \n",
       "19125  -1.073249   0.095404  -1.765402   0.573481  -1.498020   0.328973   \n",
       "19126   0.099157   0.960486   0.348756  -1.396898  -0.243330   1.558035   \n",
       "...          ...        ...        ...        ...        ...        ...   \n",
       "19691   1.442962  -0.604585  -1.027843   0.425269  -0.725805   1.282401   \n",
       "19692  -0.487461  -1.027843   0.227967  -0.336870  -0.384717   1.273946   \n",
       "19693   0.720952  -0.937049   0.437569  -0.760269   1.086635  -0.795660   \n",
       "19694  -0.178385   0.336870  -1.241002  -1.021528  -0.479065   0.529981   \n",
       "19695   0.301491  -1.326123  -0.129239   0.785448  -0.159439  -2.134683   \n",
       "\n",
       "       GTEX-ZVZQ  GTEX-ZXES  GTEX-ZXG5  \n",
       "19122  -0.954578   0.001868   0.919804  \n",
       "19123  -0.133007   0.144322   1.209352  \n",
       "19124  -1.197791  -0.095404   0.061677  \n",
       "19125  -0.336870   0.227967  -0.340827  \n",
       "19126  -0.325032  -0.151876   0.569083  \n",
       "...          ...        ...        ...  \n",
       "19691  -0.035496  -0.842686   0.328973  \n",
       "19692   0.551601   0.730675   0.197397  \n",
       "19693   0.449935   0.009339  -1.034199  \n",
       "19694   1.335178  -0.024284   0.148098  \n",
       "19695   1.372539  -0.258751   1.128013  \n",
       "\n",
       "[574 rows x 674 columns]"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "gene_expression_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/59557912.py:7: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  fam_df = pd.read_csv(fam_file, delim_whitespace=True, header=None, names=['FID', 'IID', 'PID', 'MID', 'Sex', 'Phenotype'])\n"
     ]
    }
   ],
   "source": [
    "# Define paths\n",
    "output_prefix = \"./output/clumped_results\"\n",
    "\n",
    "\n",
    "# Load FAM file to sample individual IDs\n",
    "fam_file = './chr22_data.fam'\n",
    "fam_df = pd.read_csv(fam_file, delim_whitespace=True, header=None, names=['FID', 'IID', 'PID', 'MID', 'Sex', 'Phenotype'])\n",
    "\n",
    "# Extract the genotype sample IDs\n",
    "genotype_sample_ids = set(fam_df['IID'])\n",
    "\n",
    "# Filter expression data for the gene of interest\n",
    "gene_expression = gene_expression_data[gene_expression_data['gene_id'] == gene_of_interest]\n",
    "gene_expression = gene_expression.iloc[:, 4:]  # Select expression values only, remove metadata\n",
    "gene_expression.columns = gene_expression.columns.str.strip()  # Remove any extra whitespace\n",
    "\n",
    "# Get sample IDs from expression data (assuming columns start from the 5th position)\n",
    "expression_sample_ids = set(gene_expression.columns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get the intersection of sample IDs and filter datasets\n",
    "common_sample_ids = genotype_sample_ids.intersection(expression_sample_ids)\n",
    "\n",
    "filtered_fam_df = fam_df[fam_df['IID'].isin(common_sample_ids)]\n",
    "filtered_expression_df = gene_expression[list(common_sample_ids)]\n",
    "\n",
    "# Sample 80% for training\n",
    "train_ids = filtered_fam_df.sample(frac=0.8, random_state=42)[['FID', 'IID']]\n",
    "train_ids_file = './train_ids.txt'\n",
    "train_ids.to_csv(train_ids_file, sep=' ', index=False, header=False)\n",
    "\n",
    "# Sample 10% of individuals for validation from the remaining 20%\n",
    "remaining_20_ids = filtered_fam_df[~filtered_fam_df['IID'].isin(train_ids['IID'])]\n",
    "validation_ids = remaining_20_ids.sample(frac=0.5, random_state=42)[['FID', 'IID']]\n",
    "validation_ids_file = './validation_ids.txt'\n",
    "validation_ids.to_csv(validation_ids_file, sep=' ', index=False, header=False)\n",
    "\n",
    "# Extract expression values for validation individuals\n",
    "validation_expression = filtered_expression_df[validation_ids['IID']].iloc[0]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Sample 10% of individuals for validation from the remaining 20%\n",
    "test_ids = remaining_20_ids[~remaining_20_ids['IID'].isin(validation_ids['IID'])]\n",
    "test_ids_file = './test_ids.txt'\n",
    "test_ids.to_csv(test_ids_file, sep=' ', index=False, header=False)\n",
    "\n",
    "# Extract expression values for validation individuals\n",
    "test_expression = filtered_expression_df[test_ids['IID']].iloc[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(459, 2)"
      ]
     },
     "execution_count": 54,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "train_ids.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(58, 2)"
      ]
     },
     "execution_count": 55,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "validation_ids.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'output/eQTL_Results_Transformed.txt'"
      ]
     },
     "execution_count": 56,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "summary_stats_file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 195,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1_r20.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1_r20.1.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 21 clumps formed from 265 top variants.\n",
      "Results written to ./output/clumped_results_p1_r20.1.clumped .\n",
      "Clumping completed successfully for p-value threshold 1 and r^2 threshold 0.1.\n",
      "Valid SNPs for p-value threshold 1 and r^2 threshold 0.1 saved to ./output/clumped_results_p1_r20.1_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.05_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.05\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.05_r20.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.05_r20.1.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 10 clumps formed from 157 top variants.\n",
      "Results written to ./output/clumped_results_p0.05_r20.1.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.05 and r^2 threshold 0.1.\n",
      "Valid SNPs for p-value threshold 0.05 and r^2 threshold 0.1 saved to ./output/clumped_results_p0.05_r20.1_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.01_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.01\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.01_r20.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.01_r20.1.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 7 clumps formed from 134 top variants.\n",
      "Results written to ./output/clumped_results_p0.01_r20.1.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.01 and r^2 threshold 0.1.\n",
      "Valid SNPs for p-value threshold 0.01 and r^2 threshold 0.1 saved to ./output/clumped_results_p0.01_r20.1_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.001_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.001\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.001_r20.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.001_r20.1.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 6 clumps formed from 131 top variants.\n",
      "Results written to ./output/clumped_results_p0.001_r20.1.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.001 and r^2 threshold 0.1.\n",
      "Valid SNPs for p-value threshold 0.001 and r^2 threshold 0.1 saved to ./output/clumped_results_p0.001_r20.1_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1e-05_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1e-05\n",
      "  --clump-r2 0.1\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1e-05_r20.1\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1e-05_r20.1.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 3 clumps formed from 107 top variants.\n",
      "Results written to ./output/clumped_results_p1e-05_r20.1.clumped .\n",
      "Clumping completed successfully for p-value threshold 1e-05 and r^2 threshold 0.1.\n",
      "Valid SNPs for p-value threshold 1e-05 and r^2 threshold 0.1 saved to ./output/clumped_results_p1e-05_r20.1_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1\n",
      "  --clump-r2 0.2\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1_r20.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1_r20.2.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 31 clumps formed from 265 top variants.\n",
      "Results written to ./output/clumped_results_p1_r20.2.clumped .\n",
      "Clumping completed successfully for p-value threshold 1 and r^2 threshold 0.2.\n",
      "Valid SNPs for p-value threshold 1 and r^2 threshold 0.2 saved to ./output/clumped_results_p1_r20.2_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.05_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.05\n",
      "  --clump-r2 0.2\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.05_r20.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.05_r20.2.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 13 clumps formed from 157 top variants.\n",
      "Results written to ./output/clumped_results_p0.05_r20.2.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.05 and r^2 threshold 0.2.\n",
      "Valid SNPs for p-value threshold 0.05 and r^2 threshold 0.2 saved to ./output/clumped_results_p0.05_r20.2_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.01_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.01\n",
      "  --clump-r2 0.2\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.01_r20.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.01_r20.2.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 9 clumps formed from 134 top variants.\n",
      "Results written to ./output/clumped_results_p0.01_r20.2.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.01 and r^2 threshold 0.2.\n",
      "Valid SNPs for p-value threshold 0.01 and r^2 threshold 0.2 saved to ./output/clumped_results_p0.01_r20.2_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.001_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.001\n",
      "  --clump-r2 0.2\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.001_r20.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.001_r20.2.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 8 clumps formed from 131 top variants.\n",
      "Results written to ./output/clumped_results_p0.001_r20.2.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.001 and r^2 threshold 0.2.\n",
      "Valid SNPs for p-value threshold 0.001 and r^2 threshold 0.2 saved to ./output/clumped_results_p0.001_r20.2_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1e-05_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1e-05\n",
      "  --clump-r2 0.2\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1e-05_r20.2\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1e-05_r20.2.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 4 clumps formed from 107 top variants.\n",
      "Results written to ./output/clumped_results_p1e-05_r20.2.clumped .\n",
      "Clumping completed successfully for p-value threshold 1e-05 and r^2 threshold 0.2.\n",
      "Valid SNPs for p-value threshold 1e-05 and r^2 threshold 0.2 saved to ./output/clumped_results_p1e-05_r20.2_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1\n",
      "  --clump-r2 0.5\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1_r20.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1_r20.5.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 47 clumps formed from 265 top variants.\n",
      "Results written to ./output/clumped_results_p1_r20.5.clumped .\n",
      "Clumping completed successfully for p-value threshold 1 and r^2 threshold 0.5.\n",
      "Valid SNPs for p-value threshold 1 and r^2 threshold 0.5 saved to ./output/clumped_results_p1_r20.5_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.05_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.05\n",
      "  --clump-r2 0.5\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.05_r20.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.05_r20.5.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 20 clumps formed from 157 top variants.\n",
      "Results written to ./output/clumped_results_p0.05_r20.5.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.05 and r^2 threshold 0.5.\n",
      "Valid SNPs for p-value threshold 0.05 and r^2 threshold 0.5 saved to ./output/clumped_results_p0.05_r20.5_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.01_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.01\n",
      "  --clump-r2 0.5\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.01_r20.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.01_r20.5.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 14 clumps formed from 134 top variants.\n",
      "Results written to ./output/clumped_results_p0.01_r20.5.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.01 and r^2 threshold 0.5.\n",
      "Valid SNPs for p-value threshold 0.01 and r^2 threshold 0.5 saved to ./output/clumped_results_p0.01_r20.5_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.001_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.001\n",
      "  --clump-r2 0.5\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.001_r20.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.001_r20.5.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 11 clumps formed from 131 top variants.\n",
      "Results written to ./output/clumped_results_p0.001_r20.5.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.001 and r^2 threshold 0.5.\n",
      "Valid SNPs for p-value threshold 0.001 and r^2 threshold 0.5 saved to ./output/clumped_results_p0.001_r20.5_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1e-05_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1e-05\n",
      "  --clump-r2 0.5\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1e-05_r20.5\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1e-05_r20.5.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 7 clumps formed from 107 top variants.\n",
      "Results written to ./output/clumped_results_p1e-05_r20.5.clumped .\n",
      "Clumping completed successfully for p-value threshold 1e-05 and r^2 threshold 0.5.\n",
      "Valid SNPs for p-value threshold 1e-05 and r^2 threshold 0.5 saved to ./output/clumped_results_p1e-05_r20.5_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1\n",
      "  --clump-r2 0.8\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1_r20.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1_r20.8.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 73 clumps formed from 265 top variants.\n",
      "Results written to ./output/clumped_results_p1_r20.8.clumped .\n",
      "Clumping completed successfully for p-value threshold 1 and r^2 threshold 0.8.\n",
      "Valid SNPs for p-value threshold 1 and r^2 threshold 0.8 saved to ./output/clumped_results_p1_r20.8_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.05_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.05\n",
      "  --clump-r2 0.8\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.05_r20.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.05_r20.8.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 34 clumps formed from 157 top variants.\n",
      "Results written to ./output/clumped_results_p0.05_r20.8.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.05 and r^2 threshold 0.8.\n",
      "Valid SNPs for p-value threshold 0.05 and r^2 threshold 0.8 saved to ./output/clumped_results_p0.05_r20.8_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.01_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.01\n",
      "  --clump-r2 0.8\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.01_r20.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.01_r20.8.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 25 clumps formed from 134 top variants.\n",
      "Results written to ./output/clumped_results_p0.01_r20.8.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.01 and r^2 threshold 0.8.\n",
      "Valid SNPs for p-value threshold 0.01 and r^2 threshold 0.8 saved to ./output/clumped_results_p0.01_r20.8_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p0.001_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 0.001\n",
      "  --clump-r2 0.8\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p0.001_r20.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p0.001_r20.8.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 22 clumps formed from 131 top variants.\n",
      "Results written to ./output/clumped_results_p0.001_r20.8.clumped .\n",
      "Clumping completed successfully for p-value threshold 0.001 and r^2 threshold 0.8.\n",
      "Valid SNPs for p-value threshold 0.001 and r^2 threshold 0.8 saved to ./output/clumped_results_p0.001_r20.8_valid_snps.txt\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_p1e-05_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --clump output/eQTL_Results_Transformed.txt\n",
      "  --clump-field P\n",
      "  --clump-p1 1e-05\n",
      "  --clump-r2 0.8\n",
      "  --clump-snp-field SNP\n",
      "  --keep ./train_ids.txt\n",
      "  --out ./output/clumped_results_p1e-05_r20.8\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_p1e-05_r20.8.nosex .\n",
      "--keep: 459 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 459 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is 0.999983.\n",
      "15209 variants and 459 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--clump: 12 clumps formed from 107 top variants.\n",
      "Results written to ./output/clumped_results_p1e-05_r20.8.clumped .\n",
      "Clumping completed successfully for p-value threshold 1e-05 and r^2 threshold 0.8.\n",
      "Valid SNPs for p-value threshold 1e-05 and r^2 threshold 0.8 saved to ./output/clumped_results_p1e-05_r20.8_valid_snps.txt\n"
     ]
    }
   ],
   "source": [
    "# Run clumping for each p-value and R^2 threshold on trainning set\n",
    "# Define p-value thresholds\n",
    "p_value_thresholds = [1, 0.05, 0.01, 0.001, 0.00001]\n",
    "r2_thresholds = [0.1, 0.2, 0.5, 0.8]\n",
    "\n",
    "for r2 in r2_thresholds:\n",
    "    for threshold in p_value_thresholds:\n",
    "        clump_output_prefix = f\"{output_prefix}_p{threshold}_r2{r2}\"\n",
    "        clump_command = (\n",
    "            f\"{plink_path} --bfile {genotype_prefix} --clump {summary_stats_file} \"\n",
    "            f\"--clump-p1 {threshold} \"\n",
    "            f\"--clump-r2 {r2}  \"\n",
    "            f\"--clump-snp-field SNP --clump-field P --keep {train_ids_file} --out {clump_output_prefix}\"\n",
    "        )\n",
    "        \n",
    "        # Run PLINK clumping command\n",
    "        result = os.system(clump_command)\n",
    "        if result == 0:\n",
    "            print(f\"Clumping completed successfully for p-value threshold {threshold} and r^2 threshold {r2}.\")\n",
    "        else:\n",
    "            print(f\"Clumping encountered an error for p-value threshold {threshold} and r^2 threshold {r2}.\")\n",
    "        \n",
    "        # Extract SNP IDs from each clumped file\n",
    "        clumped_file = f\"{clump_output_prefix}.clumped\"\n",
    "        valid_snps_file = f\"{clump_output_prefix}_valid_snps.txt\"\n",
    "        \n",
    "        # Check if clumped file exists before proceeding\n",
    "        if os.path.exists(clumped_file):\n",
    "            with open(clumped_file, 'r') as infile, open(valid_snps_file, 'w') as outfile:\n",
    "                next(infile)  # Skip header\n",
    "                for line in infile:\n",
    "                    parts = line.split()\n",
    "                    if len(parts) >= 3:  # Ensure there are at least three columns\n",
    "                        snp_id = parts[2]  # SNP ID is in the third column\n",
    "                        outfile.write(f\"{snp_id}\\n\")\n",
    "            print(f\"Valid SNPs for p-value threshold {threshold} and r^2 threshold {r2} saved to {valid_snps_file}\")\n",
    "        else:\n",
    "            print(f\"Warning: Clumped file not found for p-value threshold {threshold} and r^2 threshold {r2}.\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 196,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'./output/clumped_results_p1e-05_r20.8'"
      ]
     },
     "execution_count": 196,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clump_output_prefix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 197,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1_r20.1_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1_r20.1.nosex .\n",
      "--extract: 21 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "21 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 21 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1_r20.1.profile .\n",
      "R² for p-value threshold 1 and r² threshold 0.1: 0.0826729998911334\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.05_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.05_r20.1_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.05_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.05_r20.1.nosex .\n",
      "--extract: 10 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "10 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 10 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.05_r20.1.profile .\n",
      "R² for p-value threshold 0.05 and r² threshold 0.1: 0.0817969323059132\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.01_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.01_r20.1_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.01_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.01_r20.1.nosex .\n",
      "--extract: 7 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "7 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 7 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.01_r20.1.profile .\n",
      "R² for p-value threshold 0.01 and r² threshold 0.1: 0.08381644083973307\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.001_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.001_r20.1_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.001_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.001_r20.1.nosex .\n",
      "--extract: 6 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "6 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 6 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.001_r20.1.profile .\n",
      "R² for p-value threshold 0.001 and r² threshold 0.1: 0.08025902901461775\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1e-05_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1e-05_r20.1_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1e-05_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1e-05_r20.1.nosex .\n",
      "--extract: 3 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "3 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 3 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1e-05_r20.1.profile .\n",
      "R² for p-value threshold 1e-05 and r² threshold 0.1: 0.09022624911014665\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1_r20.2_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1_r20.2\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1_r20.2.nosex .\n",
      "--extract: 31 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "31 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 31 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1_r20.2.profile .\n",
      "R² for p-value threshold 1 and r² threshold 0.2: 0.07646957514870426\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.05_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.05_r20.2_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.05_r20.2\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.05_r20.2.nosex .\n",
      "--extract: 13 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "13 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 13 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.05_r20.2.profile .\n",
      "R² for p-value threshold 0.05 and r² threshold 0.2: 0.07107482024510511\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: 244 lines skipped in --score file (244 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1_r20.1.nopred for\n",
      "details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 255 lines skipped in --score file (255 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.05_r20.1.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 258 lines skipped in --score file (258 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.01_r20.1.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 259 lines skipped in --score file (259 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.001_r20.1.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 262 lines skipped in --score file (262 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1e-05_r20.1.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 234 lines skipped in --score file (234 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1_r20.2.nopred for\n",
      "details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 252 lines skipped in --score file (252 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.05_r20.2.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.01_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.01_r20.2_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.01_r20.2\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.01_r20.2.nosex .\n",
      "--extract: 9 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "9 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 9 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.01_r20.2.profile .\n",
      "R² for p-value threshold 0.01 and r² threshold 0.2: 0.07338613887471103\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.001_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.001_r20.2_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.001_r20.2\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.001_r20.2.nosex .\n",
      "--extract: 8 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "8 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 8 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.001_r20.2.profile .\n",
      "R² for p-value threshold 0.001 and r² threshold 0.2: 0.07196599638206556\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1e-05_r20.2.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1e-05_r20.2_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1e-05_r20.2\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1e-05_r20.2.nosex .\n",
      "--extract: 4 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "4 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 4 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1e-05_r20.2.profile .\n",
      "R² for p-value threshold 1e-05 and r² threshold 0.2: 0.08561562738224246\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1_r20.5_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1_r20.5\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1_r20.5.nosex .\n",
      "--extract: 47 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "47 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 47 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1_r20.5.profile .\n",
      "R² for p-value threshold 1 and r² threshold 0.5: 0.07740607765913797\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.05_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.05_r20.5_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.05_r20.5\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.05_r20.5.nosex .\n",
      "--extract: 20 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "20 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 20 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.05_r20.5.profile .\n",
      "R² for p-value threshold 0.05 and r² threshold 0.5: 0.07282360919613107\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.01_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.01_r20.5_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.01_r20.5\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.01_r20.5.nosex .\n",
      "--extract: 14 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "14 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 14 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.01_r20.5.profile .\n",
      "R² for p-value threshold 0.01 and r² threshold 0.5: 0.07864502429383954\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.001_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.001_r20.5_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.001_r20.5\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.001_r20.5.nosex .\n",
      "--extract: 11 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "11 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 11 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.001_r20.5.profile .\n",
      "R² for p-value threshold 0.001 and r² threshold 0.5: 0.07861148145765424\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1e-05_r20.5.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1e-05_r20.5_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1e-05_r20.5\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1e-05_r20.5.nosex .\n",
      "--extract: 7 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "7 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 7 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1e-05_r20.5.profile .\n",
      "R² for p-value threshold 1e-05 and r² threshold 0.5: 0.08882216137025042\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1_r20.8_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1_r20.8\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1_r20.8.nosex .\n",
      "--extract: 73 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "73 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 73 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1_r20.8.profile .\n",
      "R² for p-value threshold 1 and r² threshold 0.8: 0.07133407045240273\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: 256 lines skipped in --score file (256 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.01_r20.2.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 257 lines skipped in --score file (257 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.001_r20.2.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 261 lines skipped in --score file (261 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1e-05_r20.2.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 218 lines skipped in --score file (218 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1_r20.5.nopred for\n",
      "details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 245 lines skipped in --score file (245 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.05_r20.5.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 251 lines skipped in --score file (251 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.01_r20.5.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 254 lines skipped in --score file (254 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.001_r20.5.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 258 lines skipped in --score file (258 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1e-05_r20.5.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 192 lines skipped in --score file (192 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1_r20.8.nopred for\n",
      "details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.05_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.05_r20.8_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.05_r20.8\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.05_r20.8.nosex .\n",
      "--extract: 34 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "34 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 34 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.05_r20.8.profile .\n",
      "R² for p-value threshold 0.05 and r² threshold 0.8: 0.07015221934088692\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.01_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.01_r20.8_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.01_r20.8\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.01_r20.8.nosex .\n",
      "--extract: 25 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "25 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 25 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.01_r20.8.profile .\n",
      "R² for p-value threshold 0.01 and r² threshold 0.8: 0.07692240021122794\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_0.001_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p0.001_r20.8_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_0.001_r20.8\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_0.001_r20.8.nosex .\n",
      "--extract: 22 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "22 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 22 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_0.001_r20.8.profile .\n",
      "R² for p-value threshold 0.001 and r² threshold 0.8: 0.07679711672649255\n",
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1e-05_r20.8.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1e-05_r20.8_valid_snps.txt\n",
      "  --keep ./validation_ids.txt\n",
      "  --out ./output/clumped_results_1e-05_r20.8\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1e-05_r20.8.nosex .\n",
      "--extract: 12 variants remaining.\n",
      "--keep: 58 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 58 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "12 variants and 58 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 12 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1e-05_r20.8.profile .\n",
      "R² for p-value threshold 1e-05 and r² threshold 0.8: 0.07894632515035782\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: 231 lines skipped in --score file (231 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.05_r20.8.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 240 lines skipped in --score file (240 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.01_r20.8.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 243 lines skipped in --score file (243 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_0.001_r20.8.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
      "Warning: 253 lines skipped in --score file (253 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1e-05_r20.8.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/1656101192.py:24: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n"
     ]
    }
   ],
   "source": [
    "# Run PRS calculation for each threshold using the clumped SNPs and evaluate on validation set\n",
    "r_squared_values = []\n",
    "\n",
    "for r2 in r2_thresholds:\n",
    "    for threshold in p_value_thresholds:\n",
    "        # Define output for PRS calculation\n",
    "        prs_output_prefix = f\"{output_prefix}_{threshold}_r2{r2}\"\n",
    "        valid_snps_file = f\"./output/clumped_results_p{threshold}_r2{r2}_valid_snps.txt\"\n",
    "        \n",
    "        # Set up PLINK PRS calculation command for each threshold\n",
    "        prs_command = (\n",
    "            f\"{plink_path} --bfile {genotype_prefix} --score {summary_stats_file} 1 4 3 header \"\n",
    "            f\"--extract {valid_snps_file} --keep {validation_ids_file} \"\n",
    "            f\"--out {prs_output_prefix}\"\n",
    "        )\n",
    "        os.system(prs_command)\n",
    "        \n",
    "        # Load the predicted PRS values for correlation with true values\n",
    "        prs_results_file = f\"{prs_output_prefix}.profile\"\n",
    "        if not os.path.exists(prs_results_file):\n",
    "            print(f\"Warning: PRS results file not found for p-value threshold {threshold} and r² threshold {r2}. Skipping this threshold.\")\n",
    "            continue\n",
    "        \n",
    "        prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
    "        \n",
    "        # Calculate R^2 between predicted and true expression values for validation individuals\n",
    "        predicted_expression = prs_results['SCORE']\n",
    "        correlation = np.corrcoef(predicted_expression, validation_expression)[0, 1]\n",
    "        r_squared = correlation ** 2\n",
    "        r_squared_values.append((threshold, r2, r_squared))\n",
    "        print(f\"R² for p-value threshold {threshold} and r² threshold {r2}: {r_squared}\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.0826729998911334 0.02863090963319189\n",
      "1.3145121914844429\n",
      "0.07646957514870426 0.03561273529068606\n",
      "1.3063587910073104\n",
      "0.07740607765913797 0.03445725845733506\n",
      "1.3012316385160556\n",
      "0.07133407045240273 0.04268738738905023\n",
      "1.3037245193145421\n",
      "0.0817969323059132 0.029525951816655408\n",
      "1.344308411068152\n",
      "0.07107482024510511 0.04308037245970486\n",
      "1.3428890507708025\n",
      "0.07282360919613107 0.04049928756192078\n",
      "1.3305514455640117\n",
      "0.07015221934088692 0.04450912807161987\n",
      "1.3355215192999996\n",
      "0.08381644083973307 0.027503780213100244\n",
      "1.3836206905452175\n",
      "0.07338613887471103 0.03970299700841746\n",
      "1.3659478225938606\n",
      "0.07864502429383954 0.03298682207449194\n",
      "1.3549118820321315\n",
      "0.07692240021122794 0.03504919890644187\n",
      "1.3555764257659446\n",
      "0.08025902901461775 0.03116614368123204\n",
      "1.4176199790242232\n",
      "0.07196599638206556 0.041744738793026086\n",
      "1.370587680167897\n",
      "0.07861148145765424 0.03302578198211649\n",
      "1.3831502577161294\n",
      "0.07679711672649255 0.03520420097989366\n",
      "1.3691389404370014\n",
      "0.09022624911014665 0.021964076206933043\n",
      "1.5745492345066452\n",
      "0.08561562738224246 0.025820166618178778\n",
      "1.4615728548207114\n",
      "0.08882216137025042 0.023072693538417205\n",
      "1.446393387005263\n",
      "0.07894632515035782 0.03263894335708079\n",
      "1.4217492398797842\n",
      "Best p-value threshold: 1e-05, Best r2 threshold: 0.1, R^2: 0.09022624911014665\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/4196131160.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n"
     ]
    }
   ],
   "source": [
    "# Directory where profile files are stored\n",
    "output_prefix = \"./output/clumped_results\"\n",
    "\n",
    "# Define p-value and r2 thresholds\n",
    "p_value_thresholds = [\"1\", \"0.05\", \"0.01\", \"0.001\", \"1e-05\"]\n",
    "r2_thresholds = [\"0.1\", \"0.2\", \"0.5\", \"0.8\"]\n",
    "\n",
    "# Initialize a list to store R^2 results for each threshold combination\n",
    "r_squared_results = []\n",
    "\n",
    "# Loop through each p-value and r2 threshold, loading PRS and calculating R^2\n",
    "for p_threshold in p_value_thresholds:\n",
    "    for r2 in r2_thresholds:\n",
    "        # Construct the path to the profile file\n",
    "        profile_file = f\"{output_prefix}_{p_threshold}_r2{r2}.profile\"\n",
    "        \n",
    "        # Check if the file exists\n",
    "        if os.path.exists(profile_file):\n",
    "            # Load PRS results\n",
    "            prs_results = pd.read_csv(profile_file, delim_whitespace=True)\n",
    "            \n",
    "            # Ensure PRS results contain 'SCORE' column\n",
    "            if 'SCORE' in prs_results.columns:\n",
    "                # Calculate correlation and R^2\n",
    "                predicted_expression = prs_results['SCORE']\n",
    "                correlation = np.corrcoef(predicted_expression, validation_expression)[0, 1]\n",
    "                r_squared = correlation ** 2\n",
    "                corr, p_value = pearsonr(predicted_expression, validation_expression)\n",
    "                print(r_squared, p_value)\n",
    "                print(mean_squared_error(predicted_expression, validation_expression))\n",
    "                \n",
    "                # Store the results with both thresholds\n",
    "                r_squared_results.append((p_threshold, r2, r_squared))\n",
    "            else:\n",
    "                print(f\"Warning: 'SCORE' column missing in {profile_file}\")\n",
    "        else:\n",
    "            print(f\"Warning: Profile file not found: {profile_file}\")\n",
    "\n",
    "# Convert results to DataFrame for easy analysis\n",
    "r_squared_df = pd.DataFrame(r_squared_results, columns=['P_Threshold', 'R2_Threshold', 'R^2'])\n",
    "\n",
    "# Check if r_squared_df is not empty to avoid errors\n",
    "if not r_squared_df.empty:\n",
    "    # Find the best combination of p-value and r2 threshold based on maximum R^2\n",
    "    best_result = r_squared_df.loc[r_squared_df['R^2'].idxmax()]\n",
    "    best_p_threshold = best_result['P_Threshold']\n",
    "    best_r2_threshold = best_result['R2_Threshold']\n",
    "    best_r_squared = best_result['R^2']\n",
    "\n",
    "    print(f\"Best p-value threshold: {best_p_threshold}, Best r2 threshold: {best_r2_threshold}, R^2: {best_r_squared}\")\n",
    "else:\n",
    "    print(\"No valid R^2 results found. Please check input files.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>P_Threshold</th>\n",
       "      <th>R2_Threshold</th>\n",
       "      <th>R^2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.082673</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.076470</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.077406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.071334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0.05</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.081797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>0.05</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.071075</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>0.05</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.072824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>0.05</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.070152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>0.01</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.083816</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>0.01</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.073386</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>0.01</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.078645</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>0.01</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.076922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>0.001</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.080259</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>0.001</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.071966</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>0.001</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.078611</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>0.001</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.076797</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>1e-05</td>\n",
       "      <td>0.1</td>\n",
       "      <td>0.090226</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>1e-05</td>\n",
       "      <td>0.2</td>\n",
       "      <td>0.085616</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>1e-05</td>\n",
       "      <td>0.5</td>\n",
       "      <td>0.088822</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>1e-05</td>\n",
       "      <td>0.8</td>\n",
       "      <td>0.078946</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   P_Threshold R2_Threshold       R^2\n",
       "0            1          0.1  0.082673\n",
       "1            1          0.2  0.076470\n",
       "2            1          0.5  0.077406\n",
       "3            1          0.8  0.071334\n",
       "4         0.05          0.1  0.081797\n",
       "5         0.05          0.2  0.071075\n",
       "6         0.05          0.5  0.072824\n",
       "7         0.05          0.8  0.070152\n",
       "8         0.01          0.1  0.083816\n",
       "9         0.01          0.2  0.073386\n",
       "10        0.01          0.5  0.078645\n",
       "11        0.01          0.8  0.076922\n",
       "12       0.001          0.1  0.080259\n",
       "13       0.001          0.2  0.071966\n",
       "14       0.001          0.5  0.078611\n",
       "15       0.001          0.8  0.076797\n",
       "16       1e-05          0.1  0.090226\n",
       "17       1e-05          0.2  0.085616\n",
       "18       1e-05          0.5  0.088822\n",
       "19       1e-05          0.8  0.078946"
      ]
     },
     "execution_count": 199,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "r_squared_df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Make prediction for validation set with best parameter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 310,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/3997836465.py:20: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n"
     ]
    }
   ],
   "source": [
    "r2 = 0.1\n",
    "threshold = 1e-05\n",
    "output_prefix = \"./output/clumped_results\"\n",
    "# Define output for PRS calculation\n",
    "prs_output_prefix = f\"{output_prefix}_{threshold}_r2{r2}\"\n",
    "valid_snps_file = f\"./output/clumped_results_p{threshold}_r2{r2}_valid_snps.txt\"\n",
    "\n",
    "# # Set up PLINK PRS calculation command for each threshold\n",
    "# prs_command = (\n",
    "#     f\"{plink_path} --bfile {genotype_prefix} --score {summary_stats_file} 1 4 3 header \"\n",
    "#     f\"--extract {valid_snps_file} --keep {test_ids_file} \"\n",
    "#     f\"--out {prs_output_prefix}\"\n",
    "# )\n",
    "# os.system(prs_command)\n",
    "\n",
    "# # Load the predicted PRS values for correlation with true values\n",
    "prs_results_file = f\"{prs_output_prefix}.profile\"\n",
    "\n",
    "\n",
    "prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
    "\n",
    "# Calculate R^2 between predicted and true expression values for validation individuals\n",
    "predicted_expression = prs_results['SCORE']\n",
    "# correlation = np.corrcoef(predicted_expression, test_expression)[0, 1]\n",
    "# r_squared = correlation ** 2\n",
    "# r_squared_values.append((threshold, r2, r_squared))\n",
    "# print(f\"R² for p-value threshold {threshold} and r² threshold {r2}: {r_squared}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLINK v1.90b7.2 64-bit (11 Dec 2023)           www.cog-genomics.org/plink/1.9/\n",
      "(C) 2005-2023 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
      "Logging to ./output/clumped_results_1e-05_r20.1.log.\n",
      "Options in effect:\n",
      "  --bfile ./chr22_data\n",
      "  --extract ./output/clumped_results_p1e-05_r20.1_valid_snps.txt\n",
      "  --keep ./test_ids.txt\n",
      "  --out ./output/clumped_results_1e-05_r20.1\n",
      "  --score output/eQTL_Results_Transformed.txt 1 4 3 header\n",
      "\n",
      "16384 MB RAM detected; reserving 8192 MB for main workspace.\n",
      "15209 variants loaded from .bim file.\n",
      "715 people (0 males, 0 females, 715 ambiguous) loaded from .fam.\n",
      "Ambiguous sex IDs written to ./output/clumped_results_1e-05_r20.1.nosex .\n",
      "--extract: 3 variants remaining.\n",
      "--keep: 57 people remaining.\n",
      "Using 1 thread (no multithreaded calculations invoked).\n",
      "Before main variant filters, 57 founders and 0 nonfounders present.\n",
      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
      "Total genotyping rate in remaining samples is exactly 1.\n",
      "3 variants and 57 people pass filters and QC.\n",
      "Note: No phenotypes present.\n",
      "--score: 3 valid predictors loaded.\n",
      "--score: Results written to ./output/clumped_results_1e-05_r20.1.profile .\n",
      "R² for p-value threshold 1e-05 and r² threshold 0.1: 0.09172523009205664\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Warning: 262 lines skipped in --score file (262 due to variant ID mismatch, 0\n",
      "due to allele code mismatch); see ./output/clumped_results_1e-05_r20.1.nopred\n",
      "for details.\n",
      "/var/folders/w6/bg9y2v016jx0q_g3bk713y100000gn/T/ipykernel_23647/633983627.py:19: FutureWarning: The 'delim_whitespace' keyword in pd.read_csv is deprecated and will be removed in a future version. Use ``sep='\\s+'`` instead\n",
      "  prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n"
     ]
    }
   ],
   "source": [
    "r2 = 0.1\n",
    "threshold = 1e-05\n",
    "# Define output for PRS calculation\n",
    "prs_output_prefix = f\"{output_prefix}_{threshold}_r2{r2}\"\n",
    "valid_snps_file = f\"./output/clumped_results_p{threshold}_r2{r2}_valid_snps.txt\"\n",
    "\n",
    "# Set up PLINK PRS calculation command for each threshold\n",
    "prs_command = (\n",
    "    f\"{plink_path} --bfile {genotype_prefix} --score {summary_stats_file} 1 4 3 header \"\n",
    "    f\"--extract {valid_snps_file} --keep {test_ids_file} \"\n",
    "    f\"--out {prs_output_prefix}\"\n",
    ")\n",
    "os.system(prs_command)\n",
    "\n",
    "# Load the predicted PRS values for correlation with true values\n",
    "prs_results_file = f\"{prs_output_prefix}.profile\"\n",
    "\n",
    "\n",
    "prs_results = pd.read_csv(prs_results_file, delim_whitespace=True)\n",
    "\n",
    "# Calculate R^2 between predicted and true expression values for test individuals\n",
    "predicted_expression = prs_results['SCORE']\n",
    "correlation = np.corrcoef(predicted_expression, test_expression)[0, 1]\n",
    "r_squared = correlation ** 2\n",
    "\n",
    "print(f\"R² for p-value threshold {threshold} and r² threshold {r2}: {r_squared}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.3028617342815968)"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corr, p_value = pearsonr(predicted_expression, test_expression)\n",
    "corr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.022023082624041557)"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>output</th>\n",
       "      <th>true</th>\n",
       "      <th>mean_prediction</th>\n",
       "      <th>gene_exp_tracks</th>\n",
       "      <th>blood_mean</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GTEX-111YS</td>\n",
       "      <td>[[0.16305864, 0.16405018, 0.1471794, 0.1427891...</td>\n",
       "      <td>-1.335178</td>\n",
       "      <td>1.400116</td>\n",
       "      <td>0.711904</td>\n",
       "      <td>0.585999</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GTEX-1122O</td>\n",
       "      <td>[[0.16311526, 0.16411506, 0.14720538, 0.142785...</td>\n",
       "      <td>-1.391960</td>\n",
       "      <td>1.399944</td>\n",
       "      <td>0.711846</td>\n",
       "      <td>0.586019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GTEX-1128S</td>\n",
       "      <td>[[0.16300814, 0.16397445, 0.14724311, 0.142760...</td>\n",
       "      <td>-0.136777</td>\n",
       "      <td>1.397944</td>\n",
       "      <td>0.697863</td>\n",
       "      <td>0.575213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GTEX-113JC</td>\n",
       "      <td>[[0.16300814, 0.16397445, 0.14724311, 0.142760...</td>\n",
       "      <td>-1.299592</td>\n",
       "      <td>1.397944</td>\n",
       "      <td>0.697863</td>\n",
       "      <td>0.575213</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GTEX-117XS</td>\n",
       "      <td>[[0.16311526, 0.16411506, 0.14720538, 0.142785...</td>\n",
       "      <td>1.073249</td>\n",
       "      <td>1.399944</td>\n",
       "      <td>0.711846</td>\n",
       "      <td>0.586019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>569</th>\n",
       "      <td>GTEX-ZVT4</td>\n",
       "      <td>[[0.16292255, 0.16394998, 0.14779815, 0.142348...</td>\n",
       "      <td>-0.140549</td>\n",
       "      <td>1.400173</td>\n",
       "      <td>0.708866</td>\n",
       "      <td>0.573078</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>570</th>\n",
       "      <td>GTEX-ZVZP</td>\n",
       "      <td>[[0.16293928, 0.1640075, 0.14774908, 0.1423469...</td>\n",
       "      <td>-0.212657</td>\n",
       "      <td>1.402295</td>\n",
       "      <td>0.722701</td>\n",
       "      <td>0.583901</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>571</th>\n",
       "      <td>GTEX-ZVZQ</td>\n",
       "      <td>[[0.16293928, 0.1640075, 0.14774908, 0.1423469...</td>\n",
       "      <td>1.282401</td>\n",
       "      <td>1.402295</td>\n",
       "      <td>0.722701</td>\n",
       "      <td>0.583901</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>572</th>\n",
       "      <td>GTEX-ZXES</td>\n",
       "      <td>[[0.16298208, 0.1640323, 0.14716163, 0.1426878...</td>\n",
       "      <td>-1.217151</td>\n",
       "      <td>1.400111</td>\n",
       "      <td>0.712105</td>\n",
       "      <td>0.585968</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>GTEX-ZXG5</td>\n",
       "      <td>[[0.1629249, 0.16385481, 0.14769062, 0.1423390...</td>\n",
       "      <td>0.591185</td>\n",
       "      <td>1.402486</td>\n",
       "      <td>0.722541</td>\n",
       "      <td>0.583647</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>574 rows × 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "             id                                             output      true  \\\n",
       "0    GTEX-111YS  [[0.16305864, 0.16405018, 0.1471794, 0.1427891... -1.335178   \n",
       "1    GTEX-1122O  [[0.16311526, 0.16411506, 0.14720538, 0.142785... -1.391960   \n",
       "2    GTEX-1128S  [[0.16300814, 0.16397445, 0.14724311, 0.142760... -0.136777   \n",
       "3    GTEX-113JC  [[0.16300814, 0.16397445, 0.14724311, 0.142760... -1.299592   \n",
       "4    GTEX-117XS  [[0.16311526, 0.16411506, 0.14720538, 0.142785...  1.073249   \n",
       "..          ...                                                ...       ...   \n",
       "569   GTEX-ZVT4  [[0.16292255, 0.16394998, 0.14779815, 0.142348... -0.140549   \n",
       "570   GTEX-ZVZP  [[0.16293928, 0.1640075, 0.14774908, 0.1423469... -0.212657   \n",
       "571   GTEX-ZVZQ  [[0.16293928, 0.1640075, 0.14774908, 0.1423469...  1.282401   \n",
       "572   GTEX-ZXES  [[0.16298208, 0.1640323, 0.14716163, 0.1426878... -1.217151   \n",
       "573   GTEX-ZXG5  [[0.1629249, 0.16385481, 0.14769062, 0.1423390...  0.591185   \n",
       "\n",
       "     mean_prediction  gene_exp_tracks  blood_mean  \n",
       "0           1.400116         0.711904    0.585999  \n",
       "1           1.399944         0.711846    0.586019  \n",
       "2           1.397944         0.697863    0.575213  \n",
       "3           1.397944         0.697863    0.575213  \n",
       "4           1.399944         0.711846    0.586019  \n",
       "..               ...              ...         ...  \n",
       "569         1.400173         0.708866    0.573078  \n",
       "570         1.402295         0.722701    0.583901  \n",
       "571         1.402295         0.722701    0.583901  \n",
       "572         1.400111         0.712105    0.585968  \n",
       "573         1.402486         0.722541    0.583647  \n",
       "\n",
       "[574 rows x 6 columns]"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_pickle('sum_stats_gene447.pkl')\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Assuming df is a pandas DataFrame with the specified columns\n",
    "features = ['mean_prediction', 'gene_exp_tracks', 'blood_mean']\n",
    "df_norm = df.copy()\n",
    "\n",
    "for col in features:\n",
    "    df_norm[col] = (df[col] - df[col].mean()) / df[col].std()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {},
   "outputs": [],
   "source": [
    "for col in features:\n",
    "    df_norm[col] = np.log1p(df_norm[col]+1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>id</th>\n",
       "      <th>output</th>\n",
       "      <th>true</th>\n",
       "      <th>mean_prediction</th>\n",
       "      <th>gene_exp_tracks</th>\n",
       "      <th>blood_mean</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GTEX-111YS</td>\n",
       "      <td>[[0.16305864, 0.16405018, 0.1471794, 0.1427891...</td>\n",
       "      <td>-1.335178</td>\n",
       "      <td>0.258137</td>\n",
       "      <td>0.400999</td>\n",
       "      <td>1.292277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GTEX-1122O</td>\n",
       "      <td>[[0.16311526, 0.16411506, 0.14720538, 0.142785...</td>\n",
       "      <td>-1.391960</td>\n",
       "      <td>0.158097</td>\n",
       "      <td>0.395186</td>\n",
       "      <td>1.295943</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GTEX-1128S</td>\n",
       "      <td>[[0.16300814, 0.16397445, 0.14724311, 0.142760...</td>\n",
       "      <td>-0.136777</td>\n",
       "      <td>-1.004491</td>\n",
       "      <td>-1.019331</td>\n",
       "      <td>-0.767893</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GTEX-113JC</td>\n",
       "      <td>[[0.16300814, 0.16397445, 0.14724311, 0.142760...</td>\n",
       "      <td>-1.299592</td>\n",
       "      <td>-1.004491</td>\n",
       "      <td>-1.019331</td>\n",
       "      <td>-0.767893</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GTEX-117XS</td>\n",
       "      <td>[[0.16311526, 0.16411506, 0.14720538, 0.142785...</td>\n",
       "      <td>1.073249</td>\n",
       "      <td>0.158097</td>\n",
       "      <td>0.395186</td>\n",
       "      <td>1.295943</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           id                                             output      true  \\\n",
       "0  GTEX-111YS  [[0.16305864, 0.16405018, 0.1471794, 0.1427891... -1.335178   \n",
       "1  GTEX-1122O  [[0.16311526, 0.16411506, 0.14720538, 0.142785... -1.391960   \n",
       "2  GTEX-1128S  [[0.16300814, 0.16397445, 0.14724311, 0.142760... -0.136777   \n",
       "3  GTEX-113JC  [[0.16300814, 0.16397445, 0.14724311, 0.142760... -1.299592   \n",
       "4  GTEX-117XS  [[0.16311526, 0.16411506, 0.14720538, 0.142785...  1.073249   \n",
       "\n",
       "   mean_prediction  gene_exp_tracks  blood_mean  \n",
       "0         0.258137         0.400999    1.292277  \n",
       "1         0.158097         0.395186    1.295943  \n",
       "2        -1.004491        -1.019331   -0.767893  \n",
       "3        -1.004491        -1.019331   -0.767893  \n",
       "4         0.158097         0.395186    1.295943  "
      ]
     },
     "execution_count": 146,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_norm.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "metadata": {},
   "outputs": [],
   "source": [
    "train = df_norm[df_norm['id'].isin(train_ids['IID'])]\n",
    "test = df_norm[df_norm['id'].isin(test_ids['IID'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.linear_model import LinearRegression\n",
    "\n",
    "linreg = LinearRegression()\n",
    "linreg.fit(train['mean_prediction'].values.reshape(-1, 1), train['true'].values.reshape(-1, 1))               # train the model\n",
    "y_pred_lin = linreg.predict(test['mean_prediction'].values.reshape(-1, 1))        # predictions on test set\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 180,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/nancyh/opt/anaconda3/envs/enformer/lib/python3.9/site-packages/sklearn/base.py:1389: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples,), for example using ravel().\n",
      "  return fit_method(estimator, *args, **kwargs)\n"
     ]
    }
   ],
   "source": [
    "from sklearn.ensemble import RandomForestRegressor\n",
    "\n",
    "rf = RandomForestRegressor(n_estimators=50, random_state=42)\n",
    "rf.fit(train['mean_prediction'].values.reshape(-1, 1), train['true'].values.reshape(-1, 1))\n",
    "y_pred_rf = rf.predict(test['mean_prediction'].values.reshape(-1, 1))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 182,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/nancyh/opt/anaconda3/envs/enformer/lib/python3.9/site-packages/sklearn/ensemble/_gb.py:672: DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().\n",
      "  y = column_or_1d(y, warn=True)  # TODO: Is this still required?\n"
     ]
    }
   ],
   "source": [
    "from sklearn.ensemble import GradientBoostingRegressor\n",
    "\n",
    "gbr = GradientBoostingRegressor(n_estimators=50, random_state=42, learning_rate=0.05,  # Lower learning rate \n",
    "                                max_depth=8,  # Increase depth to capture complex relationship\n",
    "                        )\n",
    "gbr.fit(train['mean_prediction'].values.reshape(-1, 1), train['true'].values.reshape(-1, 1))\n",
    "y_pred_gbr = gbr.predict(test['mean_prediction'].values.reshape(-1, 1))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "metadata": {},
   "outputs": [],
   "source": [
    "y_test = test['true']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 183,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Linear Regression - MSE: 1.2419599067573257 , R2: 0.04328103805625372\n",
      "Random Forest - MSE: 1.1917429697815904 , R2: 0.08196465058997737\n",
      "Gradient Boosting - MSE: 1.1637373521626226 , R2: 0.10353821771660743\n"
     ]
    }
   ],
   "source": [
    "from sklearn.metrics import mean_squared_error, r2_score\n",
    "\n",
    "# Compute MSE and R2 for each model\n",
    "mse_lin  = mean_squared_error(y_test, y_pred_lin)\n",
    "r2_lin   = r2_score(y_test, y_pred_lin)\n",
    "\n",
    "mse_rf   = mean_squared_error(y_test, y_pred_rf)\n",
    "r2_rf    = r2_score(y_test, y_pred_rf)\n",
    "\n",
    "mse_gbr  = mean_squared_error(y_test, y_pred_gbr)\n",
    "r2_gbr   = r2_score(y_test, y_pred_gbr)\n",
    "\n",
    "print(\"Linear Regression - MSE:\", mse_lin, \", R2:\", r2_lin)\n",
    "print(\"Random Forest - MSE:\", mse_rf, \", R2:\", r2_rf)\n",
    "print(\"Gradient Boosting - MSE:\", mse_gbr, \", R2:\", r2_gbr)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.013416332357817135)"
      ]
     },
     "execution_count": 143,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "p_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.3257246911837879)"
      ]
     },
     "execution_count": 127,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {},
   "outputs": [],
   "source": [
    "norm_prs = (predicted_expression - predicted_expression.mean()) / predicted_expression.std()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 186,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Text(0.5, 1.0, 'PRS with Pruning and Thresholding vs. True Gene Expression')"
      ]
     },
     "execution_count": 186,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAABAgAAAImCAYAAADe77hmAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjkuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8ekN5oAAAACXBIWXMAAA9hAAAPYQGoP6dpAACuzUlEQVR4nOzdeXgT5fr/8U/SnS60RQooIKtF9rIUkK0gKCqCcDgqi6AH3ADZEVFEEBGRTXZEcUXcBVFRFBU8R2UVRUQKfkEEgRZtS1lKC01+f/BrbNq0TdNMk7Tv13VxAZNnZu7cM5nlnnlmTFar1SoAAAAAAFCumT0dAAAAAAAA8DwKBAAAAAAAgAIBAAAAAACgQAAAAAAAAESBAAAAAAAAiAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQI4CFWq9XTIfgkX86bL8cOAJ7GNhQAUBooEOTxyCOPKDY2tsA/7du3L/Y058yZo/j4eDVv3lzr1q1zf9BebvHixYqNjbX9f9euXbrvvvts/z927JhiY2P1wQcfuDyPzz77TPfdd586duyoxo0bq0OHDho9erT27NlTotidERsbq8WLF0tyz3cpyJdffqlJkyYV2iYn17n/NG/eXL1799Zbb73l9picZcQy9yYffPBBoduNnD+elJ2drXfffVd33XWX2rdvr8aNGyshIUGTJ0/W//3f/3k0Nke6du1aaC7Hjh3r6RDdKvd2BK6566678q0nOev59OnTdfr0aVvbgvb1cXFxuvXWW/Xyyy/nm/6BAwc0duxY2++nQ4cOGjNmjPbv3+/W75F3+5ienq6HH35YO3futPuud911V7GnXZwcGe2RRx5R165dS21+RipseWzbts2p/cOxY8ds+5Jjx46V8jf4h7u2Rc4s37zfN+/xoq9z9HvL++eRRx7xaIwnT57U3Llz1bNnT8XFxSkuLk59+vTRypUrlZGR4dHY8nLmt/TNN994Oky38eT2wL/U5+gDKleurCVLljj8LCAgoFjTOnDggF588UXdfvvt6t27t+rUqeOOEH3Kv//9b3Xs2NH2/3fffddtJySXLl3S+PHj9cUXX6hXr156/PHHFRUVpePHj+udd97RnXfeqblz5+rmm292y/yKEhMTo7fffls1a9Z0+7RfeeUVp9u+/fbbkiSLxaKzZ8/qm2++0RNPPCE/Pz/9+9//dntsRcm7zI3MkyckJCTYci5Jmzdv1vLly7VkyRJVrlzZg5FddvbsWT344IP66aefdPvtt+uee+5ReHi4fv/9d73++uv617/+pRdffFGtWrXydKh2OnfurOHDhzv8LCoqqpSjMdbbb7+tqlWrejoMn9ewYUM98cQTtv9fvHhRv/zyi+bPn69ff/1Vb775pkwmk6T8+3qr1aq//vpLb731lp555hkFBQVpwIABkqSDBw/qjjvuUPPmzTVlyhRVqlRJJ0+e1OrVq3X77bfrtddeU/Pmzd3yHfJuH3/99Vd9+OGH+te//uWW6RcnR0YaPny4Bg8ebPh8PK1Ro0Z2+4dffvlFTz75pKZOnapGjRrZhsfExHgiPK+S93jR1z3xxBM6e/as7f/Tp0+3Dc8RHR1d6nHl2LZtm0aNGqWKFStqwIABio2NlcVi0bZt27R8+XJ9/vnneuONNxQUFOSxGB3J+9vJrW7duqUcjXFyji09sW2gQOBAYGCg23b0aWlpkqRbbrnF6w6+S0vVqlUNO/BdsWKFPvvsMy1atEg33nij3We33nqrRowYoenTp6tr164KDg42JIbc3LnulETeGDp16qT9+/frrbfe8kiBIC9vyZO7REdH2+3kDx06JEm69tprVb16dU+FZTNz5kzt3r1bq1evtst769at1atXLw0YMECPP/64Pv30U88F6UB0dHSZWk8KU16+p9HCwsLy5bJ169Y6d+6cFi1apJ9++sn2eUHboYSEBHXr1k0ffPCBrUDw8ssvKyoqSi+88IL8/f85dOrWrZt69OihZcuWaeXKlW75DkZvH4uTIyOVlQJxUfLmOzMzU5JUr149fvd5GHm86An16tWz+39YWJgk79jep6SkaOzYsapVq5ZefvllVahQwfZZ+/btdf3116t///569dVX7e4A9Qbl5beT99iyNNHFoATuuusuPfbYY1q5cqUSEhLUpEkT3Xnnnbbb2hcvXmy75WzIkCG2W62ys7P1xhtv6NZbb1XTpk2VkJCguXPn2nYa0uVbs4YMGaInnnhCLVq00M0336zs7GzFxsbqzTff1COPPKKWLVsqPj5eTz31lC5cuKDZs2erbdu2atOmjR577DG76VksFq1cuVLdu3dX48aNdeONN+r111/P930mTJigUaNGqXnz5rrnnnvyfefXXntNDRo0UGpqqm3Y0qVLFRsbq++//942bNOmTWrQoIGSkpLsbhl75JFHtHbtWv3555/5bjE/deqURo0apbi4OMXHx+vxxx/XuXPnCsx/RkaGVq1apR49euQrDkiS2WzWmDFj1KZNG/3999+SLt+u07BhQ7377rtq37694uPj9dtvvyk7O1srV65Uz5491bRpUzVv3lx33nmntm7dajfN7du364477lCzZs1044036rvvvrP73NGt88ePH9e4ceMUHx+vZs2aaciQIdq3b1++cT799FO77z9lyhSdP3/etmy2b9+u7du3KzY2Vtu2bSswLwWJiIjId1Xo559/1tChQ9WmTRu1aNFCDzzwgA4ePGjXJjk5WZMnT1bnzp3VtGlT9evXT19++aVdm2+//Va333674uLi1Lp1az344IO2OwYcLfO8ecpZLj/99JPuuOMONWnSRF26dNGqVavyxTJ27FjFx8erdevWmjp1qhYsWFDgbYyZmZlq2bKlZs+ebTf80qVLatu2rZ566ilJ0t69ezVkyBC1bNlScXFxuvvuu/Xjjz8WL8FOyPneL7/8snr06KFmzZrp/fffd3grpqN1KS0tTVOnTtV1112nJk2a6Pbbb7f73Tny559/au3atRo8eLDDHWpQUJDGjh2rxo0b68yZM7bhBw4c0P33368WLVqoRYsWGjFihI4ePWr7POdWv++//17/+c9/1KxZM7Vv315z5sxRdna2rZ0z256SeO211/LlaevWrWrQoIGWLl0q6fI6eNddd+m9995Tly5dFBcXpyFDhtjdGl7QtkG6vD3r27evmjRpovbt2+upp56y/TYl6cKFC5o2bZo6deqkxo0bq0ePHvnW3VdffVU9evRQkyZN1LFjR02bNs3uylLe23qd+d3FxsbqjTfe0GOPPab4+HjFxcVp9OjR+uuvv9yQ2bKlcePGki5vj4sSEBCgkJAQu+3lX3/9JavVKovFYte2QoUKevTRR3XTTTc5nFZaWpoaNmxodwfYiRMnFBsbq4kTJ9qGWSwWtWnTRs8//7zdb3/btm22q+yDBw+2u43darXqhRdeUEJCgpo2bao77rijRN3q8ubI0TFBzu8+7z4o7y32Xbt21aJFizR79mxdd911atq0qYYOHarff//d1ibvds+ZcSRp7dq1uvnmm9WkSRP16tVL33//vRo2bFholzVn9vGLFy9W9+7dtXnzZt1666227VXerqHHjx/XyJEj1bJlS7Vv395hd5SS+umnn3TnnXeqSZMmSkhI0Isvvmj7rKD9iFT0dlsqelskXb7rLPd2ZdSoUfm2Kxs2bFDfvn0VFxen9u3ba+rUqYV2UbFYLFq2bJkSEhLUrFkzDR8+PF/7vF0MijrOzrF582b17dtXTZs21Y033qiPP/5Y3bt3L7CrxEcffaTY2FgdOHDAbvimTZsUGxtrO0ZzJlfukLPuLVmyRPHx8erQoYNOnz7tsLuHo24YO3fu1KBBg9SsWTPFx8dr0qRJSklJKXSea9as0d9//62nnnrKrjiQI+d4Ne9n7777rm655RZb16TFixfb7fMfeeQR3X333Xr//fd14403qnHjxurdu3e+W/+LOjYuiezsbPXr109t2rSxy8Mjjzyi5s2b2y7ixMbGavXq1Zo0aZLi4uJ03XXXaebMmXbnTwWdG2VmZurZZ59V586d1bhxY916663asGGDXRxFHVumpKRo/Pjxat++vZo0aaLevXvbbW8cdTH49ttvNWDAALVs2VJt2rTR+PHjdeLECbtxnDmeLgoFggJcunTJ4Z+8DwnauHGjvvzyS02ZMkXz58/XX3/9pYceekjZ2dn697//ralTp0q6fDtMzq2MU6dO1axZs9StWzctX75cAwcO1OrVqzV8+HC76e/cuVMnTpzQ0qVLNX78ePn5+Um6/EyDwMBALVmyRLfddptef/113XbbbTpx4oTmzp1rOxDOfRA+bdo0LVq0SL169dKKFSvUo0cPPf3007YD6ByffvqpQkNDtXz5cg0bNixfXhISEmS1Wu12qjn/3rFjh23YN998o4YNG6pKlSp24w8fPlydO3dW5cqV9fbbbyshIcH22cKFC1WtWjUtW7ZMQ4YM0TvvvFNgVw9J+u6773T+/Hn17NmzwDaxsbFatGiRrrrqKtuw7OxsvfTSS5o5c6YmT56sunXrau7cuVq2bJnuuOMOvfjii5oxY4bS0tI0evRoWx+sX375Rf/5z38UHh6uRYsWafDgwRo3blyB85Yu//jvvPNO/fLLL3r88cc1b948WSwWDRw4MF83iyeeeEJXXXWVli1bpqFDh+q9997T8uXLbZ81bNhQDRs21Ntvv13grVU5cq+z6enp+vjjj/XNN99o0KBBtjZbt25V//79JUlPP/20nnrqKZ04cUJ33nmnLba//vpL/fr1086dOzV27FgtXrxYV111lUaMGKH169dLko4eParhw4ercePGWr58uWbOnKnDhw/rvvvuk8ViKXSZ52axWDRmzBjdfPPNWrlypVq0aKFnn31W//3vfyVJWVlZGjJkiH744Qc9+uijmjVrlvbv36+XXnqpwDwEBQXpxhtv1Keffmr32/r222+Vmpqq3r176+zZsxo2bJiioqK0ePFiLViwQBkZGRo6dKjdCbM7LV68WPfee6+effZZp59rkpmZqSFDhujLL7/U2LFjtWTJElWtWlXDhg0rtEjw5Zdfymq1Fvo76dChg+bMmaPw8HBJ0uHDh3XnnXfq77//1uzZszVz5kwdPXpU/fv3txXbckyYMEEtW7bUihUr1LNnT7344ot69913bZ87u+1xxGq1FrgtznHXXXepdevWmj17tlJSUnT27Fk9+uijat68uR544AFbu19//VULFizQyJEjNWfOHKWmpmrQoEFKTk62tXG0bfjoo480YsQI1alTR0uXLtXIkSO1fv16u+31008/rW+++UaTJk3SqlWrdP311+vZZ5+1HbR//PHHmjNnjgYOHKhVq1ZpxIgR+vDDDzVjxgyH39uZ312OBQsWyGKxaP78+Xr44Yf19ddf6+mnny4yt+XN4cOHJUk1atSwG557ncrKytKxY8c0a9YsHT58WLfddputXUJCgo4fP64777xTb7zxhv7v//7Ptvx79OihPn36OJxvZGSkmjdvbldMzvm95n6mwE8//aS0tLR828dGjRrZHUfkvjV5165d+uKLL/T4449rzpw5Sk5O1oMPPmj3+ygORzkq6pigMK+99poOHTqkWbNm6amnntLevXuLfI5OUeOsW7dOjzzyiFq0aKFly5bpxhtv1PDhw+1OUBxxZh8vXb5Q8eSTT2rw4MFauXKlqlevrkmTJtn2iefPn9egQYN04MABzZgxQ48//rjeffdd7d69u1i5Kcq0adN0yy23aOXKlYqLi9OcOXP09ddf27XJux9xZrvt7Lbotdde08WLF7Vw4UKNHz9eX331lZ588knb58uWLdO4cePUvHlzLVq0SCNGjNDGjRt111136cKFCw6/05w5c7R06VL169dPS5YsUWRkpObNm1dkLgo7zpYuH8sMHz5c1apV0+LFizVw4EA98cQTdidNeXXr1k0VKlTQJ598Yjf8448/Vv369dWwYcNib7dL6vjx49qyZYsWLFigyZMnq2LFik6Nt2PHDt19990KDg7Wc889p0cffVTbt2/X4MGDC1wW0uVjg9jYWNWvX7/ANpMmTbI7bnz++ef1+OOPq127dlqxYoUGDhyoF154QY8//rjdeHv37tWqVas0atQoLV26VH5+fnrooYdsBaHiHBs7YrFYHB4X5KwTfn5+euaZZ3T+/HnbBaJNmzZp7dq1evjhh+26ey9cuFB///23nnvuOQ0bNkxvv/12vu1U3u2g1WrViBEj9NZbb+mee+7R8uXLFRcXp7Fjx9pO8J05tpw4caL+7//+T9OnT9cLL7yghg0batKkSfkuTuZYt26d/vOf/6hatWqaP3++Jk+erN27d+uOO+6wOzYr6njaGXQxcODPP/8s8ATs4Ycf1tChQ23/v3TpklatWmW7bejcuXOaNGmSfv31VzVu3Nh2e1G9evXUsGFD/fbbb3rvvfc0fvx42y077du3V0xMjB5++GF988036ty5s23aTz75ZL7brerVq2fbUMfHx+vdd9/VxYsXNXfuXPn7+6tDhw7auHGjfvjhB0mXd/rvvPOOxo0bZ5tnhw4dZDKZ9Pzzz2vAgAG2/rwBAQGaPn26AgMDHX7/mjVrqnbt2vr+++910003KSMjQ7t371ajRo3sCgT//e9/1bdvX4fjR0dH291CmXMl7sYbb9TkyZMlSe3atdO3335b4I9Ekq0qXqtWLbvhFosl31Ues9kss/mfetgDDzxgdyCWc2U69xWQoKAgPfTQQ0pMTFTz5s31/PPPq1KlSlq+fLntWRRRUVGFPizt1VdfVVpamt58801bkaJTp066+eabtXDhQi1atMjWtnPnzraNUs7337x5s8aPH6969eoV69Y0R+tv165d7Z7FMG/ePF199dVauXKlrfjUoUMHde/eXYsWLdLChQv18ssvKyUlRRs3brTF37lzZ91999169tln1bNnT+3Zs0cXLlzQ/fffbysIVa1aVV9++aXOnz9f6DLPzWq1avjw4bYuEC1bttQXX3yhzZs3q2PHjlq/fr0OHTqk999/33alq23bturWrVuhuejdu7fef/997dq1y9bN55NPPlGdOnXUpEkT/fjjj0pNTdXgwYPVokULSVKdOnX09ttv69y5c7aTZne66aabit2f+MMPP9T+/fv1zjvvqFmzZpIur0t33XWX5s6dazsZzeuPP/6Q5NzvxM/PTyaTSUuWLFFISIheeeUV23rXrl07devWTS+++KLdzvPf//63RowYYWuzadMmbd68WXfeeWextj2OrFu3rsAHu7733ntq0qSJTCaTZs2apV69emnOnDny8/NTWlqaXn31Vdt6LUlnzpzRihUrbOtA06ZN1a1bN7322muaMGGCrV3ubYPVatXcuXPVsWNHzZ0719amVq1auvvuu7VlyxYlJCRo+/btat++vW655RZJUps2bVShQgVVqlRJ0uU7j6pXr66BAwfKbDYrPj5eFSpUKPBqmzO/u5zt2TXXXKNZs2bZxt2zZ48+++yzAnNa1uUUlXKcPn1a27dvtx3A5Ww7pIL39bVq1dITTzxhK6BK0oABA3Tq1CmtWrXKtv+NiopShw4dNHjwYDVt2rTAmBISErR8+XJdvHhRAQEB+v7779WoUSP98ssvOnbsmKpXr67//ve/uuqqq/JdLQoLC7M7jsh9y3JgYKBWrlypyMhISZcfZjhlyhT99ttvatCggVtylPeYoDh3r0VERGjZsmW23+Eff/yhxYsXKzU1tcDffVHjLFy4UF26dLHd/dWxY0cFBAQUeaLpzD5eunxn4syZM9WuXTtJl9eFLl26aMuWLapbt67Wrl2r48eP6+OPP7Yti2bNmql79+5O58UZ48aNs61/zZs31xdffKGtW7eqS5cutjZ59yPjx48vcrvt7LaoSZMmevbZZ23T+Omnn7RlyxZJl9eX5cuX6/bbb7cVr6TL26KBAwfq/fff18CBA+2ml56ertdff1333HOPRo4cKenysktOTi7ypKWo4+zFixerfv36WrJkie2un0qVKhV6ASckJEQ33nijNmzYYDuGO3funL7++mvb/qy42+2SunTpkiZNmlTs7sjz5s1T7dq19fzzz9t+N82aNdMtt9zicFnk+OOPPxxeoHBUYPT399eZM2dsRbYpU6ZIurw/j4yM1JQpU3TPPffYig1nzpzRBx98YOtGVKFCBQ0aNEhbt27VjTfeWKxjY0fuvvtuh8Pr16+vjz/+WNLl7eVDDz2kefPmqVu3bpo2bZoSEhJs3cZyREdHa8WKFfL391fnzp1lNps1a9YsPfTQQ7bnGeTdDn777bf673//qwULFtiOqzt27KiMjAzbAx9/++23Io8tt2/frhEjRtiOY+Pj4xUZGenwHMxisWju3Lnq0KGD3fYu5y7zVatW6eGHH5ZU9PG0MygQOFC5cmXbldu8qlWrZvf/3CdukmwnSAU9+XP79u2SZDuQzHHLLbdo8uTJ2rZtm61AEBkZ6bAvVlxcnO3ffn5+ioqKUqNGjez6RUZGRtoqVFu3bpXValXXrl3tfvhdu3bV8uXLtWvXLtvKWadOnQKLAzkSEhK0adMmSZevYAQEBGjw4MGaOnWqsrKy9Mcff+j48eMFXikuSN6NYvXq1bVr164C2+c9ucmxcOFCrVixwm7YyJEj9dBDD9n+f+2119p9nvNjS0lJ0aFDh3TkyBFbtT4rK0vS5e/apUsXuwdV3nDDDXYnIXl9//33uvbaa1WlShVb7s1mszp16pTvSmDeE/+qVavqzz//LHDahXnvvfds/87IyNDPP/+sFStWaOjQoXrllVeUmZmpn3/+WSNHjrSLPyIiwnYwJF1eX+Pi4uzuwJCkXr16afLkyTp06JCaNWumoKAg9evXTz169FCnTp3Upk2bQg+YC5J73Q4MDFR0dLStmLB161bVqFHD7uA1LCxMXbp0KfSgNT4+XldeeaU++eQTtWrVSpmZmdq0aZPthLV+/fqKjo7WAw88oB49eqhjx45q37693e2/7pZ3/XPG999/r8qVK6tRo0Z2v+MuXbro2Wef1enTpx1ecSjo1WgPP/ywPvroI7ths2bNUt++fbV161bFx8crODjYNq+wsDC1atUqX7ea3MtMurze5l5mzm57HOnSpYvtYC2v3CdKNWrU0IQJEzRjxgxZrVbNmjUr35Xi6tWr221jYmJiFBcXZ1fYlOyXzaFDh3Ty5Endf//9dvG3bt1aYWFh+vbbb5WQkKA2bdrorbfe0smTJ9W5c2d17tzZLu62bdvq7bffVt++fdWtWzd17txZt956a4EPgnPmd5fz/R1tN7ztydOlaceOHflO+s1ms6677jo9+eSTdjnPva9PT0/XsmXL9Mcff+iZZ57Jt15L0ujRo3X33Xfrv//9r77//ntt27ZNH330kT7++GM9+uijBT5wr3Pnzpo3b55++ukntWrVSlu3btWECRP0xBNPaMeOHapevbq++eabYu8z69WrZysOSLI976SoO5+KkyNnjgkK0qRJE7v9S87xTEZGRoEFgsLGSU9P1/HjxzV69Gi7cW655ZYiCwTO7ONz5P5N5cw/Z5u2c+dO1axZ0277U61aNbf3h869rQoJCdEVV1yh9PR0uzZ59yPObLed3Ra1bNnS7v/Vq1e3zf/HH39UVlZWvrvSWrVqpauuukrbt2/Pd1L6448/6uLFi3YFDulykaOoAkFhx9lZWVnavXu3RowYYfcdevToYTtZKkjv3r21du1a7dmzR02bNtWXX36prKws9erVS1Lxt9vuUNxjg4yMDP30008aOnSoXeGvRo0aqlu3rr799tsCCwSOjqEvXbrksGiamJio3bt368KFCw7359Llk+acAkF0dLTdM0Zy/46l4h0bOzJ9+nSHceZ91tjQoUO1adMmjRo1SlFRUQ7vrrv11lvtzp9uvPFGzZo1Szt27LAVCPJuB7///nuZTCZ17tw5Xy7Wr1+vgwcPOnVs2aZNGy1evFj79u1Tx44d7S4U5nX48GGdOnVK48ePtxtes2ZNxcXF2c4vcxR2PO0MCgQOBAYGqkmTJk61DQkJsft/zlWdgk5ecyqPeZ9s7u/vr6ioKLsde2hoqMNp5N5Q5nDUfyhH7gclOpKUlFTkPHPr3LmzXn75ZR07dkzff/+9WrRooXbt2ikzM1M//fST9u7dq8qVK9udyDnDUS4Le+/zlVdeKenyVaDct0gNGDDA7qSjX79++cbNm6+ff/5Z06dP188//6yQkBDVq1fPNv2cGE6fPp3voCZnuRUkLS1NR44cKfCOlNwH8sX9/oXJu/7Gx8ercuXKmjhxor788ks1a9ZMVqtVV1xxRb5xr7jiCtt6ePr06XwnWjltpMsH1fXq1dPq1au1cuVKvffee3rttdcUERGhAQMGaMyYMcXamebduOfOQWpqqu2KbG6OhuVmMpl066236t1339WUKVP09ddf6/z587r11lslXV7n33jjDS1fvlyffvqp3n77bQUHB6t3796aMmWKywfHhSns91qQtLQ0nTp1qsB16dSpUw4LBAX9TkaPHq0hQ4bYxn3wwQft5rVhw4Z8/emk/E9cLmyZFWfb40hkZKTT2+Kbb75ZzzzzjCQ5vCqSt7uTdHnd+eWXX+yG5V42OfFPnz7d9vTp3HK6Jzz22GOqWrWq1q9frxkzZmjGjBmKi4vTtGnT1KBBA918882yWCxas2aNli1bZusyMGHCBIdvWHHmd5fDnduNsqBRo0a2ZWUymRQUFKRq1ao53G/m3de3aNFC//rXv3Tvvffq3XffVe3atfONU7FiRfXs2dN2crRv3z5NnDhRc+bM0a233upwfxAbG6tq1arpu+++U1RUlJKTk3XdddepRYsW2r59uzp37qxffvkl34lvUfJuR4o6/shRnBw5c0xQkOIeHxU1Tk5f4rzbfEf7sbyc2cc7iiFn/oUdB0iXj+nc+ewPZ37XeZe/M9ttZ7dFjtat3DmQHOc99/FDbjnj5M2dM2/5KWydSEtLU3Z2dr51ws/Pz6545kibNm1UpUoVffLJJ2ratKk++eQTxcfH205mi7vddofi/t7S09NlsVj0wgsv6IUXXsj3eWFvH7jqqqvyXYTy9/e3u8D0zjvv6J133pH0z/6woAcW5u6ul3eZ5RwH5vz2nTk2zjuN3GrXru3UsYGfn5969eqln376SU2bNnV4vJj32CCnTe47RfIul7S0NFmtVtudAXklJyfr2muvLfLYcsGCBVqxYoU+/fRTbdy40a5Qm/cCQU7+C/rd5X1+Q2HHZs6gQFDKcg7gT506ZbfwL168WOhtdyUREREh6fLt7o42Pjk7SWe1atVKYWFh+v777223C1WpUkW1atXStm3btGvXLiUkJBj+mqT27dsrKChIn332md2VlypVqjg8GShITj+h2NhY263nZrNZW7Zs0caNG23tIiMj8x0AWK3WQm83Cw8PV3x8fIGVbCNOPguSU7D5/fffbbd5OzqgOXXqlG3HWrFiRZ06dcphG+mfnX3Tpk21ZMkSZWVladeuXXr77be1YsUKNWjQoMCHdxVXlSpV8j2sSlK+PvGO9O7dW88//7y2bdumDRs2qHXr1na/vzp16tgerrdnzx59+OGHevPNN1WzZs1i97t1hclkyteHNm+lNzw8XLVq1bK71T23gt6W0LVrVz377LP67LPP7AoENWrUsJ2E5n3Hbnh4uK677jqHDyrNXWkviru3PYV56qmnFBoaqsDAQE2dOlXPP/+83ee5H6ya46+//iq0wJQT/8MPP6z4+Ph8n+dszwMDA/Xggw/qwQcf1PHjx/X1119r2bJlGj9+vK2Pa85J5ZkzZ/S///1PL7zwgiZOnKiWLVvm2145+7tDfqGhoU4XlfIKCQnRM888ozvuuEOTJ0+2ve4vKSlJ//rXvzR69Oh8b4Fp2LChxo4da3sYXEHLpnPnzvr+++9VqVIl1a5dW5UrV1abNm30zjvv6H//+5+Cg4PVpk0bl+IurpLkKO+Bfo5z586VqJjgjJwTt7zb/KL2Ac7u450RFRWlI0eO5Buec/DuSc5ut4uzLXIkZ7v3119/5Xt196lTpxwWN3N+F3///bfdOCXNW6VKlRQQEJDvWCaneFAYs9msW2+9VR9//LEeeOABffvtt3bPWZBKnit3KOzYIDQ0VCaTSXfffbfDQnxhJ9ldu3bVypUrdfToUbtllnvbsHnzZtu/c/aHc+fOzddlUXKuUJejtI6NT506pcWLF+vaa6/V119/rc8++0w9evSwa5P32CBnXSrs7QHh4eGqUKGCXnvtNYefX3311ZKKPrYMDw/XxIkTNXHiRB06dEhffvmlli1bpunTp+d7K07OcXlBx+3uPi7gIYWlLOcgM++DUT755BNlZ2fnu63LHXJuVUtNTVWTJk1sf1JSUrRw4cJib6ADAgLUvn17ffnll/r1119t36lt27bavHmzdu7cme82stxyPwugJMLDw3XPPfdo3bp1+uKLLxy2yfuEWkcOHTqktLQ0DR48WPXq1bPFl/PE1ZwDoXbt2umbb76xu+r/3//+VxcvXixw2vHx8Tp8+LCt2pnz58MPP9R7771XaPeEvEqat5yn/taqVUsVKlRQ48aN9emnn9rtfM6cOaPNmzfb1sPWrVtr9+7d+arM69evV+XKlXX11VfrlVdeUZcuXZSVlaXAwEC1a9fO9hCfnKdhu2OZx8fH69ixY/r1119twy5cuODUQ1fq1q2rRo0a6ZNPPtGWLVtstxBK0meffaa2bdvq1KlT8vPzs135jYiIcOqJ5+4QGhqq1NRUuyfn5u1eEx8frxMnTqhSpUp269K3336rF198scB1qXbt2raHBxb0Zoa8b67IeYL/tddea5tP48aN9corrxT4W3PE3duegnz++ef6+OOPNXnyZE2dOlWbN2/O90yG33//3e7hR0lJSdq9e7etr7EjderUUaVKlXTs2DG7+KtUqaJ58+Zp3759unDhgm688UbbwzKvvPJKDRw4ULfccott/RkzZoyty0F4eLhuuukmDR8+XJcuXbK76pLDmd8djNG0aVPdfvvt2r17t+35F1dccYX8/f21Zs0au99ojkOHDikoKKjQ5ZKQkKCff/5Z33zzjd0+89ixY3rrrbfUvn37Ag+Ki7OfMFrOXQYnT560DTt9+rRTDxYrqapVq6pmzZr5tkGff/55oeM5u493Rs4y+/nnn23DUlJSDHnrTXE5s90u7rbIkWbNmikwMNDW1zvHzp07dfz4cYdXVePi4hQcHJzv+Sh5H7xYXH5+fmrRokW+N7x89dVXTj2ss3fv3jp58qTtQXo33HCD7TN35KqkwsLC8t1pl/N8sZzPGzZsqEOHDtnto+rXr6/FixcX2v1y4MCBioyM1COPPOLwzQzZ2dm2p/1Ll5d7QECAkpKS7Obl7++v+fPn57vQUBh3HhsXZurUqfLz89Mrr7yi66+/XtOnT8/3doevvvrK7v8bN26UyWRS27ZtC43//PnzslqtdvEfOHBAS5cu1aVLl4o8tvzzzz/VuXNn22+iTp06uvfee3Xdddc5PPbMKSzn/d0dPXpUP/74Y4F3M7iKOwgcyMrKKnRjHxsbW2hVrjD16tVTnz59tGjRImVkZKh169b69ddftWTJErVp08bph0cUR2xsrHr16qXHH39cf/75pxo3bqzDhw9rwYIFql69usNKYFE6d+6sRx991HaiKcnWDzcoKEjXXXddgeNGRETor7/+0pYtW1zqi53bqFGjdPLkST300EPq0aOHunfvrpiYGJ06dUpff/21Pv30U1WpUqXQk4DatWsrLCzM9pASf39/bdy40XabVU5BYMSIEdq0aZOGDh2qYcOGKSUlRc8995zdMwnyuvvuu/Xhhx/q7rvv1n/+8x9FRUVpw4YNeuedd2wPZHRWRESEdu/ebXulU2FPuM29/mZnZ+uXX37RokWLdM0119juthg/fryGDh2q++67TwMGDNDFixe1cuVKZWVl2XaK99xzj9avX6+7775bI0eOVGRkpNatW6etW7fq6aefltlsVtu2bTV37lyNGDFCgwYNkp+fn9566y0FBgbaCkXuWOY9e/bUypUrNWLECI0ePVoRERF6+eWX9ffffzt1Jbp3796aPXu2/P397SrILVq0kMVi0YgRI3TfffcpNDRUn376qc6cOWM7WDh79qx+++032wMX3a1Lly56/fXX9dhjj6lfv346cOCAXn75ZbudZN++fbV69Wrdc889euCBB2y3LL/wwgsaNGhQoevh9OnTdfr0aQ0aNEi9e/dWp06dFB0drT///FMbN27U119/rTp16tieGzF8+HDdeeeduv/++9W/f38FBQXp7bff1qZNm4p8eFBuJd32FHbg7efnZys2TJs2TR06dFDv3r0lXX5C9axZs9S+fXvbVUer1aoHHnhAY8eOlZ+fn5YsWaKKFSvaPbTM0TzGjh1rO8jo0qWLra96UlKSGjVqpODgYDVq1EhLlixRQECAYmNjdfjwYa1du9b2+tW2bdvqiSee0OzZs9WpUyelp6dryZIlqlWrlsOHyTnzu4NxxowZo08//VTz5s1T9+7dFRYWpmnTpmnEiBH617/+pYEDB6pu3brKyMjQt99+qzfeeEOjR48udJvctm1bmc1mbd68WfPnz5d0+Vb/0NBQ7dq1SzNnzixw3JwHpW7evFkVK1Ys9AGERsvpLrF06VKFhYXZHjjq6jFRcZhMJo0aNcr2/Ibu3btr//79tjeiFPS7cHYf74zevXvrtdde08iRIzV27FiFhYVp+fLlxSoyGMWZ7XZxt0WOREZG6r777tPSpUsVEBCgLl266NixY1q4cKHtGDev0NBQDR8+XM8995xCQkLUtm1bbdmypcQFAunyceBdd92lUaNGqV+/fjp+/LgWLlwoSUXeyXrNNdfo2muv1Zo1a3TTTTfZdbNxJld//PGHUlJS3P4MihwJCQn65JNP1KxZM1199dX64IMP8t3BkvMQ4PHjx6tXr162t/H89NNPGj58eIHTrlKlipYsWaLRo0erV69euuOOO9SoUSOZzWbt3btX77//vn7//XfbBZWoqCgNGzZMCxcu1NmzZ9WmTRslJSVp4cKFMplMxdoulfTY+Lfffiuw+0TlypV11VVXad26dfrqq680b948RUZGaurUqbr55pttb1bK8eOPP2rChAnq3bu39u/fr8WLF+v22293eCdMjs6dO6t169YaPny4hg8frrp162rPnj1atGiROnbsqOjo6CKPLa+66ipVrVpVTz31lM6ePauaNWtq79692rJli+6///588zSbzRo3bpwmT55sW9apqam2YxlHdw6VBAUCB06dOqU77rijwM/XrVtXohPbmTNn6uqrr9b777+vF154QTExMRo8eLCGDx9u2IHfrFmz9Pzzz9sepFWpUiXdfPPNGjNmjEuVus6dO8tkMqlFixa2W9fatGkjk8mkNm3aFHqw0LdvX23ZskUjRozQqFGjStSXy8/PT7Nnz1bPnj317rvvas6cOfrrr78UGhqqa6+9Vo899phuu+22QuMJDw/XsmXL9Oyzz2r06NG2cVevXq17771XO3fuVNeuXVWrVi2tXr1azzzzjMaOHatKlSpp0qRJtn7PjlSpUkVvvfWW5s2bp2nTpikzM1O1atXSzJkzHT4boTADBw7U3r17de+992rWrFm2PvSO5F5/AwICFBMTo5tvvlmjR4+2XaVq166dXn75ZS1atEjjxo1TYGCgWrVqpdmzZ9tuRa9cubLefPNNzZs3T0899ZQuXryoBg0aaNmyZbr++uslSQ0aNNCKFSu0dOlSjRs3TtnZ2WrcuLFeeukl262E7ljm/v7+WrVqlWbOnKlp06bJ399fvXr1UmRkpO31XIXp2bOnnn32WXXp0sXuzQQxMTF68cUXtXDhQj322GPKyMiwVd9zKsi//PKLBg8ebHuIn7u1b99ekyZN0uuvv66NGzfaTjjvvPNOW5sKFSrojTfe0Lx58zRnzhydOXNGV111lcaPH6///Oc/hU4/LCxML7zwgj799FOtW7fO9oqvihUr2p5WfdNNN9mKDA0aNNAbb7yhBQsW6OGHH5bVatU111yjpUuX2pa7s0qy7dmyZYvtgZl5hYeHa+fOnZo+fboyMjLsnhGQcyDw2GOP2d79e+WVV+o///mPnn76aWVkZOi6667T8uXLi+yn+u9//1uhoaF68cUX9fbbb6tChQpq0aKF5s6dazuAePLJJ/Xcc8/ppZde0qlTp1SpUiX169fP1qf8zjvv1MWLF/XWW29pzZo1Cg4OVrt27TRx4kSHhR1nfncwTlRUlEaPHq0nn3xSS5cu1aRJk5SQkKB33nlHq1at0ooVK5SSkqLAwEA1bNhQCxYssLvy6EhISIjatGljdweBv7+/WrVqVeQDCuvXr6+ePXvqjTfe0H//+998V5BKk5+fnxYtWqSnn35a48aN0xVXXKEhQ4bo0KFDTm2HS+rWW2/V+fPntWrVKr3//vuqX7++HnvsMT322GMFPtvF2X28MwIDA/Xqq6/q6aef1syZM2UymWwnE850dzOSM9vt4m6LCvLQQw/piiuu0OrVq/X2228rMjJSPXr00JgxYwpcDvfff78qVKigV199Va+++qri4uI0adIkTZs2rUTfu1WrVlq8eLEWLlyo4cOH66qrrtLjjz+usWPHOtXtpXfv3nrmmWfs7iyUnMvVsmXLtHbtWiUmJpboOxRk8uTJunTpku3ixs0336zx48fb3iIgXX6TwKpVq7RkyRKNGjVKAQEBatSokV5++eUiCxetWrXSRx99pDfffFOfffaZXnjhBWVlZalatWpq27atFixYoIYNG9rajxkzRpUrV9aaNWv04osvqmLFimrXrp3GjRtXrDc+lfTYOG9XkNwGDx6sYcOGaebMmercubPteTFVq1bV2LFj9dRTT+njjz+2DR8yZIiSkpI0cuRIRUVF6YEHHnB4gp6b2WzWypUrtXDhQj3//PP6+++/VaVKFd1zzz22C2zOHFsuWbJE8+fP18KFC5Wamqpq1app5MiRBT7noW/fvgoNDdXzzz+vESNGKCwsTB07dtS4ceOcep5HcZis5flpRgB8xsGDB3Xo0CHdcMMNdlcF+vXrp6pVq2rJkiUejA7e7JFHHtH27dvz3UoIwLd8/PHHatiwoV0/9s2bN+v+++/Xhx9+6NG7K+AZX375papWrWr3wLuDBw+qZ8+eFFVRqNjY2HxvOcNl3EEAwCecP39eo0eP1oABA9S9e3dlZ2drw4YN2rt3r9177AEAZdP69eu1YMECjRkzRtWqVdORI0e0aNEixcfHUxwop/73v/9pw4YNmjBhgmrXrq2kpCQtX75cderUUYcOHTwdHuCTKBAA8AnNmjXTc889p1WrVmndunWyWq1q2LChXnzxxUIfJgMAKBtmz55t62aVkpKiK664Qj169NCoUaM8HRo8ZNKkSQoODtby5cuVnJysyMhIdezYUePHjy/0NX8ACkYXAwAAAAAAwGsOAQAAAAAABQIAAAAAACAKBAAAAAAAQBQIAAAAAACAeIuB06xWqywW9z3P0Ww2uXV6sEd+jUNujUV+jVWW8ms2m2QymTwdRpnCvt63kF9jkV9jkV/jlKXcemJfT4HASRaLVSkp59wyLX9/s6KiQpWefl6XLlncMk38g/wah9wai/waq6zlNzo6VH5+FAjciX297yC/xiK/xiK/xilrufXEvp4uBgAAAAAAgAIBAAAAAACgQAAAAAAAAESBAAAAAAAAiIcUAgAAL5CWlqb58+dr8+bNOnv2rGJjYzV+/Hi1atXKYfvly5frueeeyzc8MTHR4EgBACi7fO4Ogueff1533XVXoW1SU1M1fvx4tW7dWvHx8Zo+fboyMjJKKUIAAFBc48aN0+7duzV//ny9//77uvbaazV06FAdOnTIYfvExET17t1b//vf/+z+AAAA1/lUgeCNN95weLUgr1GjRunIkSN65ZVXtHDhQm3ZskXTpk0zPD4AAFB8R44c0bfffqtp06apVatWql27th5//HHFxMToo48+cjjOgQMH1LBhQ1WuXNnuDwAAcJ1PdDFISkrSE088oW3btqlWrVqFtt29e7e2b9+uDRs2qG7dupKkJ598UsOGDdO4ceNUpUqVUogYAAA4KyoqSitXrlSTJk1sw0wmk0wmk9LT0/O1z8rK0u+//646deqUZpgAAJR5PnEHwS+//KKAgACtX79ezZo1K7Ttzp07VblyZVtxQJLi4+NlMpm0a9cuo0MFAADFFBERoc6dOyswMNA2bOPGjTpy5Ig6duyYr/1vv/2m7Oxsbdy4UTfeeKMSEhI0ceJEJScnl2bYAACUOT5xB0HXrl3VtWtXp9omJSWpWrVqdsMCAwMVGRmpEydOlCgOf3/31FP8/Mx2f8O9yK9xyK2xyK+xyK/v+OGHHzR58mTdcMMNSkhIyPf5gQMHJEkhISFauHCh/v77b82fP1+DBw/WunXrFBwc7PK82df7BvJrLPJrLPJrHHJbcj5RICiOjIwMuysQOYKCgpSZmenydM1mk6KiQksSWj4RESFunR7skV/jkFtjkV9jkV/vtmnTJk2YMEEtWrTQ3LlzHba57bbb1KlTJ0VHR9uG1a9fX506ddJXX32lm2++2aV5s6/3PeTXWOTXWOTXOOTWdWWuQBAcHKysrKx8wzMzM1WhQgWXp2uxWJWefr4kodn4+ZkVERGi9PQMZWdb3DJN/IP8GofcGov8Gqus5TciIqTMXSFZvXq1Zs6cqR49emj27NkOC/45chcHJCkmJkaRkZE6efKky/NnX+87yK+xyK+xyK9xylpuPbGvL3MFgqpVq2rTpk12w7KyspSWlqaYmJgSTfvSJfeuZNnZFrdPE/8gv8Yht8Yiv8Yiv95pzZo1mjFjhu666y499thjMplMBbZdsGCBPvvsM3322We2dseOHVNqaqrq1atXojjY1/sW8mss8mss8msccuu6snXpQVLr1q118uRJHTlyxDZs+/btkqSWLVt6KiwAAFCAw4cP6+mnn1b37t11//3366+//tKpU6d06tQpnTlzRllZWTp16pTtDsHu3bvrzz//1LRp03T48GHt2LFDDz30kFq0aOHwoYYAAMA5Pn8HQXZ2tlJSUhQeHq7g4GA1a9ZMLVq00NixYzVt2jSdP39eU6dO1W233cYrDgEAbmexWHXgaJrSzmUqMjRI19SIlNlc8NVv5Ldx40ZdvHhRX3zxhb744gu7z/r06aM+ffpo8ODBeu2119SmTRs1btxYL7zwghYuXKi+ffsqMDBQ119/vSZNmlTonQeAr2I7A3hWefoN+nyB4MSJE7r++us1a9Ys9e3bVyaTSUuWLNH06dM1ZMgQBQUFqUePHpo8ebKnQwUAlDG7EpO1ZtNBpZ755yG4UeFBGtCtvlrGlqxbW3nywAMP6IEHHii0TWJiot3/27Vrp3bt2hkZFuAV2M4AnlXefoMmq9Vq9XQQviA726KUlHNumZa/v1lRUaFKTT1H3xgDkF/jkFtjkV9juTu/uxKTtXTt3gI/H9GnsaEHDtHRoWXuIYWexr7ed5SX/HpqO1Ne8usp5Nc47OtLjiMLAACKyWKxas2mg4W2eXPTQVks1OABuIbtDOBZ5fU3SIEAAIBiOnA0ze5WQ0dSzmTqwNG00gkIQJnDdgbwrPL6G6RAAABAMaWdK/yAobjtACAvtjOAZ5XX3yAFAgAAiikyNMit7QAgL7YzgGeV198gBQIAAIrpmhqRigov/IAgOvzya5AAwBVsZwDPKq+/QQoEAAAUk9ls0oBu9Qtt079b/TL7jmQAxmM7A3hWef0NUiAAAMAFLWNjNKJP43xXF6LDgwx/7RGA8oHtDOBZ5fE36O/pAAAA8FUtY2MUV7+yDhxNU9q5TEWGXr7VsKxdTQDgOWxnAM8qb79BCgQAAJSA2WxSg6ujPB0GgDKM7QzgWeXpN0gXAwAAAAAAQIEAAAAAAABQIAAAAAAAAKJAAAAAAAAARIEAAAAAAACIAgEAAAAAABAFAgAAAAAAIAoEAAAAAABAFAgAAAAAAIAoEAAAAAAAAFEgAAAAAAAAokAAAAAAAABEgQAAAAAAAIgCAQAAAAAAEAUCAAAAAAAgCgQAAAAAAEAUCAAAAAAAgCgQAAAAAAAAUSAAAAAAAACiQAAAAAAAAESBAAAAAAAAiAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQIAAAAAACAKBAAAAAAAABRIAAAAAAAAKJAAAAAAAAARIEAAAAAAACIAgEAAAAAABAFAgAAAAAAIAoEAAAAAABAFAgAAAAAAIAoEAAAAAAAAFEgAAAAAAAAokAAAAAAAABEgQAAAAAAAIgCAQAAAAAAkI8UCCwWixYtWqSOHTuqefPmuvfee3X06NEC2//9998aP3682rZtqzZt2mjs2LFKSkoqxYgBAAAAAPAtPlEgWLZsmdasWaMZM2borbfeksVi0bBhw5SVleWw/ZgxY3T8+HG9/PLLevnll3X8+HGNGDGilKMGAAAAAMB3eH2BICsrSy+99JJGjRqlhIQENWjQQAsWLNDJkyf1+eef52ufnp6u7du3695779W1116rhg0b6r777tPPP/+stLS00v8CAAAAAAD4AK8vEOzfv1/nzp1Tu3btbMMiIiLUsGFD7dixI1/74OBghYaGat26dTp79qzOnj2rDz/8ULVr11ZERERphg4AAAAAgM/w93QARTl58qQkqVq1anbDY2JibJ/lFhgYqGeeeUZTp05Vq1atZDKZFBMTo9WrV8tsLlk9xN/fPfUUPz+z3d9wL/JrHHJrLPJrLPLr3dLS0jR//nxt3rxZZ8+eVWxsrMaPH69WrVo5bH/s2DHNmDFDO3bsUIUKFdSvXz899NBD8vPzK+XIAQAoO7y+QJCRkSHp8ol/bkFBQTp9+nS+9larVb/++qvi4uI0bNgwZWdna8GCBRo+fLjefPNNhYWFuRSH2WxSVFSoS+MWJCIixK3Tgz3yaxxyayzyayzy653GjRunU6dOaf78+apUqZJef/11DR06VGvXrlWdOnXs2l68eFFDhw5VrVq19NZbb+mPP/7QY489JrPZrFGjRnnoGwAA4Pu8vkAQHBws6fKzCHL+LUmZmZkKCcl/kPfpp59q9erV+vrrr23FgBUrVqhLly567733dPfdd7sUh8ViVXr6eZfGzcvPz6yIiBClp2coO9vilmniH+TXOOTWWOTXWGUtvxERIWXmbogjR47o22+/1Zo1a9SyZUtJ0uOPP67//ve/+uijjzR69Gi79hs3btTx48f1zjvvqGLFirrmmmv0999/69lnn9UDDzyQ76ICAABwjtcXCHK6FiQnJ6tmzZq24cnJyYqNjc3XfufOnapdu7bdnQIVK1ZU7dq1deTIkRLFcumSew8os7Mtbp8m/kF+jUNujUV+jUV+vU9UVJRWrlypJk2a2IaZTCaZTCalp6fna79z5041atRIFStWtA1r27atzp49q19//VXNmjUrlbgBAChrvL5A0KBBA4WFhWnbtm22AkF6err27dunQYMG5WtftWpVffLJJ8rMzFRQUJAk6fz58zp27Jh69epVqrEDAICiRUREqHPnznbDNm7cqCNHjujRRx/N1/7kyZOqWrWq3bCYmBhJ0okTJ0pUIOB5Q76B/BqL/BqL/BqH3Jac1xcIAgMDNWjQIM2dO1fR0dG66qqrNGfOHFWtWlU33HCDsrOzlZKSovDwcAUHB+u2227TqlWrNGbMGNstic8995yCgoLUt29fD38bAABQlB9++EGTJ0/WDTfcoISEhHyfX7hwId+biXIuCmRmZro8X5435HvIr7HIr7HIr3HIreu8vkAgSaNGjdKlS5c0ZcoUXbhwQa1bt9aqVasUEBCgY8eO6frrr9esWbPUt29fxcTEaM2aNZozZ46GDBkis9msVq1aac2aNQoPD/f0VwEAAIXYtGmTJkyYoBYtWmju3LkO2wQHBysrK8tuWE5hoEKFCi7Pm+cN+Q7yayzyayzya5yylltPPG/IJwoEfn5+mjhxoiZOnJjvs+rVqysxMdFuWN26dbVixYrSCg8AALjB6tWrNXPmTPXo0UOzZ88u8GGDVatW1YEDB+yGJScnS5KqVKlSohh43pBvIb/GIr/GIr/GIbeuo3MGAADwuDVr1mjGjBkaOHCg5s+fX+ibCFq3bq19+/bp7NmztmFbt25VaGioGjRoUBrhAgBQJlEgAAAAHnX48GE9/fTT6t69u+6//3799ddfOnXqlE6dOqUzZ84oKytLp06dsnUr6NatmypXrqwxY8Zo//792rRpk+bPn6///Oc/vOIQAIAS8IkuBgBQFIvFqgNH05R2LlORoUG6pkakzGaTp8MC4ISNGzfq4sWL+uKLL/TFF1/YfdanTx/16dNHgwcP1muvvaY2bdooKChIL774oqZPn67bb79dFStW1IABAzR8+HAPfQMAAMoGCgQAfN6uxGSt2XRQqWf+eXp5VHiQBnSrr5axMR6MDIAzHnjgAT3wwAOFtsn7vKGrr75aL730kpFhAQBQ7tDFAIBP25WYrKVr99oVByQp9Uymlq7dq12JyR6KDAAAAPAtFAgA+CyLxao1mw4W2ubNTQdlsVhLKSIAAADAd1EgAOCzDhxNy3fnQF4pZzJ14Gha6QQEAAAA+DAKBAB8Vtq5wosDxW0HAAAAlGcUCAD4rMjQILe2AwAAAMozCgQAfNY1NSIVFV74yX90+OVXHgIAAAAoHAUCAD7LbDZpQLf6hbbp362+zGZTKUUEAAAA+C4KBAB8WsvYGI3o0zjfnQTR4UEa0aexWsbGeCgyAAAAwLf4ezoAACiplrExiqtfWQeOpintXKYiQy93K+DOAQAAAMB5FAgAlAlms0kNro7ydBgAAACAz6KLAQAAAAAAoEAAAAAAAAAoEAAAAAAAAFEgAAAAAAAAokAAAAAAAABEgQAAAAAAAIgCAQAAAAAAEAUCAAAAAAAgCgQAAAAAAEAUCAAAAAAAgCgQAAAAAAAAUSAAAAAAAACiQAAAAAAAAESBAAAAAAAAiAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQIAAAAAACAKBAAAAAAAABRIAAAAAAAAKJAAAAAAAAARIEAAAAAAACIAgEAAAAAABAFAgAAAAAAIAoEAAAAAABAFAgAAAAAAIAkf08HAAAAAADOsFisOnA0TWnnMhUZGqRrakTKbDZ5OiygzKBAAAAAAMDr7UpM1ppNB5V6JtM2LCo8SAO61VfL2BgPRgaUHXQxAAAAAODVdiUma+navXbFAUlKPZOppWv3aldisociA8oWCgQAAAAAvJbFYtWaTQcLbfPmpoOyWKylFBFQdlEgAAAAAOC1DhxNy3fnQF4pZzJ14Gha6QQElGEUCAAAAAB4rbRzhRcHitsOQMF8okBgsVi0aNEidezYUc2bN9e9996ro0ePFtj+4sWLmjdvnq39oEGD9Ouvv5ZixAAAAADcITI0yK3tABTMJwoEy5Yt05o1azRjxgy99dZbslgsGjZsmLKyshy2nzZtmj744AM9/fTTev/99xUdHa17771XZ86cKeXIAQAAAJTENTUiFRVe+Ml/dPjlVx4CKBmvLxBkZWXppZde0qhRo5SQkKAGDRpowYIFOnnypD7//PN87Y8ePar3339fM2fOVMeOHVW3bl099dRTCgwM1N69ez3wDQAAAAC4ymw2aUC3+oW26d+tvsxmUylFBJRdXl8g2L9/v86dO6d27drZhkVERKhhw4basWNHvvbffvutwsPD1alTJ7v2X331ld00AAAAAPiGlrExGtGncb47CaLDgzSiT2O1jI3xUGRA2eLv6QCKcvLkSUlStWrV7IbHxMTYPsvt8OHDqlGjhj7//HOtXLlSSUlJatiwoR555BHVrVu3VGIGAAAA4F4tY2MUV7+yDhxNU9q5TEWGXu5WwJ0DgPt4fYEgIyNDkhQYGGg3PCgoSKdPn87X/uzZszpy5IiWLVumhx9+WBEREVq+fLkGDBigDRs2qFKlSi7H4u/vnhsu/PzMdn/DvcivccitscivscgvAPg+s9mkBldHeToMoMzy+gJBcHCwpMvPIsj5tyRlZmYqJCQkX3t/f3+dPXtWCxYssN0xsGDBAnXu3Flr167VsGHDXIrDbDYpKirUpXELEhGRP364D/k1Drk1Fvk1FvkFAABwzOsLBDldC5KTk1WzZk3b8OTkZMXGxuZrX7VqVfn7+9t1JwgODlaNGjV07Ngxl+OwWKxKTz/v8vi5+fmZFRERovT0DGVnW9wyTfyD/BqH3BqL/BqrrOU3IiKEuyEAAIBbeX2BoEGDBgoLC9O2bdtsBYL09HTt27dPgwYNyte+devWunTpkn7++Wc1adJEknThwgUdPXpUt9xyS4liuXTJvQeU2dkWt08T/yC/xiG3xiK/xiK/AAAAjnl9gSAwMFCDBg3S3LlzFR0drauuukpz5sxR1apVdcMNNyg7O1spKSkKDw9XcHCwWrVqpeuuu06TJk3Sk08+qcjISC1atEh+fn7q3bu3p78OAAAAAABeySfuTRw1apT69eunKVOmqH///vLz89OqVasUEBCgEydOqEOHDtqwYYOt/eLFixUfH6+RI0eqX79+Onv2rF577TVFR0d78FsAAABnPP/887rrrrsKbbN+/XrFxsbm+1OS7oQAAJR3Xn8HgST5+flp4sSJmjhxYr7PqlevrsTERLthYWFhmjZtmqZNm1ZKEQIAAHd444039Nxzz6lVq1aFtktMTFR8fLzmz59vN5yLAQAAuM4nCgQAAKBsS0pK0hNPPKFt27apVq1aRbY/cOCAYmNjVblyZeODAwCgnPCJLgYAAKBs++WXXxQQEKD169erWbNmRbZPTEy0e2MRAAAoOe4gAAAAHte1a1d17drVqbanT59WUlKSdu7cqTVr1ig1NVVNmzbVxIkTVbt27RLF4e/vnmsnOa+g5FWUxiC/xiK/xiK/xiG3JUeBAAAA+JSDBw9KkqxWq2bNmqULFy5o+fLlGjBggD766CNdccUVLk3XbDYpKirUnaEqIiLErdODPfJrLPJrLPJrHHLrOgoEAADAp7Rq1Urff/+9oqKiZDKZJElLlixRQkKCPvjgA913330uTddisSo9/bxbYvTzMysiIkTp6RnKzra4ZZr4B/k1Fvk1Fvk1TlnLbURESKnfDUGBAAAA+Jy8bysICQlR9erVlZSUVKLpXrrk3gPK7GyL26eJf5BfY5FfY5Ff45Bb19E5AwAA+JS3335bbdq00fnz/1ztP3v2rH7//XfVq1fPg5EBAODbKBAAAACvlp2drVOnTunChQuSpE6dOslisejhhx/WwYMH9fPPP+uhhx5SdHS0+vbt6+FoAQDwXRQIAACAVztx4oQ6dOigDRs2SJKqVaumV155RefPn1f//v119913Kzw8XK+99pqCgoI8HC0AAL6LZxAAAACv8swzz9j9v3r16kpMTLQb1qhRI7300kulGRYAAGUedxAAAAAAAAAKBAAAAAAAgAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQIAAAAAACAKBAAAAAAAABRIAAAAAAAAKJAAAAAAAAARIEAAAAAAACIAgEAAAAAAJDk7+kAANizWKw6cDRNaecyFRkapGtqRMpsNnlFTGcyLqpGtYq6MirYo/H4Am9cjrl5e3wAAAAofRQIAC+yKzFZazYdVOqZTNuwqPAgDehWXy1jY7wmpujwIPX3YEzezhuXY27eHh8AAAA8gy4GgJfYlZispWv32p20SVLqmUwtXbtXuxKTvSamFA/G5O28cTnm5u3xAQAAwHMoEABewGKxas2mg4W2eXPTQVks1lKKyDtj8nbenjNvjw8AAACeRYEA8AIHjqblu6KbV8qZTB04mlY6Ack7Y/J23p4zb48PAAAAnkWBAPACaecKP2krbjt38MaYvJ2358zb4wMAAIBnUSAAvEBkaJBb27mDN8bk7bw9Z94eHwAAADyLAgHgBa6pEamo8MJPyqLDL7+KrrR4Y0zezttz5u3xAQAAwLMoEABewGw2aUC3+oW26d+tfqm+p94bY/J23p4zb48PAAAAnkWBAPASLWNjNKJP43xXeKPDgzSiT2OPvJ++wJgiPBeTt/PG5Zibt8cHAAAAz/H3dAAA/tEyNkZx9SvrwNE0pZ3LVGTo5du9PXlFN3dMZzIuqka1iroyKphX4RXCG5djbt4eHwAAADyDAgHgZcxmkxpcHeXpMOzkxOTvb1ZUVKhSU89RICiCNy7H3Lw9PgAAAJQ+uhgAAAAAAAAKBAAAAAAAgAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQIAAAAAACAeM0hAAAAfIDFYtWBo2lKO5epShHBalOxguHz238kVfuPpspqsSo0JEAVQ4MUFR6ka2pEymw2lWjaOd8lMvTy9CTlG1aSeZQkptLIr68paplFhARKJin9fJbdv0uyLB3NM/d0ioypQqBkldIzShYHnFfUMvMFFAgAAADg1XYlJmvNpoNKPZNpG1bpo30a0P0axdW7wpD5vfLpfp27cMnh51HhQRrQrb5axsa4NO283yU0+PIhee75lWQe7ojJyPz6GmeXWUFcWZaO5pl7Oq7EVJrrVHlU1DLzFXQxAAAAgNfalZispWv32h10S9Lfpy9o8Xt7tCsx2ZD5FXbil3omU0vX7i32vAv6LucuXMo3P1fnUVylnV9fU5xlVpDiLsuC5pkznXe+OuhSTKW1TpVHRS0zX8o5BQIAAAB4JYvFqjWbDhba5s1NB2WxWN02vze+OOB0++LM25nvUtJ5FFdp59fXuLrMCuJMLp2Z58YdRw2PA84ra78jCgQAAADwSgeOpuW7IpdXyplMHTia5rb5pZ3Ncrp9cebtzHcp6TyKq7Tz62tcXWYFcSaXzszTWsLzzPK8TI1Q1n5HFAgAAADgldLOOXdy5mw7I6ZTGjG66/u5Ol2j5u/tjPjeRU2ztHJdXpepEcra74gCAQAAALxSZGiQW9sZMZ3SiNFd38/V6Ro1f29nxPcuapqllevyukyNUNZ+RxQIAAAA4JWuqRGpqPDCD6qjw/95vZs75hcZFuh0++LM25nvUtJ5FFdp59fXuLrMCuJMLp2Zp6mEb80rz8vUCGXtd+QTBQKLxaJFixapY8eOat68ue69914dPercwznWr1+v2NhYHTt2zOAoAQAA4E5ms0kDutUvtE3/bvXd9p5xs9mkgd2vcbp9cebtzHcp6TyKq7Tz62tcXWYFcSaXzszzxtY1DI8DzitrvyOfKBAsW7ZMa9as0YwZM/TWW2/JYrFo2LBhysoq/CEyf/75p5588slSihIAAADu1jI2RiP6NM53he6KyBA91K+p298vnjO/nHfKOxIdHqQRfRoXe94FfZewkIB883N1HsVV2vn1NcVZZgUp7rIsaJ4507m9a33HMQX7G7LeomhFLTNfyrnJai3pczCNlZWVpbZt22rChAkaMGCAJCk9PV0dO3bUzJkz1bNnT4fjWSwWDRo0SAEBAdq6dau+/PJLVa9e3eU4srMtSkk55/L4ufn7mxUVFarU1HO6dMnilmniH+TXOOTWWOTXWGUtv9HRofLz84k6v89gX+/dLBbr5bcMnMtUpYhgtWlWXemnzxuWX4vFqv1HUrX/aKqsFqtCQwJUMTRIUf//VuGSXA3M/V0iQ/+59TjvsNK84lja+fU1RS2ziJBAySSln8+y+7ejZens9sHRPHNPp8iYKgRKVik9w3EcZZGnt71FLbPi8sS+3rmylwft379f586dU7t27WzDIiIi1LBhQ+3YsaPAAsGKFSt08eJFjRw5Ulu3bi2tcAEAAGAAs9mkBldHSbp8EuBn8ImO2WxSw9rRalg72pBp53yX3BwNKy2lnV9f44llVtA8PRkTClfUMvMFXl8gOHnypCSpWrVqdsNjYmJsn+W1Z88evfTSS3rvvfeUlJRkeIwAAAAAAPg6ry8QZGRkSJICA+2fKBsUFKTTp0/na3/+/HlNmDBBEyZMUK1atdxaIPD3d8/tHTm3iXBrqDHIr3HIrbHIr7HILwAAQOG8vkAQHBws6fKzCHL+LUmZmZkKCQnJ1/6pp55S7dq1deedd7o1DrPZpKioULdOMyIif/xwH/JrHHJrLPJrLPILAADgmNcXCHK6FiQnJ6tmzZq24cnJyYqNjc3X/v3331dgYKDi4uIkSdnZ2ZKknj176oEHHtADDzzgUhwWi1Xp6eddGjcvPz+zIiJClJ6eoexsHvzibuTXOOTWWOTXWGUtvxERIdwNAQAA3MrrCwQNGjRQWFiYtm3bZisQpKena9++fRo0aFC+9p9//rnd/3/66SdNnDhRK1eu1DXXOP9eW0fc/STM7GwLT4Z1E0dP3jUyv0Y+2dgXsO7+o7hPGK53VUX99ufpQp9uW1R+XXmqsdlscvuTdX2VJ9Zfcg8AAHyB1xcIAgMDNWjQIM2dO1fR0dG66qqrNGfOHFWtWlU33HCDsrOzlZKSovDwcAUHB+vqq6+2Gz/nQYZXXnmlIiMjPfANYLRdiclas+mgUs9k2oZV+mifBnS/RnH1rjBkfq98ul/nLlxy+HlUeJAGdKvvU+87hWscrXu5l7+jz00mKffLZYu7vrgyz6jwILW5Nkbbfk0ucDwYp6hlBgAA4C184t7EUaNGqV+/fpoyZYr69+8vPz8/rVq1SgEBATpx4oQ6dOigDRs2eDpMeMCuxGQtXbvX7sBbkv4+fUGL39ujXYnJhsyvoOKAJKWeydTStXvdPm94l4LWvZzl/85XBx1+nrs4kLu9M+uLq/NMPZOpz7YfLXA81lXjFLXMyD0AAPAmXn8HgST5+flp4sSJmjhxYr7PqlevrsTExALHbdOmTaGfw3dZLFat2XSw0DZvbjqouPqV3XIrr8Vi1RtfHHC6vTvnDe/izLq3ccfRYk3zzU0H1fraKqU6z9zzZl11v9LeRgEAAJSUT9xBADhy4GhavqtyeaWcydSBo2lum1/a2Syn27tz3vAuzqx7ee8UKErKmUwl/pFaqvPMPW/WVfcr7W0UAABASVEggM9KO1f4gXdx2xkxHXfNG97FqOVaWAHK6HWJddX9SnsbBQAAUFIUCOCzIkOD3NrOiOm4a97wLkYt18iwwFKfZ2lNvzwq7W1UWfL888/rrrvuKrRNamqqxo8fr9atWys+Pl7Tp09XRkZGKUUIAEDZVKJnEJw+fVoZGRmyWPK/LurKK68syaSBIl1TI1JR4UGF3sIb/f9fO+iu+UWGBTrdzcCd84Z3cWbdy/u2gqJEhwcptmZUqc4z97xZV92vtLdRRirN/f0bb7yh5557Tq1atSq03ahRo5SRkaFXXnlF6enpeuyxx3T+/HnNnj3brfEAAFCeuFQgOHLkiCZNmqSffvqpwDa//vqry0EBzjCbTRrQrb6Wrt1bYJv+3eq77eFfZrNJA7tfU+j8jJo3vIsz696NrWvos+3OPzSwqPXFiHk6O2+4prS3UUYozf19UlKSnnjiCW3btk21atUqtO3u3bu1fft2bdiwQXXr1pUkPfnkkxo2bJjGjRunKlUKfuAnAAAomEsFghkzZuj333/XyJEjVbVqVZnN9FSAZ7SMjdGIPo3zvWP8isgQ9e9WX3H1rjBkfq98ur/AVx1GhwepP+83L/MKWvdyL/+6V1XM93neq/zFWV9cnWd0eJDir43Rtl+TCxwPxnBmmXmz0tzf//LLLwoICND69eu1dOlS/fnnnwW23blzpypXrmwrDkhSfHy8TCaTdu3apZtvvtmwOAEAKMtcKhDs2LFDM2fOVM+ePd0dD1BsLWNjFFe/8uW3DJzLVKWIYLVpVl3pp8/r0qX8t8O6a377j6Rq/9FUWS1WhYYEqGJokKL+/+3C3nxFEO6Td92LDLVf/o4+r3dVRf3252mH7Y2aZ87n/RLqFTgejFPUMvNmpbm/79q1q7p27epU26SkJFWrVs1uWGBgoCIjI3XixIkSxeHv754iiJ+f2e5vuBf5NRb5NRb5NQ65LTmXCgRhYWGqWLGiu2MBXGY2m9Tg6sv9t/39zfIz+MDbbDapYe1oNawdbeh84P1yr3vOfl5Ye6Pm6cx4MI6v5t5b9/cZGRkKDMz/UM+goCBlZrr+Vgiz2aSoqNCShJZPRESIW6cHe+TXWOTXWOTXOOTWdS4VCHr37q033nhDHTp0kMnk/VdAAABA8Xnr/j44OFhZWfkfGJuZmakKFSq4PF2Lxar09PMlCc3Gz8+siIgQpadnKDvb/XezlXfk11jk11jk1zhlLbcRESGlfjeESwWCkJAQ7dq1S927d1eTJk0UHBxs97nJZNLTTz/tlgABAIBneOv+vmrVqtq0aZPdsKysLKWlpSkmpmTPdXB317TsbIsh3d1wGfk1Fvk1Fvk1Drl1nUsFgrVr1yo8PFwWi8Xhk4296SoDAABwjbfu71u3bq25c+fqyJEjuvrqqyVJ27dvlyS1bNnSIzEBAFAWuFQg+Oqrr9wdBwAA8DLesr/Pzs5WSkqKwsPDFRwcrGbNmqlFixYaO3aspk2bpvPnz2vq1Km67bbbeMUhAAAl4FKBIEd6erp+/PFHnTlzRtHR0WrSpInCwsLcFRsAAPACnt7fnzhxQtdff71mzZqlvn37ymQyacmSJZo+fbqGDBmioKAg9ejRQ5MnTy61mAAAKItcLhCsXLlSy5Yt04ULF2zDAgMDdf/992vEiBFuCQ4AAHiWJ/b3zzzzjN3/q1evrsTERLthlSpV0qJFiwyZPwAA5ZVLBYL3339f8+fPV79+/dSrVy9dccUVOnXqlD788EMtWbJEV155pfr06ePuWAEAQClifw8AQPniUoHglVdeUf/+/fXEE0/YhtWpU0dt2rRRcHCwXnvtNQ4YAADwcezvAQAoX1x6qeKRI0fUrVs3h59df/31OnToUImCAgAAnsf+HgCA8sWlAkGVKlV0/Phxh58dO3aMBxUCAFAGsL8HAKB8calA0LVrVy1cuFB79uyxG/7TTz9p8eLF6tq1q1uCAwAAnsP+HgCA8sWlZxA89NBD+u6773THHXfoqquu0hVXXKG//vpLf/75p+rWravx48e7O04AAFDK2N8DAFC+uFQgCAsL03vvvaf3339fO3bs0OnTp9WkSRP95z//Ud++fRUcHOzuOAEAQCljfw8AQPniUoFAkoKCgjRgwAANGDDAnfEAAAAvwv4eAIDyw+kCweTJkzV8+HDVqFFDkydPLrStyWTS008/XeLgAABA6WJ/DwBA+eV0gWDbtm0aMmSI7d8AAKDsYX8PAED55XSB4KuvvnL4bwAAUHawvwcAoPxy6TWHkydP1tGjRx1+dujQIT3wwAMlCgoAAHge+3sAAMoXp+8gOH78uO3fa9euVbdu3eTn55ev3TfffKPvvvvOPdEBAIBSxf4eAIDyy+kCwfTp0/XNN99IuvxQopEjRzpsZ7Va1b59e/dEBwAAShX7ewAAyi+nCwRPPvmkvvvuO1mtVj366KN68MEHVbNmTbs2ZrNZERERatOmjdsDBQAAxmN/DwBA+eV0gaBKlSrq06ePpMtXFBISEhQREWG77fDChQu6ePGiwsPDjYkUAAAYjv09AADll0sPKezZs6eee+453X777bZhP/zwg9q1a6fZs2fLYrG4LUAAAOAZ7O8BAChfXCoQLF68WOvXr1fPnj1twxo2bKgJEybonXfe0Ysvvui2AAEAgGewvwcAoHxxuotBbh999JEmTZqkO++80zYsMjJSd999t/z9/fXaa6/pvvvuc1uQAACg9LG/BwCgfHHpDoLU1FTVqFHD4Wd16tTRyZMnSxQUAADwPPb3AACULy4VCOrUqaONGzc6/Oyrr77S1VdfXaKgAACA57G/BwCgfHGpi8HgwYP1yCOPKC0tTd26dVOlSpWUkpKir7/+Wp9++qlmzZrl7jgBAEApY38PAED54lKB4LbbbtO5c+e0bNkyff7557bhUVFRevzxx3Xbbbe5Kz4AAOAh7O8BAChfXCoQSNLAgQM1YMAAHT58WGlpaYqIiFCdOnVkNrvUawEAAHgh9vcAAJQfLhcIJMlkMqlOnTruigUAAHgh9vcAAJQPThcIrr32Wr399ttq2rSpGjRoIJPJVGBbk8mkffv2uSVAAABQetjfAwBQfjldIBgxYoSqVKli+3dhBwwAAMA3sb8HAKD8crpAMHLkSNu/H3roIUOCAQAAnsX+HgCA8svpAsHx48eLNeErr7yy2MEAAADPYn8PAED55XSBoGvXrsW6zfDXX391KSAAAOA57O8BACi/nC4QPP3007YDhtOnT2vu3Llq166dbrrpJlWuXFlpaWn66quvtHnzZj3yyCOGBQwAAIzD/h4AgPLLZLVarcUdacSIEYqKitJTTz2V77OZM2fq4MGDeuWVV9wRn9fIzrYoJeWcW6bl729WVFSoUlPP6dIli1umiX+QX+OQW2ORX2OVtfxGR4fKz89s6DzK2/6efb3vIL/GIr/GIr/GKWu5LY19fV4uze3bb7/VTTfd5PCzhIQE7d69u0RBAQAAz2N/DwBA+eJSgSAqKkp79uxx+NnWrVttr0dyF4vFokWLFqljx45q3ry57r33Xh09erTA9gcPHtR9992nNm3aqF27dho1alSxH7oEAEB5V9r7ewAA4FlOP4Mgt3//+99aunSpLly4oISEBEVFRemvv/7SZ599pjfffFOPPvqoW4NctmyZ1qxZo2eeeUZVq1bVnDlzNGzYMH300UcKDAy0a5uamqp77rlHLVq00Ouvv66srCw988wzGjZsmNauXaugoCC3xgYAQFlV2vt7AADgWS4VCB588EGdOXNGq1at0sqVKyVJVqtVwcHBGj16tAYOHOi2ALOysvTSSy9pwoQJSkhIkCQtWLBAHTt21Oeff66ePXvatd+0aZPOnz+vZ599VsHBwZKkOXPmKCEhQT/88IPatWvnttgAACjLSnN/DwAAPM+lAoHJZNKkSZM0fPhw/fjjjzp9+rSioqIUFxenChUquDXA/fv369y5c3Yn9hEREWrYsKF27NiRr0DQrl07LVu2zFYckCSz+XJPivT0dLfGBgBAWVaa+3sAAOB5LhUIcoSGhqpy5cqyWq1q1qyZsrKy3H7AcPLkSUlStWrV7IbHxMTYPsutevXqql69ut2wlStXKjg4WK1bty5RLP7+7nmCZM6TKEv7iZTlBfk1Drk1Fvk1Fvl1XWns7wEAgOe5XCD48MMPNW/ePJ06dUomk0nvvvuuFi9erICAAM2bNy/fswFclZGRIUn5phcUFKTTp08XOf7rr7+u1atXa8qUKYqOjnY5DrPZpKioUJfHdyQiIsSt04M98msccmss8mss8ls8pbW/BwAAnudSgWDDhg2aNGmSevXqpS5dumjs2LGSpO7du2v69OlatmyZxowZ45YAc7oKZGVl2XUbyMzMVEhIwQd5VqtVCxcu1PLly/Xggw/qrrvuKlEcFotV6ennSzSNHH5+ZkVEhCg9PUPZ2b7/fk5vQ36NQ26NRX6NVdbyGxERYvjdEKW5vwcAAJ7nUoFgxYoVuvPOOzVt2jRlZ2fbhv/rX/9SSkqK3nnnHbcdMOR0LUhOTlbNmjVtw5OTkxUbG+twnIsXL2ry5Mn6+OOPNXnyZN19991uieXSJfceUGZnW9w+TfyD/BqH3BqL/BqL/DqvNPf3AADA81y69HD48GF1797d4WfNmjVTUlJSiYLKrUGDBgoLC9O2bdtsw9LT07Vv374Cnynw8MMP67PPPtO8efPcVhwAAKC8Kc39PQAA8DyXCgSVKlXS//3f/zn87P/+7/9UqVKlEgWVW2BgoAYNGqS5c+fqyy+/1P79+zV27FhVrVpVN9xwg7Kzs3Xq1ClduHBBkvTBBx9ow4YNGjt2rOLj43Xq1Cnbn5w2AACgaKW5vwcAAJ7nUoHg5ptv1qJFi/TZZ58pKytL0uVXIe3du1fLli1Tjx493BrkqFGj1K9fP02ZMkX9+/eXn5+fVq1apYCAAJ04cUIdOnTQhg0bJEkff/yxJOnZZ59Vhw4d7P7ktAEAAEUr7f09AADwLJPVarUWd6SsrCwNHz5c//vf/2Q2m2WxWBQaGqrz58+rVatWeuGFF+weKFgWZGdblJJyzi3T8vc3KyoqVKmp5+gHawDyaxxyayzya6yylt/o6FDDH1JY3vb37Ot9B/k1Fvk1Fvk1TlnLbWns6/Ny6SGFgYGBevHFF/Xtt99q69atSktLU3h4uOLj49W5c2eZTCZ3xwkAAEoZ+3sAAMoXlwoEQ4cO1bBhw9S+fXu1b9/e3TEBAAAvwP4eAIDyxaX7FX744QeuGgAAUMaxvwcAoHxxqUDQsWNHrV+/XhcvXnR3PAAAwEuwvwcAoHxxqYtBUFCQ1q9fr08//VR169ZVhQoV7D43mUx69dVX3RIgAADwDPb3AACULy4VCE6ePKm4uDjb//O+CMGFFyMAAAAvw/4eAIDypdgFgj179mjAgAGqWbOmGjVqZERMAADAw9jfAwBQ/jhdIEhPT9f999+vH3/80TYsLi5O8+bNU7Vq1YyIDQAAlDJP7e8tFouWLFmid999V2fOnFHr1q01depU1ahRw2H79evXa+LEifmGf/nll6pevbphcQIAUJY5/ZDC5557Tvv27dNDDz2k559/XpMmTdKhQ4c0depUI+MDAAClyFP7+2XLlmnNmjWaMWOG3nrrLVksFg0bNkxZWVkO2ycmJio+Pl7/+9//7P5w0QIAANc5fQfB119/rXHjxmnIkCGSpE6dOqlKlSqaMGGCzp8/n+/BRQAAwPd4Yn+flZWll156SRMmTFBCQoIkacGCBerYsaM+//xz9ezZM984Bw4cUGxsrCpXruz2eAAAKK+cvoPg1KlT+fogtmnTRtnZ2Tpx4oTbAwMAAKXPE/v7/fv369y5c2rXrp1tWEREhBo2bKgdO3Y4HCcxMVF169Y1JB4AAMorp+8guHTpkgIDA+2GVaxYUZKUmZnp3qgAAIBHeGJ/f/LkSUnK1z0gJibG9llup0+fVlJSknbu3Kk1a9YoNTVVTZs21cSJE1W7du0SxeLv7/S1k0L5+Znt/oZ7kV9jkV9jkV/jkNuSc+k1h3nxmiMAAMo+o/b3GRkZkpSvMBEUFKTTp0/na3/w4EFbPLNmzdKFCxe0fPlyDRgwQB999JGuuOIKl+Iwm02Kigp1adyCRESEuHV6sEd+jUV+jUV+jUNuXeeWAoHJZHLHZAAAgBczan8fHBws6fKzCHL+LV2+YyEkJP9BXqtWrfT9998rKirKFtOSJUuUkJCgDz74QPfdd59LcVgsVqWnn3dp3Lz8/MyKiAhRenqGsrMtbpkm/kF+jUV+jUV+jVPWchsREVLqd0MUq0Awbdo0hYWF2f6fcyXh8ccfV2joPxV3k8mkV1991U0hAgCA0lTa+/ucrgXJycmqWbOmbXhycrJiY2MdjhMdHW33/5CQEFWvXl1JSUkliuXSJfceUGZnW9w+TfyD/BqL/BqL/BqH3LrO6XJE69atFRoaKqvVavuTM7xChQp2wy0WFgYAAL7IE/v7Bg0aKCwsTNu2bbMNS09P1759+9S6det87d9++221adNG58//c7X/7Nmz+v3331WvXj23xAQAQHnk9B0Er7/+upFxAAAAL+CJ/X1gYKAGDRqkuXPnKjo6WldddZXmzJmjqlWr6oYbblB2drZSUlIUHh6u4OBgderUSXPnztXDDz+s0aNH68KFC5o/f76io6PVt2/fUo8fAICygsc7AgAAjxs1apT69eunKVOmqH///vLz89OqVasUEBCgEydOqEOHDtqwYYOky10SXnnlFZ0/f179+/fX3XffrfDwcL322msKCgry8DcBAMB3may8gsAp2dkWpaScc8u0/P3NiooKVWrqOfrGGID8GofcGov8Gqus5Tc6OpTXOLkZ+3rfQX6NRX6NRX6NU9Zy64l9PUcWAAAAAACAAgEAAAAAAKBAAAAAAAAARIEAAAAAAACIAgEAAAAAABAFAgAAAAAAIAoEAAAAAABAFAgAAAAAAIAoEAAAAAAAAFEgAAAAAAAAokAAAAAAAABEgQAAAAAAAIgCAQAAAAAAEAUCAAAAAAAgCgQAAAAAAEAUCAAAAAAAgCgQAAAAAAAAUSAAAAAAAACS/D0dAFAQi8WqA0fTlHYuU5GhQbqmRqTMZlOR7c9kXFSNahV1ZVSw2+fh7LQiKgRKVik9I6vE03VXnO78rgAAAADKHgoE8Eq7EpO1ZtNBpZ7JtA2LCg/SgG711TI2xqn20eFB6l9Ae1fmUdx4c3N1uu6K053fFQAAAEDZRBcDeJ1diclaunZvvpPt1DOZWrp2r3YlJjvVPqWA9q7Mw5V4Szpdd8Xpzu8KAAAAoOyiQACvYrFYtWbTwULbvLnpoCwWq0vtXR2nJPG6Ml1np13U9Nz5XQEAAACUbRQI4FUOHE0r9Eq8dPnOgANH01xq7+o4JYnXlek6O+2ipufO7woAAACgbKNAAK+Sds65k+2cdsVt7+o4JWnj6jjuaOfueQEAAAAouygQwKtEhgYVq11x27s6TknauDqOO9q5e14AAAAAyi4KBPAq19SIVFR44Ser0eGXX9HnSntXxylJvK5M19lpFzU9d35XAAAAAGUbBQJ4FbPZpAHd6hfapn+3+jKbTS61d3WcksTrynSdnXZR03PndwUAAABQtlEggNdpGRujEX0a57vyHR0epBF9GqtlbIxz7SMct3dlHq7EW9LpuitOd35XAAAAAGWXyWq1ev37zSwWi5YsWaJ3331XZ86cUevWrTV16lTVqFHDYfvU1FQ99dRT+uabb2QymXTLLbfo4YcfVkhIiMsxZGdblJJyzuXxc/P3NysqKlSpqed06ZLFLdMsiywWqw4cTVPauUxFhl6+Db6wK9057c9kXFSNahV1ZVRwka/vK+48nJ1WRIVAySqlZ2SVeLruitMd02DdNRb5NVZZy290dKj8/KjzuxP7et9Bfo1Ffo1Ffo1T1nLriX29f6nOzUXLli3TmjVr9Mwzz6hq1aqaM2eOhg0bpo8++kiBgYH52o8aNUoZGRl65ZVXlJ6erscee0znz5/X7NmzPRA9XGU2m9Tg6qhit8+9YSiqQFDceZTWtIyYtpHxAQAAAPB9Xn/pISsrSy+99JJGjRqlhIQENWjQQAsWLNDJkyf1+eef52u/e/dubd++XbNnz1ajRo3Url07Pfnkk/rwww+VlJTkgW8AAAAAAID38/oCwf79+3Xu3Dm1a9fONiwiIkINGzbUjh078rXfuXOnKleurLp169qGxcfHy2QyadeuXaUSMwAAAAAAvsbruxicPHlSklStWjW74TExMbbPcktKSsrXNjAwUJGRkTpx4kSJYvH3d089JacfCX1HjUF+jUNujUV+jUV+AQAACuf1BYKMjAxJyvesgaCgIJ0+fdphe0fPJQgKClJmZqbLcZjNJkVFhbo8viMREa4/NBFFI7/GIbfGIr/GIr8AAACOeX2BIDg4WNLlZxHk/FuSMjMzHb6VIDg4WFlZWfmGZ2ZmqkKFCi7HYbFYlZ5+3uXxc/PzMysiIkTp6RnKzvb9p2t6G/JrHHJrLPJrrLKW34iIEO6GAAAAbuX1BYKc7gLJycmqWbOmbXhycrJiY2Pzta9atao2bdpkNywrK0tpaWmKiSnZ+97d/aqM7GxLmXj9hrcqK/l156sY3aWs5NZbkV9jkV8AAADHvL5A0KBBA4WFhWnbtm22AkF6err27dunQYMG5WvfunVrzZ07V0eOHNHVV18tSdq+fbskqWXLlqUXOOAGuxKTtWbTQaWe+ad7TFR4kAZ0q6+WsSUreAEAAABAbl5/b2JgYKAGDRqkuXPn6ssvv9T+/fs1duxYVa1aVTfccIOys7N16tQpXbhwQZLUrFkztWjRQmPHjtWePXu0detWTZ06VbfddpuqVKni4W8DOG9XYrKWrt1rVxyQpNQzmVq6dq92JSZ7KDIAAAAAZZHXFwgkadSoUerXr5+mTJmi/v37y8/PT6tWrVJAQIBOnDihDh06aMOGDZIkk8mkJUuWqHr16hoyZIjGjBmjTp06adq0aZ79EkAxWCxWrdl0sNA2b246KIvFWkoRAQAAACjrvL6LgST5+flp4sSJmjhxYr7PqlevrsTERLthlSpV0qJFi0orPMDtDhxNy3fnQF4pZzJ14GiaGlwdVUpRAQAAACjLfOIOAqC8STvn3Cs5nW0HAAAAAEWhQAB4ocjQILe2AwAAAICiUCAAvNA1NSIVFV74yX90+OVXHgIAAACAO1AgALyQ2WzSgG71C23Tv1t9mc2mUooIAAAAQFlHgQDwUi1jYzSiT+N8dxJEhwdpRJ/Gahkb46HIAAAAAJRFPvEWA6C8ahkbo7j6lXXgaJrSzmUqMvRytwLuHAAAAADgbhQIAC9nNpt4lSEAAAAAw9HFAAAAAAAAUCAAAAAAAAB0MQAAAAC8gsVi5blD5YDFYtXPv/2loydOKzwkgOUMr0KBAAAAAPCwXYnJWrPpoFLPZNqGRYUHaUC3+ry5qAzZlZisNzcdVArLGV6KLgYAAMDjLBaLFi1apI4dO6p58+a69957dfTo0QLbp6amavz48WrdurXi4+M1ffp0ZWRklGLEgPvsSkzW0rV77YoDkpR6JlNL1+7VrsRkD0UGd8pZziksZ3gxCgQAAMDjli1bpjVr1mjGjBl66623ZLFYNGzYMGVlZTlsP2rUKB05ckSvvPKKFi5cqC1btmjatGmlGzTgBhaLVWs2HSy0zZubDspisZZSRDACyxm+ggIBAADwqKysLL300ksaNWqUEhIS1KBBAy1YsEAnT57U559/nq/97t27tX37ds2ePVuNGjVSu3bt9OSTT+rDDz9UUlKSB74B4LoDR9Py3TmQV8qZTB04mlY6AcEQLGf4CgoEAADAo/bv369z586pXbt2tmERERFq2LChduzYka/9zp07VblyZdWtW9c2LD4+XiaTSbt27SqVmAF3STtX+EljcdvBO7Gc4SsoEAAAAI86efKkJKlatWp2w2NiYmyf5ZaUlJSvbWBgoCIjI3XixAnjAgUMEBka5NZ28E4sZ/gK3mJQinJeXXMm46JqVKuoK6OCizXu/j9Stf9IqmSSGtSIUoOro3glihM89cogo+dbWt8r93wqRQSrTcUKbp9u7vhzD4+oEChZpfSMrDL5uqe8OWhYO1rZFqt+/T1Ff6dfyPedPbHMfSXv7o7ZF3Pgy3IeLhgYGGg3PCgoSKdPn3bYPm/bnPaZmSW7+ubv755rJ35+Zru/4V5lKb8Na0crOjwo34PrcouOuLyPKK3tUFnKr7fwxuVcFrHulhwFglLi6NU10eFB6u/EK012JSbrlU/369yFS7ZhH+uIQoP9dfdNDXglSiE89cogo+dbWt/L0XwqfbRPA7pfo7h6V7h1ulHhQWpzbYy2/ZpcYB+9svQaIEc5CAsJkEzS2fMXbcNyvrMkjy1zb8+7u2P2xRz4uuDgywXzrKws278lKTMzUyEhIQ7bO3p4YWZmpipUcL2IaTabFBUV6vL4jkRE5I8f7lNW8nt/36aa9Wr+7jS2z/s0VaVKYaUY0WVlJb/ewluXc1nEuus6k9Vq5VGZTsjOtigl5ZxL4+a80qQgI/o0LvCgs6hxixq/PPL3NysqKlSff39Yi9/bU2A7o/JWkuXtDdM3ej7OrNNF8fV13h05yM3bl7mRihNzzrYhNfWcLl2ylHh6nhYdHVpmrpDs2bNH//73v/XFF1+oZs2atuH9+/dXbGxsvrcTvPDCC1q9erW2bNliG5aVlaVmzZpp3rx5uvnmm12KIzvbovR097wq0c/PrIiIEKWnZyg72/H6BteVxfzu2J+sNzYm2l1hjo4I0sAbYtW6Qelud8pifr3Fjv3JeuPzRKWke345l0Vlbd2NiAgp9X09dxAYzNlXmsTVr5zvdiJnxpWkNV8ccDh+eZZtseqNjYmFtiko7yVRkuXtDdM3ej7OrtNFMWLZlRZ35SA3b17mRnJ3zL6Yg7KiQYMGCgsL07Zt22wFgvT0dO3bt0+DBg3K175169aaO3eujhw5oquvvlqStH37dklSy5YtSxRLQcUjV2VnW9w+TfyjLOU3rt4ValanksPuTZ76jmUpv94irt4VanlNZR1PvaCjJ04rPCTA48u5LGLddV3ZuPTgxUryShNnxpWk1LNZvBIlj32H/i60j5dkzKtkjH6FTWm9Iseo+Ti7ThfFl18D5K4c5ObNy9xI7o7ZF3NQVgQGBmrQoEGaO3euvvzyS+3fv19jx45V1apVdcMNNyg7O1unTp3ShQsXJEnNmjVTixYtNHbsWO3Zs0dbt27V1KlTddttt6lKlSoe/jaA68xmkxpcHaW2DavyrKkyzGw2qUm9K9SuMcsZ3ocCgcFK8kqT4rzmhFei2EtJv+BUO3fnzehX2JTWK3KMmo878+2r67xRcXvrMjeSu2P2xRyUJaNGjVK/fv00ZcoU9e/fX35+flq1apUCAgJ04sQJdejQQRs2bJAkmUwmLVmyRNWrV9eQIUM0ZswYderUKV9XBAAAUDx0MTBYSV5pUpzXnPBKFHvREc69IcLdeTP6FTal9Yoco+bjznz76jpvVNzeusyN5O6YfTEHZYmfn58mTpyoiRMn5vusevXqSky07zZWqVIlLVq0qLTCAwCgXOAOAoNdUyNSUeGFH0xGh1/uY+bKuJIUFRbocPzyrGGdSop2Me8lUZLl7Q3TN3o+zq7TRTFi2ZUWd+UgN29e5kZyd8y+mAMAAAB3okBgMLPZZHtFWUH6d6vvsO+RM+NK0oDu19B3KQ8/s0kDb4wttE1BeS+Jkixvb5i+0fNxdp0uihHLrrS4Kwe5efMyN5K7Y/bFHAAAALgTBYJS0DI2RiP6NM53ZSo6IqjIV2bljBsanL83SFiwv1e9csvbtG5QQN7Di857SRS4vN00X6OnX9R8rogM0UP9mro8n8Li7xFfo9AruEYvu9JSUA7CQgIUViHAbljOd/bkMvfmvLs7Zl/MAQAAgLuYrFar1dNB+ILsbItSUs6VaBoWi1UHjqbpTMZF1ahWUVdGBcticS79FotV+/9I1f4jqZJJalAjiqeeFiDvu85z8p73lUFGM3q+pfW9cs+nUkSw2jSrrvTT50v86piC4s89PKJCoGSV0jOySnXZlZa8OWhYO1qRUaHa9tMx/Z1+Id939sQy95W8OxNz3m1DSafnadHRoaX+buSyzh37+hzFWd9QfOTXWOTXWOTXOGUtt57Y11MgcBIHDb4jd36zsrK9/iDflxS17vrCSZU3K8vbBm9YN8pafikQuB/7et9Bfo1Ffo1Ffo1T1nLriX09bzFAmbVjf7JWb0y0e695VHiQBnSrz23CBtiVmKw1mw6Sb+TDugEAAOAbuPSAMum7Pce1+L09dickkpR6JlNL1+7VrsRkD0VWNu1KTNbStXvJN/Jh3QAAAPAdFAhQ5lgsVq1c93Ohbd7cdNDp5z+gcBaLVWs2HSy0Dfkun1g3AAAAfAsFApQ5iX+k6u/TFwptk3ImUweOppVOQGXcgaNp+a4O50W+yyfWDQAAAN9CgQBlTtrZLOfanSv8xAXOcTaP5Lv8Yd0AAADwLRQIUOZEhgU61y40qOhGKJKzeSTf5Q/rBgAAgG+hQIAyJ7ZmlCpVDC60TXT45desoeSuqRGpqPDCT/DId/nEugEAAOBbKBCgzDGbTbrvtiaFtunfrX6pv4O9rDKbTRrQrX6hbch3+cS6AQAA4FsoEKBMuq7plXqoX9N8Vy+jw4M0ok9j3r3uZi1jYzSiT2PyjXxYNwAAAHyHv6cDAIzSukGMmtWppANH05R2LlORoZdvZeZqpTFaxsYorn5l8o18WDcAAAB8AwUClGlms0kNro7ydBjlBvlGQVg3AAAAvB9dDAAAAAAAAAUCAAAAAABAgQAAAAAAAIgCAQAAAAAAEAUCAAAAAAAgCgQAAAAAAEAUCAAAAAAAgCgQAAAAAAAA+UiBIDMzU9OnT1e7du0UFxen8ePHKyUlpdBxfvjhB911111q2bKlOnbsqMcee0xpaWmlEzAAAAAAAD7GJwoE06ZN0//+9z8tXrxYr776qg4dOqRRo0YV2P7w4cMaOnSoYmNj9c4772jBggXas2ePRo8eXYpR+w6Lxar9R1K1dd9J7Tucon2/p2jrvpPafyRVFovV0+HlkzteI2J0dvpGx2Ekb8kh4GmsqwAAAP/w93QARUlKStK6deu0YsUKtWrVSpI0f/589ejRQ7t371ZcXFy+cdatW6eYmBg99thjMplMkqQnnnhCAwcO1NGjR1WjRo1S/Q6FsVisOnA0TWnnMhUZGqRrakTKbDaV2vx3JSZrzaaDSj2T6fDzqPAgDehWXy1jY0otpsI4itedMTo7faPjMJI7Ys85qXK03vpybsozT2+LPIF1FQAAwJ7XFwh27dolSWrbtq1tWO3atVWlShXt2LHDYYGgV69e6tKli604IMn279OnT3tNgWDH/mSt3pjosYPTXYnJWrp2b6FtUs9kaunavRrRp7HHD5gLitddMTo7faPjMJI7Yv9uz3E9/8EepThYbyX5bG7Ks/J4ouzLv2MAAACjeH2BICkpSVFRUQoKCrIbHhMTo5MnTzocp27duvmGvfDCC6pcubJiY2NdjsXf3z09Mvz8zPpuz3Etfm9Pvs9yDk4f6tdUrRsYd3BqsVj15qaDTrd/88uDan1tFY9dUXQm3pwY/fwuL6ecv905/ZaxMU7H4W1XX4uTw4Ji/+HAKT33zk/5huest2EhASWafnnnyrpbUjv2F36ibPS2qDTl5NVkMvns7xgAAMBIHi8QHDt2TNdff32Bn48ePVqBgYH5hgcFBSkz0/Ft8XnNnj1bmzdv1pIlSxQQUPgJTEHMZpOiokJdGjevbItVK9f9XGibNzcd1PVtasnPoIPTn3/7y+4KcFFS0jN1PPWCmtS7wpB4iuJMvHljjIgIcfv0v9uXXOw4vIUrOcwt22LV6xsTCx3/bMZFl6ePfxRn3S2JbItVa744UGgbo7dFnnD0r/M++zsGAAAwkscLBFWqVNGGDRsK/HzLli3KysrKNzwzM1MhIYUfRF+8eFFTp07VunXrNGPGDHXr1s3lOC0Wq9LTz7s8fm6JR9P09+kLhbb5Ky1D2346pmtrRbtlnnkdPXHapXGqVyqdExdH83a23dUxoYqICFF6eoaysy1unf7vx9Ocnp6nclWQ4uTQUey//p5S5Hpbkunj8hXu4q67JeHMMjV6W1SacvL7Z1K6U+29fV2NiAgp1btNAABA2efxAkFAQIDDLgE5EhMTlZaWpqysLLs7CZKTk1WlSpUCxzt79qxGjhypnTt3av78+brppptKHOulS+45YE9x8iTr7/QLbptnXuFF3Ape0DhGxePMvJ1tl3NilZ1tcTpeZ6d/RUSw09PzVK4KUpwcOor97/SSFwcKmz7+UZx1tyScXaZGbos8IaJCyX4LAAAAZZXXX3po2bKlLBaL7WGF0uXXGCYlJal169YOx8nKytL999+vPXv2aNWqVW4pDrhTZFj+LhMO24UGFd3IRdfUiFRUuPPTjw6//FRzT3Em3pLE6Oz0u7aobmgcRippDt2xPnprbsorZ5epkdsiT4itGeWzv2MAAAAjeX2BoEqVKrrllls0ZcoUbdu2TXv27NG4ceMUHx+v5s2bS7pcEDh16pStK8Lzzz+vXbt2acaMGapTp45OnTpl++Oou0Jpi60ZpUoVC78SbfTBqdlssj113hn9u9X36MO6nIm3JDE6O31/f7OhcRippDm8pkakoos4qQoLLvymJG/NTXlldOHNWxm9PQEAAPBVXl8gkKQZM2aoXbt2GjlypIYOHao6depo0aJFts93796tDh06aPfu3ZKkjz/+WFarVePGjVOHDh3s/uS08SSz2aT7bmtSaJvSODhtGRujEX0aF3qCEB0e5DWv+yooXnfF6Oz0jY7DSCWJ3Ww2aeCNhb8FZMhNDXw2N+VReT5R9uXfMQAAgFFMVqvV6ukgfEF2tkUpKefcMi1/f7OiokL1+feHtXpjot27x6PDg9S/lN89brFYdeBomtLOZSoiJFAySennsxQZevnKobedHOSO11GMOflNTT3nUv/hoqZf3HbeyNXY/f3N+vXoaT3/wR67p8DnXW99OTeeVNJ111W7EpO1ZtNBj2+LjOYov768rkZHh/KQQjczYl9f2r/n8oL8Gov8Gov8Gqes5dYT+3qPP6SwPGvdIEbN6lTy+MGp2WxSg6ujSnWeJWF0vM5O39fylltJYr+u6ZWKvSpC+w6nFLje+nJuyqOWsTGKq1/Z49siT2BdBQAA+AcFAg/j4BS+iPW27GGZAgAAgHsTAQAAAAAABQIAAAAAAEAXA8Dn+fJD1gAAAAB4DwoEgA9z9PT5qPAgDShjT58HAAAAYDy6GAA+aldispau3WtXHJCk1DOZWrp2r3YlJnsoMgAAAAC+iDsIAB9ksVi1ZtPBQtu8uemg4upXprsBAJSinG5fZzIuqka1iroyKpiuYG5msVj16+8pung4VQEmq2pXjdCBY2nafyRVMkkNakSpwdVRdjk2YhnkTDP1TKbOnM9SWGiAosOCPbp8fXFd88WYjWRkPnw91+6O39fzYRQKBIAPyjkgKUzKmUwdOJrGq+sAoJQ46vYVFhIgq9Wqcxcu2YbRFcx1jnJskmTN1eZjHVFosL/uvqmBWsbGGNIdz9E03TVtV/lit0NfjNlIRubD13Pt7vh9PR9GoosB4IPSzhVeHChuOwBAyRTU7etsxkW74oBEVzBXFZRjq4O25y5c0tK1e/XOVwfd3h2voDjcMW1X+WK3Q1+M2UhG5sPXc+3u+H09H0ajQAD4oMjQILe2AwC4zpluX468uemgLBZHp7fIy9Ucf7b9aKGfF3cZFCeO0lq+znY79KZ1zRdjNpKR+fD1XLs7fl/PR2mgQAD4oGtqRCoqvPCT/+jwy32pAADGcqbblyM5XcFQNFdzXJTiLoPixFFay7c43Q69hS/GbCQj8+HruXZ3/L6ej9JAgQDwQWazSQO61S+0Tf9u9XnQCgCUgpJ056IrmHOMzFNxpl3cOEpj+fpit0NfjNlIRubD13Pt7vh9PR+lgQIB4KNaxsZoRJ/G+e4kiA4P0og+jcv9A1YAoLSUpDsXXcGcY2SeijPt4sZRGsvXF7sd+mLMRjIyH76ea3fH7+v5KA28xQDwYS1jYxRXvzKvaAEAD8rp9lXcW+DpCuY8V3NclOIug+LEUVrL15mYvG1d88WYjWRkPnw91+6O39fzURq4gwDwcWazSQ2ujlLbhlXzvfcZAGA8Z7p9OUJXMOe5muMe8TUK/by4y6A4cZTW8vXFboe+GLORjMyHr+fa3fH7ej5KAwUCAADgcZmZmZo+fbratWunuLg4jR8/XikpKYWOs3z5csXGxub74wkFdfsKCwlQaLD9DZt0BXNNQTl2dBgfFuyvEX0a6/au9d3eHa+gONwxbVf5YrdDX4zZSEbmw9dz7e74fT0fRjNZrdby+w6HYsjOtigl5ZxbpuXvb1ZUVKhSU8/p0iWLW6aJf5Bf45BbY5FfY5W1/EZHh8rPr+zU+SdPnqydO3dq1qxZCgwM1BNPPKHQ0FCtXr26wHHGjBmjwMBATZw40W545cqVXYrBHft6i8WqA0fTdCbjompUq6gro4J16ZKFrmBuZLFY9X/HT+ui1aQAk1W1q0bowLE07T+SKpmkBjWi8t1Rl7Nc3LkMcqaZeiZTZ85nKSw0QNFhwR5dvu76nqW5vTRi2Xi7wvJrZD58PdfOxF+cddcX8uGJfT3PIAAAAB6VlJSkdevWacWKFWrVqpUkaf78+erRo4d2796tuLg4h+MdOHBAt99+u8sFASPkdPvKfZCaMwzuYTabdG2taLuTgIa1otWwVnSh47h7GXjjcvXGmIriizEbych8+Hqu3R2/r+fDKGXn0gMAAPBJu3btkiS1bdvWNqx27dqqUqWKduzY4XCcrKws/f7776pTp06pxAgAQHnAHQQAAMCjkpKSFBUVpaAg+/6gMTExOnnypMNxfvvtN2VnZ2vjxo2aOXOmMjMz1bp1a02cOFExMa73H/X3d8+1k5xbQstSNxBvQn6NRX6NRX6NQ25LjgIBAAAw1LFjx3T99dcX+Pno0aMVGBiYb3hQUJAyMx2/iurAgQOSpJCQEC1cuFB///235s+fr8GDB2vdunUKDg4udpxms0lRUaHFHq8wEREhbp0e7JFfY5FfY5Ff45Bb11EgAAAAhqpSpYo2bNhQ4OdbtmxRVlZWvuGZmZkKCXF8kHfbbbepU6dOio7+p995/fr11alTJ3311Ve6+eabix2nxWJVevr5Yo/niJ+fWRERIUpPz1B2tu8/FNPbkF9jkV9jkV/jlLXcRkSE8JBCAABQtgQEBKhu3boFfp6YmKi0tDRlZWXZ3UmQnJysKlWqFDhe7uKAdLlLQmRkZIHdEpzh7ie2Z2dbysRbM7wV+TUW+TUW+TUOuXUdnTMAAIBHtWzZUhaLxfawQkk6fPiwkpKS1Lp1a4fjLFiwQDfeeKNyv6352LFjSk1NVb169QyPGQCAsogCAQAA8KgqVarolltu0ZQpU7Rt2zbt2bNH48aNU3x8vJo3by7p8lsLTp06ZeuK0L17d/3555+aNm2aDh8+rB07duihhx5SixYt1LFjRw9+GwAAfBcFAgAA4HEzZsxQu3btNHLkSA0dOlR16tTRokWLbJ/v3r1bHTp00O7duyVJjRs31gsvvKDExET17dtXI0eO1LXXXqsVK1bIZDJ56msAAODTTNbc9+ahQNnZFqWknHPLtPz9zYqKClVq6jn6xhiA/BqH3BqL/BqrrOU3OjqU1zi5Gft630F+jUV+jUV+jVPWcuuJfT1HFgAAAAAAgAIBAAAAAACgQAAAAAAAAESBAAAAAAAAiAIBAAAAAAAQBQIAAAAAACAKBAAAAAAAQBQIAAAAAACAKBAAAAAAAABRIAAAAAAAAKJAAAAAAAAARIEAAAAAAACIAgEAAAAAABAFAgAAAAAAIAoEAAAAAABAFAgAAAAAAIAoEAAAAAAAAFEgAAAAAAAA8oECQWZmpqZPn6527dopLi5O48ePV0pKitPjL1++XLGxsQZGCAAAAACA7/P6AsG0adP0v//9T4sXL9arr76qQ4cOadSoUU6Nu2fPHi1ZssTgCAEAAAAA8H1eXSBISkrSunXrNGXKFLVq1UpNmzbV/PnztWPHDu3evbvQcc+fP6+JEyeqVatWpRQtAAAAAAC+y6sLBLt27ZIktW3b1jasdu3aqlKlinbs2FHouDNnztQ111yj3r17GxojAAAAAABlgVcXCJKSkhQVFaWgoCC74TExMTp58mSB433++efasmWLnnzySaNDBAAAAACgTPD35MyPHTum66+/vsDPR48ercDAwHzDg4KClJmZ6XCcpKQkTZ06Vc8++6yioqLcFqsk+fu7p57i52e2+xvuRX6NQ26NRX6NRX4BAAAK59ECQZUqVbRhw4YCP9+yZYuysrLyDc/MzFRISEi+4VarVY888ohuuukmderUya2xms0mRUWFunWaERH5vwPch/wah9wai/wai/wCAAA45tECQUBAgOrWrVvg54mJiUpLS1NWVpbdnQTJycmqUqVKvvbHjx/Xd999px9++EHr1q2TJF26dEmSFBcXp+nTp6tXr14uxWqxWJWeft6lcfPy8zMrIiJE6ekZys62uGWa+Af5NQ65NRb5NVZZy29ERAh3QwAAgP/X3p3HRVXv/wN/sSM7qGCuuA2ioIYCLoWCll7RStvc0JtoiGjX7SFYCqhpqQheUkxDveWG9tDMcoVcuimYmn6z1HD7magoyerCOuf3B3cmBwYYYM6cmfH1fDx8JGf5nPe8P+O8P334nDNaJekEQV169eoFuVyOc+fOoW/fvgCAmzdv4v79+/D19a12vJubG44cOaKy7ciRI4iLi8PevXvRtGnTRsVTXq7dAWVFhVzrbdLfmF/xMLfiYn7FxfwSERERqafXEwRubm4IDg7GggULsGzZMjRp0gQxMTHw8/NDz549AQClpaUoKCiAo6MjLC0t0a5dO5U2FJMCVbcTERERERER0d/0fm3ikiVL0LdvX0yfPh2hoaHo0KEDEhMTlfvPnz+Pl156CefPn5cwSiIiIiIiIiLDZiIIgiB1EIagokKO3NzHWmnL3NwUzs62yMt7zGWuImB+xcPciov5FZex5dfFxZbPINAy1nrDwfyKi/kVF/MrHmPLrRS1niMLIiIiIiIiIuIEARERERERERFxgoCIiIiIiIiIwAkCIiIiIiIiIgInCIiIiIiIiIgInCAgIiIiIiIiIgDmUgdAJBW5XEDm7XzkPy6Bk60VZG2cYGpqInVYRET0nFLUpbyiEhQ9KYWdrQVc7Ky1Vp8U7ec+Ksajx2Wwt7GEs7209a+mWqzNGq2uLQAatV/13E6tHHHtToHOxg6NyUN9zq16bNf2LqiQC7j8/3LxsLBYbd9o8j7SRf8aAnX5lZpYfaCPfauPMekzThDQc+ncHw+wPe0q8opKlNuc7a0wdnBn9PJwlTAyIiJ6HqmrSwraqE9it6+tmJztreDv6YrTlx9opUaru4atdeXw93Fxea3tqzvXxAQQBNR6nrY0ZqxSn3PVHWtnbQ4TUxMUPSlTOV9d39TUvi761xCoy4OLvRXCRnWHZxtHvYlJrM8ZqftWH2PSdyaC8OzHHNWkokKO3NzHWmnL3NwUzs62yMt7jPJyuVbapL/Vld9zfzzA2m9+q/H8iJFe/MCoAd+74mJ+xWVs+XVxsYWZGe8U1Capan1ddUmhofVJ7PYbQtOYqlLEqEl+G3INRfv1PVfbuWvMWKU+5za0H2oTMdILABrVv8airvzOeKs7XuzUTIcRiTcO1pfx9bOfDad/z9aLmBpDilrPkQU9V+RyAdvTrtZ6zI60q5DLOW9GRETi06QuKTSkPondfkPUJ6aqNI2xodfYkXYV5eXyep+rzdw1ZqxSn3Mb0w+12Z6aKXr/GgJN8rvtyB86fb1ijYP1cXytjzEZCk4Q0HNFcW9nbXKLSpB5O183ARER0XNNk7qk0JD6JHb7DVGfmKrSNMaGXiO3qARHf8mq97nazF1jxir1Obcx/VCbvEelovevIdCoLwp1+3rFGgfr4/j6jz/z9C4mQ8FnENBzJf+xZgVL0+OIiIgao771Rt+Ob4jGXkOT8xtzjQf5Txt0nrZy15ixijGMc/Q5tvrQx74QKya9fK2PSjU7zkjeb9rECQJ6rjjZWmn1OCIiosaob73Rt+MborHX0OT8xlzD1alJg87TVu4aM1YxhnGOPsdWH/rYF2LFpJev1c5Ss+OM5P2mTbzFgJ4rsjZOcLav/YPAxf7vr0AiIiISkyZ1SaEh9Uns9huiPjFVpWmMDb2Gi70Vgnxa1/tcbeauMWOV+pzbmH6ojbOdpej9awg06gsH3b5escbB+ji+9mjrrHcxGQpOENBzxdTUBGMHd671mDGDO/O7UYmISCc0qUsKDalPYrffEPWJqSpNY2zoNcYM7gxzc9N6n6vN3DVmrFKfcxvTD7UZ+4pM9P41BJrkd9yrHjp9vWKNg/VxfK2PMRkKThDQc6eXhysiRnpVm1V0sbcyiK87ISIi41JTXVJobH0Su31txuRib4Whfm20UqNruoZdEwvYWqveZVu1/ZrONany/xJi5a4xY5X6nFtzjsxhb2NR7Xx1faOufV30ryGoMQ8OVpg/0Re+XXT/esUaB+vj+FofYzIEJoIg8LsdNCDVdyNT/WmaX7lcQObtfOQ/LoGTbeUSI84i1o7vXXExv+IytvxK8d3Ixk7qWq+oS3lFJSh6Ugo7Wwu42FlrrT4p2s99VIxHj8tgb1O5FFzK+ldTLa6rRtcnv+raAqDRGKDquZ1aOeLanQKdjR0aM1apz7lVj+3a3gVOzrY4/X9ZeFhYrLZvNHkfNbR/jY26/DZtaidpPRKrD6TuW3WfDVLH1BhS1HpOEGhI6kEDaY75FQ9zKy7mV1zGll9OEGgfa73hYH7FxfyKi/kVj7HlVopaz5EFEREREREREXGCgIiIiIiIiIg4QUBERERERERE4AQBERER6Zno6GhERUXVeVxWVhbCwsLg4+ODl156CatXr0ZFRYUOIiQiIjJOnCAgIiIivSCXyxEfH4+dO3fWeWxZWRlCQ0MBACkpKYiNjcWOHTuwdu1ascMkIiIyWuZ1H0JEREQkruvXr+Ojjz7CrVu30LJlyzqPP3z4MO7evYtdu3bB0dERMpkMDx8+xIoVKzB16lRYWlrqIGoiIiLjwhUEREREJLmMjAx07NgR33//PVq3bl3n8WfPnkW3bt3g6Oio3NanTx88evQIly9fFjNUIiIio8UVBERERCS5cePG1ev47OxstGjRQmWbq6srAODevXvo0aOH1mIjIiJ6XnCCgIiIiESVlZWFQYMG1bg/PT0dLi4u9WqzuLgYDg4OKtusrKwAACUlJfUP8n/MzbWzuNLMzFTlv6RdzK+4mF9xMb/iYW4bjxMEREREJCo3NzccOHCgxv3P3iagKWtra5SWlqpsU0wM2NjY1Ls9ADA1NYGzs22Dzq2Jg0MTrbZHqphfcTG/4mJ+xcPcNhwnCIiIiEhUFhYW6Nixo1bbbNGiBTIzM1W2PXjwAEDlhERDyOUCCgufNDo2oPK3Vw4OTVBY+BQVFXKttEl/Y37FxfyKi/kVj7Hl1sGhic5XQ3CCgIiIiAyOr68v9u7di0ePHsHOzg5A5YMObW1t0aVLlwa3W16u3QFlRYVc623S35hfcTG/4mJ+xcPcNhxvziAiIiK9V1paipycHOVtBYMHD0bz5s0xc+ZMXLlyBWlpaYiPj8ekSZP4FYdEREQNxBUEGjI1NYGLC+9LNCTMr3iYW3Exv+IylvyamppIHYJOnT9/HhMmTMBXX30Ff39/WFlZITk5GYsWLcI777wDR0dHjB07FtOmTWvwNVjrDQ/zKy7mV1zMr3iMJbdS1HoTQRAEnV+ViIiIiIiIiPQKbzEgIiIiIiIiIk4QEBEREREREREnCIiIiIiIiIgInCAgIiIiIiIiInCCgIiIiIiIiIjACQIiIiIiIiIiAicIiIiIiIiIiAicICAiIiIiIiIicIKAiIiIiIiIiMAJAiIiIiIiIiICJwiIiIiIiIiICJwgICIiIiIiIiJwgoCIiIiIiIiIwAkCSd27dw+zZ89G//794evri9DQUFy9elXqsIxSdHQ0oqKipA7DoMnlciQmJuLll19Gz549MWXKFNy+fVvqsIzS+vXrERISInUYRiM/Px/R0dEICAiAj48PxowZg7Nnz0odFj0nWOt1h7W+8VjrdYe1XvtY77WDEwQSKS0txfvvv4+cnBx8/vnn2L59O2xtbTFx4kTk5uZKHZ7RkMvliI+Px86dO6UOxeAlJSVh+/btWLJkCVJSUiCXyzF58mSUlpZKHZpR2bZtG1avXi11GEZl9uzZOH/+POLj47F79254enoiNDQUN27ckDo0MnKs9brBWq89rPW6wVovDtZ77eAEgUTOnj2LzMxMxMXFwdvbG507d8bKlSvx5MkTHD16VOrwjML169cxduxYfP3112jZsqXU4Ri00tJSbNq0CR988AEGDhyILl26ICEhAdnZ2Thy5IjU4RmF+/fvY+rUqYiLi4O7u7vU4RiNW7du4eTJk4iNjUXv3r3Rvn17LFy4EK6urvjuu++kDo+MHGu9+FjrtYe1Xnys9eJhvdceThBIpHPnztiwYQPc3NyU20xNK7ujsLBQqrCMSkZGBjp27Ijvv/8erVu3ljocg3blyhU8fvwYffv2VW5zcHBA165dcebMGQkjMx6///47LCwssG/fPvTo0UPqcIyGs7MzNmzYAG9vb+U2ExMTmJiY8LOWRMdaLz7Weu1hrRcfa714WO+1x1zqAJ5XzZs3x4ABA1S2bdmyBcXFxejfv79EURmXcePGSR2C0cjOzgYAvPDCCyrbXV1dlfuocYKCghAUFCR1GEbHwcGh2mft4cOHcevWLXz44YcSRUXPC9Z68bHWaw9rvfhY68XDeq89nCAQSVZWFgYNGlTj/vT0dLi4uCh/Tk1NxapVq/DPf/4THh4eugjRoNU3v9Q4T58+BQBYWlqqbLeyskJBQYEUIRE1yC+//IL58+fj1VdfxcCBA6UOhwwca724WOt1i7WejAnrfcNxgkAkbm5uOHDgQI37HR0dlX/fsWMHlixZgtdeew3z5s3TRXgGrz75pcaztrYGUHl/ouLvAFBSUoImTZpIFRZRvaSlpWHu3Lnw8fFBXFyc1OGQEWCtFxdrvW6x1pOxYL1vHE4QiMTCwgIdO3as87iVK1ciOTkZ7733HiIjI2FiYqKD6Ayfpvkl7VAsN3zw4AHatm2r3P7gwQP+FowMwtatW7F06VIMHToUy5cvr/YbMqKGYK0XF2u9brHWkzFgvW88PqRQQooBQ2RkJKKiojhgIL3VpUsX2NnZ4fTp08pthYWFuHTpEnx9fSWMjKhuiq/sGjduHOLj4zlYIJ1irSdDwVpPho71Xju4gkAip0+fRnJyMkJCQjBixAjk5OQo99nY2MDW1lbC6IhUWVpaYvz48YiLi4OLiwtatWqFlStXokWLFnj11VelDo+oRjdv3sSyZcvwyiuvICwsDH/99Zdyn7W1Nezt7SWMjowdaz0ZEtZ6MmSs99rDCQKJfP/99wAqn2a8ZcsWlX3Tp0/HjBkzpAiLqEYffPABysvLsWDBAhQXF8PX1xcbN26EhYWF1KER1ejw4cMoKytDamoqUlNTVfaNHDkSn376qUSR0fOAtZ4MDWs9GSrWe+0xEQRBkDoIIiIiIiIiIpIWn0FARERERERERJwgICIiIiIiIiJOEBAREREREREROEFAREREREREROAEARERERERERGBEwREREREREREBE4QEBERERERERE4QUAii4qKgoeHR61/QkJCJI0xPz8fSUlJGDlyJHx9fdGjRw8EBwcjISEB+fn5ksZWVVZWVp353LFjh9Rhas3p06fh4eGB06dP6+yaRUVFGDRoEK5fvw4ACAkJQdeuXXHx4kW1xwcFBSEqKkpn8TXUs3Eq3kd79uwR/bpV+zA9PR2vv/46ysrKRL82EekO6712sd6Lj/Veu1jvjYe51AGQcZs2bRpGjx6t/DkpKQmXLl3CmjVrlNvs7OykCA0AkJmZibCwMJSVlWH8+PHw9vaGmZkZLly4gC+//BIHDhxASkoKmjZtKlmM6oSHh2PgwIFq97Vp00a3wYioW7du2LlzJzp16qSzay5duhRBQUHo2LGjcltFRQXmz5+PPXv2wNLSUmexiMXV1RU7d+5E27ZtdX7tvn37olWrVkhKSsK//vUvnV+fiMTBei8O1nvxsN6Li/XecHGCgETVtm1blQ8lFxcXWFpaomfPntIF9T8lJSWYOXMmzMzMsHv3bri4uCj39enTB8HBwXj99deRmJiIRYsWSRhpdW3bttWLHIrNzs5Op6/z999/x3fffYcTJ06obLe3t8fVq1exdu1azJo1S2fxiEXqf4Ph4eEYO3YsxowZA1dXV8niICLtYb0XB+u9OFjvdYP13jDxFgPSC3v27EHXrl3x9ddfo3///vDz88O1a9fULufas2cPPDw8kJWVpdym+M2Aj48PfHx8EBERgdu3b9d6zYMHD+L69etYuHChymBBoU2bNggPD6+2Ly0tDaNGjYK3tzf69++Pjz/+GE+ePFHu/+yzz/DKK6/g+PHjGDFiBLy8vDBkyBDs3btXpZ38/HxER0ejX79+8Pb2xjvvvIP09HRNU1an6dOnw9vbGzdu3FCJzdPTEz///DOAymVoCQkJWLZsGXx9feHv74958+apLLWMiorCxIkTERMTAx8fHwwbNgwVFRWQy+XYsGEDXnnlFeVr3LJli0oMf/75J6ZOnQp/f3/06NED7777rkoxLi4uRmxsLAICAuDl5YWhQ4di48aNyv3qlhxevHgRoaGh8Pf3h4+PD6ZOnYqrV69WOyc9PR2TJk1Cjx490L9/f6xcuRIVFRW15mz9+vXo06cPmjVrprLd09MTb7zxBpKTk/Hbb7/V2kZFRQW2bduGESNGoHv37hg4cCDi4uJQUlJSZ04VS0ajoqLQq1cv+Pn54eOPP0ZxcTGWL1+OPn36wN/fHx999JFKe7m5uVi0aBECAwPh5eUFPz8/REREqPwbeVbVJYchISE1LmFV5F6T/gaAlJQUDBkyBN27d8f48eNx9+7dasd4e3ujZcuW2Lx5c625JCLjw3rPes96z3pP+o0rCEhvVFRUYNOmTVi6dCny8vJUlnzV5ubNmxg9ejQ6dOiA5cuXo7y8HOvWrcOYMWPw7bff1rhcMC0tDY6Ojnj55ZdrbHvKlCkqP3/33XeYO3cuRowYgZkzZ+LOnTtISEjAtWvXsHnzZpiYmAAAcnJysHjxYoSHh6NVq1bYuHEjIiMj4e3tjY4dO6KkpAQTJ07EX3/9hVmzZsHV1RW7d+/G5MmTkZycjL59+9b6muVyOcrLy6ttNzExgZmZGQAgNjYWwcHBiImJwZYtW/Dbb7/h888/x6RJk+Dn56c8Z/v27WjXrh0++eQT5ObmYtWqVbh16xZSUlKUr+fs2bOwsrLC2rVr8eTJE5iZmSE6Ohp79uxBWFgYXnzxRZw5cwbLli1DYWEhIiIiIJfLERYWBldXV6xYsQLm5ub46quvEB4ejoMHD6Jdu3ZYtmwZfvrpJ0RGRqJZs2b48ccfsWLFCjg5OeHNN9+s9voyMjIwefJk+Pv7Y9myZSgpKcH69esxevRo7Nq1S+U9M3fuXIwdOxZTpkzB8ePHkZycjDZt2qgsgX3W48ePcfToUSxcuFDt/g8//BAnT57E/PnzsXv37hqXHkZHR+Pbb7/FlClT0Lt3b1y6dAlr167F5cuXkZycXGtOAWDlypUYPnw41qxZg2PHjuHLL7/ETz/9hC5duiAuLg4XLlzAZ599hvbt22Py5MkQBAFhYWEoKCjA3Llz0axZM/zxxx9YvXo1YmJiVAZgNYmJicGjR4+UPz99+hSzZ89G8+bN4e3tDaDy/VRbfwPA1q1bsWTJEkycOBEBAQFIT0+vMZ9Dhw7Fnj17EBkZWWd8RGRcWO9Z71nvWe9JjwlEOhQZGSkEBgZW2757925BJpMJe/fuVdkeGBgoREZGqj329u3bgiAIwuzZs4V+/foJRUVFymPy8vKEXr16CZ9++mmNsYwYMUIYNWpUte3l5eVCWVmZyh9BEAS5XC4EBAQIoaGhKsefOnVKkMlkwrFjxwRBEITExERBJpMJp06dUh5z584dQSaTCRs3bhQEQRB27twpyGQy4cKFC8pj5HK5MG7cOLUxKdy+fVuQyWQ1/unZs6fK8fv37xdkMpmwa9cuITg4WHjjjTeEkpIS5f7AwEDBz89PKCwsVG5LTU0VZDKZcOLECUEQKvtMJpMJ9+7dUx5z48YNwcPDQ1i/fr3K9RISEgRvb28hNzdXePDggSCTyYR9+/Yp9xcWFgrLli0TMjMzBUEQhCFDhggLFixQaWPNmjXKXGZkZAgymUzIyMgQBEEQ3nrrLWHYsGFCeXm58viCggLBz89P+OCDD1TOSUhIUGk3KChICAsLqzG3x48fF2QymXDp0iWV7ePHjxfGjx8vCIIg/PDDD4JMJhPi4+NVcqh4j169elWQyWTV8rJ3715BJpMJx48fFwRBfU4FQRBkMpnw9ttvK38uLy8XevbsKQQFBSnfh4IgCMOHDxfCw8MFQRCE7OxsISQkRDhz5oxKW0uWLBG8vLzUxql4H+3evbtaHuRyuRARESH4+fkJf/75pyAImvW3XC4X+vbtK8ycOVPlmOjoaJU+VFC8z65du1YtBiIyfKz3rPes96z3gsB6b4i4goD0iqenZ73PycjIgJ+fH6ytrZWz7HZ2dujduzdOnTpV43mCIKjdHhgYiPv376ts++GHH1BSUoLs7GyEhYWpzOb7+vrCzs4OJ0+eVHmQ0LP3fLVo0QIAlEsT09PT0bx5c3Tr1k2lrcDAQKxYsQIFBQVwdHSsMfbp06erfWiRYlZaYdiwYTh06BCio6NhaWmp9qE7QUFBsLe3V/nZ3NwcZ86cQUBAAADAyclJ+RqAypwLgoCgoCCV+IOCgrBu3TqcO3cOgwYNQqdOnbBw4UL89NNPeOmllxAQEID58+crj/f390dKSgqys7MxYMAADBgwQDk7XdWTJ09w8eJFTJ8+XeV1Ojg4IDAwsNp9hC+++KLKzy1atFBZGlqVYnle69atazwmKCgIr732GpKTk/Hqq6+iW7duKvsVSzmDg4NVtgcHB2P+/Pk4ffo0BgwYAKB6TtXFbWZmBmdnZ3Tr1g3m5n9/XDs5OaGoqAgA4Obmhq+++gqCICArKwu3bt3CjRs38Msvv6C0tLTG11KT1atX4+jRo8rfwACa9Xf79u3x8OFDBAYGqrT3j3/8AykpKdWuo8hzVlaWxr89JCLjwXrPes96z3pP+okTBKRXbGxs6n1Ofn4+Dhw4gAMHDlTbp+5eQ4WWLVvi119/hSAIymVgALBhwwblV7IcP35c+QRmxX16ixYtUvsQowcPHqj83KRJE+XfTU0rH/ehGKTk5+cjJyenWsFRyMnJqXXA0KpVK+VSsLqMHDkShw8fhru7O9q3b19tv5ubm8rPpqamcHZ2RkFBgXKbra2tyjGKXFQtjAr379+HiYkJNm3ahHXr1iE1NRV79+6FhYUFBg8ejEWLFsHR0REfffQRWrRogX379mHJkiVYsmQJXnzxRcTGxqJLly4qbRYVFUEQhGr3CwJAs2bNlAVUwdrautrrqmmQqGgfUO03dRYsWID09HTl0sNnKXLWvHlzle3m5uZwdnZWibFqThXUPeW7rn8X+/btQ3x8PO7duwcnJyd4enpWe/2a2L9/Pz7//HNERkaiX79+yu2a9Lfi35qzs7PKvqq5UFDkuWq/EdHzgfW+Eus96/2zWO9JH3CCgPRe1QfNVJ0Vtre3R79+/fDee+9VO/fZWdiqgoKCcPz4cfz888/w9/dXbn+2UD37MBwHBwcAwLx581Tu6VOorcBXZW9vD3d3d8TFxandX9usdn08ffoUn3zyCWQyGTIzM7Fp0yZMnjxZ5Zi8vDyVnysqKpCXl1frYEuRiy+//FJt4WvZsiWAysFIbGwsYmJicOXKFRw6dAhffPEFnJ2dERMTA0tLS4SHhyM8PBx3797FsWPHkJSUhDlz5mD//v0qbdrb28PExAR//fVXtevl5OTAyclJo5zURFHoCgsLa33tjo6OiI2NRUREBJKSkqrtU8TTqlUr5faysjLk5eVVK6bacPbsWURGRiIkJAShoaHKAeCKFStw7tw5jdv59ddf8eGHH2L48OGYNGmSyj5N+ruwsBAA8PDhQ5V9NX23uGJwJUZOiMgwsd43HOu95ljvWe+pdvwWA9JrdnZ2yM7OVtlW9UNQ8QRkT09PeHt7w9vbG15eXvjPf/6D1NTUGtseMWIE3N3dERMTo7YIAaoDhg4dOqBp06bIyspSXsfb2xtubm5YtWoVLl26pPHr8vPzw71799C0aVOVtk6ePInk5ORqSwcbatWqVcjOzsZnn32G8ePHIzExEdevX1c55scff1RZmvbDDz+gvLy81gcn9e7dG0DlYOPZ+HNzc/Hvf/8b+fn5OH/+PPr164dff/0VJiYm8PT0xKxZsyCTyXD37l0UFxdjyJAh2LRpE4DKojNu3DgEBwerfRKujY0NvLy8cPDgQZVBZFFREY4fP45evXo1KleKQU7V95s6gwcPxvDhw7Fhwwbk5uYqtysGklUHO/v370dFRUWjY1Tn/PnzkMvlmDFjhnKwUFFRoVxuK5fL62zj/v37iIiIQIcOHbB06dJq+zXpb3d3d7zwwgs4dOiQyrnHjh2r8ZrA33knoucb633jsN5rjvWe9Z5qxxUEpNcCAwOxfv16rF+/Hj169MDRo0eRkZGhcsy0adMwevRohIWFYcyYMbCyssLOnTuRlpaGxMTEGtu2sbHB2rVrERERgeHDh+Pdd9+Fj48PrKyscPXqVXzzzTf4/fffERAQABcXF5iZmWHWrFmIjo6GmZkZAgMDUVhYiKSkJNy/f7/G5YPqjBo1Clu3bsV7772HqVOn4oUXXsCpU6fwxRdfYPz48bCwsKj1/D///BMXLlxQu8/R0RHt27fHzz//jK1bt2LWrFlwd3fHzJkzkZqaiqioKKSkpCgHJffu3UN4eDgmTJiAe/fuIT4+Hi+//LLKb1mq8vDwwGuvvYaFCxfizp078PLyws2bN5GQkIDWrVvD3d0d5eXlsLa2xrx58zBjxgw0a9YMp06dwuXLlzFhwgRYW1ujW7duWLNmDSwsLODh4YGbN2/im2++wZAhQ9Red86cOQgNDcX777+PsWPHoqysDBs2bEBpaWmN9zJqqnfv3rC2tsa5c+fQtWvXOo9fuHAhMjIyVAabnTp1wsiRI5GYmIinT5/C19cXly9fxpo1a+Dv71/rE7Qbqnv37gCAxYsX480330RBQQG2bduGK1euAKj8DZy6ZYwKitw9evQIH3/8MTIzM1UGGS1atNCov01MTDB37lzMmTMHCxYswNChQ3HhwgXs2LFD7XXPnTuH1q1bq10GS0TPH9Z79VjvWe8VWO9JVzhBQHotLCwMubm52LhxI8rKyjBw4EAsXboU4eHhymO6dOmCbdu2ISEhAfPmzYMgCJDJZFi7di0GDRpUa/udOnXCN998g127duHgwYNISUnB48eP4erqCl9fX0RFRaksL3z77bdha2uL5ORk7Ny5EzY2NvDx8UFcXJzyAS+asLGxwbZt27Bq1SqsXLkSRUVFaNWqFebMmVNtuZc669atw7p169TuGzRoEOLi4jB//nzIZDKEhoYCqLwHLjo6GuHh4UhOTkZYWBiAyvvMHBwcMHPmTNjY2GDkyJGYNWtWnTF88sknWL9+vfKhQ02bNsWwYcMwc+ZMmJmZwczMDJs2bcKqVauwdOlSFBYWwt3dHYsXL8aoUaMAVBa51atXY9OmTcjJyUHTpk3x1ltv4V//+pfaa/bt2xebN29GYmIiZs+eDUtLS/Tu3RvLly9H586d64y5Nk2aNEFAQABOnDiBkJCQOo93cnJCbGwspk+frrJ96dKlaNeuHXbv3o0vvvgCrq6umDBhAqZNm6a8N1Wb/P39ER0djc2bN+PQoUNo1qwZ/P39sWbNGkRERODcuXPKByWp8+DBA1y8eBEA8P7771fbP336dMyYMaPO/gaA4cOHw9TUFElJSfj2228hk8mwePFizJ49u1q7//3vfzF06FAtZYGIDB3rvXqs96z3Cqz3pCsmQm1P8SAioxYUFAQ/Pz98+umnUoeiFy5evIh3330XR44c0dp9oVTd2bNnMWnSJKSlpcHV1VXqcIiIjB7rvSrWe91gvTdMfAYBEdH/eHt7Y+jQodi4caPUoRi15ORkTJw4kYMFIiKSBOu9brDeGyZOEBARPSM6OhonTpzAtWvXpA7FKKWnp+Pu3buYMWOG1KEQEdFzjPVeXKz3hou3GBARERERERERVxAQEREREREREScIiIiIiIiIiAicICAiIiIiIiIicIKAiIiIiIiIiMAJAiIiIiIiIiICJwiIiIiIiIiICJwgICIiIiIiIiJwgoCIiIiIiIiIwAkCIiIiIiIiIgLw/wF/4Vz3L2y+fgAAAABJRU5ErkJggg==",
      "text/plain": [
       "<Figure size 1200x600 with 2 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Set up the figure and axes for subplots\n",
    "sns.set(font_scale=1)\n",
    "fig, axs = plt.subplots(1, 2, figsize=(12, 6))\n",
    "\n",
    "# First scatter plot\n",
    "axs[0].scatter(y_test, y_pred_gbr)\n",
    "axs[0].set_xlabel(\"True Gene Expression (Normalized)\")\n",
    "axs[0].set_ylabel(\"Prediction\")\n",
    "axs[0].set_title(\"Enformer with Gradient Boosting vs. True Gene Expression\")\n",
    "\n",
    "# Second scatter plot\n",
    "axs[1].scatter(test_expression, norm_prs)\n",
    "axs[1].set_xlabel(\"True Gene Expression (Normalized)\")\n",
    "axs[1].set_ylabel(\"Prediction\")\n",
    "axs[1].set_title(\"PRS with Pruning and Thresholding vs. True Gene Expression\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.022023082624041557)"
      ]
     },
     "execution_count": 156,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corr, p_value = pearsonr(y_test, norm_prs)\n",
    "p_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "np.float64(0.012476530088572095)"
      ]
     },
     "execution_count": 165,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "corr, p_value = pearsonr(y_test, y_pred_gbr)\n",
    "p_value"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1.5977101424806923"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mean_squared_error(y_test, norm_prs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "enformer",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.21"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
